{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# File Generator\n",
    "\n",
    "This notebook generates all necessary files for the modulytics site. Capitalized comments indicate things that may need to be changed, and lowercase comments just describe what the code is doing.\n",
    "\n",
    "For more instructions, see [the wiki](https://github.com/SBRG/modulytics/wiki/Adding-a-Private-Project).\n",
    "\n",
    "Be very sure to start with this file in the right place! Instructions are in the wiki.\n",
    "\n",
    "## Load ICA data\n",
    "\n",
    "If you followed the instructions in the wiki, the only path you should need to modify below is the path to ICA_CODE_DIR. It needs to point to a folder containing [the ICA codebase](https://github.com/SBRG/ICA).\n",
    "\n",
    "The cutoff is determined in the notebook [3_estimate_thresholds.ipynb](https://github.com/SBRG/ICA/blob/master/notebooks/3_estimate_thresholds.ipynb). The organism name only matters if it is 'ecoli', but fill it in regardless. \n",
    "\n",
    "The base_conds are the conditions that you used as a baseline for normalization. They are only used here to get the x-axis values in the gene scatter plot."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Organism Folder:  e_coli\n",
      "Dataset Folder:  ros\n"
     ]
    }
   ],
   "source": [
    "# first, make sure the template is in the right place.\n",
    "# ENSURE THIS OUTPUT IS CORRECT\n",
    "import os\n",
    "cwd = os.getcwd().split('/')\n",
    "assert(cwd[-3]=='organisms')\n",
    "print('Organism Folder: ', cwd[-2])\n",
    "print('Dataset Folder: ', cwd[-1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ENSURE THIS PATH IS CORRECT\n",
    "ICA_CODE_DIR = '../../../../../main/ica/'\n",
    "\n",
    "# ADD THIS INFORMATION\n",
    "cutoff = 600 \n",
    "organism = 'ecoli'\n",
    "base_conds = ['control__wt_glc__1','control__wt_glc__2']\n",
    "\n",
    "# other paths\n",
    "DATA_DIR = 'data_files/'\n",
    "GENE_DIR = '../annotation/gene_files/'\n",
    "PY_DIR = '../../../py/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load ICA codebase\n",
    "import sys, warnings\n",
    "sys.path.append(ICA_CODE_DIR)\n",
    "try:\n",
    "    from icaviz.plotting import *\n",
    "except:\n",
    "    warnings.warn('Path to ICA codebase is incorrect.')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load data\n",
    "ica_data = load_data(X=DATA_DIR+'log_tpm.csv', # NOT normalized\n",
    "                     S=DATA_DIR+'M.csv', # may be called 'S.csv'\n",
    "                     A=DATA_DIR+'A.csv',\n",
    "                     metadata=DATA_DIR+'sample_metadata.csv',\n",
    "                     cutoff=cutoff, organism=organism,\n",
    "                     \n",
    "                     # if you do not have the below info, contact Kevin.\n",
    "                     annotation=GENE_DIR+'gene_info.csv',\n",
    "                     trn=GENE_DIR+'TRN.csv')\n",
    "\n",
    "# load the curated_enrichments file\n",
    "enrich = pd.read_csv(DATA_DIR+'curated_enrichments.csv', index_col=0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# load py files\n",
    "sys.path.append(PY_DIR)\n",
    "from gene_table import *\n",
    "from gene_histogram import *\n",
    "from gene_scatter import *\n",
    "from activity_bar import *\n",
    "from gene_activity_bar import *\n",
    "from regulon_venn import *\n",
    "from regulon_scatter import *\n",
    "from general_functions import *"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Check that the files are formatted correctly\n",
    "\n",
    "### Metadata file (DATA_DIR+'sample_metadata.csv')\n",
    "\n",
    "This lists each sample (indexed the same as X and A) and must contain a condition_id and project_id column. The project_ids define groups that will be separated by vertical lines in the A matrix plot -- if you are working on the modulome, these should separate each BioProject and be named uniquely and descriptively (e.g. 'phage_infection', 'antibiotic_panel', etc). The condition_ids should describe the condition (e.g. 'wt_ctrl', 'kanamycin_supp'). Each pair of project_ids and condition_ids should define a single biological condition. Therefore, you can have 'wt_ctrl' in multiple projects, but any samples that share the same condition_id and project_id must be biological replicates from the same condition. Any additional columns will be accessible within the site, but are not required."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ADDRESS THESE IF THEY FAIL\n",
    "# can find correct columns\n",
    "assert('condition_id' in ica_data.metadata.columns)\n",
    "assert('project_id' in ica_data.metadata.columns)\n",
    "\n",
    "# can match to the data\n",
    "assert(ica_data.X.columns.isin(ica_data.metadata.index).all())\n",
    "assert(ica_data.A.columns.isin(ica_data.metadata.index).all())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To ensure that the labels are correct, run the following code block. You will see separate tables of metadata for each set of biological replicates. Make sure that all samples listed in each table are biological replicates of each other. If not, edit the metadata file with new condition_ids."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale1__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/7/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.1.124.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A1 F124 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-1_S1_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-1_S1_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale1__1.bam</td>\n",
       "      <td>98.4</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale1__1  42C Evolution        42c     42c_ale1       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale1__1  Ryan LaCroix  Richard Szubin  5/7/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale1__1  Escherichia coli 42C.1.124.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale1__1              MiSeq     42C A1 F124 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale1__1  42c/ALE-42-glu-end-point-Clone-1_S1_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale1__1  42c/ALE-42-glu-end-point-Clone-1_S1_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale1__1  42c__42c_ale1__1.bam       98.4   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale1__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale10__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale10</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/3/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.10.153.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A10 F153 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-10_S2_L001_R1_0...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-10_S2_L001_R2_0...</td>\n",
       "      <td>42c__42c_ale10__1.bam</td>\n",
       "      <td>96.91</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                            study project_id condition_id  rep_id  \\\n",
       "sample_id                                                           \n",
       "X42c__42c_ale10__1  42C Evolution        42c    42c_ale10       1   \n",
       "\n",
       "                         contact library creator       run_date  \\\n",
       "sample_id                                                         \n",
       "X42c__42c_ale10__1  Ryan LaCroix  Richard Szubin  5/3/2013 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "X42c__42c_ale10__1  Escherichia coli 42C.10.153.1   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "X42c__42c_ale10__1              MiSeq    42C A10 F153 I1   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "X42c__42c_ale10__1                     1   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "X42c__42c_ale10__1  42c/ALE-42-glu-end-point-Clone-10_S2_L001_R1_0...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "X42c__42c_ale10__1  42c/ALE-42-glu-end-point-Clone-10_S2_L001_R2_0...   \n",
       "\n",
       "                                      BAM  alignment  \\\n",
       "sample_id                                              \n",
       "X42c__42c_ale10__1  42c__42c_ale10__1.bam      96.91   \n",
       "\n",
       "                                              DOI        GEO  Public  \n",
       "sample_id                                                             \n",
       "X42c__42c_ale10__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale2__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/2/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.2.163.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A2 F163 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-2_S1_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-2_S1_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale2__1.bam</td>\n",
       "      <td>97.51</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale2__1  42C Evolution        42c     42c_ale2       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale2__1  Ryan LaCroix  Richard Szubin  5/2/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale2__1  Escherichia coli 42C.2.163.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale2__1              MiSeq     42C A2 F163 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale2__1  42c/ALE-42-glu-end-point-Clone-2_S1_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale2__1  42c/ALE-42-glu-end-point-Clone-2_S1_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale2__1  42c__42c_ale2__1.bam      97.51   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale2__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale3__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/1/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.3.120.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A3 F120 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-3_S2_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-3_S2_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale3__1.bam</td>\n",
       "      <td>97.13</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale3__1  42C Evolution        42c     42c_ale3       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale3__1  Ryan LaCroix  Richard Szubin  5/1/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale3__1  Escherichia coli 42C.3.120.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale3__1              MiSeq     42C A3 F120 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale3__1  42c/ALE-42-glu-end-point-Clone-3_S2_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale3__1  42c/ALE-42-glu-end-point-Clone-3_S2_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale3__1  42c__42c_ale3__1.bam      97.13   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale3__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale4__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale4</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/2/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.4.161.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A4 F161 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-4_S1_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-4_S1_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale4__1.bam</td>\n",
       "      <td>97.07</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale4__1  42C Evolution        42c     42c_ale4       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale4__1  Ryan LaCroix  Richard Szubin  5/2/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale4__1  Escherichia coli 42C.4.161.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale4__1              MiSeq     42C A4 F161 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale4__1  42c/ALE-42-glu-end-point-Clone-4_S1_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale4__1  42c/ALE-42-glu-end-point-Clone-4_S1_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale4__1  42c__42c_ale4__1.bam      97.07   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale4__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale5__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale5</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/2/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.5.125.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A5 F125 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-5_S2_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-5_S2_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale5__1.bam</td>\n",
       "      <td>97.06</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale5__1  42C Evolution        42c     42c_ale5       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale5__1  Ryan LaCroix  Richard Szubin  5/2/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale5__1  Escherichia coli 42C.5.125.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale5__1              MiSeq     42C A5 F125 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale5__1  42c/ALE-42-glu-end-point-Clone-5_S2_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale5__1  42c/ALE-42-glu-end-point-Clone-5_S2_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale5__1  42c__42c_ale5__1.bam      97.06   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale5__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale6__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale6</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/3/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.6.164.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A6 F164 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-6_S1_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-6_S1_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale6__1.bam</td>\n",
       "      <td>97.77</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale6__1  42C Evolution        42c     42c_ale6       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale6__1  Ryan LaCroix  Richard Szubin  5/3/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale6__1  Escherichia coli 42C.6.164.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale6__1              MiSeq     42C A6 F164 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale6__1  42c/ALE-42-glu-end-point-Clone-6_S1_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale6__1  42c/ALE-42-glu-end-point-Clone-6_S1_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale6__1  42c__42c_ale6__1.bam      97.77   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale6__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale8__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale8</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/2/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.8.164.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A8 F164 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-8_S2_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-8_S2_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale8__1.bam</td>\n",
       "      <td>97.04</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale8__1  42C Evolution        42c     42c_ale8       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale8__1  Ryan LaCroix  Richard Szubin  5/2/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale8__1  Escherichia coli 42C.8.164.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale8__1              MiSeq     42C A8 F164 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale8__1  42c/ALE-42-glu-end-point-Clone-8_S2_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale8__1  42c/ALE-42-glu-end-point-Clone-8_S2_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale8__1  42c__42c_ale8__1.bam      97.04   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale8__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__42c_ale9__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>42c_ale9</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/3/2013 0:00</td>\n",
       "      <td>Escherichia coli 42C.9.131.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>42C A9 F131 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-9_S2_L001_R1_00...</td>\n",
       "      <td>42c/ALE-42-glu-end-point-Clone-9_S2_L001_R2_00...</td>\n",
       "      <td>42c__42c_ale9__1.bam</td>\n",
       "      <td>96.38</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                           study project_id condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "X42c__42c_ale9__1  42C Evolution        42c     42c_ale9       1   \n",
       "\n",
       "                        contact library creator       run_date  \\\n",
       "sample_id                                                        \n",
       "X42c__42c_ale9__1  Ryan LaCroix  Richard Szubin  5/3/2013 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "X42c__42c_ale9__1  Escherichia coli 42C.9.131.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "X42c__42c_ale9__1              MiSeq     42C A9 F131 I1                     1   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale9__1  42c/ALE-42-glu-end-point-Clone-9_S2_L001_R1_00...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "X42c__42c_ale9__1  42c/ALE-42-glu-end-point-Clone-9_S2_L001_R2_00...   \n",
       "\n",
       "                                    BAM  alignment  \\\n",
       "sample_id                                            \n",
       "X42c__42c_ale9__1  42c__42c_ale9__1.bam      96.38   \n",
       "\n",
       "                                             DOI        GEO  Public  \n",
       "sample_id                                                            \n",
       "X42c__42c_ale9__1  doi.org/10.1093/molbev/msu209  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>X42c__wt_42c__1</td>\n",
       "      <td>42C Evolution</td>\n",
       "      <td>42c</td>\n",
       "      <td>wt_42c</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/1/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>42c/42-degrees-ALE_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>42c/42-degrees-ALE_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>42c__wt_42c__1.bam</td>\n",
       "      <td>98.37</td>\n",
       "      <td>doi.org/10.1093/molbev/msu209</td>\n",
       "      <td>GSE132442</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "X42c__wt_42c__1  42C Evolution        42c       wt_42c       1  Ryan LaCroix   \n",
       "\n",
       "                library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                      \n",
       "X42c__wt_42c__1  Richard Szubin  5/1/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "X42c__wt_42c__1   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "X42c__wt_42c__1                NaN                     1   \n",
       "\n",
       "                                                         R1  \\\n",
       "sample_id                                                     \n",
       "X42c__wt_42c__1  42c/42-degrees-ALE_S1_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                         R2  \\\n",
       "sample_id                                                     \n",
       "X42c__wt_42c__1  42c/42-degrees-ALE_S1_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                BAM  alignment                            DOI  \\\n",
       "sample_id                                                                       \n",
       "X42c__wt_42c__1  42c__wt_42c__1.bam      98.37  doi.org/10.1093/molbev/msu209   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "X42c__wt_42c__1  GSE132442     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>acid__delgade_ph5__1</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgade_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/16/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadE</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadE-pH-5-5-1_S1_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadE-pH-5-5-1_S1_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgade_ph5__1.bam</td>\n",
       "      <td>97.82</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>acid__delgade_ph5__2</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgade_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/16/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadE</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadE-pH-5-5-2_S2_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadE-pH-5-5-2_S2_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgade_ph5__2.bam</td>\n",
       "      <td>98.00</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                     study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                 \n",
       "acid__delgade_ph5__1  Acid       acid  delgade_ph5       1  Sangwoo Seo   \n",
       "acid__delgade_ph5__2  Acid       acid  delgade_ph5       2  Sangwoo Seo   \n",
       "\n",
       "                     library creator        run_date  \\\n",
       "sample_id                                              \n",
       "acid__delgade_ph5__1     Sangwoo Seo  7/16/2014 0:00   \n",
       "acid__delgade_ph5__2     Sangwoo Seo  7/16/2014 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "acid__delgade_ph5__1  Escherichia coli K-12 MG1655 del_gadE   MG1655   \n",
       "acid__delgade_ph5__2  Escherichia coli K-12 MG1655 del_gadE   MG1655   \n",
       "\n",
       "                     Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                        ...                                         \n",
       "acid__delgade_ph5__1         M9  ...              MiSeq                NaN   \n",
       "acid__delgade_ph5__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                     Biological Replicates  \\\n",
       "sample_id                                    \n",
       "acid__delgade_ph5__1                     2   \n",
       "acid__delgade_ph5__2                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "acid__delgade_ph5__1  acid/RNAseq-dgadE-pH-5-5-1_S1_L001_R1_001.fast...   \n",
       "acid__delgade_ph5__2  acid/RNAseq-dgadE-pH-5-5-2_S2_L001_R1_001.fast...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "acid__delgade_ph5__1  acid/RNAseq-dgadE-pH-5-5-1_S1_L001_R2_001.fast...   \n",
       "acid__delgade_ph5__2  acid/RNAseq-dgadE-pH-5-5-2_S2_L001_R2_001.fast...   \n",
       "\n",
       "                                           BAM  alignment  \\\n",
       "sample_id                                                   \n",
       "acid__delgade_ph5__1  acid__delgade_ph5__1.bam      97.82   \n",
       "acid__delgade_ph5__2  acid__delgade_ph5__2.bam      98.00   \n",
       "\n",
       "                                             DOI       GEO  Public  \n",
       "sample_id                                                           \n",
       "acid__delgade_ph5__1  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "acid__delgade_ph5__2  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>acid__delgadw_ph5__1</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgadw_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/16/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadW</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadW-pH-5-5-1_S1_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadW-pH-5-5-1_S1_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgadw_ph5__1.bam</td>\n",
       "      <td>97.77</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>acid__delgadw_ph5__2</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgadw_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/16/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadW</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadW-pH-5-5-2_S2_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadW-pH-5-5-2_S2_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgadw_ph5__2.bam</td>\n",
       "      <td>97.80</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                     study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadw_ph5__1  Acid       acid  delgadw_ph5       1  Sangwoo Seo   \n",
       "acid__delgadw_ph5__2  Acid       acid  delgadw_ph5       2  Sangwoo Seo   \n",
       "\n",
       "                     library creator        run_date  \\\n",
       "sample_id                                              \n",
       "acid__delgadw_ph5__1     Sangwoo Seo  7/16/2014 0:00   \n",
       "acid__delgadw_ph5__2     Sangwoo Seo  7/16/2014 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "acid__delgadw_ph5__1  Escherichia coli K-12 MG1655 del_gadW   MG1655   \n",
       "acid__delgadw_ph5__2  Escherichia coli K-12 MG1655 del_gadW   MG1655   \n",
       "\n",
       "                     Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                        ...                                         \n",
       "acid__delgadw_ph5__1         M9  ...              MiSeq                NaN   \n",
       "acid__delgadw_ph5__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                     Biological Replicates  \\\n",
       "sample_id                                    \n",
       "acid__delgadw_ph5__1                     2   \n",
       "acid__delgadw_ph5__2                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadw_ph5__1  acid/RNAseq-dgadW-pH-5-5-1_S1_L001_R1_001.fast...   \n",
       "acid__delgadw_ph5__2  acid/RNAseq-dgadW-pH-5-5-2_S2_L001_R1_001.fast...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadw_ph5__1  acid/RNAseq-dgadW-pH-5-5-1_S1_L001_R2_001.fast...   \n",
       "acid__delgadw_ph5__2  acid/RNAseq-dgadW-pH-5-5-2_S2_L001_R2_001.fast...   \n",
       "\n",
       "                                           BAM  alignment  \\\n",
       "sample_id                                                   \n",
       "acid__delgadw_ph5__1  acid__delgadw_ph5__1.bam      97.77   \n",
       "acid__delgadw_ph5__2  acid__delgadw_ph5__2.bam      97.80   \n",
       "\n",
       "                                             DOI       GEO  Public  \n",
       "sample_id                                                           \n",
       "acid__delgadw_ph5__1  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "acid__delgadw_ph5__2  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>acid__delgadx_ph5__1</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgadx_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/17/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadX</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadX-pH-5-5-1_S1_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadX-pH-5-5-1_S1_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgadx_ph5__1.bam</td>\n",
       "      <td>46.76</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>acid__delgadx_ph5__2</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>delgadx_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/17/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_gadX</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-dgadX-pH-5-5-2_S2_L001_R1_001.fast...</td>\n",
       "      <td>acid/RNAseq-dgadX-pH-5-5-2_S2_L001_R2_001.fast...</td>\n",
       "      <td>acid__delgadx_ph5__2.bam</td>\n",
       "      <td>78.31</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                     study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadx_ph5__1  Acid       acid  delgadx_ph5       1  Sangwoo Seo   \n",
       "acid__delgadx_ph5__2  Acid       acid  delgadx_ph5       2  Sangwoo Seo   \n",
       "\n",
       "                     library creator        run_date  \\\n",
       "sample_id                                              \n",
       "acid__delgadx_ph5__1     Sangwoo Seo  7/17/2014 0:00   \n",
       "acid__delgadx_ph5__2     Sangwoo Seo  7/17/2014 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "acid__delgadx_ph5__1  Escherichia coli K-12 MG1655 del_gadX   MG1655   \n",
       "acid__delgadx_ph5__2  Escherichia coli K-12 MG1655 del_gadX   MG1655   \n",
       "\n",
       "                     Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                        ...                                         \n",
       "acid__delgadx_ph5__1         M9  ...              MiSeq                NaN   \n",
       "acid__delgadx_ph5__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                     Biological Replicates  \\\n",
       "sample_id                                    \n",
       "acid__delgadx_ph5__1                     2   \n",
       "acid__delgadx_ph5__2                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadx_ph5__1  acid/RNAseq-dgadX-pH-5-5-1_S1_L001_R1_001.fast...   \n",
       "acid__delgadx_ph5__2  acid/RNAseq-dgadX-pH-5-5-2_S2_L001_R1_001.fast...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "acid__delgadx_ph5__1  acid/RNAseq-dgadX-pH-5-5-1_S1_L001_R2_001.fast...   \n",
       "acid__delgadx_ph5__2  acid/RNAseq-dgadX-pH-5-5-2_S2_L001_R2_001.fast...   \n",
       "\n",
       "                                           BAM  alignment  \\\n",
       "sample_id                                                   \n",
       "acid__delgadx_ph5__1  acid__delgadx_ph5__1.bam      46.76   \n",
       "acid__delgadx_ph5__2  acid__delgadx_ph5__2.bam      78.31   \n",
       "\n",
       "                                             DOI       GEO  Public  \n",
       "sample_id                                                           \n",
       "acid__delgadx_ph5__1  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "acid__delgadx_ph5__2  doi.org/10.1038/ncomms8970  GSE66482     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>acid__wt_ph5__1</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>wt_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/15/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-WT-pH-5-5-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>acid/RNAseq-WT-pH-5-5-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>acid__wt_ph5__1.bam</td>\n",
       "      <td>97.99</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>acid__wt_ph5__2</td>\n",
       "      <td>Acid</td>\n",
       "      <td>acid</td>\n",
       "      <td>wt_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/15/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>acid/RNAseq-WT-pH-5-5-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>acid/RNAseq-WT-pH-5-5-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>acid__wt_ph5__2.bam</td>\n",
       "      <td>98.37</td>\n",
       "      <td>doi.org/10.1038/ncomms8970</td>\n",
       "      <td>GSE66482</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                            \n",
       "acid__wt_ph5__1  Acid       acid       wt_ph5       1  Sangwoo Seo   \n",
       "acid__wt_ph5__2  Acid       acid       wt_ph5       2  Sangwoo Seo   \n",
       "\n",
       "                library creator        run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "acid__wt_ph5__1     Sangwoo Seo  7/15/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "acid__wt_ph5__2     Sangwoo Seo  7/15/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "acid__wt_ph5__1   MG1655         M9  ...              MiSeq   \n",
       "acid__wt_ph5__2   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "acid__wt_ph5__1                NaN                     2   \n",
       "acid__wt_ph5__2                NaN                     2   \n",
       "\n",
       "                                                              R1  \\\n",
       "sample_id                                                          \n",
       "acid__wt_ph5__1  acid/RNAseq-WT-pH-5-5-1_S1_L001_R1_001.fastq.gz   \n",
       "acid__wt_ph5__2  acid/RNAseq-WT-pH-5-5-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                              R2  \\\n",
       "sample_id                                                          \n",
       "acid__wt_ph5__1  acid/RNAseq-WT-pH-5-5-1_S1_L001_R2_001.fastq.gz   \n",
       "acid__wt_ph5__2  acid/RNAseq-WT-pH-5-5-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "acid__wt_ph5__1  acid__wt_ph5__1.bam      97.99  doi.org/10.1038/ncomms8970   \n",
       "acid__wt_ph5__2  acid__wt_ph5__2.bam      98.37  doi.org/10.1038/ncomms8970   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "acid__wt_ph5__1  GSE66482     1.0  \n",
       "acid__wt_ph5__2  GSE66482     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>control__wt_glc__1</td>\n",
       "      <td>Control</td>\n",
       "      <td>control</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>control/MG1655-M9-glucose-1_S1_L001_R1_001.fas...</td>\n",
       "      <td>control/MG1655-M9-glucose-1_S1_L001_R2_001.fas...</td>\n",
       "      <td>control__wt_glc__1.bam</td>\n",
       "      <td>94.33</td>\n",
       "      <td>doi.org/10.1101/080929</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>control__wt_glc__2</td>\n",
       "      <td>Control</td>\n",
       "      <td>control</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>control/MG1655-M9-glucose-2_S2_L001_R1_001.fas...</td>\n",
       "      <td>control/MG1655-M9-glucose-2_S2_L001_R2_001.fas...</td>\n",
       "      <td>control__wt_glc__2.bam</td>\n",
       "      <td>94.24</td>\n",
       "      <td>doi.org/10.1101/080929</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                  \n",
       "control__wt_glc__1  Control    control       wt_glc       1  Sangwoo Seo   \n",
       "control__wt_glc__2  Control    control       wt_glc       2  Sangwoo Seo   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "control__wt_glc__1     Sangwoo Seo  4/17/2013 0:00   \n",
       "control__wt_glc__2     Sangwoo Seo  4/17/2013 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "control__wt_glc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "control__wt_glc__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "control__wt_glc__1              MiSeq                NaN   \n",
       "control__wt_glc__2              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "control__wt_glc__1                     2   \n",
       "control__wt_glc__2                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "control__wt_glc__1  control/MG1655-M9-glucose-1_S1_L001_R1_001.fas...   \n",
       "control__wt_glc__2  control/MG1655-M9-glucose-2_S2_L001_R1_001.fas...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "control__wt_glc__1  control/MG1655-M9-glucose-1_S1_L001_R2_001.fas...   \n",
       "control__wt_glc__2  control/MG1655-M9-glucose-2_S2_L001_R2_001.fas...   \n",
       "\n",
       "                                       BAM  alignment                     DOI  \\\n",
       "sample_id                                                                       \n",
       "control__wt_glc__1  control__wt_glc__1.bam      94.33  doi.org/10.1101/080929   \n",
       "control__wt_glc__2  control__wt_glc__2.bam      94.24  doi.org/10.1101/080929   \n",
       "\n",
       "                         GEO  Public  \n",
       "sample_id                             \n",
       "control__wt_glc__1  GSE65643     1.0  \n",
       "control__wt_glc__2  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_ac__1</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_ac</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/19/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-ace-1_S1_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-ace-1_S1_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_ac__1.bam</td>\n",
       "      <td>98.71</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_ac__2</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_ac</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/19/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-ace-2_S2_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-ace-2_S2_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_ac__2.bam</td>\n",
       "      <td>98.56</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "cra_crp__delcra_ac__1  Cra/Crp    cra_crp    delcra_ac       1  Donghyuk Kim   \n",
       "cra_crp__delcra_ac__2  Cra/Crp    cra_crp    delcra_ac       2  Donghyuk Kim   \n",
       "\n",
       "                      library creator         run_date  \\\n",
       "sample_id                                                \n",
       "cra_crp__delcra_ac__1    Donghyuk Kim  11/19/2014 0:00   \n",
       "cra_crp__delcra_ac__2    Donghyuk Kim  11/19/2014 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "cra_crp__delcra_ac__1  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "cra_crp__delcra_ac__2  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "\n",
       "                      Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                         ...                                         \n",
       "cra_crp__delcra_ac__1         M9  ...              MiSeq                NaN   \n",
       "cra_crp__delcra_ac__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "cra_crp__delcra_ac__1                     2   \n",
       "cra_crp__delcra_ac__2                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "cra_crp__delcra_ac__1  cra_crp/RNAseq-dcra-ace-1_S1_L001_R1_001.fastq   \n",
       "cra_crp__delcra_ac__2  cra_crp/RNAseq-dcra-ace-2_S2_L001_R1_001.fastq   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "cra_crp__delcra_ac__1  cra_crp/RNAseq-dcra-ace-1_S1_L001_R2_001.fastq   \n",
       "cra_crp__delcra_ac__2  cra_crp/RNAseq-dcra-ace-2_S2_L001_R2_001.fastq   \n",
       "\n",
       "                                             BAM  alignment  \\\n",
       "sample_id                                                     \n",
       "cra_crp__delcra_ac__1  cra_crp__delcra_ac__1.bam      98.71   \n",
       "cra_crp__delcra_ac__2  cra_crp__delcra_ac__2.bam      98.56   \n",
       "\n",
       "                                              DOI       GEO  Public  \n",
       "sample_id                                                            \n",
       "cra_crp__delcra_ac__1  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "cra_crp__delcra_ac__2  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_fru__1</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_fru</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/17/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-fru-1_S1_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-fru-1_S1_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_fru__1.bam</td>\n",
       "      <td>98.66</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_fru__2</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_fru</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/17/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-fru-2_S2_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-fru-2_S2_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_fru__2.bam</td>\n",
       "      <td>98.76</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                       \n",
       "cra_crp__delcra_fru__1  Cra/Crp    cra_crp   delcra_fru       1  Donghyuk Kim   \n",
       "cra_crp__delcra_fru__2  Cra/Crp    cra_crp   delcra_fru       2  Donghyuk Kim   \n",
       "\n",
       "                       library creator         run_date  \\\n",
       "sample_id                                                 \n",
       "cra_crp__delcra_fru__1    Donghyuk Kim  11/17/2014 0:00   \n",
       "cra_crp__delcra_fru__2    Donghyuk Kim  11/17/2014 0:00   \n",
       "\n",
       "                                          Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "cra_crp__delcra_fru__1  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "cra_crp__delcra_fru__2  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                          ...                                         \n",
       "cra_crp__delcra_fru__1         M9  ...              MiSeq                NaN   \n",
       "cra_crp__delcra_fru__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                       Biological Replicates  \\\n",
       "sample_id                                      \n",
       "cra_crp__delcra_fru__1                     2   \n",
       "cra_crp__delcra_fru__2                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "cra_crp__delcra_fru__1  cra_crp/RNAseq-dcra-fru-1_S1_L001_R1_001.fastq   \n",
       "cra_crp__delcra_fru__2  cra_crp/RNAseq-dcra-fru-2_S2_L001_R1_001.fastq   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "cra_crp__delcra_fru__1  cra_crp/RNAseq-dcra-fru-1_S1_L001_R2_001.fastq   \n",
       "cra_crp__delcra_fru__2  cra_crp/RNAseq-dcra-fru-2_S2_L001_R2_001.fastq   \n",
       "\n",
       "                                               BAM  alignment  \\\n",
       "sample_id                                                       \n",
       "cra_crp__delcra_fru__1  cra_crp__delcra_fru__1.bam      98.66   \n",
       "cra_crp__delcra_fru__2  cra_crp__delcra_fru__2.bam      98.76   \n",
       "\n",
       "                                               DOI       GEO  Public  \n",
       "sample_id                                                             \n",
       "cra_crp__delcra_fru__1  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "cra_crp__delcra_fru__2  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_glc__1</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/18/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-glc-1_S1_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-glc-1_S1_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_glc__1.bam</td>\n",
       "      <td>98.82</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>cra_crp__delcra_glc__2</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>delcra_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>11/18/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_cra</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/RNAseq-dcra-glc-2_S2_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/RNAseq-dcra-glc-2_S2_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__delcra_glc__2.bam</td>\n",
       "      <td>98.60</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                       \n",
       "cra_crp__delcra_glc__1  Cra/Crp    cra_crp   delcra_glc       1  Donghyuk Kim   \n",
       "cra_crp__delcra_glc__2  Cra/Crp    cra_crp   delcra_glc       2  Donghyuk Kim   \n",
       "\n",
       "                       library creator         run_date  \\\n",
       "sample_id                                                 \n",
       "cra_crp__delcra_glc__1    Donghyuk Kim  11/18/2014 0:00   \n",
       "cra_crp__delcra_glc__2    Donghyuk Kim  11/18/2014 0:00   \n",
       "\n",
       "                                          Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "cra_crp__delcra_glc__1  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "cra_crp__delcra_glc__2  Escherichia coli K-12 MG1655 del_cra   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                          ...                                         \n",
       "cra_crp__delcra_glc__1         M9  ...              MiSeq                NaN   \n",
       "cra_crp__delcra_glc__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                       Biological Replicates  \\\n",
       "sample_id                                      \n",
       "cra_crp__delcra_glc__1                     2   \n",
       "cra_crp__delcra_glc__2                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "cra_crp__delcra_glc__1  cra_crp/RNAseq-dcra-glc-1_S1_L001_R1_001.fastq   \n",
       "cra_crp__delcra_glc__2  cra_crp/RNAseq-dcra-glc-2_S2_L001_R1_001.fastq   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "cra_crp__delcra_glc__1  cra_crp/RNAseq-dcra-glc-1_S1_L001_R2_001.fastq   \n",
       "cra_crp__delcra_glc__2  cra_crp/RNAseq-dcra-glc-2_S2_L001_R2_001.fastq   \n",
       "\n",
       "                                               BAM  alignment  \\\n",
       "sample_id                                                       \n",
       "cra_crp__delcra_glc__1  cra_crp__delcra_glc__1.bam      98.82   \n",
       "cra_crp__delcra_glc__2  cra_crp__delcra_glc__2.bam      98.60   \n",
       "\n",
       "                                               DOI       GEO  Public  \n",
       "sample_id                                                             \n",
       "cra_crp__delcra_glc__1  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "cra_crp__delcra_glc__2  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>cra_crp__wt_ac__1</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>wt_ac</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/MG1655acetate-1_S1_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/MG1655acetate-1_S1_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__wt_ac__1.bam</td>\n",
       "      <td>96.37</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>cra_crp__wt_ac__2</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>wt_ac</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/MG1655acetate-2_S2_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/MG1655acetate-2_S2_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__wt_ac__2.bam</td>\n",
       "      <td>96.01</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                     study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                  \n",
       "cra_crp__wt_ac__1  Cra/Crp    cra_crp        wt_ac       1  Donghyuk Kim   \n",
       "cra_crp__wt_ac__2  Cra/Crp    cra_crp        wt_ac       2  Donghyuk Kim   \n",
       "\n",
       "                  library creator       run_date  \\\n",
       "sample_id                                          \n",
       "cra_crp__wt_ac__1    Donghyuk Kim  7/8/2014 0:00   \n",
       "cra_crp__wt_ac__2    Donghyuk Kim  7/8/2014 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "cra_crp__wt_ac__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "cra_crp__wt_ac__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "cra_crp__wt_ac__1              MiSeq                NaN                     2   \n",
       "cra_crp__wt_ac__2              MiSeq                NaN                     2   \n",
       "\n",
       "                                                             R1  \\\n",
       "sample_id                                                         \n",
       "cra_crp__wt_ac__1  cra_crp/MG1655acetate-1_S1_L001_R1_001.fastq   \n",
       "cra_crp__wt_ac__2  cra_crp/MG1655acetate-2_S2_L001_R1_001.fastq   \n",
       "\n",
       "                                                             R2  \\\n",
       "sample_id                                                         \n",
       "cra_crp__wt_ac__1  cra_crp/MG1655acetate-1_S1_L001_R2_001.fastq   \n",
       "cra_crp__wt_ac__2  cra_crp/MG1655acetate-2_S2_L001_R2_001.fastq   \n",
       "\n",
       "                                     BAM  alignment  \\\n",
       "sample_id                                             \n",
       "cra_crp__wt_ac__1  cra_crp__wt_ac__1.bam      96.37   \n",
       "cra_crp__wt_ac__2  cra_crp__wt_ac__2.bam      96.01   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "cra_crp__wt_ac__1  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "cra_crp__wt_ac__2  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>cra_crp__wt_fru__1</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>wt_fru</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/MG1655fructose1_S1_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/MG1655fructose1_S1_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__wt_fru__1.bam</td>\n",
       "      <td>97.71</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>cra_crp__wt_fru__2</td>\n",
       "      <td>Cra/Crp</td>\n",
       "      <td>cra_crp</td>\n",
       "      <td>wt_fru</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>cra_crp/MG1655fructose2_S2_L001_R1_001.fastq</td>\n",
       "      <td>cra_crp/MG1655fructose2_S2_L001_R2_001.fastq</td>\n",
       "      <td>cra_crp__wt_fru__2.bam</td>\n",
       "      <td>97.65</td>\n",
       "      <td>doi.org/10.1093/nar/gky069</td>\n",
       "      <td>GSE65643</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                   \n",
       "cra_crp__wt_fru__1  Cra/Crp    cra_crp       wt_fru       1  Donghyuk Kim   \n",
       "cra_crp__wt_fru__2  Cra/Crp    cra_crp       wt_fru       2  Donghyuk Kim   \n",
       "\n",
       "                   library creator       run_date  \\\n",
       "sample_id                                           \n",
       "cra_crp__wt_fru__1    Donghyuk Kim  7/8/2014 0:00   \n",
       "cra_crp__wt_fru__2    Donghyuk Kim  7/8/2014 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "cra_crp__wt_fru__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "cra_crp__wt_fru__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "cra_crp__wt_fru__1              MiSeq                NaN   \n",
       "cra_crp__wt_fru__2              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "cra_crp__wt_fru__1                     2   \n",
       "cra_crp__wt_fru__2                     2   \n",
       "\n",
       "                                                              R1  \\\n",
       "sample_id                                                          \n",
       "cra_crp__wt_fru__1  cra_crp/MG1655fructose1_S1_L001_R1_001.fastq   \n",
       "cra_crp__wt_fru__2  cra_crp/MG1655fructose2_S2_L001_R1_001.fastq   \n",
       "\n",
       "                                                              R2  \\\n",
       "sample_id                                                          \n",
       "cra_crp__wt_fru__1  cra_crp/MG1655fructose1_S1_L001_R2_001.fastq   \n",
       "cra_crp__wt_fru__2  cra_crp/MG1655fructose2_S2_L001_R2_001.fastq   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "cra_crp__wt_fru__1  cra_crp__wt_fru__1.bam      97.71   \n",
       "cra_crp__wt_fru__2  cra_crp__wt_fru__2.bam      97.65   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "cra_crp__wt_fru__1  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "cra_crp__wt_fru__2  doi.org/10.1093/nar/gky069  GSE65643     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delar1_glyc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar1_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>crp/SRR1751295_1.fastq;crp/SRR1751296_1.fastq;...</td>\n",
       "      <td>crp/SRR1751295_2.fastq;crp/SRR1751296_2.fastq;...</td>\n",
       "      <td>crp__delar1_glyc__1.bam</td>\n",
       "      <td>94.77</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delar1_glyc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar1_glyc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>crp/SRR1751298_1.fastq;crp/SRR1751299_1.fastq;...</td>\n",
       "      <td>crp/SRR1751298_2.fastq;crp/SRR1751299_2.fastq;...</td>\n",
       "      <td>crp__delar1_glyc__2.bam</td>\n",
       "      <td>95.10</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                     \n",
       "crp__delar1_glyc__1  Crp ARs        crp  delar1_glyc       1  Haythem Latif   \n",
       "crp__delar1_glyc__2  Crp ARs        crp  delar1_glyc       2  Haythem Latif   \n",
       "\n",
       "                    library creator       run_date  \\\n",
       "sample_id                                            \n",
       "crp__delar1_glyc__1   Haythem Latif  4/8/2014 0:00   \n",
       "crp__delar1_glyc__2   Haythem Latif  4/8/2014 0:00   \n",
       "\n",
       "                                          Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "crp__delar1_glyc__1  Escherichia coli K-12 MG1655 crp_delAr1   MG1655   \n",
       "crp__delar1_glyc__2  Escherichia coli K-12 MG1655 crp_delAr1   MG1655   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "crp__delar1_glyc__1         M9  ...              MiSeq                NaN   \n",
       "crp__delar1_glyc__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "crp__delar1_glyc__1                     2   \n",
       "crp__delar1_glyc__2                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "crp__delar1_glyc__1  crp/SRR1751295_1.fastq;crp/SRR1751296_1.fastq;...   \n",
       "crp__delar1_glyc__2  crp/SRR1751298_1.fastq;crp/SRR1751299_1.fastq;...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "crp__delar1_glyc__1  crp/SRR1751295_2.fastq;crp/SRR1751296_2.fastq;...   \n",
       "crp__delar1_glyc__2  crp/SRR1751298_2.fastq;crp/SRR1751299_2.fastq;...   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "crp__delar1_glyc__1  crp__delar1_glyc__1.bam      94.77   \n",
       "crp__delar1_glyc__2  crp__delar1_glyc__2.bam      95.10   \n",
       "\n",
       "                                                      DOI       GEO  Public  \n",
       "sample_id                                                                    \n",
       "crp__delar1_glyc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delar1_glyc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delar1ar2_glyc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar1ar2_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr1delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751301_1.fastq;crp/SRR1751302_1.fastq;...</td>\n",
       "      <td>crp/SRR1751301_2.fastq;crp/SRR1751302_2.fastq;...</td>\n",
       "      <td>crp__delar1ar2_glyc__1.bam</td>\n",
       "      <td>95.49</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delar1ar2_glyc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar1ar2_glyc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr1delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751304_1.fastq;crp/SRR1751305_1.fastq;...</td>\n",
       "      <td>crp/SRR1751304_2.fastq;crp/SRR1751305_2.fastq;...</td>\n",
       "      <td>crp__delar1ar2_glyc__2.bam</td>\n",
       "      <td>95.80</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delar1ar2_glyc__3</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar1ar2_glyc</td>\n",
       "      <td>3</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr1delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751307_1.fastq;crp/SRR1751308_1.fastq;...</td>\n",
       "      <td>crp/SRR1751307_2.fastq;crp/SRR1751308_2.fastq;...</td>\n",
       "      <td>crp__delar1ar2_glyc__3.bam</td>\n",
       "      <td>95.57</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id    condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "crp__delar1ar2_glyc__1  Crp ARs        crp  delar1ar2_glyc       1   \n",
       "crp__delar1ar2_glyc__2  Crp ARs        crp  delar1ar2_glyc       2   \n",
       "crp__delar1ar2_glyc__3  Crp ARs        crp  delar1ar2_glyc       3   \n",
       "\n",
       "                              contact library creator       run_date  \\\n",
       "sample_id                                                              \n",
       "crp__delar1ar2_glyc__1  Haythem Latif   Haythem Latif  4/8/2014 0:00   \n",
       "crp__delar1ar2_glyc__2  Haythem Latif   Haythem Latif  4/8/2014 0:00   \n",
       "crp__delar1ar2_glyc__3  Haythem Latif   Haythem Latif  4/8/2014 0:00   \n",
       "\n",
       "                                                   Strain Description  \\\n",
       "sample_id                                                               \n",
       "crp__delar1ar2_glyc__1  Escherichia coli K-12 MG1655 crp_delAr1delAr2   \n",
       "crp__delar1ar2_glyc__2  Escherichia coli K-12 MG1655 crp_delAr1delAr2   \n",
       "crp__delar1ar2_glyc__3  Escherichia coli K-12 MG1655 crp_delAr1delAr2   \n",
       "\n",
       "                       Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                   ...                      \n",
       "crp__delar1ar2_glyc__1   MG1655         M9  ...              MiSeq   \n",
       "crp__delar1ar2_glyc__2   MG1655         M9  ...              MiSeq   \n",
       "crp__delar1ar2_glyc__3   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                       Additional Details Biological Replicates  \\\n",
       "sample_id                                                         \n",
       "crp__delar1ar2_glyc__1                NaN                     3   \n",
       "crp__delar1ar2_glyc__2                NaN                     3   \n",
       "crp__delar1ar2_glyc__3                NaN                     3   \n",
       "\n",
       "                                                                       R1  \\\n",
       "sample_id                                                                   \n",
       "crp__delar1ar2_glyc__1  crp/SRR1751301_1.fastq;crp/SRR1751302_1.fastq;...   \n",
       "crp__delar1ar2_glyc__2  crp/SRR1751304_1.fastq;crp/SRR1751305_1.fastq;...   \n",
       "crp__delar1ar2_glyc__3  crp/SRR1751307_1.fastq;crp/SRR1751308_1.fastq;...   \n",
       "\n",
       "                                                                       R2  \\\n",
       "sample_id                                                                   \n",
       "crp__delar1ar2_glyc__1  crp/SRR1751301_2.fastq;crp/SRR1751302_2.fastq;...   \n",
       "crp__delar1ar2_glyc__2  crp/SRR1751304_2.fastq;crp/SRR1751305_2.fastq;...   \n",
       "crp__delar1ar2_glyc__3  crp/SRR1751307_2.fastq;crp/SRR1751308_2.fastq;...   \n",
       "\n",
       "                                               BAM  alignment  \\\n",
       "sample_id                                                       \n",
       "crp__delar1ar2_glyc__1  crp__delar1ar2_glyc__1.bam      95.49   \n",
       "crp__delar1ar2_glyc__2  crp__delar1ar2_glyc__2.bam      95.80   \n",
       "crp__delar1ar2_glyc__3  crp__delar1ar2_glyc__3.bam      95.57   \n",
       "\n",
       "                                                         DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "crp__delar1ar2_glyc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delar1ar2_glyc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delar1ar2_glyc__3  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delar2_glyc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar2_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751310_1.fastq;crp/SRR1751311_1.fastq;...</td>\n",
       "      <td>crp/SRR1751310_2.fastq;crp/SRR1751311_2.fastq;...</td>\n",
       "      <td>crp__delar2_glyc__1.bam</td>\n",
       "      <td>96.07</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delar2_glyc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar2_glyc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751313_1.fastq;crp/SRR1751314_1.fastq</td>\n",
       "      <td>crp/SRR1751313_2.fastq;crp/SRR1751314_2.fastq</td>\n",
       "      <td>crp__delar2_glyc__2.bam</td>\n",
       "      <td>96.07</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delar2_glyc__3</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delar2_glyc</td>\n",
       "      <td>3</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>4/8/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 crp_delAr2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751315_1.fastq;crp/SRR1751316_1.fastq;...</td>\n",
       "      <td>crp/SRR1751315_2.fastq;crp/SRR1751316_2.fastq;...</td>\n",
       "      <td>crp__delar2_glyc__3.bam</td>\n",
       "      <td>96.60</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                     \n",
       "crp__delar2_glyc__1  Crp ARs        crp  delar2_glyc       1  Haythem Latif   \n",
       "crp__delar2_glyc__2  Crp ARs        crp  delar2_glyc       2  Haythem Latif   \n",
       "crp__delar2_glyc__3  Crp ARs        crp  delar2_glyc       3  Haythem Latif   \n",
       "\n",
       "                    library creator       run_date  \\\n",
       "sample_id                                            \n",
       "crp__delar2_glyc__1   Haythem Latif  4/8/2014 0:00   \n",
       "crp__delar2_glyc__2   Haythem Latif  4/8/2014 0:00   \n",
       "crp__delar2_glyc__3   Haythem Latif  4/8/2014 0:00   \n",
       "\n",
       "                                          Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "crp__delar2_glyc__1  Escherichia coli K-12 MG1655 crp_delAr2   MG1655   \n",
       "crp__delar2_glyc__2  Escherichia coli K-12 MG1655 crp_delAr2   MG1655   \n",
       "crp__delar2_glyc__3  Escherichia coli K-12 MG1655 crp_delAr2   MG1655   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "crp__delar2_glyc__1         M9  ...              MiSeq                NaN   \n",
       "crp__delar2_glyc__2         M9  ...              MiSeq                NaN   \n",
       "crp__delar2_glyc__3         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "crp__delar2_glyc__1                     3   \n",
       "crp__delar2_glyc__2                     3   \n",
       "crp__delar2_glyc__3                     3   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "crp__delar2_glyc__1  crp/SRR1751310_1.fastq;crp/SRR1751311_1.fastq;...   \n",
       "crp__delar2_glyc__2      crp/SRR1751313_1.fastq;crp/SRR1751314_1.fastq   \n",
       "crp__delar2_glyc__3  crp/SRR1751315_1.fastq;crp/SRR1751316_1.fastq;...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "crp__delar2_glyc__1  crp/SRR1751310_2.fastq;crp/SRR1751311_2.fastq;...   \n",
       "crp__delar2_glyc__2      crp/SRR1751313_2.fastq;crp/SRR1751314_2.fastq   \n",
       "crp__delar2_glyc__3  crp/SRR1751315_2.fastq;crp/SRR1751316_2.fastq;...   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "crp__delar2_glyc__1  crp__delar2_glyc__1.bam      96.07   \n",
       "crp__delar2_glyc__2  crp__delar2_glyc__2.bam      96.07   \n",
       "crp__delar2_glyc__3  crp__delar2_glyc__3.bam      96.60   \n",
       "\n",
       "                                                      DOI       GEO  Public  \n",
       "sample_id                                                                    \n",
       "crp__delar2_glyc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delar2_glyc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delar2_glyc__3  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_fru__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_fru</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751318_1.fastq;crp/SRR1751319_1.fastq</td>\n",
       "      <td>crp/SRR1751318_2.fastq;crp/SRR1751319_2.fastq</td>\n",
       "      <td>crp__delcrp_fru__1.bam</td>\n",
       "      <td>97.28</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_fru__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_fru</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751320_1.fastq;crp/SRR1751321_1.fastq</td>\n",
       "      <td>crp/SRR1751320_2.fastq;crp/SRR1751321_2.fastq</td>\n",
       "      <td>crp__delcrp_fru__2.bam</td>\n",
       "      <td>97.49</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_fru__3</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_fru</td>\n",
       "      <td>3</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751322_1.fastq;crp/SRR1751323_1.fastq</td>\n",
       "      <td>crp/SRR1751322_2.fastq;crp/SRR1751323_2.fastq</td>\n",
       "      <td>crp__delcrp_fru__3.bam</td>\n",
       "      <td>97.18</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                    \n",
       "crp__delcrp_fru__1  Crp ARs        crp   delcrp_fru       1  Haythem Latif   \n",
       "crp__delcrp_fru__2  Crp ARs        crp   delcrp_fru       2  Haythem Latif   \n",
       "crp__delcrp_fru__3  Crp ARs        crp   delcrp_fru       3  Haythem Latif   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "crp__delcrp_fru__1   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_fru__2   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_fru__3   Haythem Latif  3/27/2014 0:00   \n",
       "\n",
       "                                      Strain Description Organism Base Media  \\\n",
       "sample_id                                                                      \n",
       "crp__delcrp_fru__1  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_fru__2  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_fru__3  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "\n",
       "                    ... Sequencing Machine Additional Details  \\\n",
       "sample_id           ...                                         \n",
       "crp__delcrp_fru__1  ...              MiSeq                NaN   \n",
       "crp__delcrp_fru__2  ...              MiSeq                NaN   \n",
       "crp__delcrp_fru__3  ...              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "crp__delcrp_fru__1                     3   \n",
       "crp__delcrp_fru__2                     3   \n",
       "crp__delcrp_fru__3                     3   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "crp__delcrp_fru__1  crp/SRR1751318_1.fastq;crp/SRR1751319_1.fastq   \n",
       "crp__delcrp_fru__2  crp/SRR1751320_1.fastq;crp/SRR1751321_1.fastq   \n",
       "crp__delcrp_fru__3  crp/SRR1751322_1.fastq;crp/SRR1751323_1.fastq   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "crp__delcrp_fru__1  crp/SRR1751318_2.fastq;crp/SRR1751319_2.fastq   \n",
       "crp__delcrp_fru__2  crp/SRR1751320_2.fastq;crp/SRR1751321_2.fastq   \n",
       "crp__delcrp_fru__3  crp/SRR1751322_2.fastq;crp/SRR1751323_2.fastq   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "crp__delcrp_fru__1  crp__delcrp_fru__1.bam      97.28   \n",
       "crp__delcrp_fru__2  crp__delcrp_fru__2.bam      97.49   \n",
       "crp__delcrp_fru__3  crp__delcrp_fru__3.bam      97.18   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "crp__delcrp_fru__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_fru__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_fru__3  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751324_1.fastq;crp/SRR1751325_1.fastq</td>\n",
       "      <td>crp/SRR1751324_2.fastq;crp/SRR1751325_2.fastq</td>\n",
       "      <td>crp__delcrp_glc__1.bam</td>\n",
       "      <td>97.38</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751326_1.fastq;crp/SRR1751327_1.fastq</td>\n",
       "      <td>crp/SRR1751326_2.fastq;crp/SRR1751327_2.fastq</td>\n",
       "      <td>crp__delcrp_glc__2.bam</td>\n",
       "      <td>97.48</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glc__3</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glc</td>\n",
       "      <td>3</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751328_1.fastq;crp/SRR1751329_1.fastq</td>\n",
       "      <td>crp/SRR1751328_2.fastq;crp/SRR1751329_2.fastq</td>\n",
       "      <td>crp__delcrp_glc__3.bam</td>\n",
       "      <td>97.43</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                    \n",
       "crp__delcrp_glc__1  Crp ARs        crp   delcrp_glc       1  Haythem Latif   \n",
       "crp__delcrp_glc__2  Crp ARs        crp   delcrp_glc       2  Haythem Latif   \n",
       "crp__delcrp_glc__3  Crp ARs        crp   delcrp_glc       3  Haythem Latif   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "crp__delcrp_glc__1   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_glc__2   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_glc__3   Haythem Latif  3/27/2014 0:00   \n",
       "\n",
       "                                      Strain Description Organism Base Media  \\\n",
       "sample_id                                                                      \n",
       "crp__delcrp_glc__1  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_glc__2  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_glc__3  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "\n",
       "                    ... Sequencing Machine Additional Details  \\\n",
       "sample_id           ...                                         \n",
       "crp__delcrp_glc__1  ...              MiSeq                NaN   \n",
       "crp__delcrp_glc__2  ...              MiSeq                NaN   \n",
       "crp__delcrp_glc__3  ...              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "crp__delcrp_glc__1                     3   \n",
       "crp__delcrp_glc__2                     3   \n",
       "crp__delcrp_glc__3                     3   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "crp__delcrp_glc__1  crp/SRR1751324_1.fastq;crp/SRR1751325_1.fastq   \n",
       "crp__delcrp_glc__2  crp/SRR1751326_1.fastq;crp/SRR1751327_1.fastq   \n",
       "crp__delcrp_glc__3  crp/SRR1751328_1.fastq;crp/SRR1751329_1.fastq   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "crp__delcrp_glc__1  crp/SRR1751324_2.fastq;crp/SRR1751325_2.fastq   \n",
       "crp__delcrp_glc__2  crp/SRR1751326_2.fastq;crp/SRR1751327_2.fastq   \n",
       "crp__delcrp_glc__3  crp/SRR1751328_2.fastq;crp/SRR1751329_2.fastq   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "crp__delcrp_glc__1  crp__delcrp_glc__1.bam      97.38   \n",
       "crp__delcrp_glc__2  crp__delcrp_glc__2.bam      97.48   \n",
       "crp__delcrp_glc__3  crp__delcrp_glc__3.bam      97.43   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "crp__delcrp_glc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_glc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_glc__3  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glyc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751330_1.fastq;crp/SRR1751331_1.fastq</td>\n",
       "      <td>crp/SRR1751330_2.fastq;crp/SRR1751331_2.fastq</td>\n",
       "      <td>crp__delcrp_glyc__1.bam</td>\n",
       "      <td>97.47</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glyc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glyc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751332_1.fastq;crp/SRR1751333_1.fastq</td>\n",
       "      <td>crp/SRR1751332_2.fastq;crp/SRR1751333_2.fastq</td>\n",
       "      <td>crp__delcrp_glyc__2.bam</td>\n",
       "      <td>97.80</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__delcrp_glyc__3</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>delcrp_glyc</td>\n",
       "      <td>3</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>3/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_crp</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>crp/SRR1751334_1.fastq;crp/SRR1751335_1.fastq</td>\n",
       "      <td>crp/SRR1751334_2.fastq;crp/SRR1751335_2.fastq</td>\n",
       "      <td>crp__delcrp_glyc__3.bam</td>\n",
       "      <td>97.65</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                     \n",
       "crp__delcrp_glyc__1  Crp ARs        crp  delcrp_glyc       1  Haythem Latif   \n",
       "crp__delcrp_glyc__2  Crp ARs        crp  delcrp_glyc       2  Haythem Latif   \n",
       "crp__delcrp_glyc__3  Crp ARs        crp  delcrp_glyc       3  Haythem Latif   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "crp__delcrp_glyc__1   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_glyc__2   Haythem Latif  3/27/2014 0:00   \n",
       "crp__delcrp_glyc__3   Haythem Latif  3/27/2014 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "crp__delcrp_glyc__1  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_glyc__2  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "crp__delcrp_glyc__3  Escherichia coli K-12 MG1655 del_crp   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine Additional Details  \\\n",
       "sample_id            ...                                         \n",
       "crp__delcrp_glyc__1  ...              MiSeq                NaN   \n",
       "crp__delcrp_glyc__2  ...              MiSeq                NaN   \n",
       "crp__delcrp_glyc__3  ...              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "crp__delcrp_glyc__1                     3   \n",
       "crp__delcrp_glyc__2                     3   \n",
       "crp__delcrp_glyc__3                     3   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "crp__delcrp_glyc__1  crp/SRR1751330_1.fastq;crp/SRR1751331_1.fastq   \n",
       "crp__delcrp_glyc__2  crp/SRR1751332_1.fastq;crp/SRR1751333_1.fastq   \n",
       "crp__delcrp_glyc__3  crp/SRR1751334_1.fastq;crp/SRR1751335_1.fastq   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "crp__delcrp_glyc__1  crp/SRR1751330_2.fastq;crp/SRR1751331_2.fastq   \n",
       "crp__delcrp_glyc__2  crp/SRR1751332_2.fastq;crp/SRR1751333_2.fastq   \n",
       "crp__delcrp_glyc__3  crp/SRR1751334_2.fastq;crp/SRR1751335_2.fastq   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "crp__delcrp_glyc__1  crp__delcrp_glyc__1.bam      97.47   \n",
       "crp__delcrp_glyc__2  crp__delcrp_glyc__2.bam      97.80   \n",
       "crp__delcrp_glyc__3  crp__delcrp_glyc__3.bam      97.65   \n",
       "\n",
       "                                                      DOI       GEO  Public  \n",
       "sample_id                                                                    \n",
       "crp__delcrp_glyc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_glyc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__delcrp_glyc__3  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__wt_fru__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>wt_fru</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>8/15/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>crp/SRR1751336_1.fastq;crp/SRR1751337_1.fastq;...</td>\n",
       "      <td>crp/SRR1751336_2.fastq;crp/SRR1751337_2.fastq;...</td>\n",
       "      <td>crp__wt_fru__1.bam</td>\n",
       "      <td>97.94</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                \n",
       "crp__wt_fru__1  Crp ARs        crp       wt_fru       1  Haythem Latif   \n",
       "\n",
       "               library creator        run_date            Strain Description  \\\n",
       "sample_id                                                                      \n",
       "crp__wt_fru__1   Haythem Latif  8/15/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "               Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "crp__wt_fru__1   MG1655         M9  ...              MiSeq                NaN   \n",
       "\n",
       "               Biological Replicates  \\\n",
       "sample_id                              \n",
       "crp__wt_fru__1                     1   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "crp__wt_fru__1  crp/SRR1751336_1.fastq;crp/SRR1751337_1.fastq;...   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "crp__wt_fru__1  crp/SRR1751336_2.fastq;crp/SRR1751337_2.fastq;...   \n",
       "\n",
       "                               BAM  alignment  \\\n",
       "sample_id                                       \n",
       "crp__wt_fru__1  crp__wt_fru__1.bam      97.94   \n",
       "\n",
       "                                                 DOI       GEO  Public  \n",
       "sample_id                                                               \n",
       "crp__wt_fru__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>crp__wt_glyc__1</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>wt_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>8/14/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>crp/SRR1751347_1.fastq;crp/SRR1751348_1.fastq;...</td>\n",
       "      <td>crp/SRR1751347_2.fastq;crp/SRR1751348_2.fastq;...</td>\n",
       "      <td>crp__wt_glyc__1.bam</td>\n",
       "      <td>98.47</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>crp__wt_glyc__2</td>\n",
       "      <td>Crp ARs</td>\n",
       "      <td>crp</td>\n",
       "      <td>wt_glyc</td>\n",
       "      <td>2</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>8/14/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>crp/SRR1751350_1.fastq;crp/SRR1751351_1.fastq;...</td>\n",
       "      <td>crp/SRR1751350_2.fastq;crp/SRR1751351_2.fastq;...</td>\n",
       "      <td>crp__wt_glyc__2.bam</td>\n",
       "      <td>98.20</td>\n",
       "      <td>doi.org/10.1371/journal.pone.0197272</td>\n",
       "      <td>GSE64849</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   study project_id condition_id  rep_id        contact  \\\n",
       "sample_id                                                                 \n",
       "crp__wt_glyc__1  Crp ARs        crp      wt_glyc       1  Haythem Latif   \n",
       "crp__wt_glyc__2  Crp ARs        crp      wt_glyc       2  Haythem Latif   \n",
       "\n",
       "                library creator        run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "crp__wt_glyc__1   Haythem Latif  8/14/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "crp__wt_glyc__2   Haythem Latif  8/14/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "crp__wt_glyc__1   MG1655         M9  ...              MiSeq   \n",
       "crp__wt_glyc__2   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "crp__wt_glyc__1                NaN                     2   \n",
       "crp__wt_glyc__2                NaN                     2   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "crp__wt_glyc__1  crp/SRR1751347_1.fastq;crp/SRR1751348_1.fastq;...   \n",
       "crp__wt_glyc__2  crp/SRR1751350_1.fastq;crp/SRR1751351_1.fastq;...   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "crp__wt_glyc__1  crp/SRR1751347_2.fastq;crp/SRR1751348_2.fastq;...   \n",
       "crp__wt_glyc__2  crp/SRR1751350_2.fastq;crp/SRR1751351_2.fastq;...   \n",
       "\n",
       "                                 BAM  alignment  \\\n",
       "sample_id                                         \n",
       "crp__wt_glyc__1  crp__wt_glyc__1.bam      98.47   \n",
       "crp__wt_glyc__2  crp__wt_glyc__2.bam      98.20   \n",
       "\n",
       "                                                  DOI       GEO  Public  \n",
       "sample_id                                                                \n",
       "crp__wt_glyc__1  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "crp__wt_glyc__2  doi.org/10.1371/journal.pone.0197272  GSE64849     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__delmenFentC__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>delmenFentC</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_menF del_entC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218065_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218065_2.fastq.gz</td>\n",
       "      <td>efeU__delmenFentC__1.bam</td>\n",
       "      <td>95.79</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__delmenFentC__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>delmenFentC</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_menF del_entC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218066_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218066_2.fastq.gz</td>\n",
       "      <td>efeU__delmenFentC__2.bam</td>\n",
       "      <td>95.07</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "efeU__delmenFentC__1  Pseudogene Repair       efeU  delmenFentC       1   \n",
       "efeU__delmenFentC__2  Pseudogene Repair       efeU  delmenFentC       2   \n",
       "\n",
       "                            contact library creator        run_date  \\\n",
       "sample_id                                                             \n",
       "efeU__delmenFentC__1  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "efeU__delmenFentC__2  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "\n",
       "                                                  Strain Description Organism  \\\n",
       "sample_id                                                                       \n",
       "efeU__delmenFentC__1  Escherichia coli K-12 MG1655 del_menF del_entC   MG1655   \n",
       "efeU__delmenFentC__2  Escherichia coli K-12 MG1655 del_menF del_entC   MG1655   \n",
       "\n",
       "                     Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                        ...                                         \n",
       "efeU__delmenFentC__1         M9  ...         HiSeq 4000                NaN   \n",
       "efeU__delmenFentC__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                     Biological Replicates                          R1  \\\n",
       "sample_id                                                                \n",
       "efeU__delmenFentC__1                     2  efeU/SRR8218065_1.fastq.gz   \n",
       "efeU__delmenFentC__2                     2  efeU/SRR8218066_1.fastq.gz   \n",
       "\n",
       "                                              R2                       BAM  \\\n",
       "sample_id                                                                    \n",
       "efeU__delmenFentC__1  efeU/SRR8218065_2.fastq.gz  efeU__delmenFentC__1.bam   \n",
       "efeU__delmenFentC__2  efeU/SRR8218066_2.fastq.gz  efeU__delmenFentC__2.bam   \n",
       "\n",
       "                      alignment                                DOI        GEO  \\\n",
       "sample_id                                                                       \n",
       "efeU__delmenFentC__1      95.79  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "efeU__delmenFentC__2      95.07  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "\n",
       "                      Public  \n",
       "sample_id                     \n",
       "efeU__delmenFentC__1     1.0  \n",
       "efeU__delmenFentC__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__delmenFentC_citr__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>delmenFentC_citr</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_menF del_entC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218063_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218063_2.fastq.gz</td>\n",
       "      <td>efeU__delmenFentC_citr__1.bam</td>\n",
       "      <td>94.04</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__delmenFentC_citr__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>delmenFentC_citr</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_menF del_entC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218064_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218064_2.fastq.gz</td>\n",
       "      <td>efeU__delmenFentC_citr__2.bam</td>\n",
       "      <td>92.15</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       study project_id      condition_id  \\\n",
       "sample_id                                                                   \n",
       "efeU__delmenFentC_citr__1  Pseudogene Repair       efeU  delmenFentC_citr   \n",
       "efeU__delmenFentC_citr__2  Pseudogene Repair       efeU  delmenFentC_citr   \n",
       "\n",
       "                           rep_id        contact library creator  \\\n",
       "sample_id                                                          \n",
       "efeU__delmenFentC_citr__1       1  Amitesh Anand  Richard Szubin   \n",
       "efeU__delmenFentC_citr__2       2  Amitesh Anand  Richard Szubin   \n",
       "\n",
       "                                 run_date  \\\n",
       "sample_id                                   \n",
       "efeU__delmenFentC_citr__1  5/21/2018 0:00   \n",
       "efeU__delmenFentC_citr__2  5/21/2018 0:00   \n",
       "\n",
       "                                                       Strain Description  \\\n",
       "sample_id                                                                   \n",
       "efeU__delmenFentC_citr__1  Escherichia coli K-12 MG1655 del_menF del_entC   \n",
       "efeU__delmenFentC_citr__2  Escherichia coli K-12 MG1655 del_menF del_entC   \n",
       "\n",
       "                          Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                      ...                      \n",
       "efeU__delmenFentC_citr__1   MG1655         M9  ...         HiSeq 4000   \n",
       "efeU__delmenFentC_citr__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                          Additional Details Biological Replicates  \\\n",
       "sample_id                                                            \n",
       "efeU__delmenFentC_citr__1                NaN                     2   \n",
       "efeU__delmenFentC_citr__2                NaN                     2   \n",
       "\n",
       "                                                   R1  \\\n",
       "sample_id                                               \n",
       "efeU__delmenFentC_citr__1  efeU/SRR8218063_1.fastq.gz   \n",
       "efeU__delmenFentC_citr__2  efeU/SRR8218064_1.fastq.gz   \n",
       "\n",
       "                                                   R2  \\\n",
       "sample_id                                               \n",
       "efeU__delmenFentC_citr__1  efeU/SRR8218063_2.fastq.gz   \n",
       "efeU__delmenFentC_citr__2  efeU/SRR8218064_2.fastq.gz   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "efeU__delmenFentC_citr__1  efeU__delmenFentC_citr__1.bam      94.04   \n",
       "efeU__delmenFentC_citr__2  efeU__delmenFentC_citr__2.bam      92.15   \n",
       "\n",
       "                                                         DOI        GEO  \\\n",
       "sample_id                                                                 \n",
       "efeU__delmenFentC_citr__1  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "efeU__delmenFentC_citr__2  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "\n",
       "                           Public  \n",
       "sample_id                          \n",
       "efeU__delmenFentC_citr__1     1.0  \n",
       "efeU__delmenFentC_citr__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale29__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale29</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.29.82.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218069_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218069_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale29__1.bam</td>\n",
       "      <td>93.36</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale29__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale29</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.29.82.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218070_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218070_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale29__2.bam</td>\n",
       "      <td>93.10</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     study project_id    condition_id  rep_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentC_ale29__1  Pseudogene Repair       efeU  menFentC_ale29       1   \n",
       "efeU__menFentC_ale29__2  Pseudogene Repair       efeU  menFentC_ale29       2   \n",
       "\n",
       "                               contact library creator        run_date  \\\n",
       "sample_id                                                                \n",
       "efeU__menFentC_ale29__1  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "efeU__menFentC_ale29__2  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentC_ale29__1  Escherichia coli EEP_menF_entC.29.82.1   MG1655   \n",
       "efeU__menFentC_ale29__2  Escherichia coli EEP_menF_entC.29.82.1   MG1655   \n",
       "\n",
       "                        Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "efeU__menFentC_ale29__1         M9  ...         HiSeq 4000                NaN   \n",
       "efeU__menFentC_ale29__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                        Biological Replicates                          R1  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentC_ale29__1                     2  efeU/SRR8218069_1.fastq.gz   \n",
       "efeU__menFentC_ale29__2                     2  efeU/SRR8218070_1.fastq.gz   \n",
       "\n",
       "                                                 R2  \\\n",
       "sample_id                                             \n",
       "efeU__menFentC_ale29__1  efeU/SRR8218069_2.fastq.gz   \n",
       "efeU__menFentC_ale29__2  efeU/SRR8218070_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "efeU__menFentC_ale29__1  efeU__menFentC_ale29__1.bam      93.36   \n",
       "efeU__menFentC_ale29__2  efeU__menFentC_ale29__2.bam      93.10   \n",
       "\n",
       "                                                       DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "efeU__menFentC_ale29__1  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "efeU__menFentC_ale29__2  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale30__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale30</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.30.83.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218071_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218071_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale30__1.bam</td>\n",
       "      <td>93.94</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale30__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale30</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.30.83.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218072_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218072_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale30__2.bam</td>\n",
       "      <td>92.83</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     study project_id    condition_id  rep_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentC_ale30__1  Pseudogene Repair       efeU  menFentC_ale30       1   \n",
       "efeU__menFentC_ale30__2  Pseudogene Repair       efeU  menFentC_ale30       2   \n",
       "\n",
       "                               contact library creator        run_date  \\\n",
       "sample_id                                                                \n",
       "efeU__menFentC_ale30__1  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "efeU__menFentC_ale30__2  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentC_ale30__1  Escherichia coli EEP_menF_entC.30.83.1   MG1655   \n",
       "efeU__menFentC_ale30__2  Escherichia coli EEP_menF_entC.30.83.1   MG1655   \n",
       "\n",
       "                        Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "efeU__menFentC_ale30__1         M9  ...         HiSeq 4000                NaN   \n",
       "efeU__menFentC_ale30__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                        Biological Replicates                          R1  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentC_ale30__1                     2  efeU/SRR8218071_1.fastq.gz   \n",
       "efeU__menFentC_ale30__2                     2  efeU/SRR8218072_1.fastq.gz   \n",
       "\n",
       "                                                 R2  \\\n",
       "sample_id                                             \n",
       "efeU__menFentC_ale30__1  efeU/SRR8218071_2.fastq.gz   \n",
       "efeU__menFentC_ale30__2  efeU/SRR8218072_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "efeU__menFentC_ale30__1  efeU__menFentC_ale30__1.bam      93.94   \n",
       "efeU__menFentC_ale30__2  efeU__menFentC_ale30__2.bam      92.83   \n",
       "\n",
       "                                                       DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "efeU__menFentC_ale30__1  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "efeU__menFentC_ale30__2  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale39__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale39</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.39.91.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218067_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218067_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale39__1.bam</td>\n",
       "      <td>93.68</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentC_ale39__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentC_ale39</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC.39.91.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218068_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218068_2.fastq.gz</td>\n",
       "      <td>efeU__menFentC_ale39__2.bam</td>\n",
       "      <td>93.85</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                     study project_id    condition_id  rep_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentC_ale39__1  Pseudogene Repair       efeU  menFentC_ale39       1   \n",
       "efeU__menFentC_ale39__2  Pseudogene Repair       efeU  menFentC_ale39       2   \n",
       "\n",
       "                               contact library creator        run_date  \\\n",
       "sample_id                                                                \n",
       "efeU__menFentC_ale39__1  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "efeU__menFentC_ale39__2  Amitesh Anand  Richard Szubin  5/21/2018 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentC_ale39__1  Escherichia coli EEP_menF_entC.39.91.1   MG1655   \n",
       "efeU__menFentC_ale39__2  Escherichia coli EEP_menF_entC.39.91.1   MG1655   \n",
       "\n",
       "                        Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "efeU__menFentC_ale39__1         M9  ...         HiSeq 4000                NaN   \n",
       "efeU__menFentC_ale39__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                        Biological Replicates                          R1  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentC_ale39__1                     2  efeU/SRR8218067_1.fastq.gz   \n",
       "efeU__menFentC_ale39__2                     2  efeU/SRR8218068_1.fastq.gz   \n",
       "\n",
       "                                                 R2  \\\n",
       "sample_id                                             \n",
       "efeU__menFentC_ale39__1  efeU/SRR8218067_2.fastq.gz   \n",
       "efeU__menFentC_ale39__2  efeU/SRR8218068_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "efeU__menFentC_ale39__1  efeU__menFentC_ale39__1.bam      93.68   \n",
       "efeU__menFentC_ale39__2  efeU__menFentC_ale39__2.bam      93.85   \n",
       "\n",
       "                                                       DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "efeU__menFentC_ale39__1  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "efeU__menFentC_ale39__2  doi.org/10.1038/s41564-018-0340-2  GSE122779     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale36__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale36</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.36.86.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218073_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218073_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale36__1.bam</td>\n",
       "      <td>92.90</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale36__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale36</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.36.86.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218074_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218074_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale36__2.bam</td>\n",
       "      <td>93.44</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         study project_id        condition_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale36__1  Pseudogene Repair       efeU  menFentCubiC_ale36   \n",
       "efeU__menFentCubiC_ale36__2  Pseudogene Repair       efeU  menFentCubiC_ale36   \n",
       "\n",
       "                             rep_id        contact library creator  \\\n",
       "sample_id                                                            \n",
       "efeU__menFentCubiC_ale36__1       1  Amitesh Anand  Richard Szubin   \n",
       "efeU__menFentCubiC_ale36__2       2  Amitesh Anand  Richard Szubin   \n",
       "\n",
       "                                   run_date  \\\n",
       "sample_id                                     \n",
       "efeU__menFentCubiC_ale36__1  5/21/2018 0:00   \n",
       "efeU__menFentCubiC_ale36__2  5/21/2018 0:00   \n",
       "\n",
       "                                                      Strain Description  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentCubiC_ale36__1  Escherichia coli EEP_menF_entC_ubiC.36.86.1   \n",
       "efeU__menFentCubiC_ale36__2  Escherichia coli EEP_menF_entC_ubiC.36.86.1   \n",
       "\n",
       "                            Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                        ...                      \n",
       "efeU__menFentCubiC_ale36__1   MG1655         M9  ...         HiSeq 4000   \n",
       "efeU__menFentCubiC_ale36__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale36__1  Also used in quinone study                     2   \n",
       "efeU__menFentCubiC_ale36__2  Also used in quinone study                     2   \n",
       "\n",
       "                                                     R1  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale36__1  efeU/SRR8218073_1.fastq.gz   \n",
       "efeU__menFentCubiC_ale36__2  efeU/SRR8218074_1.fastq.gz   \n",
       "\n",
       "                                                     R2  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale36__1  efeU/SRR8218073_2.fastq.gz   \n",
       "efeU__menFentCubiC_ale36__2  efeU/SRR8218074_2.fastq.gz   \n",
       "\n",
       "                                                         BAM  alignment  \\\n",
       "sample_id                                                                 \n",
       "efeU__menFentCubiC_ale36__1  efeU__menFentCubiC_ale36__1.bam      92.90   \n",
       "efeU__menFentCubiC_ale36__2  efeU__menFentCubiC_ale36__2.bam      93.44   \n",
       "\n",
       "                                                           DOI        GEO  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentCubiC_ale36__1  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "efeU__menFentCubiC_ale36__2  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "\n",
       "                             Public  \n",
       "sample_id                            \n",
       "efeU__menFentCubiC_ale36__1     1.0  \n",
       "efeU__menFentCubiC_ale36__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale37__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale37</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.37.25.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218075_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218075_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale37__1.bam</td>\n",
       "      <td>94.89</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale37__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale37</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.37.25.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218076_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218076_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale37__2.bam</td>\n",
       "      <td>94.62</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         study project_id        condition_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale37__1  Pseudogene Repair       efeU  menFentCubiC_ale37   \n",
       "efeU__menFentCubiC_ale37__2  Pseudogene Repair       efeU  menFentCubiC_ale37   \n",
       "\n",
       "                             rep_id        contact library creator  \\\n",
       "sample_id                                                            \n",
       "efeU__menFentCubiC_ale37__1       1  Amitesh Anand  Richard Szubin   \n",
       "efeU__menFentCubiC_ale37__2       2  Amitesh Anand  Richard Szubin   \n",
       "\n",
       "                                   run_date  \\\n",
       "sample_id                                     \n",
       "efeU__menFentCubiC_ale37__1  5/21/2018 0:00   \n",
       "efeU__menFentCubiC_ale37__2  5/21/2018 0:00   \n",
       "\n",
       "                                                      Strain Description  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentCubiC_ale37__1  Escherichia coli EEP_menF_entC_ubiC.37.25.1   \n",
       "efeU__menFentCubiC_ale37__2  Escherichia coli EEP_menF_entC_ubiC.37.25.1   \n",
       "\n",
       "                            Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                        ...                      \n",
       "efeU__menFentCubiC_ale37__1   MG1655         M9  ...         HiSeq 4000   \n",
       "efeU__menFentCubiC_ale37__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale37__1  Also used in quinone study                     2   \n",
       "efeU__menFentCubiC_ale37__2  Also used in quinone study                     2   \n",
       "\n",
       "                                                     R1  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale37__1  efeU/SRR8218075_1.fastq.gz   \n",
       "efeU__menFentCubiC_ale37__2  efeU/SRR8218076_1.fastq.gz   \n",
       "\n",
       "                                                     R2  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale37__1  efeU/SRR8218075_2.fastq.gz   \n",
       "efeU__menFentCubiC_ale37__2  efeU/SRR8218076_2.fastq.gz   \n",
       "\n",
       "                                                         BAM  alignment  \\\n",
       "sample_id                                                                 \n",
       "efeU__menFentCubiC_ale37__1  efeU__menFentCubiC_ale37__1.bam      94.89   \n",
       "efeU__menFentCubiC_ale37__2  efeU__menFentCubiC_ale37__2.bam      94.62   \n",
       "\n",
       "                                                           DOI        GEO  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentCubiC_ale37__1  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "efeU__menFentCubiC_ale37__2  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "\n",
       "                             Public  \n",
       "sample_id                            \n",
       "efeU__menFentCubiC_ale37__1     1.0  \n",
       "efeU__menFentCubiC_ale37__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale38__1</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale38</td>\n",
       "      <td>1</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.38.28.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218077_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218077_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale38__1.bam</td>\n",
       "      <td>95.65</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>efeU__menFentCubiC_ale38__2</td>\n",
       "      <td>Pseudogene Repair</td>\n",
       "      <td>efeU</td>\n",
       "      <td>menFentCubiC_ale38</td>\n",
       "      <td>2</td>\n",
       "      <td>Amitesh Anand</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/21/2018 0:00</td>\n",
       "      <td>Escherichia coli EEP_menF_entC_ubiC.38.28.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>Also used in quinone study</td>\n",
       "      <td>2</td>\n",
       "      <td>efeU/SRR8218078_1.fastq.gz</td>\n",
       "      <td>efeU/SRR8218078_2.fastq.gz</td>\n",
       "      <td>efeU__menFentCubiC_ale38__2.bam</td>\n",
       "      <td>94.94</td>\n",
       "      <td>doi.org/10.1038/s41564-018-0340-2</td>\n",
       "      <td>GSE122779</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         study project_id        condition_id  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale38__1  Pseudogene Repair       efeU  menFentCubiC_ale38   \n",
       "efeU__menFentCubiC_ale38__2  Pseudogene Repair       efeU  menFentCubiC_ale38   \n",
       "\n",
       "                             rep_id        contact library creator  \\\n",
       "sample_id                                                            \n",
       "efeU__menFentCubiC_ale38__1       1  Amitesh Anand  Richard Szubin   \n",
       "efeU__menFentCubiC_ale38__2       2  Amitesh Anand  Richard Szubin   \n",
       "\n",
       "                                   run_date  \\\n",
       "sample_id                                     \n",
       "efeU__menFentCubiC_ale38__1  5/21/2018 0:00   \n",
       "efeU__menFentCubiC_ale38__2  5/21/2018 0:00   \n",
       "\n",
       "                                                      Strain Description  \\\n",
       "sample_id                                                                  \n",
       "efeU__menFentCubiC_ale38__1  Escherichia coli EEP_menF_entC_ubiC.38.28.1   \n",
       "efeU__menFentCubiC_ale38__2  Escherichia coli EEP_menF_entC_ubiC.38.28.1   \n",
       "\n",
       "                            Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                        ...                      \n",
       "efeU__menFentCubiC_ale38__1   MG1655         M9  ...         HiSeq 4000   \n",
       "efeU__menFentCubiC_ale38__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "efeU__menFentCubiC_ale38__1  Also used in quinone study                     2   \n",
       "efeU__menFentCubiC_ale38__2  Also used in quinone study                     2   \n",
       "\n",
       "                                                     R1  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale38__1  efeU/SRR8218077_1.fastq.gz   \n",
       "efeU__menFentCubiC_ale38__2  efeU/SRR8218078_1.fastq.gz   \n",
       "\n",
       "                                                     R2  \\\n",
       "sample_id                                                 \n",
       "efeU__menFentCubiC_ale38__1  efeU/SRR8218077_2.fastq.gz   \n",
       "efeU__menFentCubiC_ale38__2  efeU/SRR8218078_2.fastq.gz   \n",
       "\n",
       "                                                         BAM  alignment  \\\n",
       "sample_id                                                                 \n",
       "efeU__menFentCubiC_ale38__1  efeU__menFentCubiC_ale38__1.bam      95.65   \n",
       "efeU__menFentCubiC_ale38__2  efeU__menFentCubiC_ale38__2.bam      94.94   \n",
       "\n",
       "                                                           DOI        GEO  \\\n",
       "sample_id                                                                   \n",
       "efeU__menFentCubiC_ale38__1  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "efeU__menFentCubiC_ale38__2  doi.org/10.1038/s41564-018-0340-2  GSE122779   \n",
       "\n",
       "                             Public  \n",
       "sample_id                            \n",
       "efeU__menFentCubiC_ale38__1     1.0  \n",
       "efeU__menFentCubiC_ale38__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_0pq__1.bam</td>\n",
       "      <td>99.55</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_0pq__2.bam</td>\n",
       "      <td>99.44</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__11_83_0pq__1  ROS TALE       ers2    11_83_0pq       1  Kevin Rychel   \n",
       "ers2__11_83_0pq__2  ROS TALE       ers2    11_83_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                    \n",
       "ers2__11_83_0pq__1             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "ers2__11_83_0pq__2             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__11_83_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__11_83_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__11_83_0pq__1                NaN                     2   \n",
       "ers2__11_83_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__11_83_0pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_0pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__11_83_0pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_0pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__11_83_0pq__1  processed_data/ers2__11_83_0pq__1.bam      99.55  NaN   \n",
       "ers2__11_83_0pq__2  processed_data/ers2__11_83_0pq__2.bam      99.44  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__11_83_0pq__1  NaN     NaN  \n",
       "ers2__11_83_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_250pq__1.bam</td>\n",
       "      <td>99.53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_250pq__2.bam</td>\n",
       "      <td>99.48</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__11_83_250pq__1  ROS TALE       ers2  11_83_250pq       1  Kevin Rychel   \n",
       "ers2__11_83_250pq__2  ROS TALE       ers2  11_83_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__11_83_250pq__1             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "ers2__11_83_250pq__2             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__11_83_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__11_83_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__11_83_250pq__1                NaN                     2   \n",
       "ers2__11_83_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__11_83_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__11_83_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__11_83_250pq__1  processed_data/ers2__11_83_250pq__1.bam      99.53  NaN   \n",
       "ers2__11_83_250pq__2  processed_data/ers2__11_83_250pq__2.bam      99.48  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__11_83_250pq__1  NaN     NaN  \n",
       "ers2__11_83_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_750pq__1.bam</td>\n",
       "      <td>99.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__11_83_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>11_83_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.11.83</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__11_83_750pq__2.bam</td>\n",
       "      <td>99.58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__11_83_750pq__1  ROS TALE       ers2  11_83_750pq       1  Kevin Rychel   \n",
       "ers2__11_83_750pq__2  ROS TALE       ers2  11_83_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__11_83_750pq__1             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "ers2__11_83_750pq__2             NaN  10/30/2019  Escherichia Coli ERS.11.83   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__11_83_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__11_83_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__11_83_750pq__1                NaN                     2   \n",
       "ers2__11_83_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__11_83_750pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_750pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__11_83_750pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__11_83_750pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__11_83_750pq__1  processed_data/ers2__11_83_750pq__1.bam      99.54  NaN   \n",
       "ers2__11_83_750pq__2  processed_data/ers2__11_83_750pq__2.bam      99.58  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__11_83_750pq__1  NaN     NaN  \n",
       "ers2__11_83_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_0pq__1.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_0pq__2.bam</td>\n",
       "      <td>99.48</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__13_87_0pq__1  ROS TALE       ers2    13_87_0pq       1  Kevin Rychel   \n",
       "ers2__13_87_0pq__2  ROS TALE       ers2    13_87_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__13_87_0pq__1             NaN  1/21/2020  Escherichia Coli ERS.13.87   \n",
       "ers2__13_87_0pq__2             NaN  1/21/2020  Escherichia Coli ERS.13.87   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__13_87_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__13_87_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__13_87_0pq__1                NaN                     2   \n",
       "ers2__13_87_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__13_87_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__13_87_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__13_87_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__13_87_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__13_87_0pq__1  processed_data/ers2__13_87_0pq__1.bam      99.51  NaN   \n",
       "ers2__13_87_0pq__2  processed_data/ers2__13_87_0pq__2.bam      99.48  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__13_87_0pq__1  NaN     NaN  \n",
       "ers2__13_87_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_250pq__1.bam</td>\n",
       "      <td>99.57</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_250pq__2.bam</td>\n",
       "      <td>99.61</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__13_87_250pq__1  ROS TALE       ers2  13_87_250pq       1  Kevin Rychel   \n",
       "ers2__13_87_250pq__2  ROS TALE       ers2  13_87_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__13_87_250pq__1             NaN  10/30/2019  Escherichia Coli ERS.13.87   \n",
       "ers2__13_87_250pq__2             NaN  10/30/2019  Escherichia Coli ERS.13.87   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__13_87_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__13_87_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__13_87_250pq__1                NaN                     2   \n",
       "ers2__13_87_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__13_87_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__13_87_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__13_87_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__13_87_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__13_87_250pq__1  processed_data/ers2__13_87_250pq__1.bam      99.57  NaN   \n",
       "ers2__13_87_250pq__2  processed_data/ers2__13_87_250pq__2.bam      99.61  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__13_87_250pq__1  NaN     NaN  \n",
       "ers2__13_87_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_750pq__1.bam</td>\n",
       "      <td>99.52</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__13_87_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>13_87_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.13.87</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__13_87_750pq__2.bam</td>\n",
       "      <td>98.84</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__13_87_750pq__1  ROS TALE       ers2  13_87_750pq       1  Kevin Rychel   \n",
       "ers2__13_87_750pq__2  ROS TALE       ers2  13_87_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__13_87_750pq__1             NaN  1/21/2020  Escherichia Coli ERS.13.87   \n",
       "ers2__13_87_750pq__2             NaN  1/21/2020  Escherichia Coli ERS.13.87   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__13_87_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__13_87_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__13_87_750pq__1                NaN                     2   \n",
       "ers2__13_87_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__13_87_750pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__13_87_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__13_87_750pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__13_87_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__13_87_750pq__1  processed_data/ers2__13_87_750pq__1.bam      99.52  NaN   \n",
       "ers2__13_87_750pq__2  processed_data/ers2__13_87_750pq__2.bam      98.84  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__13_87_750pq__1  NaN     NaN  \n",
       "ers2__13_87_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_0pq__1.bam</td>\n",
       "      <td>99.52</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_0pq__2.bam</td>\n",
       "      <td>99.47</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__14_27_0pq__1  ROS TALE       ers2    14_27_0pq       1  Kevin Rychel   \n",
       "ers2__14_27_0pq__2  ROS TALE       ers2    14_27_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__14_27_0pq__1             NaN  8/26/2019  Escherichia Coli ERS.14.27   \n",
       "ers2__14_27_0pq__2             NaN  8/26/2019  Escherichia Coli ERS.14.27   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__14_27_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__14_27_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__14_27_0pq__1                NaN                     2   \n",
       "ers2__14_27_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__14_27_0pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__14_27_0pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__14_27_0pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__14_27_0pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__14_27_0pq__1  processed_data/ers2__14_27_0pq__1.bam      99.52  NaN   \n",
       "ers2__14_27_0pq__2  processed_data/ers2__14_27_0pq__2.bam      99.47  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__14_27_0pq__1  NaN     NaN  \n",
       "ers2__14_27_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_250pq__1.bam</td>\n",
       "      <td>99.44</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_250pq__2.bam</td>\n",
       "      <td>99.43</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__14_27_250pq__1  ROS TALE       ers2  14_27_250pq       1  Kevin Rychel   \n",
       "ers2__14_27_250pq__2  ROS TALE       ers2  14_27_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__14_27_250pq__1             NaN  8/26/2019  Escherichia Coli ERS.14.27   \n",
       "ers2__14_27_250pq__2             NaN  8/26/2019  Escherichia Coli ERS.14.27   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__14_27_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__14_27_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__14_27_250pq__1                NaN                     2   \n",
       "ers2__14_27_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__14_27_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__14_27_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__14_27_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__14_27_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__14_27_250pq__1  processed_data/ers2__14_27_250pq__1.bam      99.44  NaN   \n",
       "ers2__14_27_250pq__2  processed_data/ers2__14_27_250pq__2.bam      99.43  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__14_27_250pq__1  NaN     NaN  \n",
       "ers2__14_27_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_750pq__1.bam</td>\n",
       "      <td>99.47</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__14_27_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>14_27_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.14.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__14_27_750pq__2.bam</td>\n",
       "      <td>99.63</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__14_27_750pq__1  ROS TALE       ers2  14_27_750pq       1  Kevin Rychel   \n",
       "ers2__14_27_750pq__2  ROS TALE       ers2  14_27_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__14_27_750pq__1             NaN  9/20/2019  Escherichia Coli ERS.14.27   \n",
       "ers2__14_27_750pq__2             NaN  1/21/2020  Escherichia Coli ERS.14.27   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__14_27_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__14_27_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__14_27_750pq__1                NaN                     2   \n",
       "ers2__14_27_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__14_27_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__14_27_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__14_27_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__14_27_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__14_27_750pq__1  processed_data/ers2__14_27_750pq__1.bam      99.47  NaN   \n",
       "ers2__14_27_750pq__2  processed_data/ers2__14_27_750pq__2.bam      99.63  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__14_27_750pq__1  NaN     NaN  \n",
       "ers2__14_27_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__16_28_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_28_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.28</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/16_28/ROSHRS_16-28A_RNA_rerun_S...</td>\n",
       "      <td>/raw_data/ers2/16_28/ROSHRS_16-28A_RNA_rerun_S...</td>\n",
       "      <td>processed_data/ers2__16_28_250pq__1.bam</td>\n",
       "      <td>99.58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_28_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_28_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.28</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/16_28/ROSHRS_16-28B_RNA_rerun_S...</td>\n",
       "      <td>/raw_data/ers2/16_28/ROSHRS_16-28B_RNA_rerun_S...</td>\n",
       "      <td>processed_data/ers2__16_28_250pq__2.bam</td>\n",
       "      <td>99.43</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__16_28_250pq__1  ROS TALE       ers2  16_28_250pq       1  Kevin Rychel   \n",
       "ers2__16_28_250pq__2  ROS TALE       ers2  16_28_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__16_28_250pq__1             NaN  11/28/2017  Escherichia Coli ERS.16.28   \n",
       "ers2__16_28_250pq__2             NaN  11/28/2017  Escherichia Coli ERS.16.28   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__16_28_250pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__16_28_250pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__16_28_250pq__1                NaN                     2   \n",
       "ers2__16_28_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_28_250pq__1  /raw_data/ers2/16_28/ROSHRS_16-28A_RNA_rerun_S...   \n",
       "ers2__16_28_250pq__2  /raw_data/ers2/16_28/ROSHRS_16-28B_RNA_rerun_S...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_28_250pq__1  /raw_data/ers2/16_28/ROSHRS_16-28A_RNA_rerun_S...   \n",
       "ers2__16_28_250pq__2  /raw_data/ers2/16_28/ROSHRS_16-28B_RNA_rerun_S...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__16_28_250pq__1  processed_data/ers2__16_28_250pq__1.bam      99.58  NaN   \n",
       "ers2__16_28_250pq__2  processed_data/ers2__16_28_250pq__2.bam      99.43  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__16_28_250pq__1  NaN     NaN  \n",
       "ers2__16_28_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__16_32_0pq__1.bam</td>\n",
       "      <td>99.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__16_32_0pq__2.bam</td>\n",
       "      <td>99.43</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_0pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_0pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32A_0PQ_S0_L00...</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32A_0PQ_S0_L00...</td>\n",
       "      <td>processed_data/ers2__16_32_0pq__3.bam</td>\n",
       "      <td>98.38</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_0pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_0pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32B_0PQ_S0_L00...</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32B_0PQ_S0_L00...</td>\n",
       "      <td>processed_data/ers2__16_32_0pq__4.bam</td>\n",
       "      <td>98.14</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__16_32_0pq__1  ROS TALE       ers2    16_32_0pq       1  Kevin Rychel   \n",
       "ers2__16_32_0pq__2  ROS TALE       ers2    16_32_0pq       2  Kevin Rychel   \n",
       "ers2__16_32_0pq__3  ROS TALE       ers2    16_32_0pq       3  Kevin Rychel   \n",
       "ers2__16_32_0pq__4  ROS TALE       ers2    16_32_0pq       4  Kevin Rychel   \n",
       "\n",
       "                   library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                    \n",
       "ers2__16_32_0pq__1             NaN   9/20/2019  Escherichia Coli ERS.16.32   \n",
       "ers2__16_32_0pq__2             NaN   9/20/2019  Escherichia Coli ERS.16.32   \n",
       "ers2__16_32_0pq__3             NaN  11/28/2017  Escherichia Coli ERS.16.32   \n",
       "ers2__16_32_0pq__4             NaN  11/28/2017  Escherichia Coli ERS.16.32   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__16_32_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__16_32_0pq__2      NaN         M9  ...            NextSeq   \n",
       "ers2__16_32_0pq__3      NaN         M9  ...              HiSeq   \n",
       "ers2__16_32_0pq__4      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__16_32_0pq__1                NaN                     4   \n",
       "ers2__16_32_0pq__2                NaN                     4   \n",
       "ers2__16_32_0pq__3                NaN                     4   \n",
       "ers2__16_32_0pq__4                NaN                     4   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__16_32_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_0pq__3  /raw_data/ers2/16_32/ERS_HRS_16_32A_0PQ_S0_L00...   \n",
       "ers2__16_32_0pq__4  /raw_data/ers2/16_32/ERS_HRS_16_32B_0PQ_S0_L00...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__16_32_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_0pq__3  /raw_data/ers2/16_32/ERS_HRS_16_32A_0PQ_S0_L00...   \n",
       "ers2__16_32_0pq__4  /raw_data/ers2/16_32/ERS_HRS_16_32B_0PQ_S0_L00...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__16_32_0pq__1  processed_data/ers2__16_32_0pq__1.bam      99.46  NaN   \n",
       "ers2__16_32_0pq__2  processed_data/ers2__16_32_0pq__2.bam      99.43  NaN   \n",
       "ers2__16_32_0pq__3  processed_data/ers2__16_32_0pq__3.bam      98.38  NaN   \n",
       "ers2__16_32_0pq__4  processed_data/ers2__16_32_0pq__4.bam      98.14  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__16_32_0pq__1  NaN     NaN  \n",
       "ers2__16_32_0pq__2  NaN     NaN  \n",
       "ers2__16_32_0pq__3  NaN     NaN  \n",
       "ers2__16_32_0pq__4  NaN     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32A_PQ_S0_L000...</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32A_PQ_S0_L000...</td>\n",
       "      <td>processed_data/ers2__16_32_250pq__1.bam</td>\n",
       "      <td>98.28</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32B_PQ_S0_L000...</td>\n",
       "      <td>/raw_data/ers2/16_32/ERS_HRS_16_32B_PQ_S0_L000...</td>\n",
       "      <td>processed_data/ers2__16_32_250pq__2.bam</td>\n",
       "      <td>98.22</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__16_32_250pq__1  ROS TALE       ers2  16_32_250pq       1  Kevin Rychel   \n",
       "ers2__16_32_250pq__2  ROS TALE       ers2  16_32_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__16_32_250pq__1             NaN  11/28/2017  Escherichia Coli ERS.16.32   \n",
       "ers2__16_32_250pq__2             NaN  11/28/2017  Escherichia Coli ERS.16.32   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__16_32_250pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__16_32_250pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__16_32_250pq__1                NaN                     2   \n",
       "ers2__16_32_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_32_250pq__1  /raw_data/ers2/16_32/ERS_HRS_16_32A_PQ_S0_L000...   \n",
       "ers2__16_32_250pq__2  /raw_data/ers2/16_32/ERS_HRS_16_32B_PQ_S0_L000...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_32_250pq__1  /raw_data/ers2/16_32/ERS_HRS_16_32A_PQ_S0_L000...   \n",
       "ers2__16_32_250pq__2  /raw_data/ers2/16_32/ERS_HRS_16_32B_PQ_S0_L000...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__16_32_250pq__1  processed_data/ers2__16_32_250pq__1.bam      98.28  NaN   \n",
       "ers2__16_32_250pq__2  processed_data/ers2__16_32_250pq__2.bam      98.22  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__16_32_250pq__1  NaN     NaN  \n",
       "ers2__16_32_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__16_32_750pq__1.bam</td>\n",
       "      <td>99.62</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__16_32_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>16_32_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.16.32</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__16_32_750pq__2.bam</td>\n",
       "      <td>99.56</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__16_32_750pq__1  ROS TALE       ers2  16_32_750pq       1  Kevin Rychel   \n",
       "ers2__16_32_750pq__2  ROS TALE       ers2  16_32_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__16_32_750pq__1             NaN  9/20/2019  Escherichia Coli ERS.16.32   \n",
       "ers2__16_32_750pq__2             NaN  9/20/2019  Escherichia Coli ERS.16.32   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__16_32_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__16_32_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__16_32_750pq__1                NaN                     2   \n",
       "ers2__16_32_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_32_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__16_32_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__16_32_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__16_32_750pq__1  processed_data/ers2__16_32_750pq__1.bam      99.62  NaN   \n",
       "ers2__16_32_750pq__2  processed_data/ers2__16_32_750pq__2.bam      99.56  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__16_32_750pq__1  NaN     NaN  \n",
       "ers2__16_32_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__17_104_0pq__1.bam</td>\n",
       "      <td>99.56</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__17_104_0pq__2.bam</td>\n",
       "      <td>99.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__17_104_0pq__1  ROS TALE       ers2   17_104_0pq       1  Kevin Rychel   \n",
       "ers2__17_104_0pq__2  ROS TALE       ers2   17_104_0pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator   run_date           Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__17_104_0pq__1             NaN  1/21/2020  Escherichia Coli ERS.17.104   \n",
       "ers2__17_104_0pq__2             NaN  1/21/2020  Escherichia Coli ERS.17.104   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__17_104_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__17_104_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__17_104_0pq__1                NaN                     2   \n",
       "ers2__17_104_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__17_104_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__17_104_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__17_104_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__17_104_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__17_104_0pq__1  processed_data/ers2__17_104_0pq__1.bam      99.56  NaN   \n",
       "ers2__17_104_0pq__2  processed_data/ers2__17_104_0pq__2.bam      99.54  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__17_104_0pq__1  NaN     NaN  \n",
       "ers2__17_104_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__17_104_250pq__1.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__17_104_250pq__2.bam</td>\n",
       "      <td>99.47</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_250pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_250pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__17_104_250pq__3.bam</td>\n",
       "      <td>99.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_250pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_250pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__17_104_250pq__4.bam</td>\n",
       "      <td>99.44</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__17_104_250pq__1  ROS TALE       ers2  17_104_250pq       1   \n",
       "ers2__17_104_250pq__2  ROS TALE       ers2  17_104_250pq       2   \n",
       "ers2__17_104_250pq__3  ROS TALE       ers2  17_104_250pq       3   \n",
       "ers2__17_104_250pq__4  ROS TALE       ers2  17_104_250pq       4   \n",
       "\n",
       "                            contact library creator   run_date  \\\n",
       "sample_id                                                        \n",
       "ers2__17_104_250pq__1  Kevin Rychel             NaN  1/16/2020   \n",
       "ers2__17_104_250pq__2  Kevin Rychel             NaN  1/16/2020   \n",
       "ers2__17_104_250pq__3  Kevin Rychel             NaN  1/16/2020   \n",
       "ers2__17_104_250pq__4  Kevin Rychel             NaN  1/16/2020   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__17_104_250pq__1  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "ers2__17_104_250pq__2  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "ers2__17_104_250pq__3  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "ers2__17_104_250pq__4  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine       Additional Details  \\\n",
       "sample_id                                                           \n",
       "ers2__17_104_250pq__1            NovaSeq                      NaN   \n",
       "ers2__17_104_250pq__2            NovaSeq                      NaN   \n",
       "ers2__17_104_250pq__3            NovaSeq  Spacer development test   \n",
       "ers2__17_104_250pq__4            NovaSeq  Spacer development test   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__17_104_250pq__1                     4   \n",
       "ers2__17_104_250pq__2                     4   \n",
       "ers2__17_104_250pq__3                     4   \n",
       "ers2__17_104_250pq__4                     4   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__17_104_250pq__1  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__2  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__3  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__4  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__17_104_250pq__1  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__2  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__3  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__17_104_250pq__4  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__17_104_250pq__1  processed_data/ers2__17_104_250pq__1.bam      99.51   \n",
       "ers2__17_104_250pq__2  processed_data/ers2__17_104_250pq__2.bam      99.47   \n",
       "ers2__17_104_250pq__3  processed_data/ers2__17_104_250pq__3.bam      99.46   \n",
       "ers2__17_104_250pq__4  processed_data/ers2__17_104_250pq__4.bam      99.44   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__17_104_250pq__1  NaN  NaN     NaN  \n",
       "ers2__17_104_250pq__2  NaN  NaN     NaN  \n",
       "ers2__17_104_250pq__3  NaN  NaN     NaN  \n",
       "ers2__17_104_250pq__4  NaN  NaN     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__17_104_750pq__1.bam</td>\n",
       "      <td>99.53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__17_104_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>17_104_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.17.104</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__17_104_750pq__2.bam</td>\n",
       "      <td>99.53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__17_104_750pq__1  ROS TALE       ers2  17_104_750pq       1   \n",
       "ers2__17_104_750pq__2  ROS TALE       ers2  17_104_750pq       2   \n",
       "\n",
       "                            contact library creator   run_date  \\\n",
       "sample_id                                                        \n",
       "ers2__17_104_750pq__1  Kevin Rychel             NaN  1/21/2020   \n",
       "ers2__17_104_750pq__2  Kevin Rychel             NaN  1/21/2020   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__17_104_750pq__1  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "ers2__17_104_750pq__2  Escherichia Coli ERS.17.104      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "ers2__17_104_750pq__1            NextSeq                NaN   \n",
       "ers2__17_104_750pq__2            NextSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__17_104_750pq__1                     2   \n",
       "ers2__17_104_750pq__2                     2   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__17_104_750pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__17_104_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__17_104_750pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__17_104_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__17_104_750pq__1  processed_data/ers2__17_104_750pq__1.bam      99.53   \n",
       "ers2__17_104_750pq__2  processed_data/ers2__17_104_750pq__2.bam      99.53   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__17_104_750pq__1  NaN  NaN     NaN  \n",
       "ers2__17_104_750pq__2  NaN  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__18_24_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_24_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.24</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/18_24/ROSHRS_18-24A_RNA_rerun_S...</td>\n",
       "      <td>/raw_data/ers2/18_24/ROSHRS_18-24A_RNA_rerun_S...</td>\n",
       "      <td>processed_data/ers2__18_24_250pq__1.bam</td>\n",
       "      <td>99.39</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_24_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_24_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.24</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/18_24/ROSHRS_18-24B_RNA_rerun_S...</td>\n",
       "      <td>/raw_data/ers2/18_24/ROSHRS_18-24B_RNA_rerun_S...</td>\n",
       "      <td>processed_data/ers2__18_24_250pq__2.bam</td>\n",
       "      <td>99.37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__18_24_250pq__1  ROS TALE       ers2  18_24_250pq       1  Kevin Rychel   \n",
       "ers2__18_24_250pq__2  ROS TALE       ers2  18_24_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__18_24_250pq__1             NaN  11/28/2017  Escherichia Coli ERS.18.24   \n",
       "ers2__18_24_250pq__2             NaN  11/28/2017  Escherichia Coli ERS.18.24   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__18_24_250pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__18_24_250pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__18_24_250pq__1                NaN                     2   \n",
       "ers2__18_24_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_24_250pq__1  /raw_data/ers2/18_24/ROSHRS_18-24A_RNA_rerun_S...   \n",
       "ers2__18_24_250pq__2  /raw_data/ers2/18_24/ROSHRS_18-24B_RNA_rerun_S...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_24_250pq__1  /raw_data/ers2/18_24/ROSHRS_18-24A_RNA_rerun_S...   \n",
       "ers2__18_24_250pq__2  /raw_data/ers2/18_24/ROSHRS_18-24B_RNA_rerun_S...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__18_24_250pq__1  processed_data/ers2__18_24_250pq__1.bam      99.39  NaN   \n",
       "ers2__18_24_250pq__2  processed_data/ers2__18_24_250pq__2.bam      99.37  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__18_24_250pq__1  NaN     NaN  \n",
       "ers2__18_24_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__18_36_0pq__1.bam</td>\n",
       "      <td>99.01</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__18_36_0pq__2.bam</td>\n",
       "      <td>98.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_0pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_0pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36A_0PQ_S0_L00...</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36A_0PQ_S0_L00...</td>\n",
       "      <td>processed_data/ers2__18_36_0pq__3.bam</td>\n",
       "      <td>97.89</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_0pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_0pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36B_0PQ_S46_L0...</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36B_0PQ_S46_L0...</td>\n",
       "      <td>processed_data/ers2__18_36_0pq__4.bam</td>\n",
       "      <td>99.60</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__18_36_0pq__1  ROS TALE       ers2    18_36_0pq       1  Kevin Rychel   \n",
       "ers2__18_36_0pq__2  ROS TALE       ers2    18_36_0pq       2  Kevin Rychel   \n",
       "ers2__18_36_0pq__3  ROS TALE       ers2    18_36_0pq       3  Kevin Rychel   \n",
       "ers2__18_36_0pq__4  ROS TALE       ers2    18_36_0pq       4  Kevin Rychel   \n",
       "\n",
       "                   library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                    \n",
       "ers2__18_36_0pq__1             NaN   1/21/2020  Escherichia Coli ERS.18.36   \n",
       "ers2__18_36_0pq__2             NaN   1/21/2020  Escherichia Coli ERS.18.36   \n",
       "ers2__18_36_0pq__3             NaN  11/28/2017  Escherichia Coli ERS.18.36   \n",
       "ers2__18_36_0pq__4             NaN  11/28/2017  Escherichia Coli ERS.18.36   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__18_36_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__18_36_0pq__2      NaN         M9  ...            NextSeq   \n",
       "ers2__18_36_0pq__3      NaN         M9  ...              HiSeq   \n",
       "ers2__18_36_0pq__4      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__18_36_0pq__1                NaN                     4   \n",
       "ers2__18_36_0pq__2                NaN                     4   \n",
       "ers2__18_36_0pq__3                NaN                     4   \n",
       "ers2__18_36_0pq__4                NaN                     4   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__18_36_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__18_36_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__18_36_0pq__3  /raw_data/ers2/18_36/ERS_HRS_18_36A_0PQ_S0_L00...   \n",
       "ers2__18_36_0pq__4  /raw_data/ers2/18_36/ERS_HRS_18_36B_0PQ_S46_L0...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__18_36_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__18_36_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__18_36_0pq__3  /raw_data/ers2/18_36/ERS_HRS_18_36A_0PQ_S0_L00...   \n",
       "ers2__18_36_0pq__4  /raw_data/ers2/18_36/ERS_HRS_18_36B_0PQ_S46_L0...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__18_36_0pq__1  processed_data/ers2__18_36_0pq__1.bam      99.01  NaN   \n",
       "ers2__18_36_0pq__2  processed_data/ers2__18_36_0pq__2.bam      98.97  NaN   \n",
       "ers2__18_36_0pq__3  processed_data/ers2__18_36_0pq__3.bam      97.89  NaN   \n",
       "ers2__18_36_0pq__4  processed_data/ers2__18_36_0pq__4.bam      99.60  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__18_36_0pq__1  NaN     NaN  \n",
       "ers2__18_36_0pq__2  NaN     NaN  \n",
       "ers2__18_36_0pq__3  NaN     NaN  \n",
       "ers2__18_36_0pq__4  NaN     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36A_PQ_S46_L00...</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36A_PQ_S46_L00...</td>\n",
       "      <td>processed_data/ers2__18_36_250pq__1.bam</td>\n",
       "      <td>99.40</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36B_PQ_S46_L00...</td>\n",
       "      <td>/raw_data/ers2/18_36/ERS_HRS_18_36B_PQ_S46_L00...</td>\n",
       "      <td>processed_data/ers2__18_36_250pq__2.bam</td>\n",
       "      <td>99.37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__18_36_250pq__1  ROS TALE       ers2  18_36_250pq       1  Kevin Rychel   \n",
       "ers2__18_36_250pq__2  ROS TALE       ers2  18_36_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date          Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__18_36_250pq__1             NaN  11/28/2017  Escherichia Coli ERS.18.36   \n",
       "ers2__18_36_250pq__2             NaN  11/28/2017  Escherichia Coli ERS.18.36   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__18_36_250pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__18_36_250pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__18_36_250pq__1                NaN                     2   \n",
       "ers2__18_36_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_36_250pq__1  /raw_data/ers2/18_36/ERS_HRS_18_36A_PQ_S46_L00...   \n",
       "ers2__18_36_250pq__2  /raw_data/ers2/18_36/ERS_HRS_18_36B_PQ_S46_L00...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_36_250pq__1  /raw_data/ers2/18_36/ERS_HRS_18_36A_PQ_S46_L00...   \n",
       "ers2__18_36_250pq__2  /raw_data/ers2/18_36/ERS_HRS_18_36B_PQ_S46_L00...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__18_36_250pq__1  processed_data/ers2__18_36_250pq__1.bam      99.40  NaN   \n",
       "ers2__18_36_250pq__2  processed_data/ers2__18_36_250pq__2.bam      99.37  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__18_36_250pq__1  NaN     NaN  \n",
       "ers2__18_36_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__18_36_750pq__1.bam</td>\n",
       "      <td>99.31</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__18_36_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>18_36_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.18.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__18_36_750pq__2.bam</td>\n",
       "      <td>99.48</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__18_36_750pq__1  ROS TALE       ers2  18_36_750pq       1  Kevin Rychel   \n",
       "ers2__18_36_750pq__2  ROS TALE       ers2  18_36_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__18_36_750pq__1             NaN  1/20/2020  Escherichia Coli ERS.18.36   \n",
       "ers2__18_36_750pq__2             NaN  1/20/2020  Escherichia Coli ERS.18.36   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__18_36_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__18_36_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__18_36_750pq__1                NaN                     2   \n",
       "ers2__18_36_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_36_750pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__18_36_750pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__18_36_750pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__18_36_750pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__18_36_750pq__1  processed_data/ers2__18_36_750pq__1.bam      99.31  NaN   \n",
       "ers2__18_36_750pq__2  processed_data/ers2__18_36_750pq__2.bam      99.48  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__18_36_750pq__1  NaN     NaN  \n",
       "ers2__18_36_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_0pq__1.bam</td>\n",
       "      <td>99.25</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_0pq__2.bam</td>\n",
       "      <td>99.31</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__19_91_0pq__1  ROS TALE       ers2    19_91_0pq       1  Kevin Rychel   \n",
       "ers2__19_91_0pq__2  ROS TALE       ers2    19_91_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__19_91_0pq__1             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "ers2__19_91_0pq__2             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__19_91_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__19_91_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__19_91_0pq__1                NaN                     2   \n",
       "ers2__19_91_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__19_91_0pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_0pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__19_91_0pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_0pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__19_91_0pq__1  processed_data/ers2__19_91_0pq__1.bam      99.25  NaN   \n",
       "ers2__19_91_0pq__2  processed_data/ers2__19_91_0pq__2.bam      99.31  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__19_91_0pq__1  NaN     NaN  \n",
       "ers2__19_91_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_250pq__1.bam</td>\n",
       "      <td>99.53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_250pq__2.bam</td>\n",
       "      <td>99.50</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__19_91_250pq__1  ROS TALE       ers2  19_91_250pq       1  Kevin Rychel   \n",
       "ers2__19_91_250pq__2  ROS TALE       ers2  19_91_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__19_91_250pq__1             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "ers2__19_91_250pq__2             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__19_91_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__19_91_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__19_91_250pq__1                NaN                     2   \n",
       "ers2__19_91_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__19_91_250pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_250pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__19_91_250pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_250pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__19_91_250pq__1  processed_data/ers2__19_91_250pq__1.bam      99.53  NaN   \n",
       "ers2__19_91_250pq__2  processed_data/ers2__19_91_250pq__2.bam      99.50  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__19_91_250pq__1  NaN     NaN  \n",
       "ers2__19_91_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_750pq__1.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/20/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_750pq__2.bam</td>\n",
       "      <td>99.56</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_750pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_750pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>SPRI development test 1.8:1</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_750pq__3.bam</td>\n",
       "      <td>99.47</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__19_91_750pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>19_91_750pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.19.91</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>SPRI development test 1.8:1</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__19_91_750pq__4.bam</td>\n",
       "      <td>99.27</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__19_91_750pq__1  ROS TALE       ers2  19_91_750pq       1  Kevin Rychel   \n",
       "ers2__19_91_750pq__2  ROS TALE       ers2  19_91_750pq       2  Kevin Rychel   \n",
       "ers2__19_91_750pq__3  ROS TALE       ers2  19_91_750pq       3  Kevin Rychel   \n",
       "ers2__19_91_750pq__4  ROS TALE       ers2  19_91_750pq       4  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__19_91_750pq__1             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "ers2__19_91_750pq__2             NaN  1/20/2020  Escherichia Coli ERS.19.91   \n",
       "ers2__19_91_750pq__3             NaN  1/21/2020  Escherichia Coli ERS.19.91   \n",
       "ers2__19_91_750pq__4             NaN  1/21/2020  Escherichia Coli ERS.19.91   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__19_91_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__19_91_750pq__2      NaN         M9  ...            NextSeq   \n",
       "ers2__19_91_750pq__3      NaN         M9  ...            NextSeq   \n",
       "ers2__19_91_750pq__4      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                               Additional Details Biological Replicates  \\\n",
       "sample_id                                                                 \n",
       "ers2__19_91_750pq__1                          NaN                     4   \n",
       "ers2__19_91_750pq__2                          NaN                     4   \n",
       "ers2__19_91_750pq__3  SPRI development test 1.8:1                     4   \n",
       "ers2__19_91_750pq__4  SPRI development test 1.8:1                     4   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__19_91_750pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_750pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_750pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__19_91_750pq__4  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__19_91_750pq__1  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_750pq__2  sbrg-sequencing-files/2020-01-20-salk-nextseq/...   \n",
       "ers2__19_91_750pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__19_91_750pq__4  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__19_91_750pq__1  processed_data/ers2__19_91_750pq__1.bam      99.51  NaN   \n",
       "ers2__19_91_750pq__2  processed_data/ers2__19_91_750pq__2.bam      99.56  NaN   \n",
       "ers2__19_91_750pq__3  processed_data/ers2__19_91_750pq__3.bam      99.47  NaN   \n",
       "ers2__19_91_750pq__4  processed_data/ers2__19_91_750pq__4.bam      99.27  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__19_91_750pq__1  NaN     NaN  \n",
       "ers2__19_91_750pq__2  NaN     NaN  \n",
       "ers2__19_91_750pq__3  NaN     NaN  \n",
       "ers2__19_91_750pq__4  NaN     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_0pq__1.bam</td>\n",
       "      <td>97.57</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_0pq__2.bam</td>\n",
       "      <td>98.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__1_112_0pq__1  ROS TALE       ers2    1_112_0pq       1  Kevin Rychel   \n",
       "ers2__1_112_0pq__2  ROS TALE       ers2    1_112_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__1_112_0pq__1             NaN  8/26/2019  Escherichia Coli ERS.1.112   \n",
       "ers2__1_112_0pq__2             NaN  8/26/2019  Escherichia Coli ERS.1.112   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__1_112_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__1_112_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__1_112_0pq__1                NaN                     2   \n",
       "ers2__1_112_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__1_112_0pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__1_112_0pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__1_112_0pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__1_112_0pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__1_112_0pq__1  processed_data/ers2__1_112_0pq__1.bam      97.57  NaN   \n",
       "ers2__1_112_0pq__2  processed_data/ers2__1_112_0pq__2.bam      98.46  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__1_112_0pq__1  NaN     NaN  \n",
       "ers2__1_112_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_250pq__1.bam</td>\n",
       "      <td>98.12</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_250pq__2.bam</td>\n",
       "      <td>97.90</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__1_112_250pq__1  ROS TALE       ers2  1_112_250pq       1  Kevin Rychel   \n",
       "ers2__1_112_250pq__2  ROS TALE       ers2  1_112_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__1_112_250pq__1             NaN  8/26/2019  Escherichia Coli ERS.1.112   \n",
       "ers2__1_112_250pq__2             NaN  8/26/2019  Escherichia Coli ERS.1.112   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__1_112_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__1_112_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__1_112_250pq__1                NaN                     2   \n",
       "ers2__1_112_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__1_112_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__1_112_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__1_112_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__1_112_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__1_112_250pq__1  processed_data/ers2__1_112_250pq__1.bam      98.12  NaN   \n",
       "ers2__1_112_250pq__2  processed_data/ers2__1_112_250pq__2.bam      97.90  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__1_112_250pq__1  NaN     NaN  \n",
       "ers2__1_112_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_750pq__1.bam</td>\n",
       "      <td>97.64</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_750pq__2.bam</td>\n",
       "      <td>96.90</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__1_112_750pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>1_112_750pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.1.112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>3</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__1_112_750pq__3.bam</td>\n",
       "      <td>97.58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__1_112_750pq__1  ROS TALE       ers2  1_112_750pq       1  Kevin Rychel   \n",
       "ers2__1_112_750pq__2  ROS TALE       ers2  1_112_750pq       2  Kevin Rychel   \n",
       "ers2__1_112_750pq__3  ROS TALE       ers2  1_112_750pq       3  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__1_112_750pq__1             NaN   9/5/2019  Escherichia Coli ERS.1.112   \n",
       "ers2__1_112_750pq__2             NaN  1/21/2020  Escherichia Coli ERS.1.112   \n",
       "ers2__1_112_750pq__3             NaN  1/21/2020  Escherichia Coli ERS.1.112   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__1_112_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__1_112_750pq__2      NaN         M9  ...            NextSeq   \n",
       "ers2__1_112_750pq__3      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__1_112_750pq__1                NaN                     3   \n",
       "ers2__1_112_750pq__2                NaN                     3   \n",
       "ers2__1_112_750pq__3                NaN                     3   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__1_112_750pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__1_112_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__1_112_750pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__1_112_750pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__1_112_750pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__1_112_750pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__1_112_750pq__1  processed_data/ers2__1_112_750pq__1.bam      97.64  NaN   \n",
       "ers2__1_112_750pq__2  processed_data/ers2__1_112_750pq__2.bam      96.90  NaN   \n",
       "ers2__1_112_750pq__3  processed_data/ers2__1_112_750pq__3.bam      97.58  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__1_112_750pq__1  NaN     NaN  \n",
       "ers2__1_112_750pq__2  NaN     NaN  \n",
       "ers2__1_112_750pq__3  NaN     NaN  \n",
       "\n",
       "[3 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_0pq__1.bam</td>\n",
       "      <td>99.62</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_0pq__2.bam</td>\n",
       "      <td>99.60</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__21_97_0pq__1  ROS TALE       ers2    21_97_0pq       1  Kevin Rychel   \n",
       "ers2__21_97_0pq__2  ROS TALE       ers2    21_97_0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__21_97_0pq__1             NaN  1/21/2020  Escherichia Coli ERS.21.97   \n",
       "ers2__21_97_0pq__2             NaN  1/21/2020  Escherichia Coli ERS.21.97   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__21_97_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__21_97_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__21_97_0pq__1                NaN                     2   \n",
       "ers2__21_97_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__21_97_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__21_97_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__21_97_0pq__1  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__21_97_0pq__2  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__21_97_0pq__1  processed_data/ers2__21_97_0pq__1.bam      99.62  NaN   \n",
       "ers2__21_97_0pq__2  processed_data/ers2__21_97_0pq__2.bam      99.60  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__21_97_0pq__1  NaN     NaN  \n",
       "ers2__21_97_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/23/2019</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_250pq__1.bam</td>\n",
       "      <td>99.53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/23/2019</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_250pq__2.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__21_97_250pq__1  ROS TALE       ers2  21_97_250pq       1  Kevin Rychel   \n",
       "ers2__21_97_250pq__2  ROS TALE       ers2  21_97_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__21_97_250pq__1             NaN  9/23/2019  Escherichia Coli ERS.21.97   \n",
       "ers2__21_97_250pq__2             NaN  9/23/2019  Escherichia Coli ERS.21.97   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__21_97_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__21_97_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__21_97_250pq__1                NaN                     2   \n",
       "ers2__21_97_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__21_97_250pq__1  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "ers2__21_97_250pq__2  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__21_97_250pq__1  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "ers2__21_97_250pq__2  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__21_97_250pq__1  processed_data/ers2__21_97_250pq__1.bam      99.53  NaN   \n",
       "ers2__21_97_250pq__2  processed_data/ers2__21_97_250pq__2.bam      99.51  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__21_97_250pq__1  NaN     NaN  \n",
       "ers2__21_97_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/23/2019</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_750pq__1.bam</td>\n",
       "      <td>99.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__21_97_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>21_97_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/23/2019</td>\n",
       "      <td>Escherichia Coli ERS.21.97</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-23-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__21_97_750pq__2.bam</td>\n",
       "      <td>99.44</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__21_97_750pq__1  ROS TALE       ers2  21_97_750pq       1  Kevin Rychel   \n",
       "ers2__21_97_750pq__2  ROS TALE       ers2  21_97_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__21_97_750pq__1             NaN  9/23/2019  Escherichia Coli ERS.21.97   \n",
       "ers2__21_97_750pq__2             NaN  9/23/2019  Escherichia Coli ERS.21.97   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__21_97_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__21_97_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__21_97_750pq__1                NaN                     2   \n",
       "ers2__21_97_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__21_97_750pq__1  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "ers2__21_97_750pq__2  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__21_97_750pq__1  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "ers2__21_97_750pq__2  sbrg-sequencing-files/2019-09-23-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__21_97_750pq__1  processed_data/ers2__21_97_750pq__1.bam      99.54  NaN   \n",
       "ers2__21_97_750pq__2  processed_data/ers2__21_97_750pq__2.bam      99.44  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__21_97_750pq__1  NaN     NaN  \n",
       "ers2__21_97_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_0pq__1.bam</td>\n",
       "      <td>99.25</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_0pq__2.bam</td>\n",
       "      <td>99.15</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__25_111_0pq__1  ROS TALE       ers2   25_111_0pq       1  Kevin Rychel   \n",
       "ers2__25_111_0pq__2  ROS TALE       ers2   25_111_0pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator  run_date           Strain Description  \\\n",
       "sample_id                                                                    \n",
       "ers2__25_111_0pq__1             NaN  9/5/2019  Escherichia Coli ERS.25.111   \n",
       "ers2__25_111_0pq__2             NaN  9/5/2019  Escherichia Coli ERS.25.111   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__25_111_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__25_111_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__25_111_0pq__1                NaN                     2   \n",
       "ers2__25_111_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__25_111_0pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_0pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__25_111_0pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_0pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__25_111_0pq__1  processed_data/ers2__25_111_0pq__1.bam      99.25  NaN   \n",
       "ers2__25_111_0pq__2  processed_data/ers2__25_111_0pq__2.bam      99.15  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__25_111_0pq__1  NaN     NaN  \n",
       "ers2__25_111_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_250pq__1.bam</td>\n",
       "      <td>99.31</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_250pq__2.bam</td>\n",
       "      <td>99.68</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__25_111_250pq__1  ROS TALE       ers2  25_111_250pq       1   \n",
       "ers2__25_111_250pq__2  ROS TALE       ers2  25_111_250pq       2   \n",
       "\n",
       "                            contact library creator  run_date  \\\n",
       "sample_id                                                       \n",
       "ers2__25_111_250pq__1  Kevin Rychel             NaN  9/5/2019   \n",
       "ers2__25_111_250pq__2  Kevin Rychel             NaN  9/5/2019   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__25_111_250pq__1  Escherichia Coli ERS.25.111      NaN         M9  ...   \n",
       "ers2__25_111_250pq__2  Escherichia Coli ERS.25.111      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "ers2__25_111_250pq__1            NextSeq                NaN   \n",
       "ers2__25_111_250pq__2            NextSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__25_111_250pq__1                     2   \n",
       "ers2__25_111_250pq__2                     2   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__25_111_250pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_250pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__25_111_250pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_250pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__25_111_250pq__1  processed_data/ers2__25_111_250pq__1.bam      99.31   \n",
       "ers2__25_111_250pq__2  processed_data/ers2__25_111_250pq__2.bam      99.68   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__25_111_250pq__1  NaN  NaN     NaN  \n",
       "ers2__25_111_250pq__2  NaN  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_750pq__1.bam</td>\n",
       "      <td>99.58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__25_111_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>25_111_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/5/2019</td>\n",
       "      <td>Escherichia Coli ERS.25.111</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-05-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__25_111_750pq__2.bam</td>\n",
       "      <td>99.57</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__25_111_750pq__1  ROS TALE       ers2  25_111_750pq       1   \n",
       "ers2__25_111_750pq__2  ROS TALE       ers2  25_111_750pq       2   \n",
       "\n",
       "                            contact library creator  run_date  \\\n",
       "sample_id                                                       \n",
       "ers2__25_111_750pq__1  Kevin Rychel             NaN  9/5/2019   \n",
       "ers2__25_111_750pq__2  Kevin Rychel             NaN  9/5/2019   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__25_111_750pq__1  Escherichia Coli ERS.25.111      NaN         M9  ...   \n",
       "ers2__25_111_750pq__2  Escherichia Coli ERS.25.111      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "ers2__25_111_750pq__1            NextSeq                NaN   \n",
       "ers2__25_111_750pq__2            NextSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__25_111_750pq__1                     2   \n",
       "ers2__25_111_750pq__2                     2   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__25_111_750pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_750pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__25_111_750pq__1  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "ers2__25_111_750pq__2  sbrg-sequencing-files/2019-09-05-salk-nextseq/...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__25_111_750pq__1  processed_data/ers2__25_111_750pq__1.bam      99.58   \n",
       "ers2__25_111_750pq__2  processed_data/ers2__25_111_750pq__2.bam      99.57   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__25_111_750pq__1  NaN  NaN     NaN  \n",
       "ers2__25_111_750pq__2  NaN  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_0pq__1.bam</td>\n",
       "      <td>99.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_0pq__2.bam</td>\n",
       "      <td>99.39</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__29_119_0pq__1  ROS TALE       ers2   29_119_0pq       1  Kevin Rychel   \n",
       "ers2__29_119_0pq__2  ROS TALE       ers2   29_119_0pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator    run_date           Strain Description  \\\n",
       "sample_id                                                                      \n",
       "ers2__29_119_0pq__1             NaN  10/30/2019  Escherichia Coli ERS.29.119   \n",
       "ers2__29_119_0pq__2             NaN  10/30/2019  Escherichia Coli ERS.29.119   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__29_119_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__29_119_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__29_119_0pq__1                NaN                     2   \n",
       "ers2__29_119_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__29_119_0pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_0pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__29_119_0pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_0pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__29_119_0pq__1  processed_data/ers2__29_119_0pq__1.bam      99.54  NaN   \n",
       "ers2__29_119_0pq__2  processed_data/ers2__29_119_0pq__2.bam      99.39  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__29_119_0pq__1  NaN     NaN  \n",
       "ers2__29_119_0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_250pq__1.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_250pq__2.bam</td>\n",
       "      <td>99.56</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__29_119_250pq__1  ROS TALE       ers2  29_119_250pq       1   \n",
       "ers2__29_119_250pq__2  ROS TALE       ers2  29_119_250pq       2   \n",
       "\n",
       "                            contact library creator    run_date  \\\n",
       "sample_id                                                         \n",
       "ers2__29_119_250pq__1  Kevin Rychel             NaN  10/30/2019   \n",
       "ers2__29_119_250pq__2  Kevin Rychel             NaN  10/30/2019   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__29_119_250pq__1  Escherichia Coli ERS.29.119      NaN         M9  ...   \n",
       "ers2__29_119_250pq__2  Escherichia Coli ERS.29.119      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "ers2__29_119_250pq__1            NextSeq                NaN   \n",
       "ers2__29_119_250pq__2            NextSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__29_119_250pq__1                     2   \n",
       "ers2__29_119_250pq__2                     2   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__29_119_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__29_119_250pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_250pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__29_119_250pq__1  processed_data/ers2__29_119_250pq__1.bam      99.51   \n",
       "ers2__29_119_250pq__2  processed_data/ers2__29_119_250pq__2.bam      99.56   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__29_119_250pq__1  NaN  NaN     NaN  \n",
       "ers2__29_119_250pq__2  NaN  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_750pq__1.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__29_119_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>29_119_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>10/30/2019</td>\n",
       "      <td>Escherichia Coli ERS.29.119</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-10-30-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__29_119_750pq__2.bam</td>\n",
       "      <td>99.45</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                          \n",
       "ers2__29_119_750pq__1  ROS TALE       ers2  29_119_750pq       1   \n",
       "ers2__29_119_750pq__2  ROS TALE       ers2  29_119_750pq       2   \n",
       "\n",
       "                            contact library creator    run_date  \\\n",
       "sample_id                                                         \n",
       "ers2__29_119_750pq__1  Kevin Rychel             NaN  10/30/2019   \n",
       "ers2__29_119_750pq__2  Kevin Rychel             NaN  10/30/2019   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ers2__29_119_750pq__1  Escherichia Coli ERS.29.119      NaN         M9  ...   \n",
       "ers2__29_119_750pq__2  Escherichia Coli ERS.29.119      NaN         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "ers2__29_119_750pq__1            NextSeq                NaN   \n",
       "ers2__29_119_750pq__2            NextSeq                NaN   \n",
       "\n",
       "                      Biological Replicates  \\\n",
       "sample_id                                     \n",
       "ers2__29_119_750pq__1                     2   \n",
       "ers2__29_119_750pq__2                     2   \n",
       "\n",
       "                                                                      R1  \\\n",
       "sample_id                                                                  \n",
       "ers2__29_119_750pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_750pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                                      R2  \\\n",
       "sample_id                                                                  \n",
       "ers2__29_119_750pq__1  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "ers2__29_119_750pq__2  sbrg-sequencing-files/2019-10-30-salk-nextseq/...   \n",
       "\n",
       "                                                            BAM  alignment  \\\n",
       "sample_id                                                                    \n",
       "ers2__29_119_750pq__1  processed_data/ers2__29_119_750pq__1.bam      99.51   \n",
       "ers2__29_119_750pq__2  processed_data/ers2__29_119_750pq__2.bam      99.45   \n",
       "\n",
       "                       DOI  GEO  Public  \n",
       "sample_id                                \n",
       "ers2__29_119_750pq__1  NaN  NaN     NaN  \n",
       "ers2__29_119_750pq__2  NaN  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__3_76_0pq__1.bam</td>\n",
       "      <td>99.20</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__3_76_0pq__2.bam</td>\n",
       "      <td>99.39</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_0pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_0pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__3_76_0pq__3.bam</td>\n",
       "      <td>99.01</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_0pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_0pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/16/2020</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NovaSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...</td>\n",
       "      <td>processed_data/ers2__3_76_0pq__4.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                   \n",
       "ers2__3_76_0pq__1  ROS TALE       ers2     3_76_0pq       1  Kevin Rychel   \n",
       "ers2__3_76_0pq__2  ROS TALE       ers2     3_76_0pq       2  Kevin Rychel   \n",
       "ers2__3_76_0pq__3  ROS TALE       ers2     3_76_0pq       3  Kevin Rychel   \n",
       "ers2__3_76_0pq__4  ROS TALE       ers2     3_76_0pq       4  Kevin Rychel   \n",
       "\n",
       "                  library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                 \n",
       "ers2__3_76_0pq__1             NaN  1/16/2020  Escherichia Coli ERS.3.76   \n",
       "ers2__3_76_0pq__2             NaN  1/16/2020  Escherichia Coli ERS.3.76   \n",
       "ers2__3_76_0pq__3             NaN  1/16/2020  Escherichia Coli ERS.3.76   \n",
       "ers2__3_76_0pq__4             NaN  1/16/2020  Escherichia Coli ERS.3.76   \n",
       "\n",
       "                  Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                              ...                      \n",
       "ers2__3_76_0pq__1      NaN         M9  ...            NovaSeq   \n",
       "ers2__3_76_0pq__2      NaN         M9  ...            NovaSeq   \n",
       "ers2__3_76_0pq__3      NaN         M9  ...            NovaSeq   \n",
       "ers2__3_76_0pq__4      NaN         M9  ...            NovaSeq   \n",
       "\n",
       "                        Additional Details Biological Replicates  \\\n",
       "sample_id                                                          \n",
       "ers2__3_76_0pq__1                      NaN                     4   \n",
       "ers2__3_76_0pq__2                      NaN                     4   \n",
       "ers2__3_76_0pq__3  Spacer development test                     4   \n",
       "ers2__3_76_0pq__4  Spacer development test                     4   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "ers2__3_76_0pq__1  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__2  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__3  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__4  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "ers2__3_76_0pq__1  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__2  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__3  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "ers2__3_76_0pq__4  sbrg-sequencing-files/2020-01-16-IGM-NovaSeq/E...   \n",
       "\n",
       "                                                    BAM  alignment  DOI  GEO  \\\n",
       "sample_id                                                                      \n",
       "ers2__3_76_0pq__1  processed_data/ers2__3_76_0pq__1.bam      99.20  NaN  NaN   \n",
       "ers2__3_76_0pq__2  processed_data/ers2__3_76_0pq__2.bam      99.39  NaN  NaN   \n",
       "ers2__3_76_0pq__3  processed_data/ers2__3_76_0pq__3.bam      99.01  NaN  NaN   \n",
       "ers2__3_76_0pq__4  processed_data/ers2__3_76_0pq__4.bam      99.51  NaN  NaN   \n",
       "\n",
       "                   Public  \n",
       "sample_id                  \n",
       "ers2__3_76_0pq__1     NaN  \n",
       "ers2__3_76_0pq__2     NaN  \n",
       "ers2__3_76_0pq__3     NaN  \n",
       "ers2__3_76_0pq__4     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__3_76_250pq__1.bam</td>\n",
       "      <td>99.45</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__3_76_250pq__2.bam</td>\n",
       "      <td>99.51</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__3_76_250pq__1  ROS TALE       ers2   3_76_250pq       1  Kevin Rychel   \n",
       "ers2__3_76_250pq__2  ROS TALE       ers2   3_76_250pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__3_76_250pq__1             NaN  8/26/2019  Escherichia Coli ERS.3.76   \n",
       "ers2__3_76_250pq__2             NaN  8/26/2019  Escherichia Coli ERS.3.76   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__3_76_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__3_76_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__3_76_250pq__1                NaN                     2   \n",
       "ers2__3_76_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__3_76_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__3_76_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__3_76_250pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__3_76_250pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__3_76_250pq__1  processed_data/ers2__3_76_250pq__1.bam      99.45  NaN   \n",
       "ers2__3_76_250pq__2  processed_data/ers2__3_76_250pq__2.bam      99.51  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__3_76_250pq__1  NaN     NaN  \n",
       "ers2__3_76_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__3_76_750pq__1.bam</td>\n",
       "      <td>99.44</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__3_76_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>3_76_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>8/26/2019</td>\n",
       "      <td>Escherichia Coli ERS.3.76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-08-26-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__3_76_750pq__2.bam</td>\n",
       "      <td>99.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__3_76_750pq__1  ROS TALE       ers2   3_76_750pq       1  Kevin Rychel   \n",
       "ers2__3_76_750pq__2  ROS TALE       ers2   3_76_750pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__3_76_750pq__1             NaN  8/26/2019  Escherichia Coli ERS.3.76   \n",
       "ers2__3_76_750pq__2             NaN  8/26/2019  Escherichia Coli ERS.3.76   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__3_76_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__3_76_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__3_76_750pq__1                NaN                     2   \n",
       "ers2__3_76_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__3_76_750pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__3_76_750pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__3_76_750pq__1  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "ers2__3_76_750pq__2  sbrg-sequencing-files/2019-08-26-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__3_76_750pq__1  processed_data/ers2__3_76_750pq__1.bam      99.44  NaN   \n",
       "ers2__3_76_750pq__2  processed_data/ers2__3_76_750pq__2.bam      99.46  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__3_76_750pq__1  NaN     NaN  \n",
       "ers2__3_76_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_0pq__1.bam</td>\n",
       "      <td>99.39</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>Spacer development test</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_0pq__2.bam</td>\n",
       "      <td>99.48</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_0pq__3</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_0pq</td>\n",
       "      <td>3</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_0pq__3.bam</td>\n",
       "      <td>99.34</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_0pq__4</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_0pq</td>\n",
       "      <td>4</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1/21/2020</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2020-01-21-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_0pq__4.bam</td>\n",
       "      <td>99.50</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__7_103_0pq__1  ROS TALE       ers2    7_103_0pq       1  Kevin Rychel   \n",
       "ers2__7_103_0pq__2  ROS TALE       ers2    7_103_0pq       2  Kevin Rychel   \n",
       "ers2__7_103_0pq__3  ROS TALE       ers2    7_103_0pq       3  Kevin Rychel   \n",
       "ers2__7_103_0pq__4  ROS TALE       ers2    7_103_0pq       4  Kevin Rychel   \n",
       "\n",
       "                   library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__7_103_0pq__1             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "ers2__7_103_0pq__2             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "ers2__7_103_0pq__3             NaN  1/21/2020  Escherichia Coli ERS.7.103   \n",
       "ers2__7_103_0pq__4             NaN  1/21/2020  Escherichia Coli ERS.7.103   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__7_103_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__7_103_0pq__2      NaN         M9  ...            NextSeq   \n",
       "ers2__7_103_0pq__3      NaN         M9  ...            NextSeq   \n",
       "ers2__7_103_0pq__4      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                         Additional Details Biological Replicates  \\\n",
       "sample_id                                                           \n",
       "ers2__7_103_0pq__1  Spacer development test                     4   \n",
       "ers2__7_103_0pq__2  Spacer development test                     4   \n",
       "ers2__7_103_0pq__3                      NaN                     4   \n",
       "ers2__7_103_0pq__4                      NaN                     4   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__7_103_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_0pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__7_103_0pq__4  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__7_103_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_0pq__3  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "ers2__7_103_0pq__4  sbrg-sequencing-files/2020-01-21-salk-nextseq/...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__7_103_0pq__1  processed_data/ers2__7_103_0pq__1.bam      99.39  NaN   \n",
       "ers2__7_103_0pq__2  processed_data/ers2__7_103_0pq__2.bam      99.48  NaN   \n",
       "ers2__7_103_0pq__3  processed_data/ers2__7_103_0pq__3.bam      99.34  NaN   \n",
       "ers2__7_103_0pq__4  processed_data/ers2__7_103_0pq__4.bam      99.50  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__7_103_0pq__1  NaN     NaN  \n",
       "ers2__7_103_0pq__2  NaN     NaN  \n",
       "ers2__7_103_0pq__3  NaN     NaN  \n",
       "ers2__7_103_0pq__4  NaN     NaN  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_250pq__1.bam</td>\n",
       "      <td>99.48</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_250pq__2.bam</td>\n",
       "      <td>99.49</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__7_103_250pq__1  ROS TALE       ers2  7_103_250pq       1  Kevin Rychel   \n",
       "ers2__7_103_250pq__2  ROS TALE       ers2  7_103_250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__7_103_250pq__1             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "ers2__7_103_250pq__2             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__7_103_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__7_103_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__7_103_250pq__1                NaN                     2   \n",
       "ers2__7_103_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__7_103_250pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_250pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__7_103_250pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_250pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__7_103_250pq__1  processed_data/ers2__7_103_250pq__1.bam      99.48  NaN   \n",
       "ers2__7_103_250pq__2  processed_data/ers2__7_103_250pq__2.bam      99.49  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__7_103_250pq__1  NaN     NaN  \n",
       "ers2__7_103_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_750pq__1.bam</td>\n",
       "      <td>99.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__7_103_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>7_103_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.7.103</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__7_103_750pq__2.bam</td>\n",
       "      <td>99.52</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__7_103_750pq__1  ROS TALE       ers2  7_103_750pq       1  Kevin Rychel   \n",
       "ers2__7_103_750pq__2  ROS TALE       ers2  7_103_750pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator   run_date          Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__7_103_750pq__1             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "ers2__7_103_750pq__2             NaN  9/20/2019  Escherichia Coli ERS.7.103   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__7_103_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__7_103_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__7_103_750pq__1                NaN                     2   \n",
       "ers2__7_103_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__7_103_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__7_103_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__7_103_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__7_103_750pq__1  processed_data/ers2__7_103_750pq__1.bam      99.46  NaN   \n",
       "ers2__7_103_750pq__2  processed_data/ers2__7_103_750pq__2.bam      99.52  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__7_103_750pq__1  NaN     NaN  \n",
       "ers2__7_103_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_0pq__1.bam</td>\n",
       "      <td>99.4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_0pq__2.bam</td>\n",
       "      <td>99.6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                   \n",
       "ers2__9_77_0pq__1  ROS TALE       ers2     9_77_0pq       1  Kevin Rychel   \n",
       "ers2__9_77_0pq__2  ROS TALE       ers2     9_77_0pq       2  Kevin Rychel   \n",
       "\n",
       "                  library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                 \n",
       "ers2__9_77_0pq__1             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "ers2__9_77_0pq__2             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "\n",
       "                  Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                              ...                      \n",
       "ers2__9_77_0pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__9_77_0pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                  Additional Details Biological Replicates  \\\n",
       "sample_id                                                    \n",
       "ers2__9_77_0pq__1                NaN                     2   \n",
       "ers2__9_77_0pq__2                NaN                     2   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "ers2__9_77_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "ers2__9_77_0pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_0pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                    BAM  alignment  DOI  GEO  \\\n",
       "sample_id                                                                      \n",
       "ers2__9_77_0pq__1  processed_data/ers2__9_77_0pq__1.bam       99.4  NaN  NaN   \n",
       "ers2__9_77_0pq__2  processed_data/ers2__9_77_0pq__2.bam       99.6  NaN  NaN   \n",
       "\n",
       "                   Public  \n",
       "sample_id                  \n",
       "ers2__9_77_0pq__1     NaN  \n",
       "ers2__9_77_0pq__2     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_250pq__1.bam</td>\n",
       "      <td>99.59</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_250pq__2.bam</td>\n",
       "      <td>99.57</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__9_77_250pq__1  ROS TALE       ers2   9_77_250pq       1  Kevin Rychel   \n",
       "ers2__9_77_250pq__2  ROS TALE       ers2   9_77_250pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__9_77_250pq__1             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "ers2__9_77_250pq__2             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__9_77_250pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__9_77_250pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__9_77_250pq__1                NaN                     2   \n",
       "ers2__9_77_250pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__9_77_250pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_250pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__9_77_250pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_250pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__9_77_250pq__1  processed_data/ers2__9_77_250pq__1.bam      99.59  NaN   \n",
       "ers2__9_77_250pq__2  processed_data/ers2__9_77_250pq__2.bam      99.57  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__9_77_250pq__1  NaN     NaN  \n",
       "ers2__9_77_250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_750pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_750pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_750pq__1.bam</td>\n",
       "      <td>99.12</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__9_77_750pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>9_77_750pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>9/20/2019</td>\n",
       "      <td>Escherichia Coli ERS.9.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>sbrg-sequencing-files/2019-09-20-salk-nextseq/...</td>\n",
       "      <td>processed_data/ers2__9_77_750pq__2.bam</td>\n",
       "      <td>99.65</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "ers2__9_77_750pq__1  ROS TALE       ers2   9_77_750pq       1  Kevin Rychel   \n",
       "ers2__9_77_750pq__2  ROS TALE       ers2   9_77_750pq       2  Kevin Rychel   \n",
       "\n",
       "                    library creator   run_date         Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__9_77_750pq__1             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "ers2__9_77_750pq__2             NaN  9/20/2019  Escherichia Coli ERS.9.77   \n",
       "\n",
       "                    Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                ...                      \n",
       "ers2__9_77_750pq__1      NaN         M9  ...            NextSeq   \n",
       "ers2__9_77_750pq__2      NaN         M9  ...            NextSeq   \n",
       "\n",
       "                    Additional Details Biological Replicates  \\\n",
       "sample_id                                                      \n",
       "ers2__9_77_750pq__1                NaN                     2   \n",
       "ers2__9_77_750pq__2                NaN                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "ers2__9_77_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "ers2__9_77_750pq__1  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "ers2__9_77_750pq__2  sbrg-sequencing-files/2019-09-20-salk-nextseq/...   \n",
       "\n",
       "                                                        BAM  alignment  DOI  \\\n",
       "sample_id                                                                     \n",
       "ers2__9_77_750pq__1  processed_data/ers2__9_77_750pq__1.bam      99.12  NaN   \n",
       "ers2__9_77_750pq__2  processed_data/ers2__9_77_750pq__2.bam      99.65  NaN   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ers2__9_77_750pq__1  NaN     NaN  \n",
       "ers2__9_77_750pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__ALE4__0pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>ALE4__0pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.ALE4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4A_0PQ_RNA_S24_...</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4A_0PQ_RNA_S24_...</td>\n",
       "      <td>processed_data/ers2__ALE4__0pq__1.bam</td>\n",
       "      <td>98.95</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__ALE4__0pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>ALE4__0pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.ALE4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4B_0PQ_RNA_S23_...</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4B_0PQ_RNA_S23_...</td>\n",
       "      <td>processed_data/ers2__ALE4__0pq__2.bam</td>\n",
       "      <td>99.08</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                       study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                    \n",
       "ers2__ALE4__0pq__1  ROS TALE       ers2    ALE4__0pq       1  Kevin Rychel   \n",
       "ers2__ALE4__0pq__2  ROS TALE       ers2    ALE4__0pq       2  Kevin Rychel   \n",
       "\n",
       "                   library creator    run_date         Strain Description  \\\n",
       "sample_id                                                                   \n",
       "ers2__ALE4__0pq__1             NaN  11/28/2017  Escherichia Coli ERS.ALE4   \n",
       "ers2__ALE4__0pq__2             NaN  11/28/2017  Escherichia Coli ERS.ALE4   \n",
       "\n",
       "                   Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "ers2__ALE4__0pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__ALE4__0pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                     \n",
       "ers2__ALE4__0pq__1                NaN                     2   \n",
       "ers2__ALE4__0pq__2                NaN                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ers2__ALE4__0pq__1  /raw_data/ers2/ALE4/ERS_HRS_ALE4A_0PQ_RNA_S24_...   \n",
       "ers2__ALE4__0pq__2  /raw_data/ers2/ALE4/ERS_HRS_ALE4B_0PQ_RNA_S23_...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ers2__ALE4__0pq__1  /raw_data/ers2/ALE4/ERS_HRS_ALE4A_0PQ_RNA_S24_...   \n",
       "ers2__ALE4__0pq__2  /raw_data/ers2/ALE4/ERS_HRS_ALE4B_0PQ_RNA_S23_...   \n",
       "\n",
       "                                                      BAM  alignment  DOI  \\\n",
       "sample_id                                                                   \n",
       "ers2__ALE4__0pq__1  processed_data/ers2__ALE4__0pq__1.bam      98.95  NaN   \n",
       "ers2__ALE4__0pq__2  processed_data/ers2__ALE4__0pq__2.bam      99.08  NaN   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ers2__ALE4__0pq__1  NaN     NaN  \n",
       "ers2__ALE4__0pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ers2__ALE4__250pq__1</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>ALE4__250pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.ALE4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4A_PQ_RNA_S27_L...</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4A_PQ_RNA_S27_L...</td>\n",
       "      <td>processed_data/ers2__ALE4__250pq__1.bam</td>\n",
       "      <td>99.07</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ers2__ALE4__250pq__2</td>\n",
       "      <td>ROS TALE</td>\n",
       "      <td>ers2</td>\n",
       "      <td>ALE4__250pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Kevin Rychel</td>\n",
       "      <td>NaN</td>\n",
       "      <td>11/28/2017</td>\n",
       "      <td>Escherichia Coli ERS.ALE4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4B_PQ_RNA_S26_L...</td>\n",
       "      <td>/raw_data/ers2/ALE4/ERS_HRS_ALE4B_PQ_RNA_S26_L...</td>\n",
       "      <td>processed_data/ers2__ALE4__250pq__2.bam</td>\n",
       "      <td>99.16</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "ers2__ALE4__250pq__1  ROS TALE       ers2  ALE4__250pq       1  Kevin Rychel   \n",
       "ers2__ALE4__250pq__2  ROS TALE       ers2  ALE4__250pq       2  Kevin Rychel   \n",
       "\n",
       "                     library creator    run_date         Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ers2__ALE4__250pq__1             NaN  11/28/2017  Escherichia Coli ERS.ALE4   \n",
       "ers2__ALE4__250pq__2             NaN  11/28/2017  Escherichia Coli ERS.ALE4   \n",
       "\n",
       "                     Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                 ...                      \n",
       "ers2__ALE4__250pq__1      NaN         M9  ...              HiSeq   \n",
       "ers2__ALE4__250pq__2      NaN         M9  ...              HiSeq   \n",
       "\n",
       "                     Additional Details Biological Replicates  \\\n",
       "sample_id                                                       \n",
       "ers2__ALE4__250pq__1                NaN                     2   \n",
       "ers2__ALE4__250pq__2                NaN                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "ers2__ALE4__250pq__1  /raw_data/ers2/ALE4/ERS_HRS_ALE4A_PQ_RNA_S27_L...   \n",
       "ers2__ALE4__250pq__2  /raw_data/ers2/ALE4/ERS_HRS_ALE4B_PQ_RNA_S26_L...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "ers2__ALE4__250pq__1  /raw_data/ers2/ALE4/ERS_HRS_ALE4A_PQ_RNA_S27_L...   \n",
       "ers2__ALE4__250pq__2  /raw_data/ers2/ALE4/ERS_HRS_ALE4B_PQ_RNA_S26_L...   \n",
       "\n",
       "                                                          BAM  alignment  DOI  \\\n",
       "sample_id                                                                       \n",
       "ers2__ALE4__250pq__1  processed_data/ers2__ALE4__250pq__1.bam      99.07  NaN   \n",
       "ers2__ALE4__250pq__2  processed_data/ers2__ALE4__250pq__2.bam      99.16  NaN   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ers2__ALE4__250pq__1  NaN     NaN  \n",
       "ers2__ALE4__250pq__2  NaN     NaN  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__bw_glc__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>bw_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536294_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536294_2.fastq.gz</td>\n",
       "      <td>fps__bw_glc__1.bam</td>\n",
       "      <td>88.99</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__bw_glc__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>bw_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536295_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536295_2.fastq.gz</td>\n",
       "      <td>fps__bw_glc__2.bam</td>\n",
       "      <td>88.62</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id condition_id  rep_id  \\\n",
       "sample_id                                                         \n",
       "fps__bw_glc__1  False Positives        fps       bw_glc       1   \n",
       "fps__bw_glc__2  False Positives        fps       bw_glc       2   \n",
       "\n",
       "                        contact   library creator        run_date  \\\n",
       "sample_id                                                           \n",
       "fps__bw_glc__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__bw_glc__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                      Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                     ...   \n",
       "fps__bw_glc__1  Escherichia coli BW25113  BW25113         M9  ...   \n",
       "fps__bw_glc__2  Escherichia coli BW25113  BW25113         M9  ...   \n",
       "\n",
       "               Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                    \n",
       "fps__bw_glc__1        NextSeq 550                NaN                     2   \n",
       "fps__bw_glc__2        NextSeq 550                NaN                     2   \n",
       "\n",
       "                                       R1                         R2  \\\n",
       "sample_id                                                              \n",
       "fps__bw_glc__1  fps/SRR7536294_1.fastq.gz  fps/SRR7536294_2.fastq.gz   \n",
       "fps__bw_glc__2  fps/SRR7536295_1.fastq.gz  fps/SRR7536295_2.fastq.gz   \n",
       "\n",
       "                               BAM  alignment  \\\n",
       "sample_id                                       \n",
       "fps__bw_glc__1  fps__bw_glc__1.bam      88.99   \n",
       "fps__bw_glc__2  fps__bw_glc__2.bam      88.62   \n",
       "\n",
       "                                              DOI        GEO  Public  \n",
       "sample_id                                                             \n",
       "fps__bw_glc__1  doi.org/10.1186/s12918-018-0653-z  GSE117303     1.0  \n",
       "fps__bw_glc__2  doi.org/10.1186/s12918-018-0653-z  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_ptsI_ale1__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_ptsI_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS ptsI.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536300_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536300_2.fastq.gz</td>\n",
       "      <td>fps__fps_ptsI_ale1__1.bam</td>\n",
       "      <td>82.60</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_ptsI_ale1__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_ptsI_ale1</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS ptsI.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536301_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536301_2.fastq.gz</td>\n",
       "      <td>fps__fps_ptsI_ale1__2.bam</td>\n",
       "      <td>89.81</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_ptsI_ale1__1  False Positives        fps  fps_ptsI_ale1       1   \n",
       "fps__fps_ptsI_ale1__2  False Positives        fps  fps_ptsI_ale1       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_ptsI_ale1__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_ptsI_ale1__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_ptsI_ale1__1  Escherichia coli FPS ptsI.1.5.1  BW25113         M9   \n",
       "fps__fps_ptsI_ale1__2  Escherichia coli FPS ptsI.1.5.1  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_ptsI_ale1__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_ptsI_ale1__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_ptsI_ale1__1                     2  fps/SRR7536300_1.fastq.gz   \n",
       "fps__fps_ptsI_ale1__2                     2  fps/SRR7536301_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_ptsI_ale1__1  fps/SRR7536300_2.fastq.gz  fps__fps_ptsI_ale1__1.bam   \n",
       "fps__fps_ptsI_ale1__2  fps/SRR7536301_2.fastq.gz  fps__fps_ptsI_ale1__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_ptsI_ale1__1      82.60  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_ptsI_ale1__2      89.81  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_ptsI_ale1__1  GSE117303     1.0  \n",
       "fps__fps_ptsI_ale1__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_ptsI_ale3__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_ptsI_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS ptsI.3.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536302_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536302_2.fastq.gz</td>\n",
       "      <td>fps__fps_ptsI_ale3__1.bam</td>\n",
       "      <td>89.26</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_ptsI_ale3__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_ptsI_ale3</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS ptsI.3.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536303_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536303_2.fastq.gz</td>\n",
       "      <td>fps__fps_ptsI_ale3__2.bam</td>\n",
       "      <td>89.23</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_ptsI_ale3__1  False Positives        fps  fps_ptsI_ale3       1   \n",
       "fps__fps_ptsI_ale3__2  False Positives        fps  fps_ptsI_ale3       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_ptsI_ale3__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_ptsI_ale3__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_ptsI_ale3__1  Escherichia coli FPS ptsI.3.1.0  BW25113         M9   \n",
       "fps__fps_ptsI_ale3__2  Escherichia coli FPS ptsI.3.1.0  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_ptsI_ale3__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_ptsI_ale3__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_ptsI_ale3__1                     2  fps/SRR7536302_1.fastq.gz   \n",
       "fps__fps_ptsI_ale3__2                     2  fps/SRR7536303_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_ptsI_ale3__1  fps/SRR7536302_2.fastq.gz  fps__fps_ptsI_ale3__1.bam   \n",
       "fps__fps_ptsI_ale3__2  fps/SRR7536303_2.fastq.gz  fps__fps_ptsI_ale3__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_ptsI_ale3__1      89.26  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_ptsI_ale3__2      89.23  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_ptsI_ale3__1  GSE117303     1.0  \n",
       "fps__fps_ptsI_ale3__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_serB_ale1__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_serB_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS serB.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536298_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536298_2.fastq.gz</td>\n",
       "      <td>fps__fps_serB_ale1__1.bam</td>\n",
       "      <td>88.48</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_serB_ale1__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_serB_ale1</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS serB.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536299_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536299_2.fastq.gz</td>\n",
       "      <td>fps__fps_serB_ale1__2.bam</td>\n",
       "      <td>87.36</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_serB_ale1__1  False Positives        fps  fps_serB_ale1       1   \n",
       "fps__fps_serB_ale1__2  False Positives        fps  fps_serB_ale1       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_serB_ale1__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_serB_ale1__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_serB_ale1__1  Escherichia coli FPS serB.1.5.1  BW25113         M9   \n",
       "fps__fps_serB_ale1__2  Escherichia coli FPS serB.1.5.1  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_serB_ale1__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_serB_ale1__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_serB_ale1__1                     2  fps/SRR7536298_1.fastq.gz   \n",
       "fps__fps_serB_ale1__2                     2  fps/SRR7536299_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_serB_ale1__1  fps/SRR7536298_2.fastq.gz  fps__fps_serB_ale1__1.bam   \n",
       "fps__fps_serB_ale1__2  fps/SRR7536299_2.fastq.gz  fps__fps_serB_ale1__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_serB_ale1__1      88.48  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_serB_ale1__2      87.36  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_serB_ale1__1  GSE117303     1.0  \n",
       "fps__fps_serB_ale1__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_serB_ale4__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_serB_ale4</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS serB.4.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536306_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536306_2.fastq.gz</td>\n",
       "      <td>fps__fps_serB_ale4__1.bam</td>\n",
       "      <td>89.86</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_serB_ale4__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_serB_ale4</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS serB.4.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536307_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536307_2.fastq.gz</td>\n",
       "      <td>fps__fps_serB_ale4__2.bam</td>\n",
       "      <td>88.78</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_serB_ale4__1  False Positives        fps  fps_serB_ale4       1   \n",
       "fps__fps_serB_ale4__2  False Positives        fps  fps_serB_ale4       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_serB_ale4__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_serB_ale4__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_serB_ale4__1  Escherichia coli FPS serB.4.1.0  BW25113         M9   \n",
       "fps__fps_serB_ale4__2  Escherichia coli FPS serB.4.1.0  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_serB_ale4__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_serB_ale4__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_serB_ale4__1                     2  fps/SRR7536306_1.fastq.gz   \n",
       "fps__fps_serB_ale4__2                     2  fps/SRR7536307_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_serB_ale4__1  fps/SRR7536306_2.fastq.gz  fps__fps_serB_ale4__1.bam   \n",
       "fps__fps_serB_ale4__2  fps/SRR7536307_2.fastq.gz  fps__fps_serB_ale4__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_serB_ale4__1      89.86  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_serB_ale4__2      88.78  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_serB_ale4__1  GSE117303     1.0  \n",
       "fps__fps_serB_ale4__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_thrA_ale1__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_thrA_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS thrA.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536296_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536296_2.fastq.gz</td>\n",
       "      <td>fps__fps_thrA_ale1__1.bam</td>\n",
       "      <td>85.90</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_thrA_ale1__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_thrA_ale1</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS thrA.1.5.1</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536297_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536297_2.fastq.gz</td>\n",
       "      <td>fps__fps_thrA_ale1__2.bam</td>\n",
       "      <td>88.86</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_thrA_ale1__1  False Positives        fps  fps_thrA_ale1       1   \n",
       "fps__fps_thrA_ale1__2  False Positives        fps  fps_thrA_ale1       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_thrA_ale1__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_thrA_ale1__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_thrA_ale1__1  Escherichia coli FPS thrA.1.5.1  BW25113         M9   \n",
       "fps__fps_thrA_ale1__2  Escherichia coli FPS thrA.1.5.1  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_thrA_ale1__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_thrA_ale1__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_thrA_ale1__1                     2  fps/SRR7536296_1.fastq.gz   \n",
       "fps__fps_thrA_ale1__2                     2  fps/SRR7536297_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_thrA_ale1__1  fps/SRR7536296_2.fastq.gz  fps__fps_thrA_ale1__1.bam   \n",
       "fps__fps_thrA_ale1__2  fps/SRR7536297_2.fastq.gz  fps__fps_thrA_ale1__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_thrA_ale1__1      85.90  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_thrA_ale1__2      88.86  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_thrA_ale1__1  GSE117303     1.0  \n",
       "fps__fps_thrA_ale1__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fps__fps_thrA_ale2__1</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_thrA_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS thrA.2.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536304_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536304_2.fastq.gz</td>\n",
       "      <td>fps__fps_thrA_ale2__1.bam</td>\n",
       "      <td>88.52</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fps__fps_thrA_ale2__2</td>\n",
       "      <td>False Positives</td>\n",
       "      <td>fps</td>\n",
       "      <td>fps_thrA_ale2</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>5/29/2018 0:00</td>\n",
       "      <td>Escherichia coli FPS thrA.2.1.0</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NextSeq 550</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fps/SRR7536305_1.fastq.gz</td>\n",
       "      <td>fps/SRR7536305_2.fastq.gz</td>\n",
       "      <td>fps__fps_thrA_ale2__2.bam</td>\n",
       "      <td>88.32</td>\n",
       "      <td>doi.org/10.1186/s12918-018-0653-z</td>\n",
       "      <td>GSE117303</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_thrA_ale2__1  False Positives        fps  fps_thrA_ale2       1   \n",
       "fps__fps_thrA_ale2__2  False Positives        fps  fps_thrA_ale2       2   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "fps__fps_thrA_ale2__1  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "fps__fps_thrA_ale2__2  Gabriela Guzman  Sibei (Julia) Xu  5/29/2018 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "fps__fps_thrA_ale2__1  Escherichia coli FPS thrA.2.1.0  BW25113         M9   \n",
       "fps__fps_thrA_ale2__2  Escherichia coli FPS thrA.2.1.0  BW25113         M9   \n",
       "\n",
       "                       ... Sequencing Machine Additional Details  \\\n",
       "sample_id              ...                                         \n",
       "fps__fps_thrA_ale2__1  ...        NextSeq 550                NaN   \n",
       "fps__fps_thrA_ale2__2  ...        NextSeq 550                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "fps__fps_thrA_ale2__1                     2  fps/SRR7536304_1.fastq.gz   \n",
       "fps__fps_thrA_ale2__2                     2  fps/SRR7536305_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "fps__fps_thrA_ale2__1  fps/SRR7536304_2.fastq.gz  fps__fps_thrA_ale2__1.bam   \n",
       "fps__fps_thrA_ale2__2  fps/SRR7536305_2.fastq.gz  fps__fps_thrA_ale2__2.bam   \n",
       "\n",
       "                       alignment                                DOI  \\\n",
       "sample_id                                                             \n",
       "fps__fps_thrA_ale2__1      88.52  doi.org/10.1186/s12918-018-0653-z   \n",
       "fps__fps_thrA_ale2__2      88.32  doi.org/10.1186/s12918-018-0653-z   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "fps__fps_thrA_ale2__1  GSE117303     1.0  \n",
       "fps__fps_thrA_ale2__2  GSE117303     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fur__delfur_dpd__1</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>delfur_dpd</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>10/2/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_fur</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/delfurDPD1_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/delfurDPD1_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__delfur_dpd__1.bam</td>\n",
       "      <td>98.07</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fur__delfur_dpd__2</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>delfur_dpd</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>10/2/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_fur</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/delfurDPD2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/delfurDPD2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__delfur_dpd__2.bam</td>\n",
       "      <td>98.31</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                               \n",
       "fur__delfur_dpd__1   Fur        fur   delfur_dpd       1  Sangwoo Seo   \n",
       "fur__delfur_dpd__2   Fur        fur   delfur_dpd       2  Sangwoo Seo   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "fur__delfur_dpd__1     Sangwoo Seo  10/2/2013 0:00   \n",
       "fur__delfur_dpd__2     Sangwoo Seo  10/2/2013 0:00   \n",
       "\n",
       "                                      Strain Description Organism Base Media  \\\n",
       "sample_id                                                                      \n",
       "fur__delfur_dpd__1  Escherichia coli K-12 MG1655 del_fur   MG1655         M9   \n",
       "fur__delfur_dpd__2  Escherichia coli K-12 MG1655 del_fur   MG1655         M9   \n",
       "\n",
       "                    ... Sequencing Machine Additional Details  \\\n",
       "sample_id           ...                                         \n",
       "fur__delfur_dpd__1  ...              MiSeq                NaN   \n",
       "fur__delfur_dpd__2  ...              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "fur__delfur_dpd__1                     2   \n",
       "fur__delfur_dpd__2                     2   \n",
       "\n",
       "                                                        R1  \\\n",
       "sample_id                                                    \n",
       "fur__delfur_dpd__1  fur/delfurDPD1_S2_L001_R1_001.fastq.gz   \n",
       "fur__delfur_dpd__2  fur/delfurDPD2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                        R2  \\\n",
       "sample_id                                                    \n",
       "fur__delfur_dpd__1  fur/delfurDPD1_S2_L001_R2_001.fastq.gz   \n",
       "fur__delfur_dpd__2  fur/delfurDPD2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "fur__delfur_dpd__1  fur__delfur_dpd__1.bam      98.07   \n",
       "fur__delfur_dpd__2  fur__delfur_dpd__2.bam      98.31   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "fur__delfur_dpd__1  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "fur__delfur_dpd__2  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fur__delfur_fe2__1</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>delfur_fe2</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/5/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_fur</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/del-fur-Fe2-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/del-fur-Fe2-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__delfur_fe2__1.bam</td>\n",
       "      <td>92.80</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fur__delfur_fe2__2</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>delfur_fe2</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/5/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_fur</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/del-fur-Fe2-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/del-fur-Fe2-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__delfur_fe2__2.bam</td>\n",
       "      <td>93.24</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                               \n",
       "fur__delfur_fe2__1   Fur        fur   delfur_fe2       1  Sangwoo Seo   \n",
       "fur__delfur_fe2__2   Fur        fur   delfur_fe2       2  Sangwoo Seo   \n",
       "\n",
       "                   library creator       run_date  \\\n",
       "sample_id                                           \n",
       "fur__delfur_fe2__1     Sangwoo Seo  9/5/2013 0:00   \n",
       "fur__delfur_fe2__2     Sangwoo Seo  9/5/2013 0:00   \n",
       "\n",
       "                                      Strain Description Organism Base Media  \\\n",
       "sample_id                                                                      \n",
       "fur__delfur_fe2__1  Escherichia coli K-12 MG1655 del_fur   MG1655         M9   \n",
       "fur__delfur_fe2__2  Escherichia coli K-12 MG1655 del_fur   MG1655         M9   \n",
       "\n",
       "                    ... Sequencing Machine Additional Details  \\\n",
       "sample_id           ...                                         \n",
       "fur__delfur_fe2__1  ...              MiSeq                NaN   \n",
       "fur__delfur_fe2__2  ...              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "fur__delfur_fe2__1                     2   \n",
       "fur__delfur_fe2__2                     2   \n",
       "\n",
       "                                                           R1  \\\n",
       "sample_id                                                       \n",
       "fur__delfur_fe2__1  fur/del-fur-Fe2-1_S1_L001_R1_001.fastq.gz   \n",
       "fur__delfur_fe2__2  fur/del-fur-Fe2-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                           R2  \\\n",
       "sample_id                                                       \n",
       "fur__delfur_fe2__1  fur/del-fur-Fe2-1_S1_L001_R2_001.fastq.gz   \n",
       "fur__delfur_fe2__2  fur/del-fur-Fe2-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "fur__delfur_fe2__1  fur__delfur_fe2__1.bam      92.80   \n",
       "fur__delfur_fe2__2  fur__delfur_fe2__2.bam      93.24   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "fur__delfur_fe2__1  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "fur__delfur_fe2__2  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fur__wt_dpd__1</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>wt_dpd</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>10/2/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/WTDPD1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/WTDPD1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__wt_dpd__1.bam</td>\n",
       "      <td>98.04</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fur__wt_dpd__2</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>wt_dpd</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>10/2/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/WTDPD2_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/WTDPD2_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__wt_dpd__2.bam</td>\n",
       "      <td>98.30</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                           \n",
       "fur__wt_dpd__1   Fur        fur       wt_dpd       1  Sangwoo Seo   \n",
       "fur__wt_dpd__2   Fur        fur       wt_dpd       2  Sangwoo Seo   \n",
       "\n",
       "               library creator        run_date            Strain Description  \\\n",
       "sample_id                                                                      \n",
       "fur__wt_dpd__1     Sangwoo Seo  10/2/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "fur__wt_dpd__2     Sangwoo Seo  10/2/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "               Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "fur__wt_dpd__1   MG1655         M9  ...              MiSeq                NaN   \n",
       "fur__wt_dpd__2   MG1655         M9  ...              MiSeq                NaN   \n",
       "\n",
       "               Biological Replicates                                  R1  \\\n",
       "sample_id                                                                  \n",
       "fur__wt_dpd__1                     2  fur/WTDPD1_S1_L001_R1_001.fastq.gz   \n",
       "fur__wt_dpd__2                     2  fur/WTDPD2_S1_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                R2                 BAM  \\\n",
       "sample_id                                                                \n",
       "fur__wt_dpd__1  fur/WTDPD1_S1_L001_R2_001.fastq.gz  fur__wt_dpd__1.bam   \n",
       "fur__wt_dpd__2  fur/WTDPD2_S1_L001_R2_001.fastq.gz  fur__wt_dpd__2.bam   \n",
       "\n",
       "                alignment                         DOI       GEO  Public  \n",
       "sample_id                                                                \n",
       "fur__wt_dpd__1      98.04  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "fur__wt_dpd__2      98.30  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>fur__wt_fe__1</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>wt_fe</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/5/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/WT-Fe2-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/WT-Fe2-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__wt_fe__1.bam</td>\n",
       "      <td>93.35</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>fur__wt_fe__2</td>\n",
       "      <td>Fur</td>\n",
       "      <td>fur</td>\n",
       "      <td>wt_fe</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/5/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>fur/WT-FE2-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>fur/WT-FE2-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>fur__wt_fe__2.bam</td>\n",
       "      <td>92.38</td>\n",
       "      <td>doi.org/10.1038/ncomms5910</td>\n",
       "      <td>GSE54900</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "              study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                          \n",
       "fur__wt_fe__1   Fur        fur        wt_fe       1  Sangwoo Seo   \n",
       "fur__wt_fe__2   Fur        fur        wt_fe       2  Sangwoo Seo   \n",
       "\n",
       "              library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                    \n",
       "fur__wt_fe__1     Sangwoo Seo  9/5/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "fur__wt_fe__2     Sangwoo Seo  9/5/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "              Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                          ...                                         \n",
       "fur__wt_fe__1   MG1655         M9  ...              MiSeq                NaN   \n",
       "fur__wt_fe__2   MG1655         M9  ...              MiSeq                NaN   \n",
       "\n",
       "              Biological Replicates                                    R1  \\\n",
       "sample_id                                                                   \n",
       "fur__wt_fe__1                     2  fur/WT-Fe2-1_S1_L001_R1_001.fastq.gz   \n",
       "fur__wt_fe__2                     2  fur/WT-FE2-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                 R2                BAM  \\\n",
       "sample_id                                                                \n",
       "fur__wt_fe__1  fur/WT-Fe2-1_S1_L001_R2_001.fastq.gz  fur__wt_fe__1.bam   \n",
       "fur__wt_fe__2  fur/WT-FE2-2_S2_L001_R2_001.fastq.gz  fur__wt_fe__2.bam   \n",
       "\n",
       "               alignment                         DOI       GEO  Public  \n",
       "sample_id                                                               \n",
       "fur__wt_fe__1      93.35  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "fur__wt_fe__2      92.38  doi.org/10.1038/ncomms5910  GSE54900     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale10__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale10</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/19/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.10.418.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A10 F418 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573444_1.fastq</td>\n",
       "      <td>glu/SRR1573444_2.fastq</td>\n",
       "      <td>glu__glu_ale10__1.bam</td>\n",
       "      <td>98.42</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "glu__glu_ale10__1  Glucose Evolution        glu    glu_ale10       1   \n",
       "\n",
       "                        contact library creator        run_date  \\\n",
       "sample_id                                                         \n",
       "glu__glu_ale10__1  Ryan LaCroix  Richard Szubin  5/19/2013 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "glu__glu_ale10__1  Escherichia coli GLU.10.418.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "glu__glu_ale10__1              MiSeq    GLU A10 F418 I1                     1   \n",
       "\n",
       "                                       R1                      R2  \\\n",
       "sample_id                                                           \n",
       "glu__glu_ale10__1  glu/SRR1573444_1.fastq  glu/SRR1573444_2.fastq   \n",
       "\n",
       "                                     BAM  alignment  \\\n",
       "sample_id                                             \n",
       "glu__glu_ale10__1  glu__glu_ale10__1.bam      98.42   \n",
       "\n",
       "                                            DOI       GEO  Public  \n",
       "sample_id                                                          \n",
       "glu__glu_ale10__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale3__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>4/15/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.3.412.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A3 F412 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573437_1.fastq</td>\n",
       "      <td>glu/SRR1573437_2.fastq</td>\n",
       "      <td>glu__glu_ale3__1.bam</td>\n",
       "      <td>98.88</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale3__1  Glucose Evolution        glu     glu_ale3       1   \n",
       "\n",
       "                       contact library creator        run_date  \\\n",
       "sample_id                                                        \n",
       "glu__glu_ale3__1  Ryan LaCroix  Richard Szubin  4/15/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale3__1  Escherichia coli GLU.3.412.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale3__1              MiSeq     GLU A3 F412 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale3__1  glu/SRR1573437_1.fastq  glu/SRR1573437_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale3__1  glu__glu_ale3__1.bam      98.88   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale3__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale4__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale4</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>4/16/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.4.403.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A4 F403 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573438_1.fastq</td>\n",
       "      <td>glu/SRR1573438_2.fastq</td>\n",
       "      <td>glu__glu_ale4__1.bam</td>\n",
       "      <td>98.94</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale4__1  Glucose Evolution        glu     glu_ale4       1   \n",
       "\n",
       "                       contact library creator        run_date  \\\n",
       "sample_id                                                        \n",
       "glu__glu_ale4__1  Ryan LaCroix  Richard Szubin  4/16/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale4__1  Escherichia coli GLU.4.403.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale4__1              MiSeq     GLU A4 F403 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale4__1  glu/SRR1573438_1.fastq  glu/SRR1573438_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale4__1  glu__glu_ale4__1.bam      98.94   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale4__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale5__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale5</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>4/16/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.5.288.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A5 F288 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573439_1.fastq</td>\n",
       "      <td>glu/SRR1573439_2.fastq</td>\n",
       "      <td>glu__glu_ale5__1.bam</td>\n",
       "      <td>98.99</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale5__1  Glucose Evolution        glu     glu_ale5       1   \n",
       "\n",
       "                       contact library creator        run_date  \\\n",
       "sample_id                                                        \n",
       "glu__glu_ale5__1  Ryan LaCroix  Richard Szubin  4/16/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale5__1  Escherichia coli GLU.5.288.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale5__1              MiSeq     GLU A5 F288 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale5__1  glu/SRR1573439_1.fastq  glu/SRR1573439_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale5__1  glu__glu_ale5__1.bam      98.99   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale5__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale6__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale6</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>4/19/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.6.406.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A6 F406 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573440_1.fastq</td>\n",
       "      <td>glu/SRR1573440_2.fastq</td>\n",
       "      <td>glu__glu_ale6__1.bam</td>\n",
       "      <td>98.06</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale6__1  Glucose Evolution        glu     glu_ale6       1   \n",
       "\n",
       "                       contact library creator        run_date  \\\n",
       "sample_id                                                        \n",
       "glu__glu_ale6__1  Ryan LaCroix  Richard Szubin  4/19/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale6__1  Escherichia coli GLU.6.406.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale6__1              MiSeq     GLU A6 F406 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale6__1  glu/SRR1573440_1.fastq  glu/SRR1573440_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale6__1  glu__glu_ale6__1.bam      98.06   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale6__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale8__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale8</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/1/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.8.380.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A8 F380 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573442_1.fastq</td>\n",
       "      <td>glu/SRR1573442_2.fastq</td>\n",
       "      <td>glu__glu_ale8__1.bam</td>\n",
       "      <td>96.96</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale8__1  Glucose Evolution        glu     glu_ale8       1   \n",
       "\n",
       "                       contact library creator       run_date  \\\n",
       "sample_id                                                       \n",
       "glu__glu_ale8__1  Ryan LaCroix  Richard Szubin  5/1/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale8__1  Escherichia coli GLU.8.380.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale8__1              MiSeq     GLU A8 F380 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale8__1  glu/SRR1573442_1.fastq  glu/SRR1573442_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale8__1  glu__glu_ale8__1.bam      96.96   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale8__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>glu__glu_ale9__1</td>\n",
       "      <td>Glucose Evolution</td>\n",
       "      <td>glu</td>\n",
       "      <td>glu_ale9</td>\n",
       "      <td>1</td>\n",
       "      <td>Ryan LaCroix</td>\n",
       "      <td>Richard Szubin</td>\n",
       "      <td>5/18/2013 0:00</td>\n",
       "      <td>Escherichia coli GLU.9.433.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>GLU A9 F422 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>glu/SRR1573443_1.fastq</td>\n",
       "      <td>glu/SRR1573443_2.fastq</td>\n",
       "      <td>glu__glu_ale9__1.bam</td>\n",
       "      <td>97.96</td>\n",
       "      <td>doi.org/10.1128/AEM.02246-14</td>\n",
       "      <td>GSE61327</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "glu__glu_ale9__1  Glucose Evolution        glu     glu_ale9       1   \n",
       "\n",
       "                       contact library creator        run_date  \\\n",
       "sample_id                                                        \n",
       "glu__glu_ale9__1  Ryan LaCroix  Richard Szubin  5/18/2013 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "glu__glu_ale9__1  Escherichia coli GLU.9.433.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "glu__glu_ale9__1              MiSeq     GLU A9 F422 I1                     1   \n",
       "\n",
       "                                      R1                      R2  \\\n",
       "sample_id                                                          \n",
       "glu__glu_ale9__1  glu/SRR1573443_1.fastq  glu/SRR1573443_2.fastq   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "glu__glu_ale9__1  glu__glu_ale9__1.bam      97.96   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "glu__glu_ale9__1  doi.org/10.1128/AEM.02246-14  GSE61327     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__ade_glc__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>ade_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL010_S37_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL010_S37_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__ade_glc__1.bam</td>\n",
       "      <td>92.46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__ade_glc__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>ade_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL011_S38_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL011_S38_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__ade_glc__2.bam</td>\n",
       "      <td>92.96</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                             \n",
       "ica__ade_glc__1   ICA        ica      ade_glc       1  Anand Sastry   \n",
       "ica__ade_glc__2   ICA        ica      ade_glc       2  Anand Sastry   \n",
       "\n",
       "                  library creator         run_date  \\\n",
       "sample_id                                            \n",
       "ica__ade_glc__1  Sibei (Julia) Xu  10/17/2018 0:00   \n",
       "ica__ade_glc__2  Sibei (Julia) Xu  10/17/2018 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ica__ade_glc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__ade_glc__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ica__ade_glc__1         HiSeq 4000                NaN                     2   \n",
       "ica__ade_glc__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                  R1  \\\n",
       "sample_id                                              \n",
       "ica__ade_glc__1  ica/PAL010_S37_L002_R1_001.fastq.gz   \n",
       "ica__ade_glc__2  ica/PAL011_S38_L002_R1_001.fastq.gz   \n",
       "\n",
       "                                                  R2                  BAM  \\\n",
       "sample_id                                                                   \n",
       "ica__ade_glc__1  ica/PAL010_S37_L002_R2_001.fastq.gz  ica__ade_glc__1.bam   \n",
       "ica__ade_glc__2  ica/PAL011_S38_L002_R2_001.fastq.gz  ica__ade_glc__2.bam   \n",
       "\n",
       "                 alignment  DOI  GEO  Public  \n",
       "sample_id                                     \n",
       "ica__ade_glc__1      92.46  NaN  NaN     0.0  \n",
       "ica__ade_glc__2      92.96  NaN  NaN     0.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__arg_sbt__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>arg_sbt</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_ArgenineSorbitolSupplementationrep...</td>\n",
       "      <td>ica/ICA_HRS_ArgenineSorbitolSupplementationrep...</td>\n",
       "      <td>ica__arg_sbt__1.bam</td>\n",
       "      <td>96.28</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__arg_sbt__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>arg_sbt</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_ArgenineSorbitolSupplementationrep...</td>\n",
       "      <td>ica/ICA_HRS_ArgenineSorbitolSupplementationrep...</td>\n",
       "      <td>ica__arg_sbt__2.bam</td>\n",
       "      <td>95.54</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                             \n",
       "ica__arg_sbt__1   ICA        ica      arg_sbt       1  Anand Sastry   \n",
       "ica__arg_sbt__2   ICA        ica      arg_sbt       2  Anand Sastry   \n",
       "\n",
       "                library creator         run_date  \\\n",
       "sample_id                                          \n",
       "ica__arg_sbt__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__arg_sbt__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ica__arg_sbt__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__arg_sbt__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ica__arg_sbt__1         HiSeq 4000                NaN                     2   \n",
       "ica__arg_sbt__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "ica__arg_sbt__1  ica/ICA_HRS_ArgenineSorbitolSupplementationrep...   \n",
       "ica__arg_sbt__2  ica/ICA_HRS_ArgenineSorbitolSupplementationrep...   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "ica__arg_sbt__1  ica/ICA_HRS_ArgenineSorbitolSupplementationrep...   \n",
       "ica__arg_sbt__2  ica/ICA_HRS_ArgenineSorbitolSupplementationrep...   \n",
       "\n",
       "                                 BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                \n",
       "ica__arg_sbt__1  ica__arg_sbt__1.bam      96.28  NaN  GSE122295     1.0  \n",
       "ica__arg_sbt__2  ica__arg_sbt__2.bam      95.54  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__bw_delpurR_cytd__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>bw_delpurR_cytd</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_purR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL005_S32_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL005_S32_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__bw_delpurR_cytd__1.bam</td>\n",
       "      <td>94.80</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__bw_delpurR_cytd__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>bw_delpurR_cytd</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_purR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL012_S39_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL012_S39_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__bw_delpurR_cytd__2.bam</td>\n",
       "      <td>92.69</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id     condition_id  rep_id  \\\n",
       "sample_id                                                           \n",
       "ica__bw_delpurR_cytd__1   ICA        ica  bw_delpurR_cytd       1   \n",
       "ica__bw_delpurR_cytd__2   ICA        ica  bw_delpurR_cytd       2   \n",
       "\n",
       "                              contact   library creator         run_date  \\\n",
       "sample_id                                                                  \n",
       "ica__bw_delpurR_cytd__1  Anand Sastry  Sibei (Julia) Xu  10/17/2018 0:00   \n",
       "ica__bw_delpurR_cytd__2  Anand Sastry  Sibei (Julia) Xu  10/17/2018 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "ica__bw_delpurR_cytd__1  Escherichia coli K-12 BW25113 del_purR  BW25113   \n",
       "ica__bw_delpurR_cytd__2  Escherichia coli K-12 BW25113 del_purR  BW25113   \n",
       "\n",
       "                        Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "ica__bw_delpurR_cytd__1         M9  ...         HiSeq 4000                NaN   \n",
       "ica__bw_delpurR_cytd__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                        Biological Replicates  \\\n",
       "sample_id                                       \n",
       "ica__bw_delpurR_cytd__1                     2   \n",
       "ica__bw_delpurR_cytd__2                     2   \n",
       "\n",
       "                                                          R1  \\\n",
       "sample_id                                                      \n",
       "ica__bw_delpurR_cytd__1  ica/PAL005_S32_L002_R1_001.fastq.gz   \n",
       "ica__bw_delpurR_cytd__2  ica/PAL012_S39_L002_R1_001.fastq.gz   \n",
       "\n",
       "                                                          R2  \\\n",
       "sample_id                                                      \n",
       "ica__bw_delpurR_cytd__1  ica/PAL005_S32_L002_R2_001.fastq.gz   \n",
       "ica__bw_delpurR_cytd__2  ica/PAL012_S39_L002_R2_001.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  DOI  GEO  \\\n",
       "sample_id                                                                   \n",
       "ica__bw_delpurR_cytd__1  ica__bw_delpurR_cytd__1.bam      94.80  NaN  NaN   \n",
       "ica__bw_delpurR_cytd__2  ica__bw_delpurR_cytd__2.bam      92.69  NaN  NaN   \n",
       "\n",
       "                         Public  \n",
       "sample_id                        \n",
       "ica__bw_delpurR_cytd__1     0.0  \n",
       "ica__bw_delpurR_cytd__2     0.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__cytd_rib__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>cytd_rib</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_CytidineD_riboseSupplementationrep...</td>\n",
       "      <td>ica/ICA_HRS_CytidineD_riboseSupplementationrep...</td>\n",
       "      <td>ica__cytd_rib__1.bam</td>\n",
       "      <td>95.41</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__cytd_rib__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>cytd_rib</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_CytidineD_riboseSupplementationrep...</td>\n",
       "      <td>ica/ICA_HRS_CytidineD_riboseSupplementationrep...</td>\n",
       "      <td>ica__cytd_rib__2.bam</td>\n",
       "      <td>96.26</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                              \n",
       "ica__cytd_rib__1   ICA        ica     cytd_rib       1  Anand Sastry   \n",
       "ica__cytd_rib__2   ICA        ica     cytd_rib       2  Anand Sastry   \n",
       "\n",
       "                 library creator         run_date  \\\n",
       "sample_id                                           \n",
       "ica__cytd_rib__1     Ying Hefner  12/23/2017 0:00   \n",
       "ica__cytd_rib__2     Ying Hefner  12/23/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ica__cytd_rib__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__cytd_rib__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ica__cytd_rib__1         HiSeq 4000                NaN                     2   \n",
       "ica__cytd_rib__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "ica__cytd_rib__1  ica/ICA_HRS_CytidineD_riboseSupplementationrep...   \n",
       "ica__cytd_rib__2  ica/ICA_HRS_CytidineD_riboseSupplementationrep...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "ica__cytd_rib__1  ica/ICA_HRS_CytidineD_riboseSupplementationrep...   \n",
       "ica__cytd_rib__2  ica/ICA_HRS_CytidineD_riboseSupplementationrep...   \n",
       "\n",
       "                                   BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "ica__cytd_rib__1  ica__cytd_rib__1.bam      95.41  NaN  GSE122295     1.0  \n",
       "ica__cytd_rib__2  ica__cytd_rib__2.bam      96.26  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__gth__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>gth</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_Glutathione_SO4rep1_S15_L007_R1_00...</td>\n",
       "      <td>ica/ICA_HRS_Glutathione_SO4rep1_S15_L007_R2_00...</td>\n",
       "      <td>ica__gth__1.bam</td>\n",
       "      <td>95.67</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__gth__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>gth</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_Glutathione_SO4rep2_S10_L007_R1_00...</td>\n",
       "      <td>ica/ICA_HRS_Glutathione_SO4rep2_S10_L007_R2_00...</td>\n",
       "      <td>ica__gth__2.bam</td>\n",
       "      <td>96.02</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                         \n",
       "ica__gth__1   ICA        ica          gth       1  Anand Sastry   \n",
       "ica__gth__2   ICA        ica          gth       2  Anand Sastry   \n",
       "\n",
       "            library creator         run_date            Strain Description  \\\n",
       "sample_id                                                                    \n",
       "ica__gth__1     Ying Hefner  12/23/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__gth__2     Ying Hefner  12/23/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "            Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                        ...                                         \n",
       "ica__gth__1   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__gth__2   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "            Biological Replicates  \\\n",
       "sample_id                           \n",
       "ica__gth__1                     2   \n",
       "ica__gth__2                     2   \n",
       "\n",
       "                                                            R1  \\\n",
       "sample_id                                                        \n",
       "ica__gth__1  ica/ICA_HRS_Glutathione_SO4rep1_S15_L007_R1_00...   \n",
       "ica__gth__2  ica/ICA_HRS_Glutathione_SO4rep2_S10_L007_R1_00...   \n",
       "\n",
       "                                                            R2  \\\n",
       "sample_id                                                        \n",
       "ica__gth__1  ica/ICA_HRS_Glutathione_SO4rep1_S15_L007_R2_00...   \n",
       "ica__gth__2  ica/ICA_HRS_Glutathione_SO4rep2_S10_L007_R2_00...   \n",
       "\n",
       "                         BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                        \n",
       "ica__gth__1  ica__gth__1.bam      95.67  NaN  GSE122295     1.0  \n",
       "ica__gth__2  ica__gth__2.bam      96.02  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__leu_glcr__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>leu_glcr</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...</td>\n",
       "      <td>ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...</td>\n",
       "      <td>ica__leu_glcr__1.bam</td>\n",
       "      <td>95.15</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__leu_glcr__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>leu_glcr</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...</td>\n",
       "      <td>ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...</td>\n",
       "      <td>ica__leu_glcr__2.bam</td>\n",
       "      <td>95.59</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                              \n",
       "ica__leu_glcr__1   ICA        ica     leu_glcr       1  Anand Sastry   \n",
       "ica__leu_glcr__2   ICA        ica     leu_glcr       2  Anand Sastry   \n",
       "\n",
       "                 library creator         run_date  \\\n",
       "sample_id                                           \n",
       "ica__leu_glcr__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__leu_glcr__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ica__leu_glcr__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__leu_glcr__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ica__leu_glcr__1         HiSeq 4000                NaN                     2   \n",
       "ica__leu_glcr__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "ica__leu_glcr__1  ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...   \n",
       "ica__leu_glcr__2  ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "ica__leu_glcr__1  ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...   \n",
       "ica__leu_glcr__2  ica/ICA_HRS_LeucineSaccharicAcidSupplementatio...   \n",
       "\n",
       "                                   BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "ica__leu_glcr__1  ica__leu_glcr__1.bam      95.15  NaN  GSE122295     1.0  \n",
       "ica__leu_glcr__2  ica__leu_glcr__2.bam      95.59  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__met_glc__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>met_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_MethionineGlucoseSupplementationre...</td>\n",
       "      <td>ica/ICA_HRS_MethionineGlucoseSupplementationre...</td>\n",
       "      <td>ica__met_glc__1.bam</td>\n",
       "      <td>95.45</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__met_glc__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>met_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_MethionineGlucoseSupplementationre...</td>\n",
       "      <td>ica/ICA_HRS_MethionineGlucoseSupplementationre...</td>\n",
       "      <td>ica__met_glc__2.bam</td>\n",
       "      <td>95.65</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                             \n",
       "ica__met_glc__1   ICA        ica      met_glc       1  Anand Sastry   \n",
       "ica__met_glc__2   ICA        ica      met_glc       2  Anand Sastry   \n",
       "\n",
       "                library creator         run_date  \\\n",
       "sample_id                                          \n",
       "ica__met_glc__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__met_glc__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ica__met_glc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__met_glc__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ica__met_glc__1         HiSeq 4000                NaN                     2   \n",
       "ica__met_glc__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "ica__met_glc__1  ica/ICA_HRS_MethionineGlucoseSupplementationre...   \n",
       "ica__met_glc__2  ica/ICA_HRS_MethionineGlucoseSupplementationre...   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "ica__met_glc__1  ica/ICA_HRS_MethionineGlucoseSupplementationre...   \n",
       "ica__met_glc__2  ica/ICA_HRS_MethionineGlucoseSupplementationre...   \n",
       "\n",
       "                                 BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                \n",
       "ica__met_glc__1  ica__met_glc__1.bam      95.45  NaN  GSE122295     1.0  \n",
       "ica__met_glc__2  ica__met_glc__2.bam      95.65  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__no3_anaero__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>no3_anaero</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_Nitrateanaerobicrep1_S5_L007_R1_00...</td>\n",
       "      <td>ica/ICA_HRS_Nitrateanaerobicrep1_S5_L007_R2_00...</td>\n",
       "      <td>ica__no3_anaero__1.bam</td>\n",
       "      <td>94.15</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__no3_anaero__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>no3_anaero</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_Nitrateanaerobicrep2_S3_L007_R1_00...</td>\n",
       "      <td>ica/ICA_HRS_Nitrateanaerobicrep2_S3_L007_R2_00...</td>\n",
       "      <td>ica__no3_anaero__2.bam</td>\n",
       "      <td>94.01</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                \n",
       "ica__no3_anaero__1   ICA        ica   no3_anaero       1  Anand Sastry   \n",
       "ica__no3_anaero__2   ICA        ica   no3_anaero       2  Anand Sastry   \n",
       "\n",
       "                   library creator         run_date  \\\n",
       "sample_id                                             \n",
       "ica__no3_anaero__1     Ying Hefner  12/23/2017 0:00   \n",
       "ica__no3_anaero__2     Ying Hefner  12/23/2017 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "ica__no3_anaero__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__no3_anaero__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "ica__no3_anaero__1         HiSeq 4000                NaN   \n",
       "ica__no3_anaero__2         HiSeq 4000                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "ica__no3_anaero__1                     2   \n",
       "ica__no3_anaero__2                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "ica__no3_anaero__1  ica/ICA_HRS_Nitrateanaerobicrep1_S5_L007_R1_00...   \n",
       "ica__no3_anaero__2  ica/ICA_HRS_Nitrateanaerobicrep2_S3_L007_R1_00...   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "ica__no3_anaero__1  ica/ICA_HRS_Nitrateanaerobicrep1_S5_L007_R2_00...   \n",
       "ica__no3_anaero__2  ica/ICA_HRS_Nitrateanaerobicrep2_S3_L007_R2_00...   \n",
       "\n",
       "                                       BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "ica__no3_anaero__1  ica__no3_anaero__1.bam      94.15  NaN  GSE122295     1.0  \n",
       "ica__no3_anaero__2  ica__no3_anaero__2.bam      94.01  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__phe_acgam__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>phe_acgam</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...</td>\n",
       "      <td>ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...</td>\n",
       "      <td>ica__phe_acgam__1.bam</td>\n",
       "      <td>96.34</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__phe_acgam__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>phe_acgam</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...</td>\n",
       "      <td>ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...</td>\n",
       "      <td>ica__phe_acgam__2.bam</td>\n",
       "      <td>96.42</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "ica__phe_acgam__1   ICA        ica    phe_acgam       1  Anand Sastry   \n",
       "ica__phe_acgam__2   ICA        ica    phe_acgam       2  Anand Sastry   \n",
       "\n",
       "                  library creator         run_date  \\\n",
       "sample_id                                            \n",
       "ica__phe_acgam__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__phe_acgam__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "ica__phe_acgam__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__phe_acgam__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "ica__phe_acgam__1         HiSeq 4000                NaN                     2   \n",
       "ica__phe_acgam__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                  R1  \\\n",
       "sample_id                                                              \n",
       "ica__phe_acgam__1  ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...   \n",
       "ica__phe_acgam__2  ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...   \n",
       "\n",
       "                                                                  R2  \\\n",
       "sample_id                                                              \n",
       "ica__phe_acgam__1  ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...   \n",
       "ica__phe_acgam__2  ica/ICA_HRS_PhenylalanineN_acetylglucosamineSu...   \n",
       "\n",
       "                                     BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                    \n",
       "ica__phe_acgam__1  ica__phe_acgam__1.bam      96.34  NaN  GSE122295     1.0  \n",
       "ica__phe_acgam__2  ica__phe_acgam__2.bam      96.42  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__thm_gal__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>thm_gal</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_ThiamineGalactoseSupplementationre...</td>\n",
       "      <td>ica/ICA_HRS_ThiamineGalactoseSupplementationre...</td>\n",
       "      <td>ica__thm_gal__1.bam</td>\n",
       "      <td>95.98</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__thm_gal__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>thm_gal</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_ThiamineGalactoseSupplementationre...</td>\n",
       "      <td>ica/ICA_HRS_ThiamineGalactoseSupplementationre...</td>\n",
       "      <td>ica__thm_gal__2.bam</td>\n",
       "      <td>96.12</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                             \n",
       "ica__thm_gal__1   ICA        ica      thm_gal       1  Anand Sastry   \n",
       "ica__thm_gal__2   ICA        ica      thm_gal       2  Anand Sastry   \n",
       "\n",
       "                library creator         run_date  \\\n",
       "sample_id                                          \n",
       "ica__thm_gal__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__thm_gal__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ica__thm_gal__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__thm_gal__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ica__thm_gal__1         HiSeq 4000                NaN                     2   \n",
       "ica__thm_gal__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "ica__thm_gal__1  ica/ICA_HRS_ThiamineGalactoseSupplementationre...   \n",
       "ica__thm_gal__2  ica/ICA_HRS_ThiamineGalactoseSupplementationre...   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "ica__thm_gal__1  ica/ICA_HRS_ThiamineGalactoseSupplementationre...   \n",
       "ica__thm_gal__2  ica/ICA_HRS_ThiamineGalactoseSupplementationre...   \n",
       "\n",
       "                                 BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                \n",
       "ica__thm_gal__1  ica__thm_gal__1.bam      95.98  NaN  GSE122295     1.0  \n",
       "ica__thm_gal__2  ica__thm_gal__2.bam      96.12  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__tyr_glcn__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>tyr_glcn</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_TyrosineD_gluconateSupplementation...</td>\n",
       "      <td>ica/ICA_HRS_TyrosineD_gluconateSupplementation...</td>\n",
       "      <td>ica__tyr_glcn__1.bam</td>\n",
       "      <td>95.41</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__tyr_glcn__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>tyr_glcn</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_TyrosineD_gluconateSupplementation...</td>\n",
       "      <td>ica/ICA_HRS_TyrosineD_gluconateSupplementation...</td>\n",
       "      <td>ica__tyr_glcn__2.bam</td>\n",
       "      <td>95.47</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                              \n",
       "ica__tyr_glcn__1   ICA        ica     tyr_glcn       1  Anand Sastry   \n",
       "ica__tyr_glcn__2   ICA        ica     tyr_glcn       2  Anand Sastry   \n",
       "\n",
       "                 library creator         run_date  \\\n",
       "sample_id                                           \n",
       "ica__tyr_glcn__1     Ying Hefner  12/10/2017 0:00   \n",
       "ica__tyr_glcn__2     Ying Hefner  12/10/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ica__tyr_glcn__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__tyr_glcn__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ica__tyr_glcn__1         HiSeq 4000                NaN                     2   \n",
       "ica__tyr_glcn__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "ica__tyr_glcn__1  ica/ICA_HRS_TyrosineD_gluconateSupplementation...   \n",
       "ica__tyr_glcn__2  ica/ICA_HRS_TyrosineD_gluconateSupplementation...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "ica__tyr_glcn__1  ica/ICA_HRS_TyrosineD_gluconateSupplementation...   \n",
       "ica__tyr_glcn__2  ica/ICA_HRS_TyrosineD_gluconateSupplementation...   \n",
       "\n",
       "                                   BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "ica__tyr_glcn__1  ica__tyr_glcn__1.bam      95.41  NaN  GSE122295     1.0  \n",
       "ica__tyr_glcn__2  ica__tyr_glcn__2.bam      95.47  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__ura_pyr__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>ura_pyr</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_UracilwithSodiumPyruvaterep1_S7_L0...</td>\n",
       "      <td>ica/ICA_HRS_UracilwithSodiumPyruvaterep1_S7_L0...</td>\n",
       "      <td>ica__ura_pyr__1.bam</td>\n",
       "      <td>93.77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__ura_pyr__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>ura_pyr</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/ICA_HRS_UracilwithSodiumPyruvaterep2_S2_L0...</td>\n",
       "      <td>ica/ICA_HRS_UracilwithSodiumPyruvaterep2_S2_L0...</td>\n",
       "      <td>ica__ura_pyr__2.bam</td>\n",
       "      <td>93.96</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                             \n",
       "ica__ura_pyr__1   ICA        ica      ura_pyr       1  Anand Sastry   \n",
       "ica__ura_pyr__2   ICA        ica      ura_pyr       2  Anand Sastry   \n",
       "\n",
       "                library creator         run_date  \\\n",
       "sample_id                                          \n",
       "ica__ura_pyr__1     Ying Hefner  12/23/2017 0:00   \n",
       "ica__ura_pyr__2     Ying Hefner  12/23/2017 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ica__ura_pyr__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ica__ura_pyr__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ica__ura_pyr__1         HiSeq 4000                NaN                     2   \n",
       "ica__ura_pyr__2         HiSeq 4000                NaN                     2   \n",
       "\n",
       "                                                                R1  \\\n",
       "sample_id                                                            \n",
       "ica__ura_pyr__1  ica/ICA_HRS_UracilwithSodiumPyruvaterep1_S7_L0...   \n",
       "ica__ura_pyr__2  ica/ICA_HRS_UracilwithSodiumPyruvaterep2_S2_L0...   \n",
       "\n",
       "                                                                R2  \\\n",
       "sample_id                                                            \n",
       "ica__ura_pyr__1  ica/ICA_HRS_UracilwithSodiumPyruvaterep1_S7_L0...   \n",
       "ica__ura_pyr__2  ica/ICA_HRS_UracilwithSodiumPyruvaterep2_S2_L0...   \n",
       "\n",
       "                                 BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                \n",
       "ica__ura_pyr__1  ica__ura_pyr__1.bam      93.77  NaN  GSE122295     1.0  \n",
       "ica__ura_pyr__2  ica__ura_pyr__2.bam      93.96  NaN  GSE122295     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__1</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>ica/ICA_HRS_Control11_1M9GlucoseAerobicrep1_S4...</td>\n",
       "      <td>ica/ICA_HRS_Control11_1M9GlucoseAerobicrep1_S4...</td>\n",
       "      <td>ica__wt_glc__1.bam</td>\n",
       "      <td>96.08</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__2</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/10/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>ica/ICA_HRS_Control11_2M9GlucoseAerobicrep2_S4...</td>\n",
       "      <td>ica/ICA_HRS_Control11_2M9GlucoseAerobicrep2_S4...</td>\n",
       "      <td>ica__wt_glc__2.bam</td>\n",
       "      <td>96.29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__3</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>3</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>ica/ICA_HRS_Control15rep1_S14_L007_R1_001.fastq</td>\n",
       "      <td>ica/ICA_HRS_Control15rep1_S14_L007_R2_001.fastq</td>\n",
       "      <td>ica__wt_glc__3.bam</td>\n",
       "      <td>95.35</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__4</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>4</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>12/23/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>ica/ICA_HRS_Control15rep2_S6_L007_R1_001.fastq</td>\n",
       "      <td>ica/ICA_HRS_Control15rep2_S6_L007_R2_001.fastq</td>\n",
       "      <td>ica__wt_glc__4.bam</td>\n",
       "      <td>94.49</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122295</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__5</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>5</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL001_S28_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL001_S28_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__wt_glc__5.bam</td>\n",
       "      <td>93.94</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ica__wt_glc__6</td>\n",
       "      <td>ICA</td>\n",
       "      <td>ica</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>6</td>\n",
       "      <td>Anand Sastry</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>10/17/2018 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ica/PAL002_S29_L002_R1_001.fastq.gz</td>\n",
       "      <td>ica/PAL002_S29_L002_R2_001.fastq.gz</td>\n",
       "      <td>ica__wt_glc__6.bam</td>\n",
       "      <td>94.85</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>6 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                            \n",
       "ica__wt_glc__1   ICA        ica       wt_glc       1  Anand Sastry   \n",
       "ica__wt_glc__2   ICA        ica       wt_glc       2  Anand Sastry   \n",
       "ica__wt_glc__3   ICA        ica       wt_glc       3  Anand Sastry   \n",
       "ica__wt_glc__4   ICA        ica       wt_glc       4  Anand Sastry   \n",
       "ica__wt_glc__5   ICA        ica       wt_glc       5  Anand Sastry   \n",
       "ica__wt_glc__6   ICA        ica       wt_glc       6  Anand Sastry   \n",
       "\n",
       "               library creator         run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "ica__wt_glc__1     Ying Hefner  12/10/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__wt_glc__2     Ying Hefner  12/10/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__wt_glc__3     Ying Hefner  12/23/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__wt_glc__4     Ying Hefner  12/23/2017 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__wt_glc__5     Ying Hefner  10/17/2018 0:00  Escherichia coli K-12 MG1655   \n",
       "ica__wt_glc__6     Ying Hefner  10/17/2018 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "               Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "ica__wt_glc__1   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__wt_glc__2   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__wt_glc__3   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__wt_glc__4   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__wt_glc__5   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "ica__wt_glc__6   MG1655         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "               Biological Replicates  \\\n",
       "sample_id                              \n",
       "ica__wt_glc__1                     4   \n",
       "ica__wt_glc__2                     4   \n",
       "ica__wt_glc__3                     4   \n",
       "ica__wt_glc__4                     4   \n",
       "ica__wt_glc__5                     2   \n",
       "ica__wt_glc__6                     2   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "ica__wt_glc__1  ica/ICA_HRS_Control11_1M9GlucoseAerobicrep1_S4...   \n",
       "ica__wt_glc__2  ica/ICA_HRS_Control11_2M9GlucoseAerobicrep2_S4...   \n",
       "ica__wt_glc__3    ica/ICA_HRS_Control15rep1_S14_L007_R1_001.fastq   \n",
       "ica__wt_glc__4     ica/ICA_HRS_Control15rep2_S6_L007_R1_001.fastq   \n",
       "ica__wt_glc__5                ica/PAL001_S28_L002_R1_001.fastq.gz   \n",
       "ica__wt_glc__6                ica/PAL002_S29_L002_R1_001.fastq.gz   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "ica__wt_glc__1  ica/ICA_HRS_Control11_1M9GlucoseAerobicrep1_S4...   \n",
       "ica__wt_glc__2  ica/ICA_HRS_Control11_2M9GlucoseAerobicrep2_S4...   \n",
       "ica__wt_glc__3    ica/ICA_HRS_Control15rep1_S14_L007_R2_001.fastq   \n",
       "ica__wt_glc__4     ica/ICA_HRS_Control15rep2_S6_L007_R2_001.fastq   \n",
       "ica__wt_glc__5                ica/PAL001_S28_L002_R2_001.fastq.gz   \n",
       "ica__wt_glc__6                ica/PAL002_S29_L002_R2_001.fastq.gz   \n",
       "\n",
       "                               BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                              \n",
       "ica__wt_glc__1  ica__wt_glc__1.bam      96.08  NaN  GSE122295     1.0  \n",
       "ica__wt_glc__2  ica__wt_glc__2.bam      96.29  NaN  GSE122295     1.0  \n",
       "ica__wt_glc__3  ica__wt_glc__3.bam      95.35  NaN  GSE122295     1.0  \n",
       "ica__wt_glc__4  ica__wt_glc__4.bam      94.49  NaN  GSE122295     1.0  \n",
       "ica__wt_glc__5  ica__wt_glc__5.bam      93.94  NaN        NaN     0.0  \n",
       "ica__wt_glc__6  ica__wt_glc__6.bam      94.85  NaN        NaN     0.0  \n",
       "\n",
       "[6 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delcra_glc__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delcra_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_cra</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>minspan/SRR922270_1.fastq</td>\n",
       "      <td>minspan/SRR922270_2.fastq</td>\n",
       "      <td>minspan__bw_delcra_glc__1.bam</td>\n",
       "      <td>84.03</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delcra_glc__2</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delcra_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_cra</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>Bridging Study between MiSeq &amp; GAIIX</td>\n",
       "      <td>2</td>\n",
       "      <td>minspan/cra_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>minspan/cra_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>minspan__bw_delcra_glc__2.bam</td>\n",
       "      <td>88.78</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delcra_glc__1  MinSpan    minspan  bw_delcra_glc       1   \n",
       "minspan__bw_delcra_glc__2  MinSpan    minspan  bw_delcra_glc       2   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "minspan__bw_delcra_glc__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "minspan__bw_delcra_glc__2  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delcra_glc__1  Escherichia coli BW25113 del_cra  BW25113   \n",
       "minspan__bw_delcra_glc__2  Escherichia coli BW25113 del_cra  BW25113   \n",
       "\n",
       "                          Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "minspan__bw_delcra_glc__1         M9  ...              GAIIX   \n",
       "minspan__bw_delcra_glc__2         M9  ...              MiSeq   \n",
       "\n",
       "                                             Additional Details  \\\n",
       "sample_id                                                         \n",
       "minspan__bw_delcra_glc__1                                   NaN   \n",
       "minspan__bw_delcra_glc__2  Bridging Study between MiSeq & GAIIX   \n",
       "\n",
       "                          Biological Replicates  \\\n",
       "sample_id                                         \n",
       "minspan__bw_delcra_glc__1                     2   \n",
       "minspan__bw_delcra_glc__2                     2   \n",
       "\n",
       "                                                            R1  \\\n",
       "sample_id                                                        \n",
       "minspan__bw_delcra_glc__1            minspan/SRR922270_1.fastq   \n",
       "minspan__bw_delcra_glc__2  minspan/cra_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                            R2  \\\n",
       "sample_id                                                        \n",
       "minspan__bw_delcra_glc__1            minspan/SRR922270_2.fastq   \n",
       "minspan__bw_delcra_glc__2  minspan/cra_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "minspan__bw_delcra_glc__1  minspan__bw_delcra_glc__1.bam      84.03   \n",
       "minspan__bw_delcra_glc__2  minspan__bw_delcra_glc__2.bam      88.78   \n",
       "\n",
       "                                                     DOI        GEO  Public  \n",
       "sample_id                                                                    \n",
       "minspan__bw_delcra_glc__1  doi.org/10.15252/msb.20145243   GSE48324     1.0  \n",
       "minspan__bw_delcra_glc__2                            NaN  GSE122211     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delcra_trp__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delcra_trp</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_cra</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922271_1.fastq</td>\n",
       "      <td>minspan/SRR922271_2.fastq</td>\n",
       "      <td>minspan__bw_delcra_trp__1.bam</td>\n",
       "      <td>76.33</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delcra_trp__1  MinSpan    minspan  bw_delcra_trp       1   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "minspan__bw_delcra_trp__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delcra_trp__1  Escherichia coli BW25113 del_cra  BW25113   \n",
       "\n",
       "                          Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "minspan__bw_delcra_trp__1         M9  ...              GAIIX   \n",
       "\n",
       "                          Additional Details Biological Replicates  \\\n",
       "sample_id                                                            \n",
       "minspan__bw_delcra_trp__1                NaN                     1   \n",
       "\n",
       "                                                  R1  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delcra_trp__1  minspan/SRR922271_1.fastq   \n",
       "\n",
       "                                                  R2  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delcra_trp__1  minspan/SRR922271_2.fastq   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "minspan__bw_delcra_trp__1  minspan__bw_delcra_trp__1.bam      76.33   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "minspan__bw_delcra_trp__1  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delmntr__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delmntr</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_mntR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922272_1.fastq</td>\n",
       "      <td>minspan/SRR922272_2.fastq</td>\n",
       "      <td>minspan__bw_delmntr__1.bam</td>\n",
       "      <td>92.81</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          study project_id condition_id  rep_id  \\\n",
       "sample_id                                                         \n",
       "minspan__bw_delmntr__1  MinSpan    minspan   bw_delmntr       1   \n",
       "\n",
       "                               contact library creator        run_date  \\\n",
       "sample_id                                                                \n",
       "minspan__bw_delmntr__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "minspan__bw_delmntr__1  Escherichia coli BW25113 del_mntR  BW25113         M9   \n",
       "\n",
       "                        ... Sequencing Machine Additional Details  \\\n",
       "sample_id               ...                                         \n",
       "minspan__bw_delmntr__1  ...              GAIIX                NaN   \n",
       "\n",
       "                       Biological Replicates                         R1  \\\n",
       "sample_id                                                                 \n",
       "minspan__bw_delmntr__1                     1  minspan/SRR922272_1.fastq   \n",
       "\n",
       "                                               R2                         BAM  \\\n",
       "sample_id                                                                       \n",
       "minspan__bw_delmntr__1  minspan/SRR922272_2.fastq  minspan__bw_delmntr__1.bam   \n",
       "\n",
       "                        alignment                            DOI       GEO  \\\n",
       "sample_id                                                                    \n",
       "minspan__bw_delmntr__1      92.81  doi.org/10.15252/msb.20145243  GSE48324   \n",
       "\n",
       "                        Public  \n",
       "sample_id                       \n",
       "minspan__bw_delmntr__1     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delnac_ade__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delnac_ade</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922269_1.fastq</td>\n",
       "      <td>minspan/SRR922269_2.fastq</td>\n",
       "      <td>minspan__bw_delnac_ade__1.bam</td>\n",
       "      <td>81.84</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delnac_ade__1  MinSpan    minspan  bw_delnac_ade       1   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "minspan__bw_delnac_ade__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delnac_ade__1  Escherichia coli BW25113 del_nac  BW25113   \n",
       "\n",
       "                          Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "minspan__bw_delnac_ade__1         M9  ...              GAIIX   \n",
       "\n",
       "                          Additional Details Biological Replicates  \\\n",
       "sample_id                                                            \n",
       "minspan__bw_delnac_ade__1                NaN                     1   \n",
       "\n",
       "                                                  R1  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delnac_ade__1  minspan/SRR922269_1.fastq   \n",
       "\n",
       "                                                  R2  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delnac_ade__1  minspan/SRR922269_2.fastq   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "minspan__bw_delnac_ade__1  minspan__bw_delnac_ade__1.bam      81.84   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "minspan__bw_delnac_ade__1  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__bw_delnac_glc__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>bw_delnac_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922268_1.fastq</td>\n",
       "      <td>minspan/SRR922268_2.fastq</td>\n",
       "      <td>minspan__bw_delnac_glc__1.bam</td>\n",
       "      <td>90.26</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delnac_glc__1  MinSpan    minspan  bw_delnac_glc       1   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "minspan__bw_delnac_glc__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "minspan__bw_delnac_glc__1  Escherichia coli BW25113 del_nac  BW25113   \n",
       "\n",
       "                          Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "minspan__bw_delnac_glc__1         M9  ...              GAIIX   \n",
       "\n",
       "                          Additional Details Biological Replicates  \\\n",
       "sample_id                                                            \n",
       "minspan__bw_delnac_glc__1                NaN                     1   \n",
       "\n",
       "                                                  R1  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delnac_glc__1  minspan/SRR922268_1.fastq   \n",
       "\n",
       "                                                  R2  \\\n",
       "sample_id                                              \n",
       "minspan__bw_delnac_glc__1  minspan/SRR922268_2.fastq   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "minspan__bw_delnac_glc__1  minspan__bw_delnac_glc__1.bam      90.26   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "minspan__bw_delnac_glc__1  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__wt_ade__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_ade</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922263_1.fastq</td>\n",
       "      <td>minspan/SRR922263_2.fastq</td>\n",
       "      <td>minspan__wt_ade__1.bam</td>\n",
       "      <td>78.38</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id         contact  \\\n",
       "sample_id                                                                     \n",
       "minspan__wt_ade__1  MinSpan    minspan       wt_ade       1  Aarash Bordbar   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "minspan__wt_ade__1   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "minspan__wt_ade__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "minspan__wt_ade__1              GAIIX                NaN   \n",
       "\n",
       "                   Biological Replicates                         R1  \\\n",
       "sample_id                                                             \n",
       "minspan__wt_ade__1                     1  minspan/SRR922263_1.fastq   \n",
       "\n",
       "                                           R2                     BAM  \\\n",
       "sample_id                                                               \n",
       "minspan__wt_ade__1  minspan/SRR922263_2.fastq  minspan__wt_ade__1.bam   \n",
       "\n",
       "                    alignment                            DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "minspan__wt_ade__1      78.38  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>minspan/SRR922260_1.fastq</td>\n",
       "      <td>minspan/SRR922260_2.fastq</td>\n",
       "      <td>minspan__wt_glc__1.bam</td>\n",
       "      <td>91.85</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc__2</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>minspan/SRR922261_1.fastq</td>\n",
       "      <td>minspan/SRR922261_2.fastq</td>\n",
       "      <td>minspan__wt_glc__2.bam</td>\n",
       "      <td>91.70</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc__3</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>3</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>4</td>\n",
       "      <td>minspan/SRR922262_1.fastq</td>\n",
       "      <td>minspan/SRR922262_2.fastq</td>\n",
       "      <td>minspan__wt_glc__3.bam</td>\n",
       "      <td>92.12</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc__4</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>4</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>Bridging Study between MiSeq &amp; GAIIX</td>\n",
       "      <td>4</td>\n",
       "      <td>minspan/wt_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>minspan/wt_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>minspan__wt_glc__4.bam</td>\n",
       "      <td>94.58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>4 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id         contact  \\\n",
       "sample_id                                                                     \n",
       "minspan__wt_glc__1  MinSpan    minspan       wt_glc       1  Aarash Bordbar   \n",
       "minspan__wt_glc__2  MinSpan    minspan       wt_glc       2  Aarash Bordbar   \n",
       "minspan__wt_glc__3  MinSpan    minspan       wt_glc       3  Aarash Bordbar   \n",
       "minspan__wt_glc__4  MinSpan    minspan       wt_glc       4  Aarash Bordbar   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "minspan__wt_glc__1   Haythem Latif  10/1/2012 0:00   \n",
       "minspan__wt_glc__2   Haythem Latif  10/1/2012 0:00   \n",
       "minspan__wt_glc__3   Haythem Latif  10/1/2012 0:00   \n",
       "minspan__wt_glc__4   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "minspan__wt_glc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "minspan__wt_glc__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "minspan__wt_glc__3  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "minspan__wt_glc__4  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine                    Additional Details  \\\n",
       "sample_id                                                                     \n",
       "minspan__wt_glc__1              GAIIX                                   NaN   \n",
       "minspan__wt_glc__2              GAIIX                                   NaN   \n",
       "minspan__wt_glc__3              GAIIX                                   NaN   \n",
       "minspan__wt_glc__4              MiSeq  Bridging Study between MiSeq & GAIIX   \n",
       "\n",
       "                   Biological Replicates                                  R1  \\\n",
       "sample_id                                                                      \n",
       "minspan__wt_glc__1                     4           minspan/SRR922260_1.fastq   \n",
       "minspan__wt_glc__2                     4           minspan/SRR922261_1.fastq   \n",
       "minspan__wt_glc__3                     4           minspan/SRR922262_1.fastq   \n",
       "minspan__wt_glc__4                     4  minspan/wt_S1_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                    R2  \\\n",
       "sample_id                                                \n",
       "minspan__wt_glc__1           minspan/SRR922260_2.fastq   \n",
       "minspan__wt_glc__2           minspan/SRR922261_2.fastq   \n",
       "minspan__wt_glc__3           minspan/SRR922262_2.fastq   \n",
       "minspan__wt_glc__4  minspan/wt_S1_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                       BAM  alignment  \\\n",
       "sample_id                                               \n",
       "minspan__wt_glc__1  minspan__wt_glc__1.bam      91.85   \n",
       "minspan__wt_glc__2  minspan__wt_glc__2.bam      91.70   \n",
       "minspan__wt_glc__3  minspan__wt_glc__3.bam      92.12   \n",
       "minspan__wt_glc__4  minspan__wt_glc__4.bam      94.58   \n",
       "\n",
       "                                              DOI        GEO  Public  \n",
       "sample_id                                                             \n",
       "minspan__wt_glc__1  doi.org/10.15252/msb.20145243   GSE48324     1.0  \n",
       "minspan__wt_glc__2  doi.org/10.15252/msb.20145243   GSE48324     1.0  \n",
       "minspan__wt_glc__3  doi.org/10.15252/msb.20145243   GSE48324     1.0  \n",
       "minspan__wt_glc__4                            NaN  GSE122211     1.0  \n",
       "\n",
       "[4 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc_anaero__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc_anaero</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>minspan/SRR922266_1.fastq</td>\n",
       "      <td>minspan/SRR922266_2.fastq</td>\n",
       "      <td>minspan__wt_glc_anaero__1.bam</td>\n",
       "      <td>83.01</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>minspan__wt_glc_anaero__2</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_glc_anaero</td>\n",
       "      <td>2</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>minspan/SRR922267_1.fastq</td>\n",
       "      <td>minspan/SRR922267_2.fastq</td>\n",
       "      <td>minspan__wt_glc_anaero__2.bam</td>\n",
       "      <td>72.08</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "minspan__wt_glc_anaero__1  MinSpan    minspan  wt_glc_anaero       1   \n",
       "minspan__wt_glc_anaero__2  MinSpan    minspan  wt_glc_anaero       2   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "minspan__wt_glc_anaero__1  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "minspan__wt_glc_anaero__2  Aarash Bordbar   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                                     Strain Description Organism Base Media  \\\n",
       "sample_id                                                                     \n",
       "minspan__wt_glc_anaero__1  Escherichia coli K-12 MG1655   MG1655         M9   \n",
       "minspan__wt_glc_anaero__2  Escherichia coli K-12 MG1655   MG1655         M9   \n",
       "\n",
       "                           ... Sequencing Machine Additional Details  \\\n",
       "sample_id                  ...                                         \n",
       "minspan__wt_glc_anaero__1  ...              GAIIX                NaN   \n",
       "minspan__wt_glc_anaero__2  ...              GAIIX                NaN   \n",
       "\n",
       "                          Biological Replicates                         R1  \\\n",
       "sample_id                                                                    \n",
       "minspan__wt_glc_anaero__1                     2  minspan/SRR922266_1.fastq   \n",
       "minspan__wt_glc_anaero__2                     2  minspan/SRR922267_1.fastq   \n",
       "\n",
       "                                                  R2  \\\n",
       "sample_id                                              \n",
       "minspan__wt_glc_anaero__1  minspan/SRR922266_2.fastq   \n",
       "minspan__wt_glc_anaero__2  minspan/SRR922267_2.fastq   \n",
       "\n",
       "                                                     BAM  alignment  \\\n",
       "sample_id                                                             \n",
       "minspan__wt_glc_anaero__1  minspan__wt_glc_anaero__1.bam      83.01   \n",
       "minspan__wt_glc_anaero__2  minspan__wt_glc_anaero__2.bam      72.08   \n",
       "\n",
       "                                                     DOI       GEO  Public  \n",
       "sample_id                                                                   \n",
       "minspan__wt_glc_anaero__1  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "minspan__wt_glc_anaero__2  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>minspan__wt_trp__1</td>\n",
       "      <td>MinSpan</td>\n",
       "      <td>minspan</td>\n",
       "      <td>wt_trp</td>\n",
       "      <td>1</td>\n",
       "      <td>Aarash Bordbar</td>\n",
       "      <td>Haythem Latif</td>\n",
       "      <td>10/1/2012 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>GAIIX</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>minspan/SRR922264_1.fastq</td>\n",
       "      <td>minspan/SRR922264_2.fastq</td>\n",
       "      <td>minspan__wt_trp__1.bam</td>\n",
       "      <td>78.6</td>\n",
       "      <td>doi.org/10.15252/msb.20145243</td>\n",
       "      <td>GSE48324</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      study project_id condition_id  rep_id         contact  \\\n",
       "sample_id                                                                     \n",
       "minspan__wt_trp__1  MinSpan    minspan       wt_trp       1  Aarash Bordbar   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "minspan__wt_trp__1   Haythem Latif  10/1/2012 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "minspan__wt_trp__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "minspan__wt_trp__1              GAIIX                NaN   \n",
       "\n",
       "                   Biological Replicates                         R1  \\\n",
       "sample_id                                                             \n",
       "minspan__wt_trp__1                     1  minspan/SRR922264_1.fastq   \n",
       "\n",
       "                                           R2                     BAM  \\\n",
       "sample_id                                                               \n",
       "minspan__wt_trp__1  minspan/SRR922264_2.fastq  minspan__wt_trp__1.bam   \n",
       "\n",
       "                    alignment                            DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "minspan__wt_trp__1       78.6  doi.org/10.15252/msb.20145243  GSE48324     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>misc__bw_delcbl__1</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delcbl</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_cbl</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/SUL-RNA-dcbl-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/SUL-RNA-dcbl-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delcbl__1.bam</td>\n",
       "      <td>98.7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>misc__bw_delcbl__2</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delcbl</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_cbl</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/SUL-RNA-dcbl-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/SUL-RNA-dcbl-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delcbl__2.bam</td>\n",
       "      <td>98.6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                   study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                               \n",
       "misc__bw_delcbl__1  Misc       misc    bw_delcbl       1  Sangwoo Seo   \n",
       "misc__bw_delcbl__2  Misc       misc    bw_delcbl       2  Sangwoo Seo   \n",
       "\n",
       "                   library creator        run_date  \\\n",
       "sample_id                                            \n",
       "misc__bw_delcbl__1     Sangwoo Seo  7/29/2015 0:00   \n",
       "misc__bw_delcbl__2     Sangwoo Seo  7/29/2015 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "misc__bw_delcbl__1  Escherichia coli K-12 BW25113 del_cbl  BW25113         M9   \n",
       "misc__bw_delcbl__2  Escherichia coli K-12 BW25113 del_cbl  BW25113         M9   \n",
       "\n",
       "                    ... Sequencing Machine Additional Details  \\\n",
       "sample_id           ...                                         \n",
       "misc__bw_delcbl__1  ...              MiSeq                NaN   \n",
       "misc__bw_delcbl__2  ...              MiSeq                NaN   \n",
       "\n",
       "                   Biological Replicates  \\\n",
       "sample_id                                  \n",
       "misc__bw_delcbl__1                     2   \n",
       "misc__bw_delcbl__2                     2   \n",
       "\n",
       "                                                             R1  \\\n",
       "sample_id                                                         \n",
       "misc__bw_delcbl__1  misc/SUL-RNA-dcbl-1_S1_L001_R1_001.fastq.gz   \n",
       "misc__bw_delcbl__2  misc/SUL-RNA-dcbl-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                             R2  \\\n",
       "sample_id                                                         \n",
       "misc__bw_delcbl__1  misc/SUL-RNA-dcbl-1_S1_L001_R2_001.fastq.gz   \n",
       "misc__bw_delcbl__2  misc/SUL-RNA-dcbl-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                       BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "misc__bw_delcbl__1  misc__bw_delcbl__1.bam       98.7  NaN  GSE122211     1.0  \n",
       "misc__bw_delcbl__2  misc__bw_delcbl__2.bam       98.6  NaN  GSE122211     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>misc__bw_delfabr__1</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delfabr</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_fabR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/LIP-RNA-dfabR-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/LIP-RNA-dfabR-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delfabr__1.bam</td>\n",
       "      <td>98.50</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>misc__bw_delfabr__2</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delfabr</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_fabR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/LIP-RNA-dfabR-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/LIP-RNA-dfabR-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delfabr__2.bam</td>\n",
       "      <td>98.42</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                    study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                \n",
       "misc__bw_delfabr__1  Misc       misc   bw_delfabr       1  Sangwoo Seo   \n",
       "misc__bw_delfabr__2  Misc       misc   bw_delfabr       2  Sangwoo Seo   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "misc__bw_delfabr__1     Sangwoo Seo  7/29/2015 0:00   \n",
       "misc__bw_delfabr__2     Sangwoo Seo  7/29/2015 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "misc__bw_delfabr__1  Escherichia coli K-12 BW25113 del_fabR  BW25113   \n",
       "misc__bw_delfabr__2  Escherichia coli K-12 BW25113 del_fabR  BW25113   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "misc__bw_delfabr__1         M9  ...              MiSeq                NaN   \n",
       "misc__bw_delfabr__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "misc__bw_delfabr__1                     2   \n",
       "misc__bw_delfabr__2                     2   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "misc__bw_delfabr__1  misc/LIP-RNA-dfabR-1_S1_L001_R1_001.fastq.gz   \n",
       "misc__bw_delfabr__2  misc/LIP-RNA-dfabR-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "misc__bw_delfabr__1  misc/LIP-RNA-dfabR-1_S1_L001_R2_001.fastq.gz   \n",
       "misc__bw_delfabr__2  misc/LIP-RNA-dfabR-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                         BAM  alignment  DOI        GEO  \\\n",
       "sample_id                                                                 \n",
       "misc__bw_delfabr__1  misc__bw_delfabr__1.bam      98.50  NaN  GSE122211   \n",
       "misc__bw_delfabr__2  misc__bw_delfabr__2.bam      98.42  NaN  GSE122211   \n",
       "\n",
       "                     Public  \n",
       "sample_id                    \n",
       "misc__bw_delfabr__1     1.0  \n",
       "misc__bw_delfabr__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>misc__bw_delfadr__1</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delfadr</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_fadR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/LIP-RNA-dfadR-1_S3_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/LIP-RNA-dfadR-1_S3_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delfadr__1.bam</td>\n",
       "      <td>98.40</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>misc__bw_delfadr__2</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>bw_delfadr</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_fadR</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/LIP-RNA-dfadR-2_S4_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/LIP-RNA-dfadR-2_S4_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__bw_delfadr__2.bam</td>\n",
       "      <td>98.36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                    study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                \n",
       "misc__bw_delfadr__1  Misc       misc   bw_delfadr       1  Sangwoo Seo   \n",
       "misc__bw_delfadr__2  Misc       misc   bw_delfadr       2  Sangwoo Seo   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "misc__bw_delfadr__1     Sangwoo Seo  7/29/2015 0:00   \n",
       "misc__bw_delfadr__2     Sangwoo Seo  7/29/2015 0:00   \n",
       "\n",
       "                                         Strain Description Organism  \\\n",
       "sample_id                                                              \n",
       "misc__bw_delfadr__1  Escherichia coli K-12 BW25113 del_fadR  BW25113   \n",
       "misc__bw_delfadr__2  Escherichia coli K-12 BW25113 del_fadR  BW25113   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "misc__bw_delfadr__1         M9  ...              MiSeq                NaN   \n",
       "misc__bw_delfadr__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "misc__bw_delfadr__1                     2   \n",
       "misc__bw_delfadr__2                     2   \n",
       "\n",
       "                                                               R1  \\\n",
       "sample_id                                                           \n",
       "misc__bw_delfadr__1  misc/LIP-RNA-dfadR-1_S3_L001_R1_001.fastq.gz   \n",
       "misc__bw_delfadr__2  misc/LIP-RNA-dfadR-2_S4_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                               R2  \\\n",
       "sample_id                                                           \n",
       "misc__bw_delfadr__1  misc/LIP-RNA-dfadR-1_S3_L001_R2_001.fastq.gz   \n",
       "misc__bw_delfadr__2  misc/LIP-RNA-dfadR-2_S4_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                         BAM  alignment  DOI        GEO  \\\n",
       "sample_id                                                                 \n",
       "misc__bw_delfadr__1  misc__bw_delfadr__1.bam      98.40  NaN  GSE122211   \n",
       "misc__bw_delfadr__2  misc__bw_delfadr__2.bam      98.36  NaN  GSE122211   \n",
       "\n",
       "                     Public  \n",
       "sample_id                    \n",
       "misc__bw_delfadr__1     1.0  \n",
       "misc__bw_delfadr__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>misc__nitr_031__1</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>nitr_031</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>7/24/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>chemostat w/ dilution rate 0.31 h^-1</td>\n",
       "      <td>1</td>\n",
       "      <td>misc/0-3_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/0-3_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__nitr_031__1.bam</td>\n",
       "      <td>96.86</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE72020</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "misc__nitr_031__1  Misc       misc     nitr_031       1  Jose Utrilla   \n",
       "\n",
       "                  library creator        run_date  \\\n",
       "sample_id                                           \n",
       "misc__nitr_031__1    Jose Utrilla  7/24/2014 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "misc__nitr_031__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine                    Additional Details  \\\n",
       "sample_id                                                                    \n",
       "misc__nitr_031__1              MiSeq  chemostat w/ dilution rate 0.31 h^-1   \n",
       "\n",
       "                  Biological Replicates                                R1  \\\n",
       "sample_id                                                                   \n",
       "misc__nitr_031__1                     1  misc/0-3_S1_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                 R2                    BAM  \\\n",
       "sample_id                                                                    \n",
       "misc__nitr_031__1  misc/0-3_S1_L001_R2_001.fastq.gz  misc__nitr_031__1.bam   \n",
       "\n",
       "                   alignment  DOI       GEO  Public  \n",
       "sample_id                                            \n",
       "misc__nitr_031__1      96.86  NaN  GSE72020     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>misc__wt_no_te__1</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>wt_no_te</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>3/13/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/RNAseq-WT-NoTE-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/RNAseq-WT-NoTE-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__wt_no_te__1.bam</td>\n",
       "      <td>98.30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>misc__wt_no_te__2</td>\n",
       "      <td>Misc</td>\n",
       "      <td>misc</td>\n",
       "      <td>wt_no_te</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>3/13/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>misc/RNAseq-WT-NoTE-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>misc/RNAseq-WT-NoTE-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>misc__wt_no_te__2.bam</td>\n",
       "      <td>98.23</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                              \n",
       "misc__wt_no_te__1  Misc       misc     wt_no_te       1  Sangwoo Seo   \n",
       "misc__wt_no_te__2  Misc       misc     wt_no_te       2  Sangwoo Seo   \n",
       "\n",
       "                  library creator        run_date  \\\n",
       "sample_id                                           \n",
       "misc__wt_no_te__1     Sangwoo Seo  3/13/2015 0:00   \n",
       "misc__wt_no_te__2     Sangwoo Seo  3/13/2015 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "misc__wt_no_te__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "misc__wt_no_te__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "misc__wt_no_te__1              MiSeq                NaN                     2   \n",
       "misc__wt_no_te__2              MiSeq                NaN                     2   \n",
       "\n",
       "                                                              R1  \\\n",
       "sample_id                                                          \n",
       "misc__wt_no_te__1  misc/RNAseq-WT-NoTE-1_S1_L001_R1_001.fastq.gz   \n",
       "misc__wt_no_te__2  misc/RNAseq-WT-NoTE-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                              R2  \\\n",
       "sample_id                                                          \n",
       "misc__wt_no_te__1  misc/RNAseq-WT-NoTE-1_S1_L001_R2_001.fastq.gz   \n",
       "misc__wt_no_te__2  misc/RNAseq-WT-NoTE-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                     BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                    \n",
       "misc__wt_no_te__1  misc__wt_no_te__1.bam      98.30  NaN  GSE122211     1.0  \n",
       "misc__wt_no_te__2  misc__wt_no_te__2.bam      98.23  NaN  GSE122211     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_csn__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_csn</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytosine-1_S1_L001_R1...</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytosine-1_S1_L001_R2...</td>\n",
       "      <td>nac_ntrc__bw_delnac_csn__1.bam</td>\n",
       "      <td>97.37</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_csn__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_csn</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytosine-2_S2_L001_R1...</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytosine-2_S2_L001_R2...</td>\n",
       "      <td>nac_ntrc__bw_delnac_csn__2.bam</td>\n",
       "      <td>98.11</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__bw_delnac_csn__1  Nac/NtrC   nac_ntrc  bw_delnac_csn       1   \n",
       "nac_ntrc__bw_delnac_csn__2  Nac/NtrC   nac_ntrc  bw_delnac_csn       2   \n",
       "\n",
       "                                 contact library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "nac_ntrc__bw_delnac_csn__1  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "nac_ntrc__bw_delnac_csn__2  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "\n",
       "                                               Strain Description Organism  \\\n",
       "sample_id                                                                    \n",
       "nac_ntrc__bw_delnac_csn__1  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "nac_ntrc__bw_delnac_csn__2  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "\n",
       "                           Base Media  ... Sequencing Machine  \\\n",
       "sample_id                              ...                      \n",
       "nac_ntrc__bw_delnac_csn__1         W2  ...              MiSeq   \n",
       "nac_ntrc__bw_delnac_csn__2         W2  ...              MiSeq   \n",
       "\n",
       "                           Additional Details Biological Replicates  \\\n",
       "sample_id                                                             \n",
       "nac_ntrc__bw_delnac_csn__1                NaN                     2   \n",
       "nac_ntrc__bw_delnac_csn__2                NaN                     2   \n",
       "\n",
       "                                                                           R1  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_csn__1  nac_ntrc/NTR-RNA-del-nac-cytosine-1_S1_L001_R1...   \n",
       "nac_ntrc__bw_delnac_csn__2  nac_ntrc/NTR-RNA-del-nac-cytosine-2_S2_L001_R1...   \n",
       "\n",
       "                                                                           R2  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_csn__1  nac_ntrc/NTR-RNA-del-nac-cytosine-1_S1_L001_R2...   \n",
       "nac_ntrc__bw_delnac_csn__2  nac_ntrc/NTR-RNA-del-nac-cytosine-2_S2_L001_R2...   \n",
       "\n",
       "                                                       BAM  alignment  \\\n",
       "sample_id                                                               \n",
       "nac_ntrc__bw_delnac_csn__1  nac_ntrc__bw_delnac_csn__1.bam      97.37   \n",
       "nac_ntrc__bw_delnac_csn__2  nac_ntrc__bw_delnac_csn__2.bam      98.11   \n",
       "\n",
       "                                                                          DOI  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_csn__1  From dissertation: https://escholarship.org/uc...   \n",
       "nac_ntrc__bw_delnac_csn__2  From dissertation: https://escholarship.org/uc...   \n",
       "\n",
       "                            GEO  Public  \n",
       "sample_id                                \n",
       "nac_ntrc__bw_delnac_csn__1  NaN     1.0  \n",
       "nac_ntrc__bw_delnac_csn__2  NaN     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_cytd__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_cytd</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytidine-1_S1_L001_R1...</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytidine-1_S1_L001_R2...</td>\n",
       "      <td>nac_ntrc__bw_delnac_cytd__1.bam</td>\n",
       "      <td>98.00</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_cytd__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_cytd</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytidine-2_S2_L001_R1...</td>\n",
       "      <td>nac_ntrc/NTR-RNA-del-nac-cytidine-2_S2_L001_R2...</td>\n",
       "      <td>nac_ntrc__bw_delnac_cytd__2.bam</td>\n",
       "      <td>97.74</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id    condition_id  rep_id  \\\n",
       "sample_id                                                                  \n",
       "nac_ntrc__bw_delnac_cytd__1  Nac/NtrC   nac_ntrc  bw_delnac_cytd       1   \n",
       "nac_ntrc__bw_delnac_cytd__2  Nac/NtrC   nac_ntrc  bw_delnac_cytd       2   \n",
       "\n",
       "                                  contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "nac_ntrc__bw_delnac_cytd__1  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "nac_ntrc__bw_delnac_cytd__2  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "\n",
       "                                                Strain Description Organism  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__bw_delnac_cytd__1  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "nac_ntrc__bw_delnac_cytd__2  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "\n",
       "                            Base Media  ... Sequencing Machine  \\\n",
       "sample_id                               ...                      \n",
       "nac_ntrc__bw_delnac_cytd__1         W2  ...              MiSeq   \n",
       "nac_ntrc__bw_delnac_cytd__2         W2  ...              MiSeq   \n",
       "\n",
       "                            Additional Details Biological Replicates  \\\n",
       "sample_id                                                              \n",
       "nac_ntrc__bw_delnac_cytd__1                NaN                     2   \n",
       "nac_ntrc__bw_delnac_cytd__2                NaN                     2   \n",
       "\n",
       "                                                                            R1  \\\n",
       "sample_id                                                                        \n",
       "nac_ntrc__bw_delnac_cytd__1  nac_ntrc/NTR-RNA-del-nac-cytidine-1_S1_L001_R1...   \n",
       "nac_ntrc__bw_delnac_cytd__2  nac_ntrc/NTR-RNA-del-nac-cytidine-2_S2_L001_R1...   \n",
       "\n",
       "                                                                            R2  \\\n",
       "sample_id                                                                        \n",
       "nac_ntrc__bw_delnac_cytd__1  nac_ntrc/NTR-RNA-del-nac-cytidine-1_S1_L001_R2...   \n",
       "nac_ntrc__bw_delnac_cytd__2  nac_ntrc/NTR-RNA-del-nac-cytidine-2_S2_L001_R2...   \n",
       "\n",
       "                                                         BAM  alignment  \\\n",
       "sample_id                                                                 \n",
       "nac_ntrc__bw_delnac_cytd__1  nac_ntrc__bw_delnac_cytd__1.bam      98.00   \n",
       "nac_ntrc__bw_delnac_cytd__2  nac_ntrc__bw_delnac_cytd__2.bam      97.74   \n",
       "\n",
       "                                                                           DOI  \\\n",
       "sample_id                                                                        \n",
       "nac_ntrc__bw_delnac_cytd__1  From dissertation: https://escholarship.org/uc...   \n",
       "nac_ntrc__bw_delnac_cytd__2  From dissertation: https://escholarship.org/uc...   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "nac_ntrc__bw_delnac_cytd__1  NaN     1.0  \n",
       "nac_ntrc__bw_delnac_cytd__2  NaN     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_gln__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_gln</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/del-nac-glutamine-1_S5_L001_R1_001.fastq</td>\n",
       "      <td>nac_ntrc/del-nac-glutamine-1_S5_L001_R2_001.fastq</td>\n",
       "      <td>nac_ntrc__bw_delnac_gln__1.bam</td>\n",
       "      <td>97.28</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_gln__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_gln</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/del-nac-glutamine-2_S6_L001_R1_001.fastq</td>\n",
       "      <td>nac_ntrc/del-nac-glutamine-2_S6_L001_R2_001.fastq</td>\n",
       "      <td>nac_ntrc__bw_delnac_gln__2.bam</td>\n",
       "      <td>97.17</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__bw_delnac_gln__1  Nac/NtrC   nac_ntrc  bw_delnac_gln       1   \n",
       "nac_ntrc__bw_delnac_gln__2  Nac/NtrC   nac_ntrc  bw_delnac_gln       2   \n",
       "\n",
       "                                 contact library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "nac_ntrc__bw_delnac_gln__1  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "nac_ntrc__bw_delnac_gln__2  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "\n",
       "                                               Strain Description Organism  \\\n",
       "sample_id                                                                    \n",
       "nac_ntrc__bw_delnac_gln__1  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "nac_ntrc__bw_delnac_gln__2  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "\n",
       "                           Base Media  ... Sequencing Machine  \\\n",
       "sample_id                              ...                      \n",
       "nac_ntrc__bw_delnac_gln__1         W2  ...              MiSeq   \n",
       "nac_ntrc__bw_delnac_gln__2         W2  ...              MiSeq   \n",
       "\n",
       "                           Additional Details Biological Replicates  \\\n",
       "sample_id                                                             \n",
       "nac_ntrc__bw_delnac_gln__1                NaN                     2   \n",
       "nac_ntrc__bw_delnac_gln__2                NaN                     2   \n",
       "\n",
       "                                                                           R1  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_gln__1  nac_ntrc/del-nac-glutamine-1_S5_L001_R1_001.fastq   \n",
       "nac_ntrc__bw_delnac_gln__2  nac_ntrc/del-nac-glutamine-2_S6_L001_R1_001.fastq   \n",
       "\n",
       "                                                                           R2  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_gln__1  nac_ntrc/del-nac-glutamine-1_S5_L001_R2_001.fastq   \n",
       "nac_ntrc__bw_delnac_gln__2  nac_ntrc/del-nac-glutamine-2_S6_L001_R2_001.fastq   \n",
       "\n",
       "                                                       BAM  alignment  \\\n",
       "sample_id                                                               \n",
       "nac_ntrc__bw_delnac_gln__1  nac_ntrc__bw_delnac_gln__1.bam      97.28   \n",
       "nac_ntrc__bw_delnac_gln__2  nac_ntrc__bw_delnac_gln__2.bam      97.17   \n",
       "\n",
       "                                                                          DOI  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_gln__1  From dissertation: https://escholarship.org/uc...   \n",
       "nac_ntrc__bw_delnac_gln__2  From dissertation: https://escholarship.org/uc...   \n",
       "\n",
       "                            GEO  Public  \n",
       "sample_id                                \n",
       "nac_ntrc__bw_delnac_gln__1  NaN     1.0  \n",
       "nac_ntrc__bw_delnac_gln__2  NaN     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_nh4__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_nh4</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/del-nac-ammonia-1_S3_L001_R1_001.fastq</td>\n",
       "      <td>nac_ntrc/del-nac-ammonia-1_S3_L001_R2_001.fastq</td>\n",
       "      <td>nac_ntrc__bw_delnac_nh4__1.bam</td>\n",
       "      <td>98.03</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__bw_delnac_nh4__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>bw_delnac_nh4</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>2/16/2016 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_nac</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/del-nac-ammonia-2_S4_L001_R1_001.fastq</td>\n",
       "      <td>nac_ntrc/del-nac-ammonia-2_S4_L001_R2_001.fastq</td>\n",
       "      <td>nac_ntrc__bw_delnac_nh4__2.bam</td>\n",
       "      <td>97.93</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__bw_delnac_nh4__1  Nac/NtrC   nac_ntrc  bw_delnac_nh4       1   \n",
       "nac_ntrc__bw_delnac_nh4__2  Nac/NtrC   nac_ntrc  bw_delnac_nh4       2   \n",
       "\n",
       "                                 contact library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "nac_ntrc__bw_delnac_nh4__1  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "nac_ntrc__bw_delnac_nh4__2  Donghyuk Kim    Donghyuk Kim  2/16/2016 0:00   \n",
       "\n",
       "                                               Strain Description Organism  \\\n",
       "sample_id                                                                    \n",
       "nac_ntrc__bw_delnac_nh4__1  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "nac_ntrc__bw_delnac_nh4__2  Escherichia coli K-12 BW25113 del_nac  BW25113   \n",
       "\n",
       "                           Base Media  ... Sequencing Machine  \\\n",
       "sample_id                              ...                      \n",
       "nac_ntrc__bw_delnac_nh4__1         M9  ...              MiSeq   \n",
       "nac_ntrc__bw_delnac_nh4__2         M9  ...              MiSeq   \n",
       "\n",
       "                           Additional Details Biological Replicates  \\\n",
       "sample_id                                                             \n",
       "nac_ntrc__bw_delnac_nh4__1                NaN                     2   \n",
       "nac_ntrc__bw_delnac_nh4__2                NaN                     2   \n",
       "\n",
       "                                                                         R1  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__bw_delnac_nh4__1  nac_ntrc/del-nac-ammonia-1_S3_L001_R1_001.fastq   \n",
       "nac_ntrc__bw_delnac_nh4__2  nac_ntrc/del-nac-ammonia-2_S4_L001_R1_001.fastq   \n",
       "\n",
       "                                                                         R2  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__bw_delnac_nh4__1  nac_ntrc/del-nac-ammonia-1_S3_L001_R2_001.fastq   \n",
       "nac_ntrc__bw_delnac_nh4__2  nac_ntrc/del-nac-ammonia-2_S4_L001_R2_001.fastq   \n",
       "\n",
       "                                                       BAM  alignment  \\\n",
       "sample_id                                                               \n",
       "nac_ntrc__bw_delnac_nh4__1  nac_ntrc__bw_delnac_nh4__1.bam      98.03   \n",
       "nac_ntrc__bw_delnac_nh4__2  nac_ntrc__bw_delnac_nh4__2.bam      97.93   \n",
       "\n",
       "                                                                          DOI  \\\n",
       "sample_id                                                                       \n",
       "nac_ntrc__bw_delnac_nh4__1  From dissertation: https://escholarship.org/uc...   \n",
       "nac_ntrc__bw_delnac_nh4__2  From dissertation: https://escholarship.org/uc...   \n",
       "\n",
       "                            GEO  Public  \n",
       "sample_id                                \n",
       "nac_ntrc__bw_delnac_nh4__1  NaN     1.0  \n",
       "nac_ntrc__bw_delnac_nh4__2  NaN     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_csn__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_csn</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytosine-1_S1_L001_R1_001.f...</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytosine-1_S1_L001_R2_001.f...</td>\n",
       "      <td>nac_ntrc__wt_csn__1.bam</td>\n",
       "      <td>93.29</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_csn__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_csn</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytosine-2_S2_L001_R1_001.f...</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytosine-2_S2_L001_R2_001.f...</td>\n",
       "      <td>nac_ntrc__wt_csn__2.bam</td>\n",
       "      <td>96.30</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__wt_csn__1  Nac/NtrC   nac_ntrc       wt_csn       1  Donghyuk Kim   \n",
       "nac_ntrc__wt_csn__2  Nac/NtrC   nac_ntrc       wt_csn       2  Donghyuk Kim   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "nac_ntrc__wt_csn__1    Donghyuk Kim  4/18/2013 0:00   \n",
       "nac_ntrc__wt_csn__2    Donghyuk Kim  4/18/2013 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "nac_ntrc__wt_csn__1  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "nac_ntrc__wt_csn__2  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "nac_ntrc__wt_csn__1              MiSeq                NaN   \n",
       "nac_ntrc__wt_csn__2              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "nac_ntrc__wt_csn__1                     2   \n",
       "nac_ntrc__wt_csn__2                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__wt_csn__1  nac_ntrc/MG1655-W2-cytosine-1_S1_L001_R1_001.f...   \n",
       "nac_ntrc__wt_csn__2  nac_ntrc/MG1655-W2-cytosine-2_S2_L001_R1_001.f...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__wt_csn__1  nac_ntrc/MG1655-W2-cytosine-1_S1_L001_R2_001.f...   \n",
       "nac_ntrc__wt_csn__2  nac_ntrc/MG1655-W2-cytosine-2_S2_L001_R2_001.f...   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "nac_ntrc__wt_csn__1  nac_ntrc__wt_csn__1.bam      93.29   \n",
       "nac_ntrc__wt_csn__2  nac_ntrc__wt_csn__2.bam      96.30   \n",
       "\n",
       "                                                                   DOI  GEO  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__wt_csn__1  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "nac_ntrc__wt_csn__2  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "\n",
       "                     Public  \n",
       "sample_id                    \n",
       "nac_ntrc__wt_csn__1     1.0  \n",
       "nac_ntrc__wt_csn__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_cytd__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_cytd</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytidine-1_S1_L001_R1_001.f...</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytidine-1_S1_L001_R2_001.f...</td>\n",
       "      <td>nac_ntrc__wt_cytd__1.bam</td>\n",
       "      <td>93.16</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_cytd__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_cytd</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytidine-2_S2_L001_R1_001.f...</td>\n",
       "      <td>nac_ntrc/MG1655-W2-cytidine-2_S2_L001_R2_001.f...</td>\n",
       "      <td>nac_ntrc__wt_cytd__2.bam</td>\n",
       "      <td>92.52</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "nac_ntrc__wt_cytd__1  Nac/NtrC   nac_ntrc      wt_cytd       1  Donghyuk Kim   \n",
       "nac_ntrc__wt_cytd__2  Nac/NtrC   nac_ntrc      wt_cytd       2  Donghyuk Kim   \n",
       "\n",
       "                     library creator        run_date  \\\n",
       "sample_id                                              \n",
       "nac_ntrc__wt_cytd__1    Donghyuk Kim  4/17/2013 0:00   \n",
       "nac_ntrc__wt_cytd__2    Donghyuk Kim  4/17/2013 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "nac_ntrc__wt_cytd__1  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "nac_ntrc__wt_cytd__2  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "\n",
       "                     Sequencing Machine Additional Details  \\\n",
       "sample_id                                                    \n",
       "nac_ntrc__wt_cytd__1              MiSeq                NaN   \n",
       "nac_ntrc__wt_cytd__2              MiSeq                NaN   \n",
       "\n",
       "                     Biological Replicates  \\\n",
       "sample_id                                    \n",
       "nac_ntrc__wt_cytd__1                     2   \n",
       "nac_ntrc__wt_cytd__2                     2   \n",
       "\n",
       "                                                                     R1  \\\n",
       "sample_id                                                                 \n",
       "nac_ntrc__wt_cytd__1  nac_ntrc/MG1655-W2-cytidine-1_S1_L001_R1_001.f...   \n",
       "nac_ntrc__wt_cytd__2  nac_ntrc/MG1655-W2-cytidine-2_S2_L001_R1_001.f...   \n",
       "\n",
       "                                                                     R2  \\\n",
       "sample_id                                                                 \n",
       "nac_ntrc__wt_cytd__1  nac_ntrc/MG1655-W2-cytidine-1_S1_L001_R2_001.f...   \n",
       "nac_ntrc__wt_cytd__2  nac_ntrc/MG1655-W2-cytidine-2_S2_L001_R2_001.f...   \n",
       "\n",
       "                                           BAM  alignment  \\\n",
       "sample_id                                                   \n",
       "nac_ntrc__wt_cytd__1  nac_ntrc__wt_cytd__1.bam      93.16   \n",
       "nac_ntrc__wt_cytd__2  nac_ntrc__wt_cytd__2.bam      92.52   \n",
       "\n",
       "                                                                    DOI  GEO  \\\n",
       "sample_id                                                                      \n",
       "nac_ntrc__wt_cytd__1  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "nac_ntrc__wt_cytd__2  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "\n",
       "                      Public  \n",
       "sample_id                     \n",
       "nac_ntrc__wt_cytd__1     1.0  \n",
       "nac_ntrc__wt_cytd__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_gln__1</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_gln</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-glutamine-1_S1_L001_R1_001....</td>\n",
       "      <td>nac_ntrc/MG1655-W2-glutamine-1_S1_L001_R2_001....</td>\n",
       "      <td>nac_ntrc__wt_gln__1.bam</td>\n",
       "      <td>95.86</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>nac_ntrc__wt_gln__2</td>\n",
       "      <td>Nac/NtrC</td>\n",
       "      <td>nac_ntrc</td>\n",
       "      <td>wt_gln</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>4/17/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>W2</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>nac_ntrc/MG1655-W2-glutamine-2_S2_L001_R1_001....</td>\n",
       "      <td>nac_ntrc/MG1655-W2-glutamine-2_S2_L001_R2_001....</td>\n",
       "      <td>nac_ntrc__wt_gln__2.bam</td>\n",
       "      <td>94.27</td>\n",
       "      <td>From dissertation: https://escholarship.org/uc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__wt_gln__1  Nac/NtrC   nac_ntrc       wt_gln       1  Donghyuk Kim   \n",
       "nac_ntrc__wt_gln__2  Nac/NtrC   nac_ntrc       wt_gln       2  Donghyuk Kim   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "nac_ntrc__wt_gln__1    Donghyuk Kim  4/17/2013 0:00   \n",
       "nac_ntrc__wt_gln__2    Donghyuk Kim  4/17/2013 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "nac_ntrc__wt_gln__1  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "nac_ntrc__wt_gln__2  Escherichia coli K-12 MG1655   MG1655         W2  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "nac_ntrc__wt_gln__1              MiSeq                NaN   \n",
       "nac_ntrc__wt_gln__2              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "nac_ntrc__wt_gln__1                     2   \n",
       "nac_ntrc__wt_gln__2                     2   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__wt_gln__1  nac_ntrc/MG1655-W2-glutamine-1_S1_L001_R1_001....   \n",
       "nac_ntrc__wt_gln__2  nac_ntrc/MG1655-W2-glutamine-2_S2_L001_R1_001....   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "nac_ntrc__wt_gln__1  nac_ntrc/MG1655-W2-glutamine-1_S1_L001_R2_001....   \n",
       "nac_ntrc__wt_gln__2  nac_ntrc/MG1655-W2-glutamine-2_S2_L001_R2_001....   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "nac_ntrc__wt_gln__1  nac_ntrc__wt_gln__1.bam      95.86   \n",
       "nac_ntrc__wt_gln__2  nac_ntrc__wt_gln__2.bam      94.27   \n",
       "\n",
       "                                                                   DOI  GEO  \\\n",
       "sample_id                                                                     \n",
       "nac_ntrc__wt_gln__1  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "nac_ntrc__wt_gln__2  From dissertation: https://escholarship.org/uc...  NaN   \n",
       "\n",
       "                     Public  \n",
       "sample_id                    \n",
       "nac_ntrc__wt_gln__1     1.0  \n",
       "nac_ntrc__wt_gln__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__bw_ac__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_ac</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-ac-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-ac-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_ac__1.bam</td>\n",
       "      <td>97.80</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__bw_ac__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_ac</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-ac-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-ac-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_ac__2.bam</td>\n",
       "      <td>98.04</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                              \n",
       "omics__bw_ac__1  Omics      omics        bw_ac       1  Donghyuk Kim   \n",
       "omics__bw_ac__2  Omics      omics        bw_ac       2  Donghyuk Kim   \n",
       "\n",
       "                library creator        run_date        Strain Description  \\\n",
       "sample_id                                                                   \n",
       "omics__bw_ac__1    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "omics__bw_ac__2    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "omics__bw_ac__1  BW25113         M9  ...              MiSeq   \n",
       "omics__bw_ac__2  BW25113         M9  ...              MiSeq   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "omics__bw_ac__1                NaN                     2   \n",
       "omics__bw_ac__2                NaN                     2   \n",
       "\n",
       "                                                         R1  \\\n",
       "sample_id                                                     \n",
       "omics__bw_ac__1  omics/bw25113-ac-1_S1_L001_R1_001.fastq.gz   \n",
       "omics__bw_ac__2  omics/bw25113-ac-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                         R2  \\\n",
       "sample_id                                                     \n",
       "omics__bw_ac__1  omics/bw25113-ac-1_S1_L001_R2_001.fastq.gz   \n",
       "omics__bw_ac__2  omics/bw25113-ac-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                          DOI  \\\n",
       "sample_id                                                                      \n",
       "omics__bw_ac__1  omics__bw_ac__1.bam      97.80  doi.org/10.1038/ncomms13091   \n",
       "omics__bw_ac__2  omics__bw_ac__2.bam      98.04  doi.org/10.1038/ncomms13091   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "omics__bw_ac__1  GSE59759     1.0  \n",
       "omics__bw_ac__2  GSE59759     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__bw_fum__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_fum</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-fum1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-fum1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_fum__1.bam</td>\n",
       "      <td>97.94</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__bw_fum__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_fum</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-fum2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-fum2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_fum__2.bam</td>\n",
       "      <td>98.32</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__bw_fum__1  Omics      omics       bw_fum       1  Donghyuk Kim   \n",
       "omics__bw_fum__2  Omics      omics       bw_fum       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator        run_date        Strain Description  \\\n",
       "sample_id                                                                    \n",
       "omics__bw_fum__1    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "omics__bw_fum__2    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__bw_fum__1  BW25113         M9  ...              MiSeq   \n",
       "omics__bw_fum__2  BW25113         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__bw_fum__1                NaN                     2   \n",
       "omics__bw_fum__2                NaN                     2   \n",
       "\n",
       "                                                          R1  \\\n",
       "sample_id                                                      \n",
       "omics__bw_fum__1  omics/bw25113-fum1_S1_L001_R1_001.fastq.gz   \n",
       "omics__bw_fum__2  omics/bw25113-fum2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                          R2  \\\n",
       "sample_id                                                      \n",
       "omics__bw_fum__1  omics/bw25113-fum1_S1_L001_R2_001.fastq.gz   \n",
       "omics__bw_fum__2  omics/bw25113-fum2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "omics__bw_fum__1  omics__bw_fum__1.bam      97.94   \n",
       "omics__bw_fum__2  omics__bw_fum__2.bam      98.32   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "omics__bw_fum__1  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "omics__bw_fum__2  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__bw_glc__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>3/12/2013 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-glc1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-glc1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_glc__1.bam</td>\n",
       "      <td>97.62</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__bw_glc__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>3/12/2013 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-glc2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-glc2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_glc__2.bam</td>\n",
       "      <td>98.13</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__bw_glc__1  Omics      omics       bw_glc       1  Donghyuk Kim   \n",
       "omics__bw_glc__2  Omics      omics       bw_glc       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator        run_date        Strain Description  \\\n",
       "sample_id                                                                    \n",
       "omics__bw_glc__1    Donghyuk Kim  3/12/2013 0:00  Escherichia coli BW25113   \n",
       "omics__bw_glc__2    Donghyuk Kim  3/12/2013 0:00  Escherichia coli BW25113   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__bw_glc__1  BW25113         M9  ...              MiSeq   \n",
       "omics__bw_glc__2  BW25113         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__bw_glc__1                NaN                     2   \n",
       "omics__bw_glc__2                NaN                     2   \n",
       "\n",
       "                                                          R1  \\\n",
       "sample_id                                                      \n",
       "omics__bw_glc__1  omics/bw25113-glc1_S1_L001_R1_001.fastq.gz   \n",
       "omics__bw_glc__2  omics/bw25113-glc2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                          R2  \\\n",
       "sample_id                                                      \n",
       "omics__bw_glc__1  omics/bw25113-glc1_S1_L001_R2_001.fastq.gz   \n",
       "omics__bw_glc__2  omics/bw25113-glc2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "omics__bw_glc__1  omics__bw_glc__1.bam      97.62   \n",
       "omics__bw_glc__2  omics__bw_glc__2.bam      98.13   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "omics__bw_glc__1  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "omics__bw_glc__2  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__bw_pyr__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_pyr</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-pyr-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-pyr-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_pyr__1.bam</td>\n",
       "      <td>97.36</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__bw_pyr__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>bw_pyr</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>1/12/2015 0:00</td>\n",
       "      <td>Escherichia coli BW25113</td>\n",
       "      <td>BW25113</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/bw25113-pyr-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>omics/bw25113-pyr-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>omics__bw_pyr__2.bam</td>\n",
       "      <td>97.59</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59759</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__bw_pyr__1  Omics      omics       bw_pyr       1  Donghyuk Kim   \n",
       "omics__bw_pyr__2  Omics      omics       bw_pyr       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator        run_date        Strain Description  \\\n",
       "sample_id                                                                    \n",
       "omics__bw_pyr__1    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "omics__bw_pyr__2    Donghyuk Kim  1/12/2015 0:00  Escherichia coli BW25113   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__bw_pyr__1  BW25113         M9  ...              MiSeq   \n",
       "omics__bw_pyr__2  BW25113         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__bw_pyr__1                NaN                     2   \n",
       "omics__bw_pyr__2                NaN                     2   \n",
       "\n",
       "                                                           R1  \\\n",
       "sample_id                                                       \n",
       "omics__bw_pyr__1  omics/bw25113-pyr-1_S1_L001_R1_001.fastq.gz   \n",
       "omics__bw_pyr__2  omics/bw25113-pyr-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                           R2  \\\n",
       "sample_id                                                       \n",
       "omics__bw_pyr__1  omics/bw25113-pyr-1_S1_L001_R2_001.fastq.gz   \n",
       "omics__bw_pyr__2  omics/bw25113-pyr-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "omics__bw_pyr__1  omics__bw_pyr__1.bam      97.36   \n",
       "omics__bw_pyr__2  omics__bw_pyr__2.bam      97.59   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "omics__bw_pyr__1  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "omics__bw_pyr__2  doi.org/10.1038/ncomms13091  GSE59759     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__wt_glu__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_glu</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/2/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-glutamate-1_S1_L001_R1_0...</td>\n",
       "      <td>omics/supplementation-glutamate-1_S1_L001_R2_0...</td>\n",
       "      <td>omics__wt_glu__1.bam</td>\n",
       "      <td>96.92</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__wt_glu__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_glu</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/2/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-glutamate-2_S2_L001_R1_0...</td>\n",
       "      <td>omics/supplementation-glutamate-2_S2_L001_R2_0...</td>\n",
       "      <td>omics__wt_glu__2.bam</td>\n",
       "      <td>97.88</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GSE122211</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__wt_glu__1  Omics      omics       wt_glu       1  Donghyuk Kim   \n",
       "omics__wt_glu__2  Omics      omics       wt_glu       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "omics__wt_glu__1    Donghyuk Kim  7/2/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "omics__wt_glu__2    Donghyuk Kim  7/2/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__wt_glu__1   MG1655         M9  ...              MiSeq   \n",
       "omics__wt_glu__2   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__wt_glu__1                NaN                     2   \n",
       "omics__wt_glu__2                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "omics__wt_glu__1  omics/supplementation-glutamate-1_S1_L001_R1_0...   \n",
       "omics__wt_glu__2  omics/supplementation-glutamate-2_S2_L001_R1_0...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "omics__wt_glu__1  omics/supplementation-glutamate-1_S1_L001_R2_0...   \n",
       "omics__wt_glu__2  omics/supplementation-glutamate-2_S2_L001_R2_0...   \n",
       "\n",
       "                                   BAM  alignment  DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "omics__wt_glu__1  omics__wt_glu__1.bam      96.92  NaN  GSE122211     1.0  \n",
       "omics__wt_glu__2  omics__wt_glu__2.bam      97.88  NaN  GSE122211     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__wt_gly__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_gly</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/3/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-glycine-1_S1_L001_R1_001...</td>\n",
       "      <td>omics/supplementation-glycine-1_S1_L001_R2_001...</td>\n",
       "      <td>omics__wt_gly__1.bam</td>\n",
       "      <td>91.98</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59760</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__wt_gly__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_gly</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/3/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-glycine-2_S2_L001_R1_001...</td>\n",
       "      <td>omics/supplementation-glycine-2_S2_L001_R2_001...</td>\n",
       "      <td>omics__wt_gly__2.bam</td>\n",
       "      <td>82.97</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59760</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__wt_gly__1  Omics      omics       wt_gly       1  Donghyuk Kim   \n",
       "omics__wt_gly__2  Omics      omics       wt_gly       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "omics__wt_gly__1    Donghyuk Kim  7/3/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "omics__wt_gly__2    Donghyuk Kim  7/3/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__wt_gly__1   MG1655         M9  ...              MiSeq   \n",
       "omics__wt_gly__2   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__wt_gly__1                NaN                     2   \n",
       "omics__wt_gly__2                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "omics__wt_gly__1  omics/supplementation-glycine-1_S1_L001_R1_001...   \n",
       "omics__wt_gly__2  omics/supplementation-glycine-2_S2_L001_R1_001...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "omics__wt_gly__1  omics/supplementation-glycine-1_S1_L001_R2_001...   \n",
       "omics__wt_gly__2  omics/supplementation-glycine-2_S2_L001_R2_001...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "omics__wt_gly__1  omics__wt_gly__1.bam      91.98   \n",
       "omics__wt_gly__2  omics__wt_gly__2.bam      82.97   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "omics__wt_gly__1  doi.org/10.1038/ncomms13091  GSE59760     1.0  \n",
       "omics__wt_gly__2  doi.org/10.1038/ncomms13091  GSE59760     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>omics__wt_thr__1</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_thr</td>\n",
       "      <td>1</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/7/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-threonine-1_S1_L001_R1_0...</td>\n",
       "      <td>omics/supplementation-threonine-1_S1_L001_R2_0...</td>\n",
       "      <td>omics__wt_thr__1.bam</td>\n",
       "      <td>97.65</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59760</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>omics__wt_thr__2</td>\n",
       "      <td>Omics</td>\n",
       "      <td>omics</td>\n",
       "      <td>wt_thr</td>\n",
       "      <td>2</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>Donghyuk Kim</td>\n",
       "      <td>7/7/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>omics/supplementation-threonine-2_S2_L001_R1_0...</td>\n",
       "      <td>omics/supplementation-threonine-2_S2_L001_R2_0...</td>\n",
       "      <td>omics__wt_thr__2.bam</td>\n",
       "      <td>92.68</td>\n",
       "      <td>doi.org/10.1038/ncomms13091</td>\n",
       "      <td>GSE59760</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                               \n",
       "omics__wt_thr__1  Omics      omics       wt_thr       1  Donghyuk Kim   \n",
       "omics__wt_thr__2  Omics      omics       wt_thr       2  Donghyuk Kim   \n",
       "\n",
       "                 library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "omics__wt_thr__1    Donghyuk Kim  7/7/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "omics__wt_thr__2    Donghyuk Kim  7/7/2014 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                 Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                             ...                      \n",
       "omics__wt_thr__1   MG1655         M9  ...              MiSeq   \n",
       "omics__wt_thr__2   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                 Additional Details Biological Replicates  \\\n",
       "sample_id                                                   \n",
       "omics__wt_thr__1                NaN                     2   \n",
       "omics__wt_thr__2                NaN                     2   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "omics__wt_thr__1  omics/supplementation-threonine-1_S1_L001_R1_0...   \n",
       "omics__wt_thr__2  omics/supplementation-threonine-2_S2_L001_R1_0...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "omics__wt_thr__1  omics/supplementation-threonine-1_S1_L001_R2_0...   \n",
       "omics__wt_thr__2  omics/supplementation-threonine-2_S2_L001_R2_0...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "omics__wt_thr__1  omics__wt_thr__1.bam      97.65   \n",
       "omics__wt_thr__2  omics__wt_thr__2.bam      92.68   \n",
       "\n",
       "                                          DOI       GEO  Public  \n",
       "sample_id                                                        \n",
       "omics__wt_thr__1  doi.org/10.1038/ncomms13091  GSE59760     1.0  \n",
       "omics__wt_thr__2  doi.org/10.1038/ncomms13091  GSE59760     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ompr__bw_delompr_nacl__1</td>\n",
       "      <td>OmpR</td>\n",
       "      <td>ompr</td>\n",
       "      <td>bw_delompr_nacl</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_ompR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ompr/dompRNaCl1_S3_L001_R1_001.fastq</td>\n",
       "      <td>ompr/dompRNaCl1_S3_L001_R2_001.fastq</td>\n",
       "      <td>ompr__bw_delompr_nacl__1.bam</td>\n",
       "      <td>98.56</td>\n",
       "      <td>doi.org/10.1038/s41598-017-02110-7</td>\n",
       "      <td>GSE88980</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ompr__bw_delompr_nacl__2</td>\n",
       "      <td>OmpR</td>\n",
       "      <td>ompr</td>\n",
       "      <td>bw_delompr_nacl</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 BW25113 del_ompR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ompr/dompRNaCl2_S4_L001_R1_001.fastq</td>\n",
       "      <td>ompr/dompRNaCl2_S4_L001_R2_001.fastq</td>\n",
       "      <td>ompr__bw_delompr_nacl__2.bam</td>\n",
       "      <td>98.60</td>\n",
       "      <td>doi.org/10.1038/s41598-017-02110-7</td>\n",
       "      <td>GSE88980</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id     condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "ompr__bw_delompr_nacl__1  OmpR       ompr  bw_delompr_nacl       1   \n",
       "ompr__bw_delompr_nacl__2  OmpR       ompr  bw_delompr_nacl       2   \n",
       "\n",
       "                              contact library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "ompr__bw_delompr_nacl__1  Sangwoo Seo     Sangwoo Seo  7/29/2015 0:00   \n",
       "ompr__bw_delompr_nacl__2  Sangwoo Seo     Sangwoo Seo  7/29/2015 0:00   \n",
       "\n",
       "                                              Strain Description Organism  \\\n",
       "sample_id                                                                   \n",
       "ompr__bw_delompr_nacl__1  Escherichia coli K-12 BW25113 del_ompR   MG1655   \n",
       "ompr__bw_delompr_nacl__2  Escherichia coli K-12 BW25113 del_ompR   MG1655   \n",
       "\n",
       "                         Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ompr__bw_delompr_nacl__1         M9  ...              MiSeq   \n",
       "ompr__bw_delompr_nacl__2         M9  ...              MiSeq   \n",
       "\n",
       "                         Additional Details Biological Replicates  \\\n",
       "sample_id                                                           \n",
       "ompr__bw_delompr_nacl__1                NaN                     2   \n",
       "ompr__bw_delompr_nacl__2                NaN                     2   \n",
       "\n",
       "                                                            R1  \\\n",
       "sample_id                                                        \n",
       "ompr__bw_delompr_nacl__1  ompr/dompRNaCl1_S3_L001_R1_001.fastq   \n",
       "ompr__bw_delompr_nacl__2  ompr/dompRNaCl2_S4_L001_R1_001.fastq   \n",
       "\n",
       "                                                            R2  \\\n",
       "sample_id                                                        \n",
       "ompr__bw_delompr_nacl__1  ompr/dompRNaCl1_S3_L001_R2_001.fastq   \n",
       "ompr__bw_delompr_nacl__2  ompr/dompRNaCl2_S4_L001_R2_001.fastq   \n",
       "\n",
       "                                                   BAM  alignment  \\\n",
       "sample_id                                                           \n",
       "ompr__bw_delompr_nacl__1  ompr__bw_delompr_nacl__1.bam      98.56   \n",
       "ompr__bw_delompr_nacl__2  ompr__bw_delompr_nacl__2.bam      98.60   \n",
       "\n",
       "                                                         DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "ompr__bw_delompr_nacl__1  doi.org/10.1038/s41598-017-02110-7  GSE88980     1.0  \n",
       "ompr__bw_delompr_nacl__2  doi.org/10.1038/s41598-017-02110-7  GSE88980     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ompr__wt_nacl__1</td>\n",
       "      <td>OmpR</td>\n",
       "      <td>ompr</td>\n",
       "      <td>wt_nacl</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ompr/WTNaCl2_S2_L001_R1_001.fastq</td>\n",
       "      <td>ompr/WTNaCl2_S2_L001_R2_001.fastq</td>\n",
       "      <td>ompr__wt_nacl__1.bam</td>\n",
       "      <td>97.81</td>\n",
       "      <td>doi.org/10.1038/s41598-017-02110-7</td>\n",
       "      <td>GSE88980</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ompr__wt_nacl__2</td>\n",
       "      <td>OmpR</td>\n",
       "      <td>ompr</td>\n",
       "      <td>wt_nacl</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>7/29/2015 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ompr/WTNaCl1_S1_L001_R1_001.fastq</td>\n",
       "      <td>ompr/WTNaCl1_S1_L001_R2_001.fastq</td>\n",
       "      <td>ompr__wt_nacl__2.bam</td>\n",
       "      <td>98.81</td>\n",
       "      <td>doi.org/10.1038/s41598-017-02110-7</td>\n",
       "      <td>GSE88980</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                             \n",
       "ompr__wt_nacl__1  OmpR       ompr      wt_nacl       1  Sangwoo Seo   \n",
       "ompr__wt_nacl__2  OmpR       ompr      wt_nacl       2  Sangwoo Seo   \n",
       "\n",
       "                 library creator        run_date  \\\n",
       "sample_id                                          \n",
       "ompr__wt_nacl__1     Sangwoo Seo  7/29/2015 0:00   \n",
       "ompr__wt_nacl__2     Sangwoo Seo  7/29/2015 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ompr__wt_nacl__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "ompr__wt_nacl__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ompr__wt_nacl__1              MiSeq                NaN                     2   \n",
       "ompr__wt_nacl__2              MiSeq                NaN                     2   \n",
       "\n",
       "                                                 R1  \\\n",
       "sample_id                                             \n",
       "ompr__wt_nacl__1  ompr/WTNaCl2_S2_L001_R1_001.fastq   \n",
       "ompr__wt_nacl__2  ompr/WTNaCl1_S1_L001_R1_001.fastq   \n",
       "\n",
       "                                                 R2                   BAM  \\\n",
       "sample_id                                                                   \n",
       "ompr__wt_nacl__1  ompr/WTNaCl2_S2_L001_R2_001.fastq  ompr__wt_nacl__1.bam   \n",
       "ompr__wt_nacl__2  ompr/WTNaCl1_S1_L001_R2_001.fastq  ompr__wt_nacl__2.bam   \n",
       "\n",
       "                  alignment                                 DOI       GEO  \\\n",
       "sample_id                                                                   \n",
       "ompr__wt_nacl__1      97.81  doi.org/10.1038/s41598-017-02110-7  GSE88980   \n",
       "ompr__wt_nacl__2      98.81  doi.org/10.1038/s41598-017-02110-7  GSE88980   \n",
       "\n",
       "                  Public  \n",
       "sample_id                 \n",
       "ompr__wt_nacl__1     1.0  \n",
       "ompr__wt_nacl__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>oxidative__deloxyr_pq__1</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>deloxyr_pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/4/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_oxyR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/oxyR-PQ-RNAseq-1_S3_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/oxyR-PQ-RNAseq-1_S3_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__deloxyr_pq__1.bam</td>\n",
       "      <td>98.06</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>oxidative__deloxyr_pq__2</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>deloxyr_pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/4/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_oxyR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/oxyR-PQ-RNAseq-2_S1_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/oxyR-PQ-RNAseq-2_S1_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__deloxyr_pq__2.bam</td>\n",
       "      <td>97.80</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "oxidative__deloxyr_pq__1  Oxidative  oxidative   deloxyr_pq       1   \n",
       "oxidative__deloxyr_pq__2  Oxidative  oxidative   deloxyr_pq       2   \n",
       "\n",
       "                              contact library creator       run_date  \\\n",
       "sample_id                                                              \n",
       "oxidative__deloxyr_pq__1  Sangwoo Seo     Sangwoo Seo  9/4/2014 0:00   \n",
       "oxidative__deloxyr_pq__2  Sangwoo Seo     Sangwoo Seo  9/4/2014 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "oxidative__deloxyr_pq__1  Escherichia coli K-12 MG1655 del_oxyR   MG1655   \n",
       "oxidative__deloxyr_pq__2  Escherichia coli K-12 MG1655 del_oxyR   MG1655   \n",
       "\n",
       "                         Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "oxidative__deloxyr_pq__1         M9  ...              MiSeq   \n",
       "oxidative__deloxyr_pq__2         M9  ...              MiSeq   \n",
       "\n",
       "                         Additional Details Biological Replicates  \\\n",
       "sample_id                                                           \n",
       "oxidative__deloxyr_pq__1                NaN                     2   \n",
       "oxidative__deloxyr_pq__2                NaN                     2   \n",
       "\n",
       "                                                                         R1  \\\n",
       "sample_id                                                                     \n",
       "oxidative__deloxyr_pq__1  oxidative/oxyR-PQ-RNAseq-1_S3_L001_R1_001.fast...   \n",
       "oxidative__deloxyr_pq__2  oxidative/oxyR-PQ-RNAseq-2_S1_L001_R1_001.fast...   \n",
       "\n",
       "                                                                         R2  \\\n",
       "sample_id                                                                     \n",
       "oxidative__deloxyr_pq__1  oxidative/oxyR-PQ-RNAseq-1_S3_L001_R2_001.fast...   \n",
       "oxidative__deloxyr_pq__2  oxidative/oxyR-PQ-RNAseq-2_S1_L001_R2_001.fast...   \n",
       "\n",
       "                                                   BAM  alignment  \\\n",
       "sample_id                                                           \n",
       "oxidative__deloxyr_pq__1  oxidative__deloxyr_pq__1.bam      98.06   \n",
       "oxidative__deloxyr_pq__2  oxidative__deloxyr_pq__2.bam      97.80   \n",
       "\n",
       "                                                           DOI       GEO  \\\n",
       "sample_id                                                                  \n",
       "oxidative__deloxyr_pq__1  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "oxidative__deloxyr_pq__2  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "\n",
       "                          Public  \n",
       "sample_id                         \n",
       "oxidative__deloxyr_pq__1     1.0  \n",
       "oxidative__deloxyr_pq__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>oxidative__delsoxr_pq__1</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>delsoxr_pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/4/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_soxR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/soxR-PQ-RNAseq-1_S2_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/soxR-PQ-RNAseq-1_S2_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__delsoxr_pq__1.bam</td>\n",
       "      <td>98.33</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>oxidative__delsoxr_pq__2</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>delsoxr_pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/4/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_soxR</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/soxR-PQ-RNAseq-2_S3_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/soxR-PQ-RNAseq-2_S3_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__delsoxr_pq__2.bam</td>\n",
       "      <td>98.41</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "oxidative__delsoxr_pq__1  Oxidative  oxidative   delsoxr_pq       1   \n",
       "oxidative__delsoxr_pq__2  Oxidative  oxidative   delsoxr_pq       2   \n",
       "\n",
       "                              contact library creator       run_date  \\\n",
       "sample_id                                                              \n",
       "oxidative__delsoxr_pq__1  Sangwoo Seo     Sangwoo Seo  9/4/2014 0:00   \n",
       "oxidative__delsoxr_pq__2  Sangwoo Seo     Sangwoo Seo  9/4/2014 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "oxidative__delsoxr_pq__1  Escherichia coli K-12 MG1655 del_soxR   MG1655   \n",
       "oxidative__delsoxr_pq__2  Escherichia coli K-12 MG1655 del_soxR   MG1655   \n",
       "\n",
       "                         Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "oxidative__delsoxr_pq__1         M9  ...              MiSeq   \n",
       "oxidative__delsoxr_pq__2         M9  ...              MiSeq   \n",
       "\n",
       "                         Additional Details Biological Replicates  \\\n",
       "sample_id                                                           \n",
       "oxidative__delsoxr_pq__1                NaN                     2   \n",
       "oxidative__delsoxr_pq__2                NaN                     2   \n",
       "\n",
       "                                                                         R1  \\\n",
       "sample_id                                                                     \n",
       "oxidative__delsoxr_pq__1  oxidative/soxR-PQ-RNAseq-1_S2_L001_R1_001.fast...   \n",
       "oxidative__delsoxr_pq__2  oxidative/soxR-PQ-RNAseq-2_S3_L001_R1_001.fast...   \n",
       "\n",
       "                                                                         R2  \\\n",
       "sample_id                                                                     \n",
       "oxidative__delsoxr_pq__1  oxidative/soxR-PQ-RNAseq-1_S2_L001_R2_001.fast...   \n",
       "oxidative__delsoxr_pq__2  oxidative/soxR-PQ-RNAseq-2_S3_L001_R2_001.fast...   \n",
       "\n",
       "                                                   BAM  alignment  \\\n",
       "sample_id                                                           \n",
       "oxidative__delsoxr_pq__1  oxidative__delsoxr_pq__1.bam      98.33   \n",
       "oxidative__delsoxr_pq__2  oxidative__delsoxr_pq__2.bam      98.41   \n",
       "\n",
       "                                                           DOI       GEO  \\\n",
       "sample_id                                                                  \n",
       "oxidative__delsoxr_pq__1  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "oxidative__delsoxr_pq__2  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "\n",
       "                          Public  \n",
       "sample_id                         \n",
       "oxidative__delsoxr_pq__1     1.0  \n",
       "oxidative__delsoxr_pq__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>oxidative__delsoxs_pq__1</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>delsoxs_pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/18/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_soxS</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/soxS-PQ-RNAseq-1_S1_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/soxS-PQ-RNAseq-1_S1_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__delsoxs_pq__1.bam</td>\n",
       "      <td>97.30</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>oxidative__delsoxs_pq__2</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>delsoxs_pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>9/18/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_soxS</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/soxS-PQ-RNAseq-2_S2_L001_R1_001.fast...</td>\n",
       "      <td>oxidative/soxS-PQ-RNAseq-2_S2_L001_R2_001.fast...</td>\n",
       "      <td>oxidative__delsoxs_pq__2.bam</td>\n",
       "      <td>97.07</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                              study project_id condition_id  rep_id  \\\n",
       "sample_id                                                             \n",
       "oxidative__delsoxs_pq__1  Oxidative  oxidative   delsoxs_pq       1   \n",
       "oxidative__delsoxs_pq__2  Oxidative  oxidative   delsoxs_pq       2   \n",
       "\n",
       "                              contact library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "oxidative__delsoxs_pq__1  Sangwoo Seo     Sangwoo Seo  9/18/2014 0:00   \n",
       "oxidative__delsoxs_pq__2  Sangwoo Seo     Sangwoo Seo  9/18/2014 0:00   \n",
       "\n",
       "                                             Strain Description Organism  \\\n",
       "sample_id                                                                  \n",
       "oxidative__delsoxs_pq__1  Escherichia coli K-12 MG1655 del_soxS   MG1655   \n",
       "oxidative__delsoxs_pq__2  Escherichia coli K-12 MG1655 del_soxS   MG1655   \n",
       "\n",
       "                         Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "oxidative__delsoxs_pq__1         M9  ...              MiSeq   \n",
       "oxidative__delsoxs_pq__2         M9  ...              MiSeq   \n",
       "\n",
       "                         Additional Details Biological Replicates  \\\n",
       "sample_id                                                           \n",
       "oxidative__delsoxs_pq__1                NaN                     2   \n",
       "oxidative__delsoxs_pq__2                NaN                     2   \n",
       "\n",
       "                                                                         R1  \\\n",
       "sample_id                                                                     \n",
       "oxidative__delsoxs_pq__1  oxidative/soxS-PQ-RNAseq-1_S1_L001_R1_001.fast...   \n",
       "oxidative__delsoxs_pq__2  oxidative/soxS-PQ-RNAseq-2_S2_L001_R1_001.fast...   \n",
       "\n",
       "                                                                         R2  \\\n",
       "sample_id                                                                     \n",
       "oxidative__delsoxs_pq__1  oxidative/soxS-PQ-RNAseq-1_S1_L001_R2_001.fast...   \n",
       "oxidative__delsoxs_pq__2  oxidative/soxS-PQ-RNAseq-2_S2_L001_R2_001.fast...   \n",
       "\n",
       "                                                   BAM  alignment  \\\n",
       "sample_id                                                           \n",
       "oxidative__delsoxs_pq__1  oxidative__delsoxs_pq__1.bam      97.30   \n",
       "oxidative__delsoxs_pq__2  oxidative__delsoxs_pq__2.bam      97.07   \n",
       "\n",
       "                                                           DOI       GEO  \\\n",
       "sample_id                                                                  \n",
       "oxidative__delsoxs_pq__1  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "oxidative__delsoxs_pq__2  doi.org/10.1016/j.celrep.2015.07.043  GSE65711   \n",
       "\n",
       "                          Public  \n",
       "sample_id                         \n",
       "oxidative__delsoxs_pq__1     1.0  \n",
       "oxidative__delsoxs_pq__2     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>oxidative__wt_pq__1</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>wt_pq</td>\n",
       "      <td>1</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>8/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/WT-PQ-RNAseq-1_S1_L001_R1_001.fastq.gz</td>\n",
       "      <td>oxidative/WT-PQ-RNAseq-1_S1_L001_R2_001.fastq.gz</td>\n",
       "      <td>oxidative__wt_pq__1.bam</td>\n",
       "      <td>98.70</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>oxidative__wt_pq__2</td>\n",
       "      <td>Oxidative</td>\n",
       "      <td>oxidative</td>\n",
       "      <td>wt_pq</td>\n",
       "      <td>2</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>Sangwoo Seo</td>\n",
       "      <td>8/27/2014 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>oxidative/WT-PQ-RNAseq-2_S2_L001_R1_001.fastq.gz</td>\n",
       "      <td>oxidative/WT-PQ-RNAseq-2_S2_L001_R2_001.fastq.gz</td>\n",
       "      <td>oxidative__wt_pq__2.bam</td>\n",
       "      <td>98.51</td>\n",
       "      <td>doi.org/10.1016/j.celrep.2015.07.043</td>\n",
       "      <td>GSE65711</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id      contact  \\\n",
       "sample_id                                                                     \n",
       "oxidative__wt_pq__1  Oxidative  oxidative        wt_pq       1  Sangwoo Seo   \n",
       "oxidative__wt_pq__2  Oxidative  oxidative        wt_pq       2  Sangwoo Seo   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "oxidative__wt_pq__1     Sangwoo Seo  8/27/2014 0:00   \n",
       "oxidative__wt_pq__2     Sangwoo Seo  8/27/2014 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "oxidative__wt_pq__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "oxidative__wt_pq__2  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "oxidative__wt_pq__1              MiSeq                NaN   \n",
       "oxidative__wt_pq__2              MiSeq                NaN   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "oxidative__wt_pq__1                     2   \n",
       "oxidative__wt_pq__2                     2   \n",
       "\n",
       "                                                                   R1  \\\n",
       "sample_id                                                               \n",
       "oxidative__wt_pq__1  oxidative/WT-PQ-RNAseq-1_S1_L001_R1_001.fastq.gz   \n",
       "oxidative__wt_pq__2  oxidative/WT-PQ-RNAseq-2_S2_L001_R1_001.fastq.gz   \n",
       "\n",
       "                                                                   R2  \\\n",
       "sample_id                                                               \n",
       "oxidative__wt_pq__1  oxidative/WT-PQ-RNAseq-1_S1_L001_R2_001.fastq.gz   \n",
       "oxidative__wt_pq__2  oxidative/WT-PQ-RNAseq-2_S2_L001_R2_001.fastq.gz   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "oxidative__wt_pq__1  oxidative__wt_pq__1.bam      98.70   \n",
       "oxidative__wt_pq__2  oxidative__wt_pq__2.bam      98.51   \n",
       "\n",
       "                                                      DOI       GEO  Public  \n",
       "sample_id                                                                    \n",
       "oxidative__wt_pq__1  doi.org/10.1016/j.celrep.2015.07.043  GSE65711     1.0  \n",
       "oxidative__wt_pq__2  doi.org/10.1016/j.celrep.2015.07.043  GSE65711     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__5drib_ale10__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>5drib_ale10</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.10.118.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>pal/SRR7154632_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154632_2.fastq.gz</td>\n",
       "      <td>pal__5drib_ale10__1.bam</td>\n",
       "      <td>97.36</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "pal__5drib_ale10__1  Enzyme Promiscuity        pal  5drib_ale10       1   \n",
       "\n",
       "                             contact  library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pal__5drib_ale10__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pal__5drib_ale10__1  Escherichia coli PAL.10.118.0   MG1655         M9  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "pal__5drib_ale10__1         HiSeq 2500                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "pal__5drib_ale10__1                     1  pal/SRR7154632_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "pal__5drib_ale10__1  pal/SRR7154632_2.fastq.gz  pal__5drib_ale10__1.bam   \n",
       "\n",
       "                     alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "pal__5drib_ale10__1      97.36  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__5drib_ale10_f18__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>5drib_ale10_f18</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.10.18.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154629_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154629_2.fastq.gz</td>\n",
       "      <td>pal__5drib_ale10_f18__1.bam</td>\n",
       "      <td>98.09</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__5drib_ale10_f18__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>5drib_ale10_f18</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.10.18.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154631_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154631_2.fastq.gz</td>\n",
       "      <td>pal__5drib_ale10_f18__2.bam</td>\n",
       "      <td>98.29</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      study project_id     condition_id  \\\n",
       "sample_id                                                                 \n",
       "pal__5drib_ale10_f18__1  Enzyme Promiscuity        pal  5drib_ale10_f18   \n",
       "pal__5drib_ale10_f18__2  Enzyme Promiscuity        pal  5drib_ale10_f18   \n",
       "\n",
       "                         rep_id          contact  library creator  \\\n",
       "sample_id                                                           \n",
       "pal__5drib_ale10_f18__1       1  Gabriela Guzman  Gabriela Guzman   \n",
       "pal__5drib_ale10_f18__2       2  Gabriela Guzman  Gabriela Guzman   \n",
       "\n",
       "                               run_date            Strain Description  \\\n",
       "sample_id                                                               \n",
       "pal__5drib_ale10_f18__1  5/11/2016 0:00  Escherichia coli PAL.10.18.1   \n",
       "pal__5drib_ale10_f18__2  5/11/2016 0:00  Escherichia coli PAL.10.18.1   \n",
       "\n",
       "                        Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                    ...                      \n",
       "pal__5drib_ale10_f18__1   MG1655         M9  ...         HiSeq 2500   \n",
       "pal__5drib_ale10_f18__2   MG1655         M9  ...         HiSeq 2500   \n",
       "\n",
       "                        Additional Details Biological Replicates  \\\n",
       "sample_id                                                          \n",
       "pal__5drib_ale10_f18__1                NaN                     2   \n",
       "pal__5drib_ale10_f18__2                NaN                     2   \n",
       "\n",
       "                                                R1                         R2  \\\n",
       "sample_id                                                                       \n",
       "pal__5drib_ale10_f18__1  pal/SRR7154629_1.fastq.gz  pal/SRR7154629_2.fastq.gz   \n",
       "pal__5drib_ale10_f18__2  pal/SRR7154631_1.fastq.gz  pal/SRR7154631_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "pal__5drib_ale10_f18__1  pal__5drib_ale10_f18__1.bam      98.09   \n",
       "pal__5drib_ale10_f18__2  pal__5drib_ale10_f18__2.bam      98.29   \n",
       "\n",
       "                                            DOI        GEO  Public  \n",
       "sample_id                                                           \n",
       "pal__5drib_ale10_f18__1  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__5drib_ale10_f18__2  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale14__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale14</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.14.78.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154635_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154635_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale14__1.bam</td>\n",
       "      <td>97.76</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale14__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale14</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.14.78.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154636_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154636_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale14__2.bam</td>\n",
       "      <td>97.75</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                \n",
       "pal__arab_ale14__1  Enzyme Promiscuity        pal   arab_ale14       1   \n",
       "pal__arab_ale14__2  Enzyme Promiscuity        pal   arab_ale14       2   \n",
       "\n",
       "                            contact  library creator        run_date  \\\n",
       "sample_id                                                              \n",
       "pal__arab_ale14__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__arab_ale14__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "pal__arab_ale14__1  Escherichia coli PAL.14.78.0   MG1655         M9  ...   \n",
       "pal__arab_ale14__2  Escherichia coli PAL.14.78.0   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "pal__arab_ale14__1         HiSeq 2500                NaN   \n",
       "pal__arab_ale14__2         HiSeq 2500                NaN   \n",
       "\n",
       "                   Biological Replicates                         R1  \\\n",
       "sample_id                                                             \n",
       "pal__arab_ale14__1                     2  pal/SRR7154635_1.fastq.gz   \n",
       "pal__arab_ale14__2                     2  pal/SRR7154636_1.fastq.gz   \n",
       "\n",
       "                                           R2                     BAM  \\\n",
       "sample_id                                                               \n",
       "pal__arab_ale14__1  pal/SRR7154635_2.fastq.gz  pal__arab_ale14__1.bam   \n",
       "pal__arab_ale14__2  pal/SRR7154636_2.fastq.gz  pal__arab_ale14__2.bam   \n",
       "\n",
       "                    alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                 \n",
       "pal__arab_ale14__1      97.76  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__arab_ale14__2      97.75  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale14_f9__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale14_f9</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.14.9.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154633_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154633_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale14_f9__1.bam</td>\n",
       "      <td>97.35</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale14_f9__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale14_f9</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.14.9.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154634_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154634_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale14_f9__2.bam</td>\n",
       "      <td>97.71</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                    study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                     \n",
       "pal__arab_ale14_f9__1  Enzyme Promiscuity        pal  arab_ale14_f9       1   \n",
       "pal__arab_ale14_f9__2  Enzyme Promiscuity        pal  arab_ale14_f9       2   \n",
       "\n",
       "                               contact  library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "pal__arab_ale14_f9__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__arab_ale14_f9__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pal__arab_ale14_f9__1  Escherichia coli PAL.14.9.1   MG1655         M9  ...   \n",
       "pal__arab_ale14_f9__2  Escherichia coli PAL.14.9.1   MG1655         M9  ...   \n",
       "\n",
       "                      Sequencing Machine Additional Details  \\\n",
       "sample_id                                                     \n",
       "pal__arab_ale14_f9__1         HiSeq 2500                NaN   \n",
       "pal__arab_ale14_f9__2         HiSeq 2500                NaN   \n",
       "\n",
       "                      Biological Replicates                         R1  \\\n",
       "sample_id                                                                \n",
       "pal__arab_ale14_f9__1                     2  pal/SRR7154633_1.fastq.gz   \n",
       "pal__arab_ale14_f9__2                     2  pal/SRR7154634_1.fastq.gz   \n",
       "\n",
       "                                              R2                        BAM  \\\n",
       "sample_id                                                                     \n",
       "pal__arab_ale14_f9__1  pal/SRR7154633_2.fastq.gz  pal__arab_ale14_f9__1.bam   \n",
       "pal__arab_ale14_f9__2  pal/SRR7154634_2.fastq.gz  pal__arab_ale14_f9__2.bam   \n",
       "\n",
       "                       alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                    \n",
       "pal__arab_ale14_f9__1      97.35  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__arab_ale14_f9__2      97.71  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale16__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale16</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.16.70.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154637_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154637_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale16__1.bam</td>\n",
       "      <td>98.15</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__arab_ale16__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>arab_ale16</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.16.70.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154638_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154638_2.fastq.gz</td>\n",
       "      <td>pal__arab_ale16__2.bam</td>\n",
       "      <td>98.09</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                 study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                \n",
       "pal__arab_ale16__1  Enzyme Promiscuity        pal   arab_ale16       1   \n",
       "pal__arab_ale16__2  Enzyme Promiscuity        pal   arab_ale16       2   \n",
       "\n",
       "                            contact  library creator        run_date  \\\n",
       "sample_id                                                              \n",
       "pal__arab_ale16__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__arab_ale16__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                              Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                             ...   \n",
       "pal__arab_ale16__1  Escherichia coli PAL.16.70.0   MG1655         M9  ...   \n",
       "pal__arab_ale16__2  Escherichia coli PAL.16.70.0   MG1655         M9  ...   \n",
       "\n",
       "                   Sequencing Machine Additional Details  \\\n",
       "sample_id                                                  \n",
       "pal__arab_ale16__1         HiSeq 2500                NaN   \n",
       "pal__arab_ale16__2         HiSeq 2500                NaN   \n",
       "\n",
       "                   Biological Replicates                         R1  \\\n",
       "sample_id                                                             \n",
       "pal__arab_ale16__1                     2  pal/SRR7154637_1.fastq.gz   \n",
       "pal__arab_ale16__2                     2  pal/SRR7154638_1.fastq.gz   \n",
       "\n",
       "                                           R2                     BAM  \\\n",
       "sample_id                                                               \n",
       "pal__arab_ale16__1  pal/SRR7154637_2.fastq.gz  pal__arab_ale16__1.bam   \n",
       "pal__arab_ale16__2  pal/SRR7154638_2.fastq.gz  pal__arab_ale16__2.bam   \n",
       "\n",
       "                    alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                 \n",
       "pal__arab_ale16__1      98.15  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__arab_ale16__2      98.09  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__lyx_ale2__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>lyx_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.2.73.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154625_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154625_2.fastq.gz</td>\n",
       "      <td>pal__lyx_ale2__1.bam</td>\n",
       "      <td>97.75</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__lyx_ale2__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>lyx_ale2</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.2.73.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154626_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154626_2.fastq.gz</td>\n",
       "      <td>pal__lyx_ale2__2.bam</td>\n",
       "      <td>97.57</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "pal__lyx_ale2__1  Enzyme Promiscuity        pal     lyx_ale2       1   \n",
       "pal__lyx_ale2__2  Enzyme Promiscuity        pal     lyx_ale2       2   \n",
       "\n",
       "                          contact  library creator        run_date  \\\n",
       "sample_id                                                            \n",
       "pal__lyx_ale2__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__lyx_ale2__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "pal__lyx_ale2__1  Escherichia coli PAL.2.73.0   MG1655         M9  ...   \n",
       "pal__lyx_ale2__2  Escherichia coli PAL.2.73.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "pal__lyx_ale2__1         HiSeq 2500                NaN                     2   \n",
       "pal__lyx_ale2__2         HiSeq 2500                NaN                     2   \n",
       "\n",
       "                                         R1                         R2  \\\n",
       "sample_id                                                                \n",
       "pal__lyx_ale2__1  pal/SRR7154625_1.fastq.gz  pal/SRR7154625_2.fastq.gz   \n",
       "pal__lyx_ale2__2  pal/SRR7154626_1.fastq.gz  pal/SRR7154626_2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment                     DOI  \\\n",
       "sample_id                                                                   \n",
       "pal__lyx_ale2__1  pal__lyx_ale2__1.bam      97.75  doi.org/10.1101/310946   \n",
       "pal__lyx_ale2__2  pal__lyx_ale2__2.bam      97.57  doi.org/10.1101/310946   \n",
       "\n",
       "                        GEO  Public  \n",
       "sample_id                            \n",
       "pal__lyx_ale2__1  GSE114358     1.0  \n",
       "pal__lyx_ale2__2  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__lyx_ale2_f6__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>lyx_ale2_f6</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.2.6.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>pal/SRR7154624_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154624_2.fastq.gz</td>\n",
       "      <td>pal__lyx_ale2_f6__1.bam</td>\n",
       "      <td>97.44</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "pal__lyx_ale2_f6__1  Enzyme Promiscuity        pal  lyx_ale2_f6       1   \n",
       "\n",
       "                             contact  library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pal__lyx_ale2_f6__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                             Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                            ...   \n",
       "pal__lyx_ale2_f6__1  Escherichia coli PAL.2.6.1   MG1655         M9  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "pal__lyx_ale2_f6__1         HiSeq 2500                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "pal__lyx_ale2_f6__1                     1  pal/SRR7154624_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "pal__lyx_ale2_f6__1  pal/SRR7154624_2.fastq.gz  pal__lyx_ale2_f6__1.bam   \n",
       "\n",
       "                     alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "pal__lyx_ale2_f6__1      97.44  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__lyx_ale4__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>lyx_ale4</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.4.90.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154627_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154627_2.fastq.gz</td>\n",
       "      <td>pal__lyx_ale4__1.bam</td>\n",
       "      <td>97.75</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__lyx_ale4__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>lyx_ale4</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.4.90.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154628_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154628_2.fastq.gz</td>\n",
       "      <td>pal__lyx_ale4__2.bam</td>\n",
       "      <td>97.57</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id condition_id  rep_id  \\\n",
       "sample_id                                                              \n",
       "pal__lyx_ale4__1  Enzyme Promiscuity        pal     lyx_ale4       1   \n",
       "pal__lyx_ale4__2  Enzyme Promiscuity        pal     lyx_ale4       2   \n",
       "\n",
       "                          contact  library creator        run_date  \\\n",
       "sample_id                                                            \n",
       "pal__lyx_ale4__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__lyx_ale4__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "pal__lyx_ale4__1  Escherichia coli PAL.4.90.0   MG1655         M9  ...   \n",
       "pal__lyx_ale4__2  Escherichia coli PAL.4.90.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "pal__lyx_ale4__1         HiSeq 2500                NaN                     2   \n",
       "pal__lyx_ale4__2         HiSeq 2500                NaN                     2   \n",
       "\n",
       "                                         R1                         R2  \\\n",
       "sample_id                                                                \n",
       "pal__lyx_ale4__1  pal/SRR7154627_1.fastq.gz  pal/SRR7154627_2.fastq.gz   \n",
       "pal__lyx_ale4__2  pal/SRR7154628_1.fastq.gz  pal/SRR7154628_2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment                     DOI  \\\n",
       "sample_id                                                                   \n",
       "pal__lyx_ale4__1  pal__lyx_ale4__1.bam      97.75  doi.org/10.1101/310946   \n",
       "pal__lyx_ale4__2  pal__lyx_ale4__2.bam      97.57  doi.org/10.1101/310946   \n",
       "\n",
       "                        GEO  Public  \n",
       "sample_id                            \n",
       "pal__lyx_ale4__1  GSE114358     1.0  \n",
       "pal__lyx_ale4__2  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale26__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale26</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.26.86.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154643_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154643_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale26__1.bam</td>\n",
       "      <td>97.24</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale26__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale26</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.26.86.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154644_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154644_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale26__2.bam</td>\n",
       "      <td>97.59</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale26__1  Enzyme Promiscuity        pal  tartr_ale26       1   \n",
       "pal__tartr_ale26__2  Enzyme Promiscuity        pal  tartr_ale26       2   \n",
       "\n",
       "                             contact  library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pal__tartr_ale26__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__tartr_ale26__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "pal__tartr_ale26__1  Escherichia coli PAL.26.86.0   MG1655         M9  ...   \n",
       "pal__tartr_ale26__2  Escherichia coli PAL.26.86.0   MG1655         M9  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "pal__tartr_ale26__1         HiSeq 2500                NaN   \n",
       "pal__tartr_ale26__2         HiSeq 2500                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "pal__tartr_ale26__1                     2  pal/SRR7154643_1.fastq.gz   \n",
       "pal__tartr_ale26__2                     2  pal/SRR7154644_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale26__1  pal/SRR7154643_2.fastq.gz  pal__tartr_ale26__1.bam   \n",
       "pal__tartr_ale26__2  pal/SRR7154644_2.fastq.gz  pal__tartr_ale26__2.bam   \n",
       "\n",
       "                     alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "pal__tartr_ale26__1      97.24  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__tartr_ale26__2      97.59  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale26_f10__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale26_f10</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.26.10.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154639_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154639_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale26_f10__1.bam</td>\n",
       "      <td>98.17</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale26_f10__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale26_f10</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.26.10.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154640_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154640_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale26_f10__2.bam</td>\n",
       "      <td>98.07</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      study project_id     condition_id  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale26_f10__1  Enzyme Promiscuity        pal  tartr_ale26_f10   \n",
       "pal__tartr_ale26_f10__2  Enzyme Promiscuity        pal  tartr_ale26_f10   \n",
       "\n",
       "                         rep_id          contact  library creator  \\\n",
       "sample_id                                                           \n",
       "pal__tartr_ale26_f10__1       1  Gabriela Guzman  Gabriela Guzman   \n",
       "pal__tartr_ale26_f10__2       2  Gabriela Guzman  Gabriela Guzman   \n",
       "\n",
       "                               run_date            Strain Description  \\\n",
       "sample_id                                                               \n",
       "pal__tartr_ale26_f10__1  5/11/2016 0:00  Escherichia coli PAL.26.10.1   \n",
       "pal__tartr_ale26_f10__2  5/11/2016 0:00  Escherichia coli PAL.26.10.1   \n",
       "\n",
       "                        Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                    ...                      \n",
       "pal__tartr_ale26_f10__1   MG1655         M9  ...         HiSeq 2500   \n",
       "pal__tartr_ale26_f10__2   MG1655         M9  ...         HiSeq 2500   \n",
       "\n",
       "                        Additional Details Biological Replicates  \\\n",
       "sample_id                                                          \n",
       "pal__tartr_ale26_f10__1                NaN                     2   \n",
       "pal__tartr_ale26_f10__2                NaN                     2   \n",
       "\n",
       "                                                R1                         R2  \\\n",
       "sample_id                                                                       \n",
       "pal__tartr_ale26_f10__1  pal/SRR7154639_1.fastq.gz  pal/SRR7154639_2.fastq.gz   \n",
       "pal__tartr_ale26_f10__2  pal/SRR7154640_1.fastq.gz  pal/SRR7154640_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "pal__tartr_ale26_f10__1  pal__tartr_ale26_f10__1.bam      98.17   \n",
       "pal__tartr_ale26_f10__2  pal__tartr_ale26_f10__2.bam      98.07   \n",
       "\n",
       "                                            DOI        GEO  Public  \n",
       "sample_id                                                           \n",
       "pal__tartr_ale26_f10__1  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__tartr_ale26_f10__2  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale28__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale28</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.28.88.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154645_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154645_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale28__1.bam</td>\n",
       "      <td>98.13</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale28__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale28</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.28.88.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154646_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154646_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale28__2.bam</td>\n",
       "      <td>98.04</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                  study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale28__1  Enzyme Promiscuity        pal  tartr_ale28       1   \n",
       "pal__tartr_ale28__2  Enzyme Promiscuity        pal  tartr_ale28       2   \n",
       "\n",
       "                             contact  library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pal__tartr_ale28__1  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "pal__tartr_ale28__2  Gabriela Guzman  Gabriela Guzman  5/11/2016 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "pal__tartr_ale28__1  Escherichia coli PAL.28.88.0   MG1655         M9  ...   \n",
       "pal__tartr_ale28__2  Escherichia coli PAL.28.88.0   MG1655         M9  ...   \n",
       "\n",
       "                    Sequencing Machine Additional Details  \\\n",
       "sample_id                                                   \n",
       "pal__tartr_ale28__1         HiSeq 2500                NaN   \n",
       "pal__tartr_ale28__2         HiSeq 2500                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "pal__tartr_ale28__1                     2  pal/SRR7154645_1.fastq.gz   \n",
       "pal__tartr_ale28__2                     2  pal/SRR7154646_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale28__1  pal/SRR7154645_2.fastq.gz  pal__tartr_ale28__1.bam   \n",
       "pal__tartr_ale28__2  pal/SRR7154646_2.fastq.gz  pal__tartr_ale28__2.bam   \n",
       "\n",
       "                     alignment                     DOI        GEO  Public  \n",
       "sample_id                                                                  \n",
       "pal__tartr_ale28__1      98.13  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__tartr_ale28__2      98.04  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale28_f10__1</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale28_f10</td>\n",
       "      <td>1</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.28.10.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154641_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154641_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale28_f10__1.bam</td>\n",
       "      <td>97.47</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>pal__tartr_ale28_f10__2</td>\n",
       "      <td>Enzyme Promiscuity</td>\n",
       "      <td>pal</td>\n",
       "      <td>tartr_ale28_f10</td>\n",
       "      <td>2</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>Gabriela Guzman</td>\n",
       "      <td>5/11/2016 0:00</td>\n",
       "      <td>Escherichia coli PAL.28.10.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2500</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>pal/SRR7154642_1.fastq.gz</td>\n",
       "      <td>pal/SRR7154642_2.fastq.gz</td>\n",
       "      <td>pal__tartr_ale28_f10__2.bam</td>\n",
       "      <td>97.83</td>\n",
       "      <td>doi.org/10.1101/310946</td>\n",
       "      <td>GSE114358</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      study project_id     condition_id  \\\n",
       "sample_id                                                                 \n",
       "pal__tartr_ale28_f10__1  Enzyme Promiscuity        pal  tartr_ale28_f10   \n",
       "pal__tartr_ale28_f10__2  Enzyme Promiscuity        pal  tartr_ale28_f10   \n",
       "\n",
       "                         rep_id          contact  library creator  \\\n",
       "sample_id                                                           \n",
       "pal__tartr_ale28_f10__1       1  Gabriela Guzman  Gabriela Guzman   \n",
       "pal__tartr_ale28_f10__2       2  Gabriela Guzman  Gabriela Guzman   \n",
       "\n",
       "                               run_date            Strain Description  \\\n",
       "sample_id                                                               \n",
       "pal__tartr_ale28_f10__1  5/11/2016 0:00  Escherichia coli PAL.28.10.1   \n",
       "pal__tartr_ale28_f10__2  5/11/2016 0:00  Escherichia coli PAL.28.10.1   \n",
       "\n",
       "                        Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                    ...                      \n",
       "pal__tartr_ale28_f10__1   MG1655         M9  ...         HiSeq 2500   \n",
       "pal__tartr_ale28_f10__2   MG1655         M9  ...         HiSeq 2500   \n",
       "\n",
       "                        Additional Details Biological Replicates  \\\n",
       "sample_id                                                          \n",
       "pal__tartr_ale28_f10__1                NaN                     2   \n",
       "pal__tartr_ale28_f10__2                NaN                     2   \n",
       "\n",
       "                                                R1                         R2  \\\n",
       "sample_id                                                                       \n",
       "pal__tartr_ale28_f10__1  pal/SRR7154641_1.fastq.gz  pal/SRR7154641_2.fastq.gz   \n",
       "pal__tartr_ale28_f10__2  pal/SRR7154642_1.fastq.gz  pal/SRR7154642_2.fastq.gz   \n",
       "\n",
       "                                                 BAM  alignment  \\\n",
       "sample_id                                                         \n",
       "pal__tartr_ale28_f10__1  pal__tartr_ale28_f10__1.bam      97.47   \n",
       "pal__tartr_ale28_f10__2  pal__tartr_ale28_f10__2.bam      97.83   \n",
       "\n",
       "                                            DOI        GEO  Public  \n",
       "sample_id                                                           \n",
       "pal__tartr_ale28_f10__1  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "pal__tartr_ale28_f10__2  doi.org/10.1101/310946  GSE114358     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__glu4_delpgi__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>glu4_delpgi</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/16/2014 0:00</td>\n",
       "      <td>Escherichia coli GLU.4.403.1 del_pgi</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GLU A4 F403 I1 w/ pgi KO</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140716_0_OxicEvo04pgiEcoliGlcM9_Broth-1_R1...</td>\n",
       "      <td>pgi/140716_0_OxicEvo04pgiEcoliGlcM9_Broth-1_R2...</td>\n",
       "      <td>pgi__glu4_delpgi__1.bam</td>\n",
       "      <td>93.97</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id condition_id  rep_id  \\\n",
       "sample_id                                                               \n",
       "pgi__glu4_delpgi__1  PGI KO Evolution        pgi  glu4_delpgi       1   \n",
       "\n",
       "                               contact   library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "pgi__glu4_delpgi__1  Douglas McCloskey  Sibei (Julia) Xu  7/16/2014 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "pgi__glu4_delpgi__1  Escherichia coli GLU.4.403.1 del_pgi   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine        Additional Details  \\\n",
       "sample_id            ...                                                \n",
       "pgi__glu4_delpgi__1  ...                NaN  GLU A4 F403 I1 w/ pgi KO   \n",
       "\n",
       "                    Biological Replicates  \\\n",
       "sample_id                                   \n",
       "pgi__glu4_delpgi__1                     1   \n",
       "\n",
       "                                                                    R1  \\\n",
       "sample_id                                                                \n",
       "pgi__glu4_delpgi__1  pgi/140716_0_OxicEvo04pgiEcoliGlcM9_Broth-1_R1...   \n",
       "\n",
       "                                                                    R2  \\\n",
       "sample_id                                                                \n",
       "pgi__glu4_delpgi__1  pgi/140716_0_OxicEvo04pgiEcoliGlcM9_Broth-1_R2...   \n",
       "\n",
       "                                         BAM  alignment  \\\n",
       "sample_id                                                 \n",
       "pgi__glu4_delpgi__1  pgi__glu4_delpgi__1.bam      93.97   \n",
       "\n",
       "                                              DOI  GEO  Public  \n",
       "sample_id                                                       \n",
       "pgi__glu4_delpgi__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale1__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/28/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.1.113.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A1 F113 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140728_11_OxicEvo04pgiEvo01EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140728_11_OxicEvo04pgiEvo01EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale1__1.bam</td>\n",
       "      <td>92.57</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale1__1  PGI KO Evolution        pgi     pgi_ale1       1   \n",
       "\n",
       "                            contact   library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pgi__pgi_ale1__1  Douglas McCloskey  Sibei (Julia) Xu  7/28/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale1__1  Escherichia coli CCK_PGI.1.113.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale1__1                NaN  CCK PGI A1 F113 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale1__1  pgi/140728_11_OxicEvo04pgiEvo01EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale1__1  pgi/140728_11_OxicEvo04pgiEvo01EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale1__1  pgi__pgi_ale1__1.bam      92.57   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale1__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale2__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/28/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.2.107.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A2 F107 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140728_11_OxicEvo04pgiEvo02EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140728_11_OxicEvo04pgiEvo02EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale2__1.bam</td>\n",
       "      <td>90.17</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale2__1  PGI KO Evolution        pgi     pgi_ale2       1   \n",
       "\n",
       "                            contact   library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pgi__pgi_ale2__1  Douglas McCloskey  Sibei (Julia) Xu  7/28/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale2__1  Escherichia coli CCK_PGI.2.107.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale2__1                NaN  CCK PGI A2 F107 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale2__1  pgi/140728_11_OxicEvo04pgiEvo02EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale2__1  pgi/140728_11_OxicEvo04pgiEvo02EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale2__1  pgi__pgi_ale2__1.bam      90.17   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale2__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale3__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/30/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.3.108.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A3 F108 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140730_11_OxicEvo04pgiEvo03EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140730_11_OxicEvo04pgiEvo03EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale3__1.bam</td>\n",
       "      <td>95.97</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale3__1  PGI KO Evolution        pgi     pgi_ale3       1   \n",
       "\n",
       "                            contact   library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pgi__pgi_ale3__1  Douglas McCloskey  Sibei (Julia) Xu  7/30/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale3__1  Escherichia coli CCK_PGI.3.108.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale3__1                NaN  CCK PGI A3 F108 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale3__1  pgi/140730_11_OxicEvo04pgiEvo03EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale3__1  pgi/140730_11_OxicEvo04pgiEvo03EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale3__1  pgi__pgi_ale3__1.bam      95.97   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale3__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale5__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale5</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/31/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.5.109.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A5 F109 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140731_11_OxicEvo04pgiEvo05EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140731_11_OxicEvo04pgiEvo05EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale5__1.bam</td>\n",
       "      <td>92.92</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale5__1  PGI KO Evolution        pgi     pgi_ale5       1   \n",
       "\n",
       "                            contact   library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pgi__pgi_ale5__1  Douglas McCloskey  Sibei (Julia) Xu  7/31/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale5__1  Escherichia coli CCK_PGI.5.109.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale5__1                NaN  CCK PGI A5 F109 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale5__1  pgi/140731_11_OxicEvo04pgiEvo05EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale5__1  pgi/140731_11_OxicEvo04pgiEvo05EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale5__1  pgi__pgi_ale5__1.bam      92.92   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale5__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale6__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale6</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>7/31/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.6.110.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A6 F110 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140731_11_OxicEvo04pgiEvo06EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140731_11_OxicEvo04pgiEvo06EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale6__1.bam</td>\n",
       "      <td>86.39</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale6__1  PGI KO Evolution        pgi     pgi_ale6       1   \n",
       "\n",
       "                            contact   library creator        run_date  \\\n",
       "sample_id                                                               \n",
       "pgi__pgi_ale6__1  Douglas McCloskey  Sibei (Julia) Xu  7/31/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale6__1  Escherichia coli CCK_PGI.6.110.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale6__1                NaN  CCK PGI A6 F110 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale6__1  pgi/140731_11_OxicEvo04pgiEvo06EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale6__1  pgi/140731_11_OxicEvo04pgiEvo06EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale6__1  pgi__pgi_ale6__1.bam      86.39   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale6__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale7__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale7</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>8/1/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.7.97.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A7 F97 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140801_10_OxicEvo04pgiEvo07EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140801_10_OxicEvo04pgiEvo07EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale7__1.bam</td>\n",
       "      <td>95.15</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale7__1  PGI KO Evolution        pgi     pgi_ale7       1   \n",
       "\n",
       "                            contact   library creator       run_date  \\\n",
       "sample_id                                                              \n",
       "pgi__pgi_ale7__1  Douglas McCloskey  Sibei (Julia) Xu  8/1/2014 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "pgi__pgi_ale7__1  Escherichia coli CCK_PGI.7.97.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "pgi__pgi_ale7__1                NaN  CCK PGI A7 F97 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale7__1  pgi/140801_10_OxicEvo04pgiEvo07EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale7__1  pgi/140801_10_OxicEvo04pgiEvo07EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale7__1  pgi__pgi_ale7__1.bam      95.15   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale7__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>pgi__pgi_ale8__1</td>\n",
       "      <td>PGI KO Evolution</td>\n",
       "      <td>pgi</td>\n",
       "      <td>pgi_ale8</td>\n",
       "      <td>1</td>\n",
       "      <td>Douglas McCloskey</td>\n",
       "      <td>Sibei (Julia) Xu</td>\n",
       "      <td>8/1/2014 0:00</td>\n",
       "      <td>Escherichia coli CCK_PGI.8.110.0</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>CCK PGI A8 F110 I0</td>\n",
       "      <td>1</td>\n",
       "      <td>pgi/140801_11_OxicEvo04pgiEvo08EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi/140801_11_OxicEvo04pgiEvo08EPEcoliGlcM9_Br...</td>\n",
       "      <td>pgi__pgi_ale8__1.bam</td>\n",
       "      <td>95.25</td>\n",
       "      <td>doi.org/10.1128/AEM.00823-18</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                             study project_id condition_id  rep_id  \\\n",
       "sample_id                                                            \n",
       "pgi__pgi_ale8__1  PGI KO Evolution        pgi     pgi_ale8       1   \n",
       "\n",
       "                            contact   library creator       run_date  \\\n",
       "sample_id                                                              \n",
       "pgi__pgi_ale8__1  Douglas McCloskey  Sibei (Julia) Xu  8/1/2014 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "pgi__pgi_ale8__1  Escherichia coli CCK_PGI.8.110.0   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine  Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "pgi__pgi_ale8__1                NaN  CCK PGI A8 F110 I0                     1   \n",
       "\n",
       "                                                                 R1  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale8__1  pgi/140801_11_OxicEvo04pgiEvo08EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                                                 R2  \\\n",
       "sample_id                                                             \n",
       "pgi__pgi_ale8__1  pgi/140801_11_OxicEvo04pgiEvo08EPEcoliGlcM9_Br...   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "pgi__pgi_ale8__1  pgi__pgi_ale8__1.bam      95.25   \n",
       "\n",
       "                                           DOI  GEO  Public  \n",
       "sample_id                                                    \n",
       "pgi__pgi_ale8__1  doi.org/10.1128/AEM.00823-18  NaN     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE546V_031__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE546V_031</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>8/13/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE546V</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>chemostat w/ dilution rate 0.31 h^-1</td>\n",
       "      <td>1</td>\n",
       "      <td>rpoB/SRR1514298_1.fastq</td>\n",
       "      <td>rpoB/SRR1514298_2.fastq</td>\n",
       "      <td>rpoB__rpoBE546V_031__1.bam</td>\n",
       "      <td>98.2</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_031__1  RpoB Knock-in       rpoB  rpoBE546V_031       1   \n",
       "\n",
       "                             contact library creator        run_date  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE546V_031__1  Jose Utrilla    Jose Utrilla  8/13/2013 0:00   \n",
       "\n",
       "                                            Strain Description Organism  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_031__1  Escherichia coli K-12 MG1655 rpoBE546V   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine  \\\n",
       "sample_id                          ...                      \n",
       "rpoB__rpoBE546V_031__1         M9  ...              MiSeq   \n",
       "\n",
       "                                          Additional Details  \\\n",
       "sample_id                                                      \n",
       "rpoB__rpoBE546V_031__1  chemostat w/ dilution rate 0.31 h^-1   \n",
       "\n",
       "                       Biological Replicates                       R1  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_031__1                     1  rpoB/SRR1514298_1.fastq   \n",
       "\n",
       "                                             R2                         BAM  \\\n",
       "sample_id                                                                     \n",
       "rpoB__rpoBE546V_031__1  rpoB/SRR1514298_2.fastq  rpoB__rpoBE546V_031__1.bam   \n",
       "\n",
       "                        alignment                                 DOI  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_031__1       98.2  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "rpoB__rpoBE546V_031__1  GSE59377     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE546V_044__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE546V_044</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>8/13/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE546V</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>chemostat w/ dilution rate 0.44 h^-1</td>\n",
       "      <td>1</td>\n",
       "      <td>rpoB/SRR1514299_1.fastq</td>\n",
       "      <td>rpoB/SRR1514299_2.fastq</td>\n",
       "      <td>rpoB__rpoBE546V_044__1.bam</td>\n",
       "      <td>97.57</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_044__1  RpoB Knock-in       rpoB  rpoBE546V_044       1   \n",
       "\n",
       "                             contact library creator        run_date  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE546V_044__1  Jose Utrilla    Jose Utrilla  8/13/2013 0:00   \n",
       "\n",
       "                                            Strain Description Organism  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_044__1  Escherichia coli K-12 MG1655 rpoBE546V   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine  \\\n",
       "sample_id                          ...                      \n",
       "rpoB__rpoBE546V_044__1         M9  ...              MiSeq   \n",
       "\n",
       "                                          Additional Details  \\\n",
       "sample_id                                                      \n",
       "rpoB__rpoBE546V_044__1  chemostat w/ dilution rate 0.44 h^-1   \n",
       "\n",
       "                       Biological Replicates                       R1  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_044__1                     1  rpoB/SRR1514299_1.fastq   \n",
       "\n",
       "                                             R2                         BAM  \\\n",
       "sample_id                                                                     \n",
       "rpoB__rpoBE546V_044__1  rpoB/SRR1514299_2.fastq  rpoB__rpoBE546V_044__1.bam   \n",
       "\n",
       "                        alignment                                 DOI  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_044__1      97.57  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "rpoB__rpoBE546V_044__1  GSE59377     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE546V_glc__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE546V_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/20/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE546V</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>rpoB/SRR1514295_1.fastq</td>\n",
       "      <td>rpoB/SRR1514295_2.fastq</td>\n",
       "      <td>rpoB__rpoBE546V_glc__1.bam</td>\n",
       "      <td>87.63</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_glc__1  RpoB Knock-in       rpoB  rpoBE546V_glc       1   \n",
       "\n",
       "                             contact library creator         run_date  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_glc__1  Jose Utrilla    Jose Utrilla  12/20/2013 0:00   \n",
       "\n",
       "                                            Strain Description Organism  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE546V_glc__1  Escherichia coli K-12 MG1655 rpoBE546V   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                          ...                                         \n",
       "rpoB__rpoBE546V_glc__1         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                       Biological Replicates                       R1  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_glc__1                     1  rpoB/SRR1514295_1.fastq   \n",
       "\n",
       "                                             R2                         BAM  \\\n",
       "sample_id                                                                     \n",
       "rpoB__rpoBE546V_glc__1  rpoB/SRR1514295_2.fastq  rpoB__rpoBE546V_glc__1.bam   \n",
       "\n",
       "                        alignment                                 DOI  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_glc__1      87.63  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "rpoB__rpoBE546V_glc__1  GSE59377     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE546V_lb__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE546V_lb</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE546V</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514296_1.fastq</td>\n",
       "      <td>rpoB/SRR1514296_2.fastq</td>\n",
       "      <td>rpoB__rpoBE546V_lb__1.bam</td>\n",
       "      <td>98.55</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE546V_lb__2</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE546V_lb</td>\n",
       "      <td>2</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE546V</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514297_1.fastq</td>\n",
       "      <td>rpoB/SRR1514297_2.fastq</td>\n",
       "      <td>rpoB__rpoBE546V_lb__2.bam</td>\n",
       "      <td>98.60</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE546V_lb__1  RpoB Knock-in       rpoB  rpoBE546V_lb       1   \n",
       "rpoB__rpoBE546V_lb__2  RpoB Knock-in       rpoB  rpoBE546V_lb       2   \n",
       "\n",
       "                            contact library creator         run_date  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE546V_lb__1  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "rpoB__rpoBE546V_lb__2  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "\n",
       "                                           Strain Description Organism  \\\n",
       "sample_id                                                                \n",
       "rpoB__rpoBE546V_lb__1  Escherichia coli K-12 MG1655 rpoBE546V   MG1655   \n",
       "rpoB__rpoBE546V_lb__2  Escherichia coli K-12 MG1655 rpoBE546V   MG1655   \n",
       "\n",
       "                      Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                         ...                                         \n",
       "rpoB__rpoBE546V_lb__1         LB  ...              MiSeq                NaN   \n",
       "rpoB__rpoBE546V_lb__2         LB  ...              MiSeq                NaN   \n",
       "\n",
       "                      Biological Replicates                       R1  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE546V_lb__1                     2  rpoB/SRR1514296_1.fastq   \n",
       "rpoB__rpoBE546V_lb__2                     2  rpoB/SRR1514297_1.fastq   \n",
       "\n",
       "                                            R2                        BAM  \\\n",
       "sample_id                                                                   \n",
       "rpoB__rpoBE546V_lb__1  rpoB/SRR1514296_2.fastq  rpoB__rpoBE546V_lb__1.bam   \n",
       "rpoB__rpoBE546V_lb__2  rpoB/SRR1514297_2.fastq  rpoB__rpoBE546V_lb__2.bam   \n",
       "\n",
       "                       alignment                                 DOI  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE546V_lb__1      98.55  doi.org/10.1016/j.cels.2016.04.003   \n",
       "rpoB__rpoBE546V_lb__2      98.60  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                            GEO  Public  \n",
       "sample_id                                \n",
       "rpoB__rpoBE546V_lb__1  GSE59377     1.0  \n",
       "rpoB__rpoBE546V_lb__2  GSE59377     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE672K_glc__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE672K_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE672K</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514300_1.fastq</td>\n",
       "      <td>rpoB/SRR1514300_2.fastq</td>\n",
       "      <td>rpoB__rpoBE672K_glc__1.bam</td>\n",
       "      <td>97.14</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE672K_glc__2</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE672K_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE672K</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514301_1.fastq</td>\n",
       "      <td>rpoB/SRR1514301_2.fastq</td>\n",
       "      <td>rpoB__rpoBE672K_glc__2.bam</td>\n",
       "      <td>97.30</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                study project_id   condition_id  rep_id  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE672K_glc__1  RpoB Knock-in       rpoB  rpoBE672K_glc       1   \n",
       "rpoB__rpoBE672K_glc__2  RpoB Knock-in       rpoB  rpoBE672K_glc       2   \n",
       "\n",
       "                             contact library creator         run_date  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE672K_glc__1  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "rpoB__rpoBE672K_glc__2  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "\n",
       "                                            Strain Description Organism  \\\n",
       "sample_id                                                                 \n",
       "rpoB__rpoBE672K_glc__1  Escherichia coli K-12 MG1655 rpoBE672K   MG1655   \n",
       "rpoB__rpoBE672K_glc__2  Escherichia coli K-12 MG1655 rpoBE672K   MG1655   \n",
       "\n",
       "                       Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                          ...                                         \n",
       "rpoB__rpoBE672K_glc__1         M9  ...              MiSeq                NaN   \n",
       "rpoB__rpoBE672K_glc__2         M9  ...              MiSeq                NaN   \n",
       "\n",
       "                       Biological Replicates                       R1  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE672K_glc__1                     2  rpoB/SRR1514300_1.fastq   \n",
       "rpoB__rpoBE672K_glc__2                     2  rpoB/SRR1514301_1.fastq   \n",
       "\n",
       "                                             R2                         BAM  \\\n",
       "sample_id                                                                     \n",
       "rpoB__rpoBE672K_glc__1  rpoB/SRR1514300_2.fastq  rpoB__rpoBE672K_glc__1.bam   \n",
       "rpoB__rpoBE672K_glc__2  rpoB/SRR1514301_2.fastq  rpoB__rpoBE672K_glc__2.bam   \n",
       "\n",
       "                        alignment                                 DOI  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE672K_glc__1      97.14  doi.org/10.1016/j.cels.2016.04.003   \n",
       "rpoB__rpoBE672K_glc__2      97.30  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                             GEO  Public  \n",
       "sample_id                                 \n",
       "rpoB__rpoBE672K_glc__1  GSE59377     1.0  \n",
       "rpoB__rpoBE672K_glc__2  GSE59377     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE672K_lb__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE672K_lb</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE672K</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514302_1.fastq</td>\n",
       "      <td>rpoB/SRR1514302_2.fastq</td>\n",
       "      <td>rpoB__rpoBE672K_lb__1.bam</td>\n",
       "      <td>98.52</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>rpoB__rpoBE672K_lb__2</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>rpoBE672K_lb</td>\n",
       "      <td>2</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 rpoBE672K</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514303_1.fastq</td>\n",
       "      <td>rpoB/SRR1514303_2.fastq</td>\n",
       "      <td>rpoB__rpoBE672K_lb__2.bam</td>\n",
       "      <td>98.56</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                               study project_id  condition_id  rep_id  \\\n",
       "sample_id                                                               \n",
       "rpoB__rpoBE672K_lb__1  RpoB Knock-in       rpoB  rpoBE672K_lb       1   \n",
       "rpoB__rpoBE672K_lb__2  RpoB Knock-in       rpoB  rpoBE672K_lb       2   \n",
       "\n",
       "                            contact library creator         run_date  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE672K_lb__1  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "rpoB__rpoBE672K_lb__2  Jose Utrilla    Jose Utrilla  12/18/2013 0:00   \n",
       "\n",
       "                                           Strain Description Organism  \\\n",
       "sample_id                                                                \n",
       "rpoB__rpoBE672K_lb__1  Escherichia coli K-12 MG1655 rpoBE672K   MG1655   \n",
       "rpoB__rpoBE672K_lb__2  Escherichia coli K-12 MG1655 rpoBE672K   MG1655   \n",
       "\n",
       "                      Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                         ...                                         \n",
       "rpoB__rpoBE672K_lb__1         LB  ...              MiSeq                NaN   \n",
       "rpoB__rpoBE672K_lb__2         LB  ...              MiSeq                NaN   \n",
       "\n",
       "                      Biological Replicates                       R1  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE672K_lb__1                     2  rpoB/SRR1514302_1.fastq   \n",
       "rpoB__rpoBE672K_lb__2                     2  rpoB/SRR1514303_1.fastq   \n",
       "\n",
       "                                            R2                        BAM  \\\n",
       "sample_id                                                                   \n",
       "rpoB__rpoBE672K_lb__1  rpoB/SRR1514302_2.fastq  rpoB__rpoBE672K_lb__1.bam   \n",
       "rpoB__rpoBE672K_lb__2  rpoB/SRR1514303_2.fastq  rpoB__rpoBE672K_lb__2.bam   \n",
       "\n",
       "                       alignment                                 DOI  \\\n",
       "sample_id                                                              \n",
       "rpoB__rpoBE672K_lb__1      98.52  doi.org/10.1016/j.cels.2016.04.003   \n",
       "rpoB__rpoBE672K_lb__2      98.56  doi.org/10.1016/j.cels.2016.04.003   \n",
       "\n",
       "                            GEO  Public  \n",
       "sample_id                                \n",
       "rpoB__rpoBE672K_lb__1  GSE59377     1.0  \n",
       "rpoB__rpoBE672K_lb__2  GSE59377     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__wt_031__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>wt_031</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>8/7/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>chemostat w/ dilution rate 0.31 h^-1</td>\n",
       "      <td>1</td>\n",
       "      <td>rpoB/SRR1514293_1.fastq</td>\n",
       "      <td>rpoB/SRR1514293_2.fastq</td>\n",
       "      <td>rpoB__wt_031__1.bam</td>\n",
       "      <td>98.26</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "rpoB__wt_031__1  RpoB Knock-in       rpoB       wt_031       1  Jose Utrilla   \n",
       "\n",
       "                library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                      \n",
       "rpoB__wt_031__1    Jose Utrilla  8/7/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "rpoB__wt_031__1   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "rpoB__wt_031__1  chemostat w/ dilution rate 0.31 h^-1                     1   \n",
       "\n",
       "                                      R1                       R2  \\\n",
       "sample_id                                                           \n",
       "rpoB__wt_031__1  rpoB/SRR1514293_1.fastq  rpoB/SRR1514293_2.fastq   \n",
       "\n",
       "                                 BAM  alignment  \\\n",
       "sample_id                                         \n",
       "rpoB__wt_031__1  rpoB__wt_031__1.bam      98.26   \n",
       "\n",
       "                                                DOI       GEO  Public  \n",
       "sample_id                                                              \n",
       "rpoB__wt_031__1  doi.org/10.1016/j.cels.2016.04.003  GSE59377     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__wt_044__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>wt_044</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>8/7/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>chemostat w/ dilution rate 0.44 h^-1</td>\n",
       "      <td>1</td>\n",
       "      <td>rpoB/SRR1514294_1.fastq</td>\n",
       "      <td>rpoB/SRR1514294_2.fastq</td>\n",
       "      <td>rpoB__wt_044__1.bam</td>\n",
       "      <td>97.53</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                         study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                      \n",
       "rpoB__wt_044__1  RpoB Knock-in       rpoB       wt_044       1  Jose Utrilla   \n",
       "\n",
       "                library creator       run_date            Strain Description  \\\n",
       "sample_id                                                                      \n",
       "rpoB__wt_044__1    Jose Utrilla  8/7/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "rpoB__wt_044__1   MG1655         M9  ...              MiSeq   \n",
       "\n",
       "                                   Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "rpoB__wt_044__1  chemostat w/ dilution rate 0.44 h^-1                     1   \n",
       "\n",
       "                                      R1                       R2  \\\n",
       "sample_id                                                           \n",
       "rpoB__wt_044__1  rpoB/SRR1514294_1.fastq  rpoB/SRR1514294_2.fastq   \n",
       "\n",
       "                                 BAM  alignment  \\\n",
       "sample_id                                         \n",
       "rpoB__wt_044__1  rpoB__wt_044__1.bam      97.53   \n",
       "\n",
       "                                                DOI       GEO  Public  \n",
       "sample_id                                                              \n",
       "rpoB__wt_044__1  doi.org/10.1016/j.cels.2016.04.003  GSE59377     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>rpoB__wt_lb__1</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>wt_lb</td>\n",
       "      <td>1</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514291_1.fastq</td>\n",
       "      <td>rpoB/SRR1514291_2.fastq</td>\n",
       "      <td>rpoB__wt_lb__1.bam</td>\n",
       "      <td>98.31</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>rpoB__wt_lb__2</td>\n",
       "      <td>RpoB Knock-in</td>\n",
       "      <td>rpoB</td>\n",
       "      <td>wt_lb</td>\n",
       "      <td>2</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>Jose Utrilla</td>\n",
       "      <td>12/18/2013 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>LB</td>\n",
       "      <td>...</td>\n",
       "      <td>MiSeq</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>rpoB/SRR1514292_1.fastq</td>\n",
       "      <td>rpoB/SRR1514292_2.fastq</td>\n",
       "      <td>rpoB__wt_lb__2.bam</td>\n",
       "      <td>98.46</td>\n",
       "      <td>doi.org/10.1016/j.cels.2016.04.003</td>\n",
       "      <td>GSE59377</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                        study project_id condition_id  rep_id       contact  \\\n",
       "sample_id                                                                     \n",
       "rpoB__wt_lb__1  RpoB Knock-in       rpoB        wt_lb       1  Jose Utrilla   \n",
       "rpoB__wt_lb__2  RpoB Knock-in       rpoB        wt_lb       2  Jose Utrilla   \n",
       "\n",
       "               library creator         run_date            Strain Description  \\\n",
       "sample_id                                                                       \n",
       "rpoB__wt_lb__1    Jose Utrilla  12/18/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "rpoB__wt_lb__2    Jose Utrilla  12/18/2013 0:00  Escherichia coli K-12 MG1655   \n",
       "\n",
       "               Organism Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                           ...                                         \n",
       "rpoB__wt_lb__1   MG1655         LB  ...              MiSeq                NaN   \n",
       "rpoB__wt_lb__2   MG1655         LB  ...              MiSeq                NaN   \n",
       "\n",
       "               Biological Replicates                       R1  \\\n",
       "sample_id                                                       \n",
       "rpoB__wt_lb__1                     2  rpoB/SRR1514291_1.fastq   \n",
       "rpoB__wt_lb__2                     2  rpoB/SRR1514292_1.fastq   \n",
       "\n",
       "                                     R2                 BAM  alignment  \\\n",
       "sample_id                                                                \n",
       "rpoB__wt_lb__1  rpoB/SRR1514291_2.fastq  rpoB__wt_lb__1.bam      98.31   \n",
       "rpoB__wt_lb__2  rpoB/SRR1514292_2.fastq  rpoB__wt_lb__2.bam      98.46   \n",
       "\n",
       "                                               DOI       GEO  Public  \n",
       "sample_id                                                             \n",
       "rpoB__wt_lb__1  doi.org/10.1016/j.cels.2016.04.003  GSE59377     1.0  \n",
       "rpoB__wt_lb__2  doi.org/10.1016/j.cels.2016.04.003  GSE59377     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ac_ac2__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ac_ac2</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_AC.9.92.4</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU AC A9 F92 I4</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ac_ac2_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ac_ac2_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ac_ac2__1.bam</td>\n",
       "      <td>98.1</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                            study project_id condition_id  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_ac_ac2__1  Substrate-switching Evolution        ssw   glc_ac_ac2   \n",
       "\n",
       "                    rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_ac_ac2__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                    Strain Description Organism Base Media  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_ac_ac2__1  Escherichia coli SSW_GLU_AC.9.92.4   MG1655         M9   \n",
       "\n",
       "                    ... Sequencing Machine    Additional Details  \\\n",
       "sample_id           ...                                            \n",
       "ssw__glc_ac_ac2__1  ...         HiSeq 2000  SSW GLU AC A9 F92 I4   \n",
       "\n",
       "                   Biological Replicates                          R1  \\\n",
       "sample_id                                                              \n",
       "ssw__glc_ac_ac2__1                     1  ssw/glc_ac_ac2_R1.fastq.gz   \n",
       "\n",
       "                                            R2                     BAM  \\\n",
       "sample_id                                                                \n",
       "ssw__glc_ac_ac2__1  ssw/glc_ac_ac2_R2.fastq.gz  ssw__glc_ac_ac2__1.bam   \n",
       "\n",
       "                    alignment                           DOI       GEO  Public  \n",
       "sample_id                                                                      \n",
       "ssw__glc_ac_ac2__1       98.1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ac_glc1__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ac_glc1</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_AC.9.92.3</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU AC A9 F92 I3</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ac_glc1_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ac_glc1_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ac_glc1__1.bam</td>\n",
       "      <td>56.96</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             study project_id condition_id  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_ac_glc1__1  Substrate-switching Evolution        ssw  glc_ac_glc1   \n",
       "\n",
       "                     rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_ac_glc1__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                     Strain Description Organism Base Media  \\\n",
       "sample_id                                                                     \n",
       "ssw__glc_ac_glc1__1  Escherichia coli SSW_GLU_AC.9.92.3   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine    Additional Details  \\\n",
       "sample_id            ...                                            \n",
       "ssw__glc_ac_glc1__1  ...         HiSeq 2000  SSW GLU AC A9 F92 I3   \n",
       "\n",
       "                    Biological Replicates                           R1  \\\n",
       "sample_id                                                                \n",
       "ssw__glc_ac_glc1__1                     1  ssw/glc_ac_glc1_R1.fastq.gz   \n",
       "\n",
       "                                              R2                      BAM  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_ac_glc1__1  ssw/glc_ac_glc1_R2.fastq.gz  ssw__glc_ac_glc1__1.bam   \n",
       "\n",
       "                     alignment                           DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "ssw__glc_ac_glc1__1      56.96  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ac_glc2__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ac_glc2</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_AC.9.92.4</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU AC A9 F92 I4</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ac_glc2_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ac_glc2_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ac_glc2__1.bam</td>\n",
       "      <td>98.06</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             study project_id condition_id  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_ac_glc2__1  Substrate-switching Evolution        ssw  glc_ac_glc2   \n",
       "\n",
       "                     rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_ac_glc2__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                     Strain Description Organism Base Media  \\\n",
       "sample_id                                                                     \n",
       "ssw__glc_ac_glc2__1  Escherichia coli SSW_GLU_AC.9.92.4   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine    Additional Details  \\\n",
       "sample_id            ...                                            \n",
       "ssw__glc_ac_glc2__1  ...         HiSeq 2000  SSW GLU AC A9 F92 I4   \n",
       "\n",
       "                    Biological Replicates                           R1  \\\n",
       "sample_id                                                                \n",
       "ssw__glc_ac_glc2__1                     1  ssw/glc_ac_glc2_R1.fastq.gz   \n",
       "\n",
       "                                              R2                      BAM  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_ac_glc2__1  ssw/glc_ac_glc2_R2.fastq.gz  ssw__glc_ac_glc2__1.bam   \n",
       "\n",
       "                     alignment                           DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "ssw__glc_ac_glc2__1      98.06  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ale1__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli C13.1.133.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>C13 A1 F133 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ale1_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ale1_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ale1__1.bam</td>\n",
       "      <td>55.43</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale1__1  Substrate-switching Evolution        ssw     glc_ale1   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale1__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ssw__glc_ale1__1  Escherichia coli C13.1.133.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__glc_ale1__1         HiSeq 2000     C13 A1 F133 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__glc_ale1__1  ssw/glc_ale1_R1.fastq.gz  ssw/glc_ale1_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__glc_ale1__1  ssw__glc_ale1__1.bam      55.43   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__glc_ale1__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ale2__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli C13.1.134.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>C13 A1 F134 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ale2_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ale2_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ale2__1.bam</td>\n",
       "      <td>60.57</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale2__1  Substrate-switching Evolution        ssw     glc_ale2   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale2__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ssw__glc_ale2__1  Escherichia coli C13.1.134.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__glc_ale2__1         HiSeq 2000     C13 A1 F134 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__glc_ale2__1  ssw/glc_ale2_R1.fastq.gz  ssw/glc_ale2_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__glc_ale2__1  ssw__glc_ale2__1.bam      60.57   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__glc_ale2__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_ale3__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli C13.6.133.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>C13 A6 F133 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_ale3_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_ale3_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_ale3__1.bam</td>\n",
       "      <td>59.6</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale3__1  Substrate-switching Evolution        ssw     glc_ale3   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_ale3__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                            Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                           ...   \n",
       "ssw__glc_ale3__1  Escherichia coli C13.6.133.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__glc_ale3__1         HiSeq 2000     C13 A6 F133 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__glc_ale3__1  ssw/glc_ale3_R1.fastq.gz  ssw/glc_ale3_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__glc_ale3__1  ssw__glc_ale3__1.bam       59.6   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__glc_ale3__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_glyc_glc__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_glyc_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_GLY.4.165.2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU GLY A4 F165 I2</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_glyc_glc_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_glyc_glc_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_glyc_glc__1.bam</td>\n",
       "      <td>98.34</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                              study project_id  condition_id  \\\n",
       "sample_id                                                                      \n",
       "ssw__glc_glyc_glc__1  Substrate-switching Evolution        ssw  glc_glyc_glc   \n",
       "\n",
       "                      rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                     \n",
       "ssw__glc_glyc_glc__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                        Strain Description Organism  \\\n",
       "sample_id                                                             \n",
       "ssw__glc_glyc_glc__1  Escherichia coli SSW_GLU_GLY.4.165.2   MG1655   \n",
       "\n",
       "                     Base Media  ... Sequencing Machine  \\\n",
       "sample_id                        ...                      \n",
       "ssw__glc_glyc_glc__1         M9  ...         HiSeq 2000   \n",
       "\n",
       "                          Additional Details Biological Replicates  \\\n",
       "sample_id                                                            \n",
       "ssw__glc_glyc_glc__1  SSW GLU GLY A4 F165 I2                     1   \n",
       "\n",
       "                                                R1  \\\n",
       "sample_id                                            \n",
       "ssw__glc_glyc_glc__1  ssw/glc_glyc_glc_R1.fastq.gz   \n",
       "\n",
       "                                                R2                       BAM  \\\n",
       "sample_id                                                                      \n",
       "ssw__glc_glyc_glc__1  ssw/glc_glyc_glc_R2.fastq.gz  ssw__glc_glyc_glc__1.bam   \n",
       "\n",
       "                      alignment                           DOI       GEO  \\\n",
       "sample_id                                                                 \n",
       "ssw__glc_glyc_glc__1      98.34  doi.org/10.1128/AEM.00410-17  GSE97944   \n",
       "\n",
       "                      Public  \n",
       "sample_id                     \n",
       "ssw__glc_glyc_glc__1     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_glyc_glyc__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_glyc_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_GLY.4.165.2</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU GLY A4 F165 I2</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_glyc_glyc_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_glyc_glyc_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_glyc_glyc__1.bam</td>\n",
       "      <td>97.8</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               study project_id  \\\n",
       "sample_id                                                         \n",
       "ssw__glc_glyc_glyc__1  Substrate-switching Evolution        ssw   \n",
       "\n",
       "                        condition_id  rep_id        contact library creator  \\\n",
       "sample_id                                                                     \n",
       "ssw__glc_glyc_glyc__1  glc_glyc_glyc       1  Troy Sandberg     Ying Hefner   \n",
       "\n",
       "                             run_date                    Strain Description  \\\n",
       "sample_id                                                                     \n",
       "ssw__glc_glyc_glyc__1  4/18/2017 0:00  Escherichia coli SSW_GLU_GLY.4.165.2   \n",
       "\n",
       "                      Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                                  ...                      \n",
       "ssw__glc_glyc_glyc__1   MG1655         M9  ...         HiSeq 2000   \n",
       "\n",
       "                           Additional Details Biological Replicates  \\\n",
       "sample_id                                                             \n",
       "ssw__glc_glyc_glyc__1  SSW GLU GLY A4 F165 I2                     1   \n",
       "\n",
       "                                                  R1  \\\n",
       "sample_id                                              \n",
       "ssw__glc_glyc_glyc__1  ssw/glc_glyc_glyc_R1.fastq.gz   \n",
       "\n",
       "                                                  R2  \\\n",
       "sample_id                                              \n",
       "ssw__glc_glyc_glyc__1  ssw/glc_glyc_glyc_R2.fastq.gz   \n",
       "\n",
       "                                             BAM  alignment  \\\n",
       "sample_id                                                     \n",
       "ssw__glc_glyc_glyc__1  ssw__glc_glyc_glyc__1.bam       97.8   \n",
       "\n",
       "                                                DOI       GEO  Public  \n",
       "sample_id                                                              \n",
       "ssw__glc_glyc_glyc__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_xyl_glc__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_xyl_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_XYL.2.165.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU GLY A2 F165 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_xyl_glc_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_xyl_glc_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_xyl_glc__1.bam</td>\n",
       "      <td>98.26</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             study project_id condition_id  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_xyl_glc__1  Substrate-switching Evolution        ssw  glc_xyl_glc   \n",
       "\n",
       "                     rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_xyl_glc__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "ssw__glc_xyl_glc__1  Escherichia coli SSW_GLU_XYL.2.165.1   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine      Additional Details  \\\n",
       "sample_id            ...                                              \n",
       "ssw__glc_xyl_glc__1  ...         HiSeq 2000  SSW GLU GLY A2 F165 I1   \n",
       "\n",
       "                    Biological Replicates                           R1  \\\n",
       "sample_id                                                                \n",
       "ssw__glc_xyl_glc__1                     1  ssw/glc_xyl_glc_R1.fastq.gz   \n",
       "\n",
       "                                              R2                      BAM  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_xyl_glc__1  ssw/glc_xyl_glc_R2.fastq.gz  ssw__glc_xyl_glc__1.bam   \n",
       "\n",
       "                     alignment                           DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "ssw__glc_xyl_glc__1      98.26  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glc_xyl_xyl__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glc_xyl_xyl</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLU_XYL.2.165.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLU XYL A2 F165 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glc_xyl_xyl_R1.fastq.gz</td>\n",
       "      <td>ssw/glc_xyl_xyl_R2.fastq.gz</td>\n",
       "      <td>ssw__glc_xyl_xyl__1.bam</td>\n",
       "      <td>98.13</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                             study project_id condition_id  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_xyl_xyl__1  Substrate-switching Evolution        ssw  glc_xyl_xyl   \n",
       "\n",
       "                     rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                    \n",
       "ssw__glc_xyl_xyl__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                       Strain Description Organism Base Media  \\\n",
       "sample_id                                                                       \n",
       "ssw__glc_xyl_xyl__1  Escherichia coli SSW_GLU_XYL.2.165.1   MG1655         M9   \n",
       "\n",
       "                     ... Sequencing Machine      Additional Details  \\\n",
       "sample_id            ...                                              \n",
       "ssw__glc_xyl_xyl__1  ...         HiSeq 2000  SSW GLU XYL A2 F165 I1   \n",
       "\n",
       "                    Biological Replicates                           R1  \\\n",
       "sample_id                                                                \n",
       "ssw__glc_xyl_xyl__1                     1  ssw/glc_xyl_xyl_R1.fastq.gz   \n",
       "\n",
       "                                              R2                      BAM  \\\n",
       "sample_id                                                                   \n",
       "ssw__glc_xyl_xyl__1  ssw/glc_xyl_xyl_R2.fastq.gz  ssw__glc_xyl_xyl__1.bam   \n",
       "\n",
       "                     alignment                           DOI       GEO  Public  \n",
       "sample_id                                                                       \n",
       "ssw__glc_xyl_xyl__1      98.13  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glyc_ale1__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glyc_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLY.1.45.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLY A1 F45 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glyc_ale1_R1.fastq.gz</td>\n",
       "      <td>ssw/glyc_ale1_R2.fastq.gz</td>\n",
       "      <td>ssw__glyc_ale1__1.bam</td>\n",
       "      <td>58.83</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                           study project_id condition_id  \\\n",
       "sample_id                                                                  \n",
       "ssw__glyc_ale1__1  Substrate-switching Evolution        ssw    glyc_ale1   \n",
       "\n",
       "                   rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "ssw__glyc_ale1__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ssw__glyc_ale1__1  Escherichia coli SSW_GLY.1.45.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "ssw__glyc_ale1__1         HiSeq 2000  SSW GLY A1 F45 I1                     1   \n",
       "\n",
       "                                          R1                         R2  \\\n",
       "sample_id                                                                 \n",
       "ssw__glyc_ale1__1  ssw/glyc_ale1_R1.fastq.gz  ssw/glyc_ale1_R2.fastq.gz   \n",
       "\n",
       "                                     BAM  alignment  \\\n",
       "sample_id                                             \n",
       "ssw__glyc_ale1__1  ssw__glyc_ale1__1.bam      58.83   \n",
       "\n",
       "                                            DOI       GEO  Public  \n",
       "sample_id                                                          \n",
       "ssw__glyc_ale1__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__glyc_ale2__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>glyc_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_GLY.2.45.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW GLY A2 F45 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/glyc_ale2_R1.fastq.gz</td>\n",
       "      <td>ssw/glyc_ale2_R2.fastq.gz</td>\n",
       "      <td>ssw__glyc_ale2__1.bam</td>\n",
       "      <td>57.67</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                           study project_id condition_id  \\\n",
       "sample_id                                                                  \n",
       "ssw__glyc_ale2__1  Substrate-switching Evolution        ssw    glyc_ale2   \n",
       "\n",
       "                   rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                  \n",
       "ssw__glyc_ale2__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                                Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                               ...   \n",
       "ssw__glyc_ale2__1  Escherichia coli SSW_GLY.2.45.1   MG1655         M9  ...   \n",
       "\n",
       "                  Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                       \n",
       "ssw__glyc_ale2__1         HiSeq 2000  SSW GLY A2 F45 I1                     1   \n",
       "\n",
       "                                          R1                         R2  \\\n",
       "sample_id                                                                 \n",
       "ssw__glyc_ale2__1  ssw/glyc_ale2_R1.fastq.gz  ssw/glyc_ale2_R2.fastq.gz   \n",
       "\n",
       "                                     BAM  alignment  \\\n",
       "sample_id                                             \n",
       "ssw__glyc_ale2__1  ssw__glyc_ale2__1.bam      57.67   \n",
       "\n",
       "                                            DOI       GEO  Public  \n",
       "sample_id                                                          \n",
       "ssw__glyc_ale2__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__wt_ac__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>wt_ac</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/wt_ac_R1.fastq.gz</td>\n",
       "      <td>ssw/wt_ac_R2.fastq.gz</td>\n",
       "      <td>ssw__wt_ac__1.bam</td>\n",
       "      <td>98.19</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                       study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                      \n",
       "ssw__wt_ac__1  Substrate-switching Evolution        ssw        wt_ac       1   \n",
       "\n",
       "                     contact library creator        run_date  \\\n",
       "sample_id                                                      \n",
       "ssw__wt_ac__1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                         Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                        ...   \n",
       "ssw__wt_ac__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "              Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                   \n",
       "ssw__wt_ac__1         HiSeq 2000                NaN                     1   \n",
       "\n",
       "                                  R1                     R2  \\\n",
       "sample_id                                                     \n",
       "ssw__wt_ac__1  ssw/wt_ac_R1.fastq.gz  ssw/wt_ac_R2.fastq.gz   \n",
       "\n",
       "                             BAM  alignment                           DOI  \\\n",
       "sample_id                                                                   \n",
       "ssw__wt_ac__1  ssw__wt_ac__1.bam      98.19  doi.org/10.1128/AEM.00410-17   \n",
       "\n",
       "                    GEO  Public  \n",
       "sample_id                        \n",
       "ssw__wt_ac__1  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__wt_glc__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/wt_glc_R1.fastq.gz</td>\n",
       "      <td>ssw/wt_glc_R2.fastq.gz</td>\n",
       "      <td>ssw__wt_glc__1.bam</td>\n",
       "      <td>95.37</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                       \n",
       "ssw__wt_glc__1  Substrate-switching Evolution        ssw       wt_glc       1   \n",
       "\n",
       "                      contact library creator        run_date  \\\n",
       "sample_id                                                       \n",
       "ssw__wt_glc__1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                          Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                         ...   \n",
       "ssw__wt_glc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "               Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                    \n",
       "ssw__wt_glc__1         HiSeq 2000                NaN                     1   \n",
       "\n",
       "                                    R1                      R2  \\\n",
       "sample_id                                                        \n",
       "ssw__wt_glc__1  ssw/wt_glc_R1.fastq.gz  ssw/wt_glc_R2.fastq.gz   \n",
       "\n",
       "                               BAM  alignment                           DOI  \\\n",
       "sample_id                                                                     \n",
       "ssw__wt_glc__1  ssw__wt_glc__1.bam      95.37  doi.org/10.1128/AEM.00410-17   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ssw__wt_glc__1  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__wt_glyc__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>wt_glyc</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/wt_glyc_R1.fastq.gz</td>\n",
       "      <td>ssw/wt_glyc_R2.fastq.gz</td>\n",
       "      <td>ssw__wt_glyc__1.bam</td>\n",
       "      <td>98.01</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                         study project_id condition_id  \\\n",
       "sample_id                                                                \n",
       "ssw__wt_glyc__1  Substrate-switching Evolution        ssw      wt_glyc   \n",
       "\n",
       "                 rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                \n",
       "ssw__wt_glyc__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                           Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                          ...   \n",
       "ssw__wt_glyc__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "                Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                     \n",
       "ssw__wt_glyc__1         HiSeq 2000                NaN                     1   \n",
       "\n",
       "                                      R1                       R2  \\\n",
       "sample_id                                                           \n",
       "ssw__wt_glyc__1  ssw/wt_glyc_R1.fastq.gz  ssw/wt_glyc_R2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                           DOI  \\\n",
       "sample_id                                                                       \n",
       "ssw__wt_glyc__1  ssw__wt_glyc__1.bam      98.01  doi.org/10.1128/AEM.00410-17   \n",
       "\n",
       "                      GEO  Public  \n",
       "sample_id                          \n",
       "ssw__wt_glyc__1  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__wt_xyl__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>wt_xyl</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/wt_xyl_R1.fastq.gz</td>\n",
       "      <td>ssw/wt_xyl_R2.fastq.gz</td>\n",
       "      <td>ssw__wt_xyl__1.bam</td>\n",
       "      <td>97.99</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        study project_id condition_id  rep_id  \\\n",
       "sample_id                                                                       \n",
       "ssw__wt_xyl__1  Substrate-switching Evolution        ssw       wt_xyl       1   \n",
       "\n",
       "                      contact library creator        run_date  \\\n",
       "sample_id                                                       \n",
       "ssw__wt_xyl__1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                          Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                         ...   \n",
       "ssw__wt_xyl__1  Escherichia coli K-12 MG1655   MG1655         M9  ...   \n",
       "\n",
       "               Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                    \n",
       "ssw__wt_xyl__1         HiSeq 2000                NaN                     1   \n",
       "\n",
       "                                    R1                      R2  \\\n",
       "sample_id                                                        \n",
       "ssw__wt_xyl__1  ssw/wt_xyl_R1.fastq.gz  ssw/wt_xyl_R2.fastq.gz   \n",
       "\n",
       "                               BAM  alignment                           DOI  \\\n",
       "sample_id                                                                     \n",
       "ssw__wt_xyl__1  ssw__wt_xyl__1.bam      97.99  doi.org/10.1128/AEM.00410-17   \n",
       "\n",
       "                     GEO  Public  \n",
       "sample_id                         \n",
       "ssw__wt_xyl__1  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__xyl_ale1__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>xyl_ale1</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_XYL.1.20.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW XYL A1 F20 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/xyl_ale1_R1.fastq.gz</td>\n",
       "      <td>ssw/xyl_ale1_R2.fastq.gz</td>\n",
       "      <td>ssw__xyl_ale1__1.bam</td>\n",
       "      <td>98.15</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale1__1  Substrate-switching Evolution        ssw     xyl_ale1   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale1__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "ssw__xyl_ale1__1  Escherichia coli SSW_XYL.1.20.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__xyl_ale1__1         HiSeq 2000  SSW XYL A1 F20 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__xyl_ale1__1  ssw/xyl_ale1_R1.fastq.gz  ssw/xyl_ale1_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__xyl_ale1__1  ssw__xyl_ale1__1.bam      98.15   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__xyl_ale1__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__xyl_ale2__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>xyl_ale2</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_XYL.2.20.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW XYL A2 F20 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/xyl_ale2_R1.fastq.gz</td>\n",
       "      <td>ssw/xyl_ale2_R2.fastq.gz</td>\n",
       "      <td>ssw__xyl_ale2__1.bam</td>\n",
       "      <td>98.31</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale2__1  Substrate-switching Evolution        ssw     xyl_ale2   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale2__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "ssw__xyl_ale2__1  Escherichia coli SSW_XYL.2.20.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__xyl_ale2__1         HiSeq 2000  SSW XYL A2 F20 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__xyl_ale2__1  ssw/xyl_ale2_R1.fastq.gz  ssw/xyl_ale2_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__xyl_ale2__1  ssw__xyl_ale2__1.bam      98.31   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__xyl_ale2__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ssw__xyl_ale3__1</td>\n",
       "      <td>Substrate-switching Evolution</td>\n",
       "      <td>ssw</td>\n",
       "      <td>xyl_ale3</td>\n",
       "      <td>1</td>\n",
       "      <td>Troy Sandberg</td>\n",
       "      <td>Ying Hefner</td>\n",
       "      <td>4/18/2017 0:00</td>\n",
       "      <td>Escherichia coli SSW_XYL.3.20.1</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 2000</td>\n",
       "      <td>SSW XYL A3 F20 I1</td>\n",
       "      <td>1</td>\n",
       "      <td>ssw/xyl_ale3_R1.fastq.gz</td>\n",
       "      <td>ssw/xyl_ale3_R2.fastq.gz</td>\n",
       "      <td>ssw__xyl_ale3__1.bam</td>\n",
       "      <td>98.06</td>\n",
       "      <td>doi.org/10.1128/AEM.00410-17</td>\n",
       "      <td>GSE97944</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          study project_id condition_id  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale3__1  Substrate-switching Evolution        ssw     xyl_ale3   \n",
       "\n",
       "                  rep_id        contact library creator        run_date  \\\n",
       "sample_id                                                                 \n",
       "ssw__xyl_ale3__1       1  Troy Sandberg     Ying Hefner  4/18/2017 0:00   \n",
       "\n",
       "                               Strain Description Organism Base Media  ...  \\\n",
       "sample_id                                                              ...   \n",
       "ssw__xyl_ale3__1  Escherichia coli SSW_XYL.3.20.1   MG1655         M9  ...   \n",
       "\n",
       "                 Sequencing Machine Additional Details Biological Replicates  \\\n",
       "sample_id                                                                      \n",
       "ssw__xyl_ale3__1         HiSeq 2000  SSW XYL A3 F20 I1                     1   \n",
       "\n",
       "                                        R1                        R2  \\\n",
       "sample_id                                                              \n",
       "ssw__xyl_ale3__1  ssw/xyl_ale3_R1.fastq.gz  ssw/xyl_ale3_R2.fastq.gz   \n",
       "\n",
       "                                   BAM  alignment  \\\n",
       "sample_id                                           \n",
       "ssw__xyl_ale3__1  ssw__xyl_ale3__1.bam      98.06   \n",
       "\n",
       "                                           DOI       GEO  Public  \n",
       "sample_id                                                         \n",
       "ssw__xyl_ale3__1  doi.org/10.1128/AEM.00410-17  GSE97944     1.0  \n",
       "\n",
       "[1 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyafC__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyafC</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yafC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6781999_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6781999_2.fastq.gz</td>\n",
       "      <td>ytf__delyafC__1.bam</td>\n",
       "      <td>94.86</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyafC__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyafC</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yafC</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782000_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782000_2.fastq.gz</td>\n",
       "      <td>ytf__delyafC__2.bam</td>\n",
       "      <td>95.25</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyafC__1   yTF        ytf      delyafC       1  Ye Gao          Ye Gao   \n",
       "ytf__delyafC__2   yTF        ytf      delyafC       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyafC__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yafC   \n",
       "ytf__delyafC__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yafC   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyafC__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyafC__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyafC__1                NaN                     2   \n",
       "ytf__delyafC__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyafC__1  ytf/SRR6781999_1.fastq.gz  ytf/SRR6781999_2.fastq.gz   \n",
       "ytf__delyafC__2  ytf/SRR6782000_1.fastq.gz  ytf/SRR6782000_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyafC__1  ytf__delyafC__1.bam      94.86  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyafC__2  ytf__delyafC__2.bam      95.25  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyafC__1  GSE111094     1.0  \n",
       "ytf__delyafC__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delybaO__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybaO</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybaO</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057989_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057989_2.fastq.gz</td>\n",
       "      <td>ytf__delybaO__1.bam</td>\n",
       "      <td>94.71</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delybaO__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybaO</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybaO</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057990_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057990_2.fastq.gz</td>\n",
       "      <td>ytf__delybaO__2.bam</td>\n",
       "      <td>95.01</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delybaO__1   yTF        ytf      delybaO       1  Ye Gao          Ye Gao   \n",
       "ytf__delybaO__2   yTF        ytf      delybaO       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delybaO__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybaO   \n",
       "ytf__delybaO__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybaO   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delybaO__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delybaO__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delybaO__1                NaN                     2   \n",
       "ytf__delybaO__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delybaO__1  ytf/SRR7057989_1.fastq.gz  ytf/SRR7057989_2.fastq.gz   \n",
       "ytf__delybaO__2  ytf/SRR7057990_1.fastq.gz  ytf/SRR7057990_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delybaO__1  ytf__delybaO__1.bam      94.71  doi.org/10.1093/nar/gky752   \n",
       "ytf__delybaO__2  ytf__delybaO__2.bam      95.01  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delybaO__1  GSE111094     1.0  \n",
       "ytf__delybaO__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delybaQ__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybaQ</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybaQ</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057991_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057991_2.fastq.gz</td>\n",
       "      <td>ytf__delybaQ__1.bam</td>\n",
       "      <td>94.76</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delybaQ__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybaQ</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybaQ</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057992_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057992_2.fastq.gz</td>\n",
       "      <td>ytf__delybaQ__2.bam</td>\n",
       "      <td>94.63</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delybaQ__1   yTF        ytf      delybaQ       1  Ye Gao          Ye Gao   \n",
       "ytf__delybaQ__2   yTF        ytf      delybaQ       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delybaQ__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybaQ   \n",
       "ytf__delybaQ__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybaQ   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delybaQ__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delybaQ__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delybaQ__1                NaN                     2   \n",
       "ytf__delybaQ__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delybaQ__1  ytf/SRR7057991_1.fastq.gz  ytf/SRR7057991_2.fastq.gz   \n",
       "ytf__delybaQ__2  ytf/SRR7057992_1.fastq.gz  ytf/SRR7057992_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delybaQ__1  ytf__delybaQ__1.bam      94.76  doi.org/10.1093/nar/gky752   \n",
       "ytf__delybaQ__2  ytf__delybaQ__2.bam      94.63  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delybaQ__1  GSE111094     1.0  \n",
       "ytf__delybaQ__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delybiH__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybiH</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybiH</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057993_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057993_2.fastq.gz</td>\n",
       "      <td>ytf__delybiH__1.bam</td>\n",
       "      <td>96.35</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delybiH__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delybiH</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ybiH</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057994_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057994_2.fastq.gz</td>\n",
       "      <td>ytf__delybiH__2.bam</td>\n",
       "      <td>95.24</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delybiH__1   yTF        ytf      delybiH       1  Ye Gao          Ye Gao   \n",
       "ytf__delybiH__2   yTF        ytf      delybiH       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delybiH__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybiH   \n",
       "ytf__delybiH__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_ybiH   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delybiH__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delybiH__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delybiH__1                NaN                     2   \n",
       "ytf__delybiH__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delybiH__1  ytf/SRR7057993_1.fastq.gz  ytf/SRR7057993_2.fastq.gz   \n",
       "ytf__delybiH__2  ytf/SRR7057994_1.fastq.gz  ytf/SRR7057994_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delybiH__1  ytf__delybiH__1.bam      96.35  doi.org/10.1093/nar/gky752   \n",
       "ytf__delybiH__2  ytf__delybiH__2.bam      95.24  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delybiH__1  GSE111094     1.0  \n",
       "ytf__delybiH__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delydcI_ph5__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delydcI_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ydcI</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057995_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057995_2.fastq.gz</td>\n",
       "      <td>ytf__delydcI_ph5__1.bam</td>\n",
       "      <td>94.20</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delydcI_ph5__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delydcI_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ydcI</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057996_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057996_2.fastq.gz</td>\n",
       "      <td>ytf__delydcI_ph5__2.bam</td>\n",
       "      <td>94.54</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                    study project_id condition_id  rep_id contact  \\\n",
       "sample_id                                                           \n",
       "ytf__delydcI_ph5__1   yTF        ytf  delydcI_ph5       1  Ye Gao   \n",
       "ytf__delydcI_ph5__2   yTF        ytf  delydcI_ph5       2  Ye Gao   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "ytf__delydcI_ph5__1          Ye Gao  10/2/2017 0:00   \n",
       "ytf__delydcI_ph5__2          Ye Gao  10/2/2017 0:00   \n",
       "\n",
       "                                        Strain Description Organism  \\\n",
       "sample_id                                                             \n",
       "ytf__delydcI_ph5__1  Escherichia coli K-12 MG1655 del_ydcI   MG1655   \n",
       "ytf__delydcI_ph5__2  Escherichia coli K-12 MG1655 del_ydcI   MG1655   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "ytf__delydcI_ph5__1         M9  ...         HiSeq 4000                NaN   \n",
       "ytf__delydcI_ph5__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "ytf__delydcI_ph5__1                     2  ytf/SRR7057995_1.fastq.gz   \n",
       "ytf__delydcI_ph5__2                     2  ytf/SRR7057996_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "ytf__delydcI_ph5__1  ytf/SRR7057995_2.fastq.gz  ytf__delydcI_ph5__1.bam   \n",
       "ytf__delydcI_ph5__2  ytf/SRR7057996_2.fastq.gz  ytf__delydcI_ph5__2.bam   \n",
       "\n",
       "                     alignment                         DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "ytf__delydcI_ph5__1      94.20  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "ytf__delydcI_ph5__2      94.54  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delydcI_ph8__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delydcI_ph8</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ydcI</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057997_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057997_2.fastq.gz</td>\n",
       "      <td>ytf__delydcI_ph8__1.bam</td>\n",
       "      <td>95.15</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delydcI_ph8__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delydcI_ph8</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_ydcI</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057998_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057998_2.fastq.gz</td>\n",
       "      <td>ytf__delydcI_ph8__2.bam</td>\n",
       "      <td>95.50</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                    study project_id condition_id  rep_id contact  \\\n",
       "sample_id                                                           \n",
       "ytf__delydcI_ph8__1   yTF        ytf  delydcI_ph8       1  Ye Gao   \n",
       "ytf__delydcI_ph8__2   yTF        ytf  delydcI_ph8       2  Ye Gao   \n",
       "\n",
       "                    library creator        run_date  \\\n",
       "sample_id                                             \n",
       "ytf__delydcI_ph8__1          Ye Gao  10/2/2017 0:00   \n",
       "ytf__delydcI_ph8__2          Ye Gao  10/2/2017 0:00   \n",
       "\n",
       "                                        Strain Description Organism  \\\n",
       "sample_id                                                             \n",
       "ytf__delydcI_ph8__1  Escherichia coli K-12 MG1655 del_ydcI   MG1655   \n",
       "ytf__delydcI_ph8__2  Escherichia coli K-12 MG1655 del_ydcI   MG1655   \n",
       "\n",
       "                    Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                       ...                                         \n",
       "ytf__delydcI_ph8__1         M9  ...         HiSeq 4000                NaN   \n",
       "ytf__delydcI_ph8__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "                    Biological Replicates                         R1  \\\n",
       "sample_id                                                              \n",
       "ytf__delydcI_ph8__1                     2  ytf/SRR7057997_1.fastq.gz   \n",
       "ytf__delydcI_ph8__2                     2  ytf/SRR7057998_1.fastq.gz   \n",
       "\n",
       "                                            R2                      BAM  \\\n",
       "sample_id                                                                 \n",
       "ytf__delydcI_ph8__1  ytf/SRR7057997_2.fastq.gz  ytf__delydcI_ph8__1.bam   \n",
       "ytf__delydcI_ph8__2  ytf/SRR7057998_2.fastq.gz  ytf__delydcI_ph8__2.bam   \n",
       "\n",
       "                     alignment                         DOI        GEO  Public  \n",
       "sample_id                                                                      \n",
       "ytf__delydcI_ph8__1      95.15  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "ytf__delydcI_ph8__2      95.50  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyddM__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyddM</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yddM</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057999_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057999_2.fastq.gz</td>\n",
       "      <td>ytf__delyddM__1.bam</td>\n",
       "      <td>95.08</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyddM__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyddM</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yddM</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7058000_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7058000_2.fastq.gz</td>\n",
       "      <td>ytf__delyddM__2.bam</td>\n",
       "      <td>95.41</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyddM__1   yTF        ytf      delyddM       1  Ye Gao          Ye Gao   \n",
       "ytf__delyddM__2   yTF        ytf      delyddM       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyddM__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yddM   \n",
       "ytf__delyddM__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yddM   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyddM__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyddM__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyddM__1                NaN                     2   \n",
       "ytf__delyddM__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyddM__1  ytf/SRR7057999_1.fastq.gz  ytf/SRR7057999_2.fastq.gz   \n",
       "ytf__delyddM__2  ytf/SRR7058000_1.fastq.gz  ytf/SRR7058000_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyddM__1  ytf__delyddM__1.bam      95.08  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyddM__2  ytf__delyddM__2.bam      95.41  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyddM__1  GSE111094     1.0  \n",
       "ytf__delyddM__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyeiE__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyeiE</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yeiE</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782001_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782001_2.fastq.gz</td>\n",
       "      <td>ytf__delyeiE__1.bam</td>\n",
       "      <td>95.52</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyeiE__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyeiE</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yeiE</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782002_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782002_2.fastq.gz</td>\n",
       "      <td>ytf__delyeiE__2.bam</td>\n",
       "      <td>95.95</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyeiE__1   yTF        ytf      delyeiE       1  Ye Gao          Ye Gao   \n",
       "ytf__delyeiE__2   yTF        ytf      delyeiE       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyeiE__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yeiE   \n",
       "ytf__delyeiE__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yeiE   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyeiE__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyeiE__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyeiE__1                NaN                     2   \n",
       "ytf__delyeiE__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyeiE__1  ytf/SRR6782001_1.fastq.gz  ytf/SRR6782001_2.fastq.gz   \n",
       "ytf__delyeiE__2  ytf/SRR6782002_1.fastq.gz  ytf/SRR6782002_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyeiE__1  ytf__delyeiE__1.bam      95.52  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyeiE__2  ytf__delyeiE__2.bam      95.95  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyeiE__1  GSE111094     1.0  \n",
       "ytf__delyeiE__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyheO__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyheO</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yheO</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7058001_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7058001_2.fastq.gz</td>\n",
       "      <td>ytf__delyheO__1.bam</td>\n",
       "      <td>95.09</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyheO__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyheO</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yheO</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7058002_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7058002_2.fastq.gz</td>\n",
       "      <td>ytf__delyheO__2.bam</td>\n",
       "      <td>95.13</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyheO__1   yTF        ytf      delyheO       1  Ye Gao          Ye Gao   \n",
       "ytf__delyheO__2   yTF        ytf      delyheO       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyheO__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yheO   \n",
       "ytf__delyheO__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yheO   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyheO__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyheO__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyheO__1                NaN                     2   \n",
       "ytf__delyheO__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyheO__1  ytf/SRR7058001_1.fastq.gz  ytf/SRR7058001_2.fastq.gz   \n",
       "ytf__delyheO__2  ytf/SRR7058002_1.fastq.gz  ytf/SRR7058002_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyheO__1  ytf__delyheO__1.bam      95.09  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyheO__2  ytf__delyheO__2.bam      95.13  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyheO__1  GSE111094     1.0  \n",
       "ytf__delyheO__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyiaJ__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyiaJ</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yiaJ</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782003_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782003_2.fastq.gz</td>\n",
       "      <td>ytf__delyiaJ__1.bam</td>\n",
       "      <td>94.63</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyiaJ__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyiaJ</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yiaJ</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782004_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782004_2.fastq.gz</td>\n",
       "      <td>ytf__delyiaJ__2.bam</td>\n",
       "      <td>95.07</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyiaJ__1   yTF        ytf      delyiaJ       1  Ye Gao          Ye Gao   \n",
       "ytf__delyiaJ__2   yTF        ytf      delyiaJ       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyiaJ__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yiaJ   \n",
       "ytf__delyiaJ__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yiaJ   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyiaJ__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyiaJ__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyiaJ__1                NaN                     2   \n",
       "ytf__delyiaJ__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyiaJ__1  ytf/SRR6782003_1.fastq.gz  ytf/SRR6782003_2.fastq.gz   \n",
       "ytf__delyiaJ__2  ytf/SRR6782004_1.fastq.gz  ytf/SRR6782004_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyiaJ__1  ytf__delyiaJ__1.bam      94.63  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyiaJ__2  ytf__delyiaJ__2.bam      95.07  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyiaJ__1  GSE111094     1.0  \n",
       "ytf__delyiaJ__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__delyieP__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyieP</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yieP</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782005_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782005_2.fastq.gz</td>\n",
       "      <td>ytf__delyieP__1.bam</td>\n",
       "      <td>94.07</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__delyieP__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>delyieP</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655 del_yieP</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6782006_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6782006_2.fastq.gz</td>\n",
       "      <td>ytf__delyieP__2.bam</td>\n",
       "      <td>94.80</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                       \n",
       "ytf__delyieP__1   yTF        ytf      delyieP       1  Ye Gao          Ye Gao   \n",
       "ytf__delyieP__2   yTF        ytf      delyieP       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                       run_date                     Strain Description  \\\n",
       "sample_id                                                                \n",
       "ytf__delyieP__1  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yieP   \n",
       "ytf__delyieP__2  10/2/2017 0:00  Escherichia coli K-12 MG1655 del_yieP   \n",
       "\n",
       "                Organism Base Media  ... Sequencing Machine  \\\n",
       "sample_id                            ...                      \n",
       "ytf__delyieP__1   MG1655         M9  ...         HiSeq 4000   \n",
       "ytf__delyieP__2   MG1655         M9  ...         HiSeq 4000   \n",
       "\n",
       "                Additional Details Biological Replicates  \\\n",
       "sample_id                                                  \n",
       "ytf__delyieP__1                NaN                     2   \n",
       "ytf__delyieP__2                NaN                     2   \n",
       "\n",
       "                                        R1                         R2  \\\n",
       "sample_id                                                               \n",
       "ytf__delyieP__1  ytf/SRR6782005_1.fastq.gz  ytf/SRR6782005_2.fastq.gz   \n",
       "ytf__delyieP__2  ytf/SRR6782006_1.fastq.gz  ytf/SRR6782006_2.fastq.gz   \n",
       "\n",
       "                                 BAM  alignment                         DOI  \\\n",
       "sample_id                                                                     \n",
       "ytf__delyieP__1  ytf__delyieP__1.bam      94.07  doi.org/10.1093/nar/gky752   \n",
       "ytf__delyieP__2  ytf__delyieP__2.bam      94.80  doi.org/10.1093/nar/gky752   \n",
       "\n",
       "                       GEO  Public  \n",
       "sample_id                           \n",
       "ytf__delyieP__1  GSE111094     1.0  \n",
       "ytf__delyieP__2  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__wt_glc__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6781997_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6781997_2.fastq.gz</td>\n",
       "      <td>ytf__wt_glc__1.bam</td>\n",
       "      <td>95.74</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__wt_glc__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_glc</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR6781998_1.fastq.gz</td>\n",
       "      <td>ytf/SRR6781998_2.fastq.gz</td>\n",
       "      <td>ytf__wt_glc__2.bam</td>\n",
       "      <td>97.01</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                      \n",
       "ytf__wt_glc__1   yTF        ytf       wt_glc       1  Ye Gao          Ye Gao   \n",
       "ytf__wt_glc__2   yTF        ytf       wt_glc       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                      run_date            Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "ytf__wt_glc__1  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "ytf__wt_glc__2  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "\n",
       "               Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                  ...                                         \n",
       "ytf__wt_glc__1         M9  ...         HiSeq 4000                NaN   \n",
       "ytf__wt_glc__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "               Biological Replicates                         R1  \\\n",
       "sample_id                                                         \n",
       "ytf__wt_glc__1                     2  ytf/SRR6781997_1.fastq.gz   \n",
       "ytf__wt_glc__2                     2  ytf/SRR6781998_1.fastq.gz   \n",
       "\n",
       "                                       R2                 BAM  alignment  \\\n",
       "sample_id                                                                  \n",
       "ytf__wt_glc__1  ytf/SRR6781997_2.fastq.gz  ytf__wt_glc__1.bam      95.74   \n",
       "ytf__wt_glc__2  ytf/SRR6781998_2.fastq.gz  ytf__wt_glc__2.bam      97.01   \n",
       "\n",
       "                                       DOI        GEO  Public  \n",
       "sample_id                                                      \n",
       "ytf__wt_glc__1  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "ytf__wt_glc__2  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__wt_ph5__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_ph5</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057985_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057985_2.fastq.gz</td>\n",
       "      <td>ytf__wt_ph5__1.bam</td>\n",
       "      <td>94.96</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__wt_ph5__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_ph5</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057986_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057986_2.fastq.gz</td>\n",
       "      <td>ytf__wt_ph5__2.bam</td>\n",
       "      <td>94.67</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                      \n",
       "ytf__wt_ph5__1   yTF        ytf       wt_ph5       1  Ye Gao          Ye Gao   \n",
       "ytf__wt_ph5__2   yTF        ytf       wt_ph5       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                      run_date            Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "ytf__wt_ph5__1  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "ytf__wt_ph5__2  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "\n",
       "               Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                  ...                                         \n",
       "ytf__wt_ph5__1         M9  ...         HiSeq 4000                NaN   \n",
       "ytf__wt_ph5__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "               Biological Replicates                         R1  \\\n",
       "sample_id                                                         \n",
       "ytf__wt_ph5__1                     2  ytf/SRR7057985_1.fastq.gz   \n",
       "ytf__wt_ph5__2                     2  ytf/SRR7057986_1.fastq.gz   \n",
       "\n",
       "                                       R2                 BAM  alignment  \\\n",
       "sample_id                                                                  \n",
       "ytf__wt_ph5__1  ytf/SRR7057985_2.fastq.gz  ytf__wt_ph5__1.bam      94.96   \n",
       "ytf__wt_ph5__2  ytf/SRR7057986_2.fastq.gz  ytf__wt_ph5__2.bam      94.67   \n",
       "\n",
       "                                       DOI        GEO  Public  \n",
       "sample_id                                                      \n",
       "ytf__wt_ph5__1  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "ytf__wt_ph5__2  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>study</th>\n",
       "      <th>project_id</th>\n",
       "      <th>condition_id</th>\n",
       "      <th>rep_id</th>\n",
       "      <th>contact</th>\n",
       "      <th>library creator</th>\n",
       "      <th>run_date</th>\n",
       "      <th>Strain Description</th>\n",
       "      <th>Organism</th>\n",
       "      <th>Base Media</th>\n",
       "      <th>...</th>\n",
       "      <th>Sequencing Machine</th>\n",
       "      <th>Additional Details</th>\n",
       "      <th>Biological Replicates</th>\n",
       "      <th>R1</th>\n",
       "      <th>R2</th>\n",
       "      <th>BAM</th>\n",
       "      <th>alignment</th>\n",
       "      <th>DOI</th>\n",
       "      <th>GEO</th>\n",
       "      <th>Public</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>sample_id</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>ytf__wt_ph8__1</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_ph8</td>\n",
       "      <td>1</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057987_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057987_2.fastq.gz</td>\n",
       "      <td>ytf__wt_ph8__1.bam</td>\n",
       "      <td>94.96</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>ytf__wt_ph8__2</td>\n",
       "      <td>yTF</td>\n",
       "      <td>ytf</td>\n",
       "      <td>wt_ph8</td>\n",
       "      <td>2</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>Ye Gao</td>\n",
       "      <td>10/2/2017 0:00</td>\n",
       "      <td>Escherichia coli K-12 MG1655</td>\n",
       "      <td>MG1655</td>\n",
       "      <td>M9</td>\n",
       "      <td>...</td>\n",
       "      <td>HiSeq 4000</td>\n",
       "      <td>NaN</td>\n",
       "      <td>2</td>\n",
       "      <td>ytf/SRR7057988_1.fastq.gz</td>\n",
       "      <td>ytf/SRR7057988_2.fastq.gz</td>\n",
       "      <td>ytf__wt_ph8__2.bam</td>\n",
       "      <td>94.50</td>\n",
       "      <td>doi.org/10.1093/nar/gky752</td>\n",
       "      <td>GSE111094</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows Ã— 32 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               study project_id condition_id  rep_id contact library creator  \\\n",
       "sample_id                                                                      \n",
       "ytf__wt_ph8__1   yTF        ytf       wt_ph8       1  Ye Gao          Ye Gao   \n",
       "ytf__wt_ph8__2   yTF        ytf       wt_ph8       2  Ye Gao          Ye Gao   \n",
       "\n",
       "                      run_date            Strain Description Organism  \\\n",
       "sample_id                                                               \n",
       "ytf__wt_ph8__1  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "ytf__wt_ph8__2  10/2/2017 0:00  Escherichia coli K-12 MG1655   MG1655   \n",
       "\n",
       "               Base Media  ... Sequencing Machine Additional Details  \\\n",
       "sample_id                  ...                                         \n",
       "ytf__wt_ph8__1         M9  ...         HiSeq 4000                NaN   \n",
       "ytf__wt_ph8__2         M9  ...         HiSeq 4000                NaN   \n",
       "\n",
       "               Biological Replicates                         R1  \\\n",
       "sample_id                                                         \n",
       "ytf__wt_ph8__1                     2  ytf/SRR7057987_1.fastq.gz   \n",
       "ytf__wt_ph8__2                     2  ytf/SRR7057988_1.fastq.gz   \n",
       "\n",
       "                                       R2                 BAM  alignment  \\\n",
       "sample_id                                                                  \n",
       "ytf__wt_ph8__1  ytf/SRR7057987_2.fastq.gz  ytf__wt_ph8__1.bam      94.96   \n",
       "ytf__wt_ph8__2  ytf/SRR7057988_2.fastq.gz  ytf__wt_ph8__2.bam      94.50   \n",
       "\n",
       "                                       DOI        GEO  Public  \n",
       "sample_id                                                      \n",
       "ytf__wt_ph8__1  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "ytf__wt_ph8__2  doi.org/10.1093/nar/gky752  GSE111094     1.0  \n",
       "\n",
       "[2 rows x 32 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# CHECK THAT THIS GROUPING IS CORRECT\n",
    "for name, group in ica_data.metadata.groupby(['project_id', 'condition_id']):\n",
    "    display(group)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# add a biological replicates count column to the metadata\n",
    "# this is needed later on\n",
    "for name, group in ica_data.metadata.groupby(['project_id', 'condition_id']):\n",
    "    ica_data.metadata.loc[group.index, 'Biological Replicates'] = group.shape[0]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Gene info (GENE_DIR+'gene_info.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "# ADDRESS THESE IF THEY FAIL\n",
    "# has a gene name column\n",
    "assert('gene_name' in ica_data.gene_info.columns)\n",
    "\n",
    "# can match to the data\n",
    "assert(ica_data.X.index.isin(ica_data.gene_info.index).all())\n",
    "assert(ica_data.S.index.isin(ica_data.gene_info.index).all())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Check for gene names. We expect some genes to be named, but we do not require all of them to be. The gene_name column for an unnamed gene will be the named the same as its locus tag in the following code block."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3923 (100.00%) genes had names.\n",
      "0 (0.00%) genes were assigned their locus tag as a name.\n"
     ]
    }
   ],
   "source": [
    "# MAKE SURE THE OUTPUT IS EXPECTED FOR YOU\n",
    "not_in = 0 # counts the missing gene loci\n",
    "had_nan = 0 # counts the unassigned gene names\n",
    "name = 0 # counts the genes with everything\n",
    "for g in ica_data.S.index:\n",
    "    if not(g in ica_data.num2name.keys()):\n",
    "        ica_data.num2name[g] = g # assigns the locus tag as the name\n",
    "        not_in += 1\n",
    "    elif not(type(ica_data.num2name[g])==str):\n",
    "        ica_data.num2name[g] = g # assigns the locus tag as the name\n",
    "        had_nan += 1\n",
    "    else:\n",
    "        name += 1\n",
    "no_name = not_in + had_nan\n",
    "tot = no_name + name\n",
    "print('%i (%.2f%%) genes had names.'%(name, name*100/tot))\n",
    "print('%i (%.2f%%) genes were assigned their locus tag as a name.'%(not_in, not_in*100/tot))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Optional: Gene Links\n",
    "\n",
    "This part is potentially the trickiest. We offer 'additional information' on our gene pages, which links each gene to a more informative database. For example:\n",
    "\n",
    "- E. coli: [EcoCyc](https://ecocyc.org/)\n",
    "- S. aureus: [AureoWiki](https://aureowiki.med.uni-greifswald.de/Main_Page)\n",
    "- B. subtilis: [SubtiWiki](http://subtiwiki.uni-goettingen.de/)\n",
    "\n",
    "If you would like to use this feature, you need to create a dictionary that takes gene loci as keys and returns links as items. There are two ways to do that:\n",
    "\n",
    "1. Create a .csv file in which one column is the gene loci and the other is the corresponding links\n",
    "2. Write a function that produces the links. The example from bacillus is shown.\n",
    "\n",
    "If this organism is new to the website and it has a database, play around with the database and urls to see which option is best for you. If this organism is already online, use the same option as the other datasets for this organism.\n",
    "\n",
    "### Gene links option 1: gene_links.csv\n",
    "\n",
    "If you are choosing this option, name your file 'gene_links.csv' and place it in the organism annotation folder, but outside of the gene_files folder. This is because the gene_files folder is downloadable from the site, but we don't want people to download this links file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'b0002': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10998',\n",
       " 'b0003': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10999',\n",
       " 'b0004': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11000',\n",
       " 'b0005': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6081',\n",
       " 'b0006': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10011',\n",
       " 'b0007': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11555',\n",
       " 'b0008': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11556',\n",
       " 'b0009': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11511',\n",
       " 'b0010': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11512',\n",
       " 'b0011': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6082',\n",
       " 'b0014': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10241',\n",
       " 'b0015': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10240',\n",
       " 'b0016': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6083',\n",
       " 'b0018': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10373',\n",
       " 'b0019': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10652',\n",
       " 'b0020': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11078',\n",
       " 'b0021': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6085',\n",
       " 'b0022': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6086',\n",
       " 'b0023': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10919',\n",
       " 'b0025': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11079',\n",
       " 'b0026': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10492',\n",
       " 'b0027': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10548',\n",
       " 'b0028': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11080',\n",
       " 'b0029': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11081',\n",
       " 'b0030': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11082',\n",
       " 'b0031': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10206',\n",
       " 'b0032': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10134',\n",
       " 'b0033': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10135',\n",
       " 'b0034': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6088',\n",
       " 'b0036': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11557',\n",
       " 'b0037': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11558',\n",
       " 'b0038': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11559',\n",
       " 'b0039': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11560',\n",
       " 'b0040': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11561',\n",
       " 'b0041': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11562',\n",
       " 'b0042': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11563',\n",
       " 'b0043': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11564',\n",
       " 'b0045': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11566',\n",
       " 'b0046': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11568',\n",
       " 'b0047': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10521',\n",
       " 'b0048': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10326',\n",
       " 'b0049': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10048',\n",
       " 'b0050': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10047',\n",
       " 'b0051': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10523',\n",
       " 'b0052': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10691',\n",
       " 'b0053': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10985',\n",
       " 'b0054': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11569',\n",
       " 'b0055': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11570',\n",
       " 'b0058': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12609',\n",
       " 'b0059': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11083',\n",
       " 'b0060': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10747',\n",
       " 'b0061': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10055',\n",
       " 'b0062': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10052',\n",
       " 'b0063': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10053',\n",
       " 'b0064': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10054',\n",
       " 'b0065': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11571',\n",
       " 'b0066': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11572',\n",
       " 'b0067': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11573',\n",
       " 'b0068': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11574',\n",
       " 'b0069': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12094',\n",
       " 'b0070': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11754',\n",
       " 'b0071': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11575',\n",
       " 'b0072': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11576',\n",
       " 'b0073': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11577',\n",
       " 'b0074': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11226',\n",
       " 'b0076': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10531',\n",
       " 'b0077': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10500',\n",
       " 'b0078': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10499',\n",
       " 'b0080': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10338',\n",
       " 'b0081': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11084',\n",
       " 'b0082': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11085',\n",
       " 'b0083': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11086',\n",
       " 'b0084': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10341',\n",
       " 'b0085': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10621',\n",
       " 'b0086': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10622',\n",
       " 'b0087': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10604',\n",
       " 'b0088': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10620',\n",
       " 'b0089': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10344',\n",
       " 'b0090': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10623',\n",
       " 'b0091': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10619',\n",
       " 'b0092': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10214',\n",
       " 'b0093': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10342',\n",
       " 'b0094': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10339',\n",
       " 'b0095': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10347',\n",
       " 'b0096': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10265',\n",
       " 'b0097': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11087',\n",
       " 'b0098': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10936',\n",
       " 'b0099': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10626',\n",
       " 'b0101': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12314',\n",
       " 'b0102': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12313',\n",
       " 'b0103': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12312',\n",
       " 'b0104': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10422',\n",
       " 'b0106': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11798',\n",
       " 'b0107': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12106',\n",
       " 'b0109': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11546',\n",
       " 'b0110': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10041',\n",
       " 'b0111': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10042',\n",
       " 'b0112': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10084',\n",
       " 'b0113': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11088',\n",
       " 'b0114': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10024',\n",
       " 'b0115': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10025',\n",
       " 'b0116': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10543',\n",
       " 'b0117': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12315',\n",
       " 'b0118': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12316',\n",
       " 'b0119': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12605',\n",
       " 'b0120': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10962',\n",
       " 'b0121': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10963',\n",
       " 'b0122': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11089',\n",
       " 'b0123': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12318',\n",
       " 'b0124': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10369',\n",
       " 'b0125': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG20098',\n",
       " 'b0126': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12319',\n",
       " 'b0127': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12320',\n",
       " 'b0128': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12321',\n",
       " 'b0129': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12322',\n",
       " 'b0130': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11749',\n",
       " 'b0131': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11747',\n",
       " 'b0132': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11748',\n",
       " 'b0133': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11746',\n",
       " 'b0134': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11675',\n",
       " 'b0135': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11678',\n",
       " 'b0137': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12326',\n",
       " 'b0139': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11972',\n",
       " 'b0142': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11374',\n",
       " 'b0143': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10690',\n",
       " 'b0144': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11362',\n",
       " 'b0145': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10230',\n",
       " 'b0146': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10949',\n",
       " 'b0147': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12330',\n",
       " 'b0148': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12329',\n",
       " 'b0149': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10605',\n",
       " 'b0150': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10302',\n",
       " 'b0151': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10304',\n",
       " 'b0152': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10305',\n",
       " 'b0153': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10303',\n",
       " 'b0154': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10432',\n",
       " 'b0155': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12331',\n",
       " 'b0156': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12332',\n",
       " 'b0157': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12333',\n",
       " 'b0158': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12334',\n",
       " 'b0159': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11090',\n",
       " 'b0160': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10225',\n",
       " 'b0161': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10463',\n",
       " 'b0162': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12335',\n",
       " 'b0163': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12336',\n",
       " 'b0164': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12337',\n",
       " 'b0166': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10207',\n",
       " 'b0167': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11411',\n",
       " 'b0168': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10570',\n",
       " 'b0169': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10901',\n",
       " 'b0170': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11033',\n",
       " 'b0171': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11539',\n",
       " 'b0172': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10335',\n",
       " 'b0173': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12715',\n",
       " 'b0174': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6092',\n",
       " 'b0175': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10139',\n",
       " 'b0176': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12436',\n",
       " 'b0177': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6093',\n",
       " 'b0178': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10455',\n",
       " 'b0179': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10316',\n",
       " 'b0180': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11284',\n",
       " 'b0181': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10545',\n",
       " 'b0182': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10546',\n",
       " 'b0183': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10861',\n",
       " 'b0184': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10238',\n",
       " 'b0185': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11647',\n",
       " 'b0186': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6094',\n",
       " 'b0187': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6095',\n",
       " 'b0188': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6096',\n",
       " 'b0189': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6097',\n",
       " 'b0190': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6098',\n",
       " 'b0191': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12354',\n",
       " 'b0192': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12137',\n",
       " 'b0193': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12138',\n",
       " 'b0194': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10770',\n",
       " 'b0195': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11503',\n",
       " 'b0196': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11502',\n",
       " 'b0197': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11504',\n",
       " 'b0198': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11737',\n",
       " 'b0199': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11621',\n",
       " 'b0200': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11736',\n",
       " 'b0207': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11648',\n",
       " 'b0208': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11649',\n",
       " 'b0209': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11650',\n",
       " 'b0210': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11651',\n",
       " 'b0211': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10246',\n",
       " 'b0212': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6099',\n",
       " 'b0213': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6100',\n",
       " 'b0214': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10860',\n",
       " 'b0215': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10243',\n",
       " 'b0217': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6101',\n",
       " 'b0219': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6103',\n",
       " 'b0220': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6104',\n",
       " 'b0221': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6105',\n",
       " 'b0222': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6106',\n",
       " 'b0223': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6107',\n",
       " 'b0224': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6108',\n",
       " 'b0225': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6109',\n",
       " 'b0226': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6110',\n",
       " 'b0227': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6111',\n",
       " 'b0228': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6112',\n",
       " 'b0229': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6113',\n",
       " 'b0230': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6114',\n",
       " 'b0231': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6115',\n",
       " 'b0232': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6116',\n",
       " 'b0233': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6117',\n",
       " 'b0234': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6118',\n",
       " 'b0237': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10695',\n",
       " 'b0238': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10414',\n",
       " 'b0239': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11091',\n",
       " 'b0240_2': nan,\n",
       " 'b0241': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10729',\n",
       " 'b0242': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10768',\n",
       " 'b0243': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10767',\n",
       " 'b0250': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6125',\n",
       " 'b0251': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6126',\n",
       " 'b0252': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6127',\n",
       " 'b0253': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6128',\n",
       " 'b0254': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6129',\n",
       " 'b0258': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6133',\n",
       " 'b0259': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6134',\n",
       " 'b0260': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6135',\n",
       " 'b0261': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6136',\n",
       " 'b0262': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12340',\n",
       " 'b0264': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6138',\n",
       " 'b0265': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6139',\n",
       " 'b0266': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12339',\n",
       " 'b0267': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12338',\n",
       " 'b0268': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6140',\n",
       " 'b0269': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6141',\n",
       " 'b0270': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6142',\n",
       " 'b0271': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6143',\n",
       " 'b0272': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6144',\n",
       " 'b0273': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10067',\n",
       " 'b0274': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6145',\n",
       " 'b0275': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6146',\n",
       " 'b0276': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6147',\n",
       " 'b0277': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6148',\n",
       " 'b0278': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6149',\n",
       " 'b0280': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6151',\n",
       " 'b0281': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6152',\n",
       " 'b0283': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6154',\n",
       " 'b0284': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6155',\n",
       " 'b0285': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6156',\n",
       " 'b0286': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6157',\n",
       " 'b0287': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6158',\n",
       " 'b0288': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6159',\n",
       " 'b0289': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6160',\n",
       " 'b0290': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6161',\n",
       " 'b0291': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6162',\n",
       " 'b0292': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6163',\n",
       " 'b0293': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6164',\n",
       " 'b0294': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6165',\n",
       " 'b0295': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6166',\n",
       " 'b0296': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6167',\n",
       " 'b0300': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6170',\n",
       " 'b0304': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6174',\n",
       " 'b0305': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6175',\n",
       " 'b0306': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6176',\n",
       " 'b0307': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6177',\n",
       " 'b0308': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6178',\n",
       " 'b0311': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10109',\n",
       " 'b0312': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10110',\n",
       " 'b0313': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10111',\n",
       " 'b0314': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10112',\n",
       " 'b0315': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11236',\n",
       " 'b0316': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6181',\n",
       " 'b0318': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6183',\n",
       " 'b0320': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6185',\n",
       " 'b0321': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6186',\n",
       " 'b0323': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6188',\n",
       " 'b0324': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6189',\n",
       " 'b0325': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6190',\n",
       " 'b0328': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6193',\n",
       " 'b0329': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6194',\n",
       " 'b0330': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6195',\n",
       " 'b0331': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6196',\n",
       " 'b0333': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6198',\n",
       " 'b0334': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6199',\n",
       " 'b0335': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6200',\n",
       " 'b0336': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11327',\n",
       " 'b0337': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11326',\n",
       " 'b0338': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11421',\n",
       " 'b0339': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10176',\n",
       " 'b0340': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10175',\n",
       " 'b0341': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10177',\n",
       " 'b0342': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10524',\n",
       " 'b0343': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10526',\n",
       " 'b0344': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10527',\n",
       " 'b0345': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10525',\n",
       " 'b0346': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6201',\n",
       " 'b0347': 'https://ecocyc.org/gene?orgid=ECOLI&id=M010',\n",
       " 'b0348': 'https://ecocyc.org/gene?orgid=ECOLI&id=M011',\n",
       " 'b0349': 'https://ecocyc.org/gene?orgid=ECOLI&id=M012',\n",
       " 'b0350': 'https://ecocyc.org/gene?orgid=ECOLI&id=M013',\n",
       " 'b0351': 'https://ecocyc.org/gene?orgid=ECOLI&id=M014',\n",
       " 'b0352': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6205',\n",
       " 'b0353': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6206',\n",
       " 'b0354': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6207',\n",
       " 'b0355': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6208',\n",
       " 'b0356': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG50010',\n",
       " 'b0357': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6209',\n",
       " 'b0358': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6210',\n",
       " 'b0363': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6215',\n",
       " 'b0365': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6217',\n",
       " 'b0366': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6218',\n",
       " 'b0367': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6219',\n",
       " 'b0368': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12423',\n",
       " 'b0369': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10428',\n",
       " 'b0372': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6223',\n",
       " 'b0373': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6224',\n",
       " 'b0376': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12867',\n",
       " 'b0377': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10928',\n",
       " 'b0378': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6227',\n",
       " 'b0379': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6228',\n",
       " 'b0380': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6229',\n",
       " 'b0381': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10213',\n",
       " 'b0382': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11256',\n",
       " 'b0383': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10727',\n",
       " 'b0384': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11401',\n",
       " 'b0385': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11257',\n",
       " 'b0386': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10769',\n",
       " 'b0387': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6230',\n",
       " 'b0388': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10082',\n",
       " 'b0389': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11093',\n",
       " 'b0390': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10083',\n",
       " 'b0391': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12159',\n",
       " 'b0392': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6231',\n",
       " 'b0393': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12158',\n",
       " 'b0394': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11288',\n",
       " 'b0396': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10060',\n",
       " 'b0397': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10927',\n",
       " 'b0398': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11094',\n",
       " 'b0399': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10728',\n",
       " 'b0400': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10733',\n",
       " 'b0401': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12168',\n",
       " 'b0402': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6233',\n",
       " 'b0403': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10565',\n",
       " 'b0404': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11095',\n",
       " 'b0405': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10812',\n",
       " 'b0406': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10996',\n",
       " 'b0407': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11096',\n",
       " 'b0408': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10938',\n",
       " 'b0409': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10940',\n",
       " 'b0410': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11097',\n",
       " 'b0411': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11035',\n",
       " 'b0412': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12874',\n",
       " 'b0413': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11320',\n",
       " 'b0414': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11321',\n",
       " 'b0415': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11322',\n",
       " 'b0416': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10666',\n",
       " 'b0417': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6234',\n",
       " 'b0418': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10704',\n",
       " 'b0419': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6236',\n",
       " 'b0420': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6237',\n",
       " 'b0421': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10508',\n",
       " 'b0422': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11098',\n",
       " 'b0423': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6238',\n",
       " 'b0424': 'https://ecocyc.org/gene?orgid=ECOLI&id=G198',\n",
       " 'b0425': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6239',\n",
       " 'b0426': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6240',\n",
       " 'b0427': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6241',\n",
       " 'b0428': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10182',\n",
       " 'b0429': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10181',\n",
       " 'b0430': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10180',\n",
       " 'b0431': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10179',\n",
       " 'b0432': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10178',\n",
       " 'b0433': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12183',\n",
       " 'b0434': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12182',\n",
       " 'b0435': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10125',\n",
       " 'b0436': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11003',\n",
       " 'b0437': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10158',\n",
       " 'b0438': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10159',\n",
       " 'b0439': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10542',\n",
       " 'b0440': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10467',\n",
       " 'b0441': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6242',\n",
       " 'b0442': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6243',\n",
       " 'b0443': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6244',\n",
       " 'b0444': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6245',\n",
       " 'b0445': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG13217',\n",
       " 'b0446': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6246',\n",
       " 'b0447': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6247',\n",
       " 'b0448': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6248',\n",
       " 'b0449': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11622',\n",
       " 'b0450': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12191',\n",
       " 'b0451': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11821',\n",
       " 'b0452': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10995',\n",
       " 'b0453': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6250',\n",
       " 'b0454': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6251',\n",
       " 'b0456': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11099',\n",
       " 'b0457': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6252',\n",
       " 'b0458': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6253',\n",
       " 'b0459': 'https://ecocyc.org/gene?orgid=ECOLI&id=G58',\n",
       " 'b0460': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10439',\n",
       " 'b0461': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12429',\n",
       " 'b0462': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11704',\n",
       " 'b0463': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11703',\n",
       " 'b0464': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12116',\n",
       " 'b0465': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6255',\n",
       " 'b0467': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10765',\n",
       " 'b0468': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12843',\n",
       " 'b0469': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10051',\n",
       " 'b0470': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10245',\n",
       " 'b0471': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11100',\n",
       " 'b0472': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10834',\n",
       " 'b0473': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10461',\n",
       " 'b0474': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10032',\n",
       " 'b0475': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10431',\n",
       " 'b0476': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11101',\n",
       " 'b0477': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11102',\n",
       " 'b0478': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12623',\n",
       " 'b0479': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6256',\n",
       " 'b0480': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11060',\n",
       " 'b0481': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12454',\n",
       " 'b0482': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6258',\n",
       " 'b0483': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6259',\n",
       " 'b0484': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6260',\n",
       " 'b0485': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6261',\n",
       " 'b0486': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6262',\n",
       " 'b0487': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6263',\n",
       " 'b0488': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6264',\n",
       " 'b0489': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6265',\n",
       " 'b0490': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6266',\n",
       " 'b0491': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6267',\n",
       " 'b0492': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6268',\n",
       " 'b0493': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6269',\n",
       " 'b0494': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11542',\n",
       " 'b0495': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11657',\n",
       " 'b0496': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6270',\n",
       " 'b0497': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10849',\n",
       " 'b0498': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11769',\n",
       " 'b0499': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6271',\n",
       " 'b0502_1': nan,\n",
       " 'b0503': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11768',\n",
       " 'b0504': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6274',\n",
       " 'b0505': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6275',\n",
       " 'b0506': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6276',\n",
       " 'b0507': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11583',\n",
       " 'b0508': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6277',\n",
       " 'b0509': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6278',\n",
       " 'b0511': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6280',\n",
       " 'b0512': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6281',\n",
       " 'b0513': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6282',\n",
       " 'b0514': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6283',\n",
       " 'b0515': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6284',\n",
       " 'b0516': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6285',\n",
       " 'b0517': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6286',\n",
       " 'b0518': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6287',\n",
       " 'b0520': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6289',\n",
       " 'b0521': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12384',\n",
       " 'b0522': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10796',\n",
       " 'b0523': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10793',\n",
       " 'b0524': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12666',\n",
       " 'b0525': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10758',\n",
       " 'b0526': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10196',\n",
       " 'b0527': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12708',\n",
       " 'b0528': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12879',\n",
       " 'b0529': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10328',\n",
       " 'b0532': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6292',\n",
       " 'b0533': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6293',\n",
       " 'b0537': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10507',\n",
       " 'b0543': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10629',\n",
       " 'b0544': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6300',\n",
       " 'b0545': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6301',\n",
       " 'b0546': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6302',\n",
       " 'b0552': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6308',\n",
       " 'b0553_1': nan,\n",
       " 'b0555': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6310',\n",
       " 'b0557': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6312',\n",
       " 'b0559': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6314',\n",
       " 'b0562_2': nan,\n",
       " 'b0564': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10050',\n",
       " 'b0565': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10673',\n",
       " 'b0566': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10268',\n",
       " 'b0567': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12448',\n",
       " 'b0568': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11740',\n",
       " 'b0569': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11739',\n",
       " 'b0570': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6318',\n",
       " 'b0571': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6319',\n",
       " 'b0572': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6320',\n",
       " 'b0573': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6321',\n",
       " 'b0574': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6322',\n",
       " 'b0575': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12367',\n",
       " 'b0576': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10708',\n",
       " 'b0577': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6323',\n",
       " 'b0578': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG50005',\n",
       " 'b0579': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6324',\n",
       " 'b0580': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6325',\n",
       " 'b0581': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6326',\n",
       " 'b0582': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6327',\n",
       " 'b0583': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10262',\n",
       " 'b0584': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10293',\n",
       " 'b0585': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10299',\n",
       " 'b0586': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10264',\n",
       " 'b0587': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10297',\n",
       " 'b0588': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10295',\n",
       " 'b0589': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10298',\n",
       " 'b0590': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10296',\n",
       " 'b0591': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11104',\n",
       " 'b0592': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10294',\n",
       " 'b0593': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10261',\n",
       " 'b0594': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10263',\n",
       " 'b0595': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10260',\n",
       " 'b0596': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10259',\n",
       " 'b0597': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11105',\n",
       " 'b0598': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10167',\n",
       " 'b0599': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12692',\n",
       " 'b0600': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6329',\n",
       " 'b0601': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6330',\n",
       " 'b0602': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6331',\n",
       " 'b0603': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6332',\n",
       " 'b0604': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6333',\n",
       " 'b0605': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11384',\n",
       " 'b0606': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11385',\n",
       " 'b0607': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6334',\n",
       " 'b0608': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6335',\n",
       " 'b0610': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6337',\n",
       " 'b0611': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10856',\n",
       " 'b0612': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6338',\n",
       " 'b0615': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6341',\n",
       " 'b0618': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6344',\n",
       " 'b0619': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6345',\n",
       " 'b0620': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6346',\n",
       " 'b0621': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6347',\n",
       " 'b0622': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12180',\n",
       " 'b0623': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12179',\n",
       " 'b0624': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12209',\n",
       " 'b0627': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11305',\n",
       " 'b0628': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11306',\n",
       " 'b0629': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11594',\n",
       " 'b0630': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11591',\n",
       " 'b0631': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11592',\n",
       " 'b0632': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10201',\n",
       " 'b0633': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10854',\n",
       " 'b0634': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10607',\n",
       " 'b0635': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10606',\n",
       " 'b0636': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11254',\n",
       " 'b0637': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11255',\n",
       " 'b0638': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6349',\n",
       " 'b0639': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6350',\n",
       " 'b0640': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11412',\n",
       " 'b0641': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10855',\n",
       " 'b0642': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10532',\n",
       " 'b0643': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12851',\n",
       " 'b0644': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6351',\n",
       " 'b0646': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6353',\n",
       " 'b0649': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6356',\n",
       " 'b0650': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6357',\n",
       " 'b0651': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6358',\n",
       " 'b0652': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12663',\n",
       " 'b0653': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12662',\n",
       " 'b0654': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12661',\n",
       " 'b0655': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6359',\n",
       " 'b0656': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6360',\n",
       " 'b0657': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10168',\n",
       " 'b0658': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6361',\n",
       " 'b0659': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6362',\n",
       " 'b0660': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6363',\n",
       " 'b0661': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6364',\n",
       " 'b0662': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6365',\n",
       " 'b0674': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10092',\n",
       " 'b0675': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10634',\n",
       " 'b0676': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10636',\n",
       " 'b0677': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10632',\n",
       " 'b0678': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10633',\n",
       " 'b0679': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10635',\n",
       " 'b0680': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10390',\n",
       " 'b0681': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6370',\n",
       " 'b0682': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6371',\n",
       " 'b0683': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10359',\n",
       " 'b0684': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10318',\n",
       " 'b0685': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11775',\n",
       " 'b0686': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11776',\n",
       " 'b0687': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12197',\n",
       " 'b0688': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12144',\n",
       " 'b0689': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6374',\n",
       " 'b0692': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10753',\n",
       " 'b0693': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10964',\n",
       " 'b0694': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10517',\n",
       " 'b0695': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10516',\n",
       " 'b0696': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10515',\n",
       " 'b0697': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10514',\n",
       " 'b0698': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10513',\n",
       " 'b0699': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11521',\n",
       " 'b0700': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10848',\n",
       " 'b0703': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6376',\n",
       " 'b0704': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11523',\n",
       " 'b0705': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6377',\n",
       " 'b0707': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11108',\n",
       " 'b0708': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10736',\n",
       " 'b0709': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6378',\n",
       " 'b0710': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6379',\n",
       " 'b0711': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6380',\n",
       " 'b0712': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6381',\n",
       " 'b0713': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6382',\n",
       " 'b0714': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6383',\n",
       " 'b0715': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6384',\n",
       " 'b0716': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6385',\n",
       " 'b0718': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6387',\n",
       " 'b0720': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10402',\n",
       " 'b0721': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10933',\n",
       " 'b0722': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10934',\n",
       " 'b0723': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10931',\n",
       " 'b0724': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10932',\n",
       " 'b0726': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10979',\n",
       " 'b0727': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10980',\n",
       " 'b0728': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10981',\n",
       " 'b0729': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10982',\n",
       " 'b0730': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11109',\n",
       " 'b0731': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG13235',\n",
       " 'b0732': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG13236',\n",
       " 'b0733': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10173',\n",
       " 'b0734': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10174',\n",
       " 'b0735': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12395',\n",
       " 'b0736': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11110',\n",
       " 'b0737': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11010',\n",
       " 'b0738': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11011',\n",
       " 'b0739': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11007',\n",
       " 'b0740': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11008',\n",
       " 'b0741': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10684',\n",
       " 'b0742': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12854',\n",
       " 'b0750': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10630',\n",
       " 'b0751': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11700',\n",
       " 'b0752': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6393',\n",
       " 'b0753': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6394',\n",
       " 'b0754': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10079',\n",
       " 'b0755': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11699',\n",
       " 'b0756': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11698',\n",
       " 'b0757': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10363',\n",
       " 'b0758': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10366',\n",
       " 'b0759': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10362',\n",
       " 'b0760': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11677',\n",
       " 'b0761': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6395',\n",
       " 'b0762': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6396',\n",
       " 'b0763': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12427',\n",
       " 'b0764': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10002',\n",
       " 'b0765': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10152',\n",
       " 'b0766': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11239',\n",
       " 'b0767': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6397',\n",
       " 'b0768': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6398',\n",
       " 'b0770': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6400',\n",
       " 'b0771': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6401',\n",
       " 'b0772': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12875',\n",
       " 'b0773': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11238',\n",
       " 'b0774': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10117',\n",
       " 'b0775': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10118',\n",
       " 'b0776': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10121',\n",
       " 'b0777': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10119',\n",
       " 'b0778': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10120',\n",
       " 'b0779': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11062',\n",
       " 'b0780': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6402',\n",
       " 'b0781': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11595',\n",
       " 'b0782': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11596',\n",
       " 'b0783': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11666',\n",
       " 'b0784': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11597',\n",
       " 'b0785': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11598',\n",
       " 'b0786': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6403',\n",
       " 'b0787': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6404',\n",
       " 'b0788': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6405',\n",
       " 'b0789': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6406',\n",
       " 'b0790': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6407',\n",
       " 'b0791': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6408',\n",
       " 'b0792': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6409',\n",
       " 'b0793': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6410',\n",
       " 'b0794': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6411',\n",
       " 'b0795': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6412',\n",
       " 'b0796': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12406',\n",
       " 'b0797': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11235',\n",
       " 'b0798': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11579',\n",
       " 'b0799': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11357',\n",
       " 'b0800': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11580',\n",
       " 'b0801': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11581',\n",
       " 'b0802': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12422',\n",
       " 'b0803': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12421',\n",
       " 'b0804': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6413',\n",
       " 'b0805': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6414',\n",
       " 'b0806': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6415',\n",
       " 'b0807': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6416',\n",
       " 'b0808': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6417',\n",
       " 'b0809': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10389',\n",
       " 'b0810': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10388',\n",
       " 'b0811': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10386',\n",
       " 'b0812': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11415',\n",
       " 'b0813': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12134',\n",
       " 'b0814': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12117',\n",
       " 'b0815': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6418',\n",
       " 'b0817': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6420',\n",
       " 'b0818': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6421',\n",
       " 'b0819': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6422',\n",
       " 'b0820': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6423',\n",
       " 'b0821': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6424',\n",
       " 'b0822': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6425',\n",
       " 'b0823': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6426',\n",
       " 'b0824': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6427',\n",
       " 'b0825': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6428',\n",
       " 'b0826': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10154',\n",
       " 'b0827': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10153',\n",
       " 'b0828': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12407',\n",
       " 'b0829': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6429',\n",
       " 'b0830': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6430',\n",
       " 'b0831': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6431',\n",
       " 'b0832': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6432',\n",
       " 'b0833': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6433',\n",
       " 'b0834': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6434',\n",
       " 'b0835': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6435',\n",
       " 'b0836': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6436',\n",
       " 'b0837': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6437',\n",
       " 'b0838': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6438',\n",
       " 'b0839': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10203',\n",
       " 'b0840': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10223',\n",
       " 'b0841': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6439',\n",
       " 'b0842': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6440',\n",
       " 'b0843': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6441',\n",
       " 'b0844': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6442',\n",
       " 'b0845': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6443',\n",
       " 'b0846': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6444',\n",
       " 'b0847': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6445',\n",
       " 'b0848': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6446',\n",
       " 'b0849': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10417',\n",
       " 'b0850': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12842',\n",
       " 'b0851': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11261',\n",
       " 'b0852': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10852',\n",
       " 'b0853': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6447',\n",
       " 'b0854': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11629',\n",
       " 'b0855': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11630',\n",
       " 'b0856': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11631',\n",
       " 'b0857': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11632',\n",
       " 'b0858': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6448',\n",
       " 'b0859': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6449',\n",
       " 'b0860': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11628',\n",
       " 'b0861': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11627',\n",
       " 'b0862': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11626',\n",
       " 'b0863': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11625',\n",
       " 'b0864': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11624',\n",
       " 'b0865': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6450',\n",
       " 'b0866': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6451',\n",
       " 'b0867': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6452',\n",
       " 'b0868': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6453',\n",
       " 'b0869': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6454',\n",
       " 'b0870': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6455',\n",
       " 'b0871': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10754',\n",
       " 'b0872': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6456',\n",
       " 'b0873': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6457',\n",
       " 'b0874': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6458',\n",
       " 'b0875': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG13270',\n",
       " 'b0876': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6459',\n",
       " 'b0877': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6460',\n",
       " 'b0878': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6461',\n",
       " 'b0879': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6462',\n",
       " 'b0880': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11111',\n",
       " 'b0881': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6463',\n",
       " 'b0882': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10156',\n",
       " 'b0884': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10504',\n",
       " 'b0885': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11112',\n",
       " 'b0886': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10012',\n",
       " 'b0887': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11405',\n",
       " 'b0888': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11032',\n",
       " 'b0889': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10547',\n",
       " 'b0890': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6464',\n",
       " 'b0891': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6465',\n",
       " 'b0892': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12690',\n",
       " 'b0893': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10947',\n",
       " 'b0894': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10232',\n",
       " 'b0895': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10233',\n",
       " 'b0896': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10234',\n",
       " 'b0897': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11241',\n",
       " 'b0898': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11242',\n",
       " 'b0899': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6466',\n",
       " 'b0900': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6467',\n",
       " 'b0901': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12702',\n",
       " 'b0902': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10028',\n",
       " 'b0903': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10701',\n",
       " 'b0904': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11258',\n",
       " 'b0905': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6468',\n",
       " 'b0906': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6469',\n",
       " 'b0907': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10946',\n",
       " 'b0908': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10073',\n",
       " 'b0909': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6470',\n",
       " 'b0910': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11265',\n",
       " 'b0911': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10900',\n",
       " 'b0912': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10441',\n",
       " 'b0913': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12375',\n",
       " 'b0914': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10613',\n",
       " 'b0915': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11409',\n",
       " 'b0916': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6471',\n",
       " 'b0917': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6472',\n",
       " 'b0918': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10519',\n",
       " 'b0919': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6473',\n",
       " 'b0920': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12166',\n",
       " 'b0921': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12167',\n",
       " 'b0922': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12165',\n",
       " 'b0923': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11252',\n",
       " 'b0924': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10618',\n",
       " 'b0925': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11253',\n",
       " 'b0926': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6474',\n",
       " 'b0927': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6475',\n",
       " 'b0928': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10096',\n",
       " 'b0929': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10671',\n",
       " 'b0930': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10094',\n",
       " 'b0931': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10742',\n",
       " 'b0932': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10696',\n",
       " 'b0933': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12358',\n",
       " 'b0934': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6476',\n",
       " 'b0935': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6477',\n",
       " 'b0936': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6478',\n",
       " 'b0937': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6479',\n",
       " 'b0940': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6482',\n",
       " 'b0941': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6483',\n",
       " 'b0942': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6484',\n",
       " 'b0943': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6485',\n",
       " 'b0944': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12397',\n",
       " 'b0945': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10807',\n",
       " 'b0946': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6486',\n",
       " 'b0947': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6487',\n",
       " 'b0948': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6488',\n",
       " 'b0949': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6489',\n",
       " 'b0950': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6490',\n",
       " 'b0951': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6491',\n",
       " 'b0952': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6492',\n",
       " 'b0953': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG50004',\n",
       " 'b0954': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10273',\n",
       " 'b0955': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6493',\n",
       " 'b0956': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12855',\n",
       " 'b0957': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10669',\n",
       " 'b0958': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10984',\n",
       " 'b0959': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6494',\n",
       " 'b0960': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6495',\n",
       " 'b0961': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6496',\n",
       " 'b0962': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10426',\n",
       " 'b0963': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6497',\n",
       " 'b0964': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6498',\n",
       " 'b0965': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6499',\n",
       " 'b0966': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6500',\n",
       " 'b0967': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6501',\n",
       " 'b0968': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6502',\n",
       " 'b0969': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12876',\n",
       " 'b0970': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11113',\n",
       " 'b0972': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10468',\n",
       " 'b0973': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10469',\n",
       " 'b0974': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10470',\n",
       " 'b0975': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10471',\n",
       " 'b0976': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10472',\n",
       " 'b0977': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10473',\n",
       " 'b0978': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11380',\n",
       " 'b0979': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11379',\n",
       " 'b0980': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10049',\n",
       " 'b0981': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11826',\n",
       " 'b0982': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6503',\n",
       " 'b0983': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6504',\n",
       " 'b0984': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6505',\n",
       " 'b0985': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6506',\n",
       " 'b0986': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6507',\n",
       " 'b0987': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6508',\n",
       " 'b0990': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6511',\n",
       " 'b0991': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6512',\n",
       " 'b0992': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6513',\n",
       " 'b0993': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6514',\n",
       " 'b0994': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6515',\n",
       " 'b0995': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12615',\n",
       " 'b0996': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11815',\n",
       " 'b0997': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11814',\n",
       " 'b0998': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12195',\n",
       " 'b0999': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12194',\n",
       " 'b1000': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12193',\n",
       " 'b1001': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12196',\n",
       " 'b1002': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10033',\n",
       " 'b1003': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12703',\n",
       " 'b1004': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11540',\n",
       " 'b1006': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6517',\n",
       " 'b1007': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6518',\n",
       " 'b1008': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6519',\n",
       " 'b1009': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6520',\n",
       " 'b1010': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6521',\n",
       " 'b1011': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6522',\n",
       " 'b1012': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6523',\n",
       " 'b1013': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12301',\n",
       " 'b1014': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10801',\n",
       " 'b1015': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10802',\n",
       " 'b1018': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6527',\n",
       " 'b1019': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11735',\n",
       " 'b1020': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11734',\n",
       " 'b1021': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6528',\n",
       " 'b1022': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6529',\n",
       " 'b1023': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6530',\n",
       " 'b1024': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6531',\n",
       " 'b1025': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6532',\n",
       " 'b1026': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6533',\n",
       " 'b1027': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6534',\n",
       " 'b1033': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6539',\n",
       " 'b1034': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6540',\n",
       " 'b1035': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6541',\n",
       " 'b1036': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6542',\n",
       " 'b1037': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6543',\n",
       " 'b1038': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6544',\n",
       " 'b1039': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6545',\n",
       " 'b1040': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6546',\n",
       " 'b1042': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11489',\n",
       " 'b1044': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6549',\n",
       " 'b1045': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6550',\n",
       " 'b1046': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6551',\n",
       " 'b1047': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6552',\n",
       " 'b1048': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11885',\n",
       " 'b1049': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11886',\n",
       " 'b1050': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12689',\n",
       " 'b1051': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11338',\n",
       " 'b1053': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11343',\n",
       " 'b1054': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10464',\n",
       " 'b1055': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11116',\n",
       " 'b1056': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11820',\n",
       " 'b1057': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6554',\n",
       " 'b1059': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6556',\n",
       " 'b1060': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6557',\n",
       " 'b1061': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6558',\n",
       " 'b1062': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10806',\n",
       " 'b1063': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11117',\n",
       " 'b1064': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12688',\n",
       " 'b1065': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6559',\n",
       " 'b1066': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10851',\n",
       " 'b1067': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11537',\n",
       " 'b1068': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6560',\n",
       " 'b1069': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6561',\n",
       " 'b1070': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6562',\n",
       " 'b1071': 'https://ecocyc.org/gene?orgid=ECOLI&id=G369',\n",
       " 'b1072': 'https://ecocyc.org/gene?orgid=ECOLI&id=G357',\n",
       " 'b1073': 'https://ecocyc.org/gene?orgid=ECOLI&id=G358',\n",
       " 'b1074': 'https://ecocyc.org/gene?orgid=ECOLI&id=G359',\n",
       " 'b1075': 'https://ecocyc.org/gene?orgid=ECOLI&id=G360',\n",
       " 'b1076': 'https://ecocyc.org/gene?orgid=ECOLI&id=G361',\n",
       " 'b1077': 'https://ecocyc.org/gene?orgid=ECOLI&id=G362',\n",
       " 'b1078': 'https://ecocyc.org/gene?orgid=ECOLI&id=G363',\n",
       " 'b1079': 'https://ecocyc.org/gene?orgid=ECOLI&id=G364',\n",
       " 'b1080': 'https://ecocyc.org/gene?orgid=ECOLI&id=G365',\n",
       " 'b1081': 'https://ecocyc.org/gene?orgid=ECOLI&id=G366',\n",
       " 'b1082': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11967',\n",
       " 'b1083': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11545',\n",
       " 'b1084': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10859',\n",
       " 'b1086': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11118',\n",
       " 'b1087': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11438',\n",
       " 'b1088': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11119',\n",
       " 'b1089': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10890',\n",
       " 'b1090': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11437',\n",
       " 'b1091': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10277',\n",
       " 'b1092': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11317',\n",
       " 'b1093': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11318',\n",
       " 'b1094': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG50003',\n",
       " 'b1095': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12606',\n",
       " 'b1096': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11493',\n",
       " 'b1097': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11494',\n",
       " 'b1098': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12302',\n",
       " 'b1099': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11500',\n",
       " 'b1100': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12303',\n",
       " 'b1101': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10787',\n",
       " 'b1102': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10306',\n",
       " 'b1103': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12172',\n",
       " 'b1104': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6564',\n",
       " 'b1105': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6565',\n",
       " 'b1106': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6566',\n",
       " 'b1107': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6567',\n",
       " 'b1108': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6568',\n",
       " 'b1109': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10649',\n",
       " 'b1110': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12444',\n",
       " 'b1111': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6569',\n",
       " 'b1112': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6570',\n",
       " 'b1113': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6571',\n",
       " 'b1114': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11619',\n",
       " 'b1115': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6572',\n",
       " 'b1116': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6573',\n",
       " 'b1117': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6574',\n",
       " 'b1118': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6575',\n",
       " 'b1119': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6576',\n",
       " 'b1120': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6577',\n",
       " 'b1123': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10752',\n",
       " 'b1124': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10751',\n",
       " 'b1125': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10750',\n",
       " 'b1126': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10749',\n",
       " 'b1127': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11549',\n",
       " 'b1128': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11430',\n",
       " 'b1129': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10732',\n",
       " 'b1130': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10731',\n",
       " 'b1131': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11314',\n",
       " 'b1132': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11345',\n",
       " 'b1133': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG11344',\n",
       " 'b1134': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6580',\n",
       " 'b1135': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6581',\n",
       " 'b1136': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10489',\n",
       " 'b1137': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6582',\n",
       " 'b1138': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6583',\n",
       " 'b1139': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10535',\n",
       " 'b1140': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG12370',\n",
       " 'b1141': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6585',\n",
       " 'b1143': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6587',\n",
       " 'b1144': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6588',\n",
       " 'b1145': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6589',\n",
       " 'b1147': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6591',\n",
       " 'b1155': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6598',\n",
       " 'b1159': 'https://ecocyc.org/gene?orgid=ECOLI&id=EG10573',\n",
       " 'b1161': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6601',\n",
       " 'b1162': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6602',\n",
       " 'b1163': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6603',\n",
       " 'b1164': 'https://ecocyc.org/gene?orgid=ECOLI&id=G6604',\n",
       " ...}"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# IF USING THIS, PLACE YOUR FILE AT THIS PATH\n",
    "# YOU MAY NEED TO READ IT IN DIFFERENTLY DEPENDING ON FORMATTING\n",
    "links = pd.read_csv('../annotation/gene_links.csv', index_col=0, header=None).to_dict()[1]\n",
    "links"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Gene links option 2: get_links() function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# IF USING THIS, MODIFY THIS FUNCTION\n",
    "def get_links(locus):\n",
    "    # by looking at the db, I noticed that I could construct my own \n",
    "    # search url for it\n",
    "    prefix = 'http://subtiwiki.uni-goettingen.de/v3/gene/search/exact/'\n",
    "    return prefix+locus\n",
    "\n",
    "# use the function to fill in a links dictionary\n",
    "links = {}\n",
    "for g in ica_data.gene_info.index:\n",
    "    links[g] = get_links(g)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Gene links option 3: No links"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# IF USING THIS OPTION, UNCOMMENT AND RUN THE FOLLOWING\n",
    "#links = {}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Test your gene links\n",
    "\n",
    "This may take some time (~25 min for E. coli), but we want to make sure the links work -- broken links are not cool on a website."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "3c02e953460e4be99c1ce5a730a182bf",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(IntProgress(value=0, max=3923), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "3865 (98.52%) genes have working links.\n",
      "58 (1.48%) genes don't have links.\n",
      "0 (0.00%) genes had broken links!\n"
     ]
    }
   ],
   "source": [
    "# IF USING GENE LINKS, MAKE SURE THIS OUTPUT MATCHES YOUR EXPECTATIONS\n",
    "import requests\n",
    "from tqdm import tqdm_notebook as tqdm\n",
    "had_link = 0\n",
    "no_link = 0\n",
    "bad_links = pd.Series()\n",
    "for g in tqdm(ica_data.S.index):\n",
    "    if (g in links.keys()) and (type(links[g]) == str):\n",
    "        request = requests.get(links[g])\n",
    "        if request.status_code == 200: # good link\n",
    "            had_link += 1\n",
    "        else: # bad link\n",
    "            bad_links.loc[g] = links[g] # store so you can fix\n",
    "            links[g] = np.nan # remove the link from links dictionary\n",
    "    else:\n",
    "        no_link += 1\n",
    "num_genes = ica_data.S.shape[0]\n",
    "print('%i (%.2f%%) genes have working links.'%(had_link, had_link*100/num_genes))\n",
    "print('%i (%.2f%%) genes don\\'t have links.'%(no_link, no_link*100/num_genes))\n",
    "print('%i (%.2f%%) genes had broken links!'%(bad_links.shape[0], bad_links.shape[0]*100/num_genes))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## TF links\n",
    "\n",
    "When an iModulon is annotated (in curated_enrichments.csv) as having enriched transcription factors, we convert those TF strings into links. For all organisms except E. coli, we use the same gene database to assign those links. In E. coli, we prefer [RegulonDB](http://regulondb.ccg.unam.mx/) over EcoCyc, since the details there are more specific to regulatory activity. \n",
    "\n",
    "Note that the 'links' dictionary has LOCI has keys (ex: b0010), whereas tfs in the tf strings have names (ex: glpR). This will need to be reflected in the dictionaries."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "# CHOOSE TF LINKS METHOD BELOW\n",
    "\n",
    "# if using the same links \n",
    "# IF USING, DOUBLE CHECK BEHAVIOR\n",
    "#tf_links = {tf:links[ica_data.name2num[tf]] for tf in ica_data.gene_info.gene_name}\n",
    "\n",
    "# if reading in the links\n",
    "tf_links = pd.read_csv('../annotation/tf_links.csv', index_col = 1, header=None).to_dict()[2]\n",
    "\n",
    "# if using no tf_links\n",
    "#tf_links = {}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The code block below will print the HTML string that we will use for each iModulon. It will show 'nan' if the iModulon has no assigned regulator, plain text if there are no links, and a string with urls in it if the string has links. Make sure that the regulators you expect to have links do have links in this output."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120015690&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">dhaR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011240&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">mlc</a>\n",
      "histidine\n",
      "Sigma28\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013750&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cbl</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011312&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cysB</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125145850&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">btsR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125158264&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">ypdB</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012246&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">pdhR</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012854&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">purR</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125160661&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">puuR</a>\n",
      "argP or nadR\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125285350&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fur</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011265&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">nac</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125285350&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fur</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011345&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">arcA</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011953&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cdaR</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "L-tryptophan or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012176&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">trpR</a> or trp-tRNA\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120015234&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">prpR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011575&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">evgA</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011670&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">argR</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013786&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">xylR</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125285350&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fur</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011229&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fnr</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011437&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">glcC</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013014&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">atoC</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011502&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">narL</a>\n",
      "nan\n",
      "Sigma38\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125230705&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cecR</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012465&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gcvA</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120015630&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">allR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012316&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">araC</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013118&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fucR</a>\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013229&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">rbsR</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012092&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gadE</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011465&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gadW</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011458&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gadX</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012096&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gntR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011194&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">tyrR</a>\n",
      "L-tryptophan or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012176&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">trpR</a> or trp-tRNA\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012141&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">exuR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013118&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fucR</a>\n",
      "Sigma32\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011298&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">soxS</a>\n",
      "birA\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011302&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">oxyR</a>\n",
      "ile-tRNA or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012965&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">ilvY</a> or leu-tRNA\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011383&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">lrp</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120023539&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">nikR</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012730&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">glpR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011265&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">nac</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257191&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">zntR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012020&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">zur</a>\n",
      "gutR and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013141&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gutM</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013112&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cra</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011502&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">narL</a>\n",
      "nan\n",
      "TPP\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011908&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">nagC</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011194&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">tyrR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257192&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">ntrC</a> and Sigma54\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012218&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">malT</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257195&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">rcsA;rcsB</a>\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257194&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">flhD;flhC</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120014044&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">yiaJ</a>\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120048948&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">metJ</a>\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125285349&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">csqR</a>\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011235&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">crp</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011824&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fadR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011897&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">iclR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011458&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gadX</a> and <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011465&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">gadW</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011210&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cpxR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120013112&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cra</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011312&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cysB</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120012854&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">purR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011941&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cusR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125228798&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">hprR</a> or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257186&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">cueR</a>\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125257192&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">ntrC</a> and Sigma54\n",
      "nan\n",
      "nan\n",
      "Sigma19\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "fhlA or <a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011502&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">narL</a>\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011298&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">soxS</a>\n",
      "bluR\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK125285350&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">fur</a> and nrdR\n",
      "<a href=\"http://regulondb.ccg.unam.mx/regulon?term=ECK120011235&organism=ECK12&format=jsp&type=regulon\" target=\"_blank\">crp</a>\n"
     ]
    }
   ],
   "source": [
    "# TEST: MAKE SURE OUTPUT MAKES SENSE\n",
    "\n",
    "for tf in enrich.TF:\n",
    "    # if desired, find and modify the below function in general_functions.py\n",
    "    print(tf_with_links(tf, tf_links))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loop through all iModulons & Generate Files\n",
    "\n",
    "This is where errors are most likely to occur. This website is a work in progress - please email krychel@eng.ucsd.edu if you run into issues here."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_directory(ica_data, k, row, links, tf_links, base_conds):\n",
    "    \n",
    "    # generate the plot files\n",
    "    gene_table = gene_table_df(ica_data, k, row, links = links)\n",
    "    gene_hist = gene_hist_df(ica_data, k, row)\n",
    "    gene_scatter = gene_scatter_df(ica_data, k, base_conds, links)\n",
    "    act_bar = activity_bar_df(ica_data, k, ica_data.metadata.reset_index())\n",
    "\n",
    "    reg_venn = regulon_venn_df(ica_data, k, row)\n",
    "    reg_scatter = regulon_scatter_df(ica_data, k, row)\n",
    "\n",
    "    # generate a basic data df\n",
    "    res = pd.Series(index=['name', 'TF', 'Regulator',\n",
    "                           'Function', 'Category', \n",
    "                           'has_venn', 'scatter', 'has_meme'])\n",
    "    res.loc['name'] = row.loc['name']\n",
    "    res.loc['TF'] = row.TF\n",
    "    res.loc['Regulator'] = tf_with_links(row.TF, tf_links)\n",
    "    res.loc['Function'] = row.Function\n",
    "    res.loc['Category'] = row.Category\n",
    "    res.loc['has_venn'] = not(reg_venn is None)\n",
    "    if reg_scatter is None:\n",
    "        res.loc['scatter'] = 0\n",
    "    else:\n",
    "        res.loc['scatter'] = reg_scatter.shape[1] - 1\n",
    "    res.loc['has_meme'] = False #memes will be added to the website later.\n",
    "\n",
    "    # save output\n",
    "    folder = 'iModulon_files/'+str(k)+'/'\n",
    "    if not(os.path.isdir('iModulon_files')):\n",
    "        os.mkdir('iModulon_files')\n",
    "    if not(os.path.isdir(folder)):\n",
    "        os.mkdir(folder)\n",
    "    res.to_csv(folder+str(k)+'_meta.csv')\n",
    "    gene_table.to_csv(folder+str(k)+'_gene_table.csv')\n",
    "    gene_hist.to_csv(folder+str(k)+'_gene_hist.csv')\n",
    "    gene_scatter.to_csv(folder+str(k)+'_gene_scatter.csv')\n",
    "    act_bar.to_csv(folder+str(k)+'_activity_bar.csv')\n",
    "    if not(reg_venn is None):\n",
    "        reg_venn.to_csv(folder+str(k)+'_reg_venn.csv')\n",
    "    if not(reg_scatter is None):\n",
    "        reg_scatter.to_csv(folder+str(k)+'_reg_scatter.csv')\n",
    "    ica_data.S[k].to_csv(folder+str(k)+'_gene_weights.csv')\n",
    "    ica_data.A.loc[k].to_csv(folder+str(k)+'_activity.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "# suppress some warnings\n",
    "from scipy.optimize import OptimizeWarning\n",
    "warnings.filterwarnings('ignore', category=FutureWarning, append=True)\n",
    "warnings.filterwarnings('ignore', category=OptimizeWarning, append=True)\n",
    "pd.options.mode.chained_assignment = None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MISSING TF: histidine\n",
      "MISSING TF: histidine\n",
      "MISSING TF: Sigma28\n",
      "MISSING TF: Sigma28\n",
      "MISSING TF: Sigma38\n",
      "MISSING TF: Sigma38\n",
      "MISSING TF: Sigma32\n",
      "MISSING TF: Sigma32\n",
      "MISSING TF: gutR\n",
      "MISSING TF: gutR\n",
      "MISSING TF: Sigma54\n",
      "MISSING TF: Sigma54\n",
      "MISSING TF: Sigma54\n",
      "MISSING TF: Sigma54\n",
      "MISSING TF: Sigma19\n",
      "MISSING TF: Sigma19\n"
     ]
    }
   ],
   "source": [
    "# generate all files\n",
    "for k, row in enrich.iterrows():\n",
    "    make_directory(ica_data, k, row, links, tf_links, base_conds)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The output above will indicate if any of the TFs did not have genes with expression profiles associated with them. You can ignore missing TF expression profiles, unless you think a TF listed should have a gene and expression profile. If so, add the tf to ica_data.name2num with its appropriate index in X."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loop through all genes & Generate Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make these two data structures\n",
    "\n",
    "# generate df with all iM genes indicated T/F \n",
    "im_genes= ica_data.S.T.copy()\n",
    "for k,row in ica_data.S.T.iterrows():\n",
    "    im_genes.loc[k,:] = abs(row) > abs(ica_data.thresholds[k])\n",
    "    \n",
    "# iM_table: start with main component of iM page\n",
    "im_table = enrich[['name', 'Regulator', 'Function', 'Category']]\n",
    "im_table = im_table.rename(columns={'name':'Name'})\n",
    "im_table.index.name = 'k'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [],
   "source": [
    "# IF USING LINKS, PUT THE DATABASE NAME HERE\n",
    "gene_db = 'EcoCyc'\n",
    "\n",
    "def make_gene_files(ica_data, g, row, links, im_table, im_genes, gene_db):\n",
    "    \n",
    "    # generate the activity bar df\n",
    "    act_bar = gene_activity_bar_df(ica_data, g, ica_data.metadata.reset_index())\n",
    "    \n",
    "    # generate the iModulon table\n",
    "    perGene_table = im_table.copy()\n",
    "    perGene_table.insert(column ='in_iM', value = im_genes.loc[:, g], loc = 1)\n",
    "    perGene_table.insert(column ='gene_weight', value = ica_data.S.loc[g, :], loc = 2)\n",
    "    perGene_table = (perGene_table.assign(A=perGene_table['gene_weight'].abs())\n",
    "            .sort_values(['in_iM','A'],ascending=[False, False])\n",
    "            .drop('A', 1))\n",
    "    \n",
    "    # generate metadata df\n",
    "    res = pd.Series(index=['gene_id', 'name', 'operon', 'product',\n",
    "                           'cog', 'regulator(s)', 'link'])\n",
    "    res.loc['gene_id'] = g\n",
    "    res.loc['name'] = row.gene_name\n",
    "    res.loc['operon'] = row.operon\n",
    "    res.loc['product'] = row['product']\n",
    "    res.loc['cog'] = row.cog\n",
    "    res.loc['regulator(s)'] = \", \".join(ica_data.trn[ica_data.trn.gene_id == g].TF.to_list())\n",
    "    res.loc['link'] = '<a href=\"' + str(links[g]) + '\">'+ gene_db + '</a>'\n",
    "    res.fillna(value=\"<i> Not Available </i>\", inplace = True) \n",
    "    \n",
    "    # save output\n",
    "    folder = 'gene_page_files/'+str(g)+'/'\n",
    "    if not(os.path.isdir('gene_page_files')):\n",
    "        os.mkdir('gene_page_files')\n",
    "    if not(os.path.isdir(folder)):\n",
    "        os.mkdir(folder)\n",
    "    \n",
    "    res.to_csv(folder+str(g)+'_meta.csv', header=True)\n",
    "    act_bar.to_csv(folder+str(g)+'_activity_bar.csv')\n",
    "    ica_data.X.loc[g].to_csv(folder+str(g)+'_expression.csv')\n",
    "    perGene_table.to_csv(folder + str(g) + '_perGene_table.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Filter out genes with no ICA data\n",
    "reduced_gene_info = ica_data.gene_info.copy()\n",
    "reduced_gene_info = reduced_gene_info[reduced_gene_info.index.isin(ica_data.X.index)]\n",
    "\n",
    "# generate files for each gene\n",
    "for g, row in reduced_gene_info.iterrows():\n",
    "    make_gene_files(ica_data, g, row, links, im_table, im_genes, gene_db)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "# make json for gene search\n",
    "\n",
    "# Get df in correct format\n",
    "gene_df = reduced_gene_info.copy()\n",
    "gene_df[\"gene_id\"] = reduced_gene_info.index\n",
    "gene_df = gene_df[['gene_name', \"gene_id\", \"product\"]]\n",
    "gene_df = gene_df.sort_values(by=\"gene_name\").fillna(value = \"not available\")\n",
    "\n",
    "#create gene info json\n",
    "gene_df.to_json('./gene_page_files/gene_list.json', orient=\"records\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Dataset home: iModulon table\n",
    "\n",
    "Now, we have generated all the detailed individual pages. Let's generate a table of iModulons from curated_enrichments.csv, which will be the main part of the dataset home page.\n",
    "\n",
    "Default behavior is to sort the iModulons by category. You can use whatever category names you choose, but make sure the list in cat_order is exhaustive if you want to use this feature. You can also fill in the 'category_num' column with individual indices to put your iModulons in any specific order of your choice."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "# iM_table: this is the main component of the dataset page\n",
    "\n",
    "# CHOOSE THE ORDER YOU WOULD LIKE YOUR CATEGORIES IN\n",
    "cat_order = ['Carbon Source Utilization', \n",
    "             'Amino Acid and Nucleotide Biosynthesis',\n",
    "             'Energy Metabolism',\n",
    "             'Metal Homeostasis',\n",
    "             'Miscellaneous Metabolism',\n",
    "             'Structural Components',\n",
    "             'Stress Response',\n",
    "             'Regulator Discovery',\n",
    "             'Biological Enrichment',\n",
    "             'Genomic Alterations',\n",
    "             'Uncharacterized']\n",
    "cat_dict = {cat_order[i]:i for i in range(len(cat_order))}\n",
    "\n",
    "# make the table - note the required columns for enrich are here.\n",
    "im_table = enrich[['name', 'Regulator', 'Function', 'Category', 'n_genes', 'precision', 'recall']]\n",
    "im_table = im_table.rename(columns={'name':'Name'})\n",
    "im_table.Category = im_table.Category.fillna('Uncharacterized')\n",
    "im_table.index.name = 'k'\n",
    "im_table['category_num'] = [cat_dict[im_table.Category[k]] for k in im_table.index]\n",
    "im_table.to_csv('iM_table.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## General Dataset Details\n",
    "\n",
    "Almost done! Fill in these last bits of information for display on your dataset page.\n",
    "\n",
    "Note that in HTML, text can be formatted [like this](https://www.w3schools.com/html/html_formatting.asp). It is good practice to italicize organism names and use the correct capitalization (<i>Genus species</i>).\n",
    "\n",
    "The dataset_meta Series is read very simply in javascript - if you would like to add any additional notes in this space, feel free to append to the series (for example, you could include a link to another site that you may want to have here, or other project details)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "# dataset page header\n",
    "\n",
    "# FILL THESE DETAILS IN\n",
    "title = '<i>E. coli</i> ROS TALE'\n",
    "organism = '<i>Escherichia coli</i> K-12'\n",
    "strain = 'MG1655 and BW25113'\n",
    "\n",
    "# get these statistics\n",
    "num_genes = ica_data.X.shape[0]\n",
    "num_samps = ica_data.X.shape[1]\n",
    "num_conds = 0;\n",
    "for n, g in ica_data.metadata.groupby(['condition_id', 'project_id']):\n",
    "    num_conds += 1\n",
    "num_ims = ica_data.S.shape[1]\n",
    "\n",
    "dataset_meta = pd.Series({'Title':title,\n",
    "                          'Organism': organism,\n",
    "                          'Strain': strain,\n",
    "                          'Number of Samples': num_samps,\n",
    "                          'Number of Unique Conditions': num_conds,\n",
    "                          'Number of Genes':num_genes,\n",
    "                          'Number of iModulons': num_ims})\n",
    "dataset_meta.to_csv('dataset_meta.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "# num_ims - used so that the 'next iModulon' button doesn't overflow\n",
    "file = open('num_ims.txt', 'w')\n",
    "file.write(str(num_ims))\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Setting up the splash page\n",
    "\n",
    "The splash page, encoded by index.html, is the first thing we see when we go to the website. If you go to it now, you'll notice only the links to public datasets, even though we have now generated everything for these pages to exist. The following code will produce a bit of html that needs to be put in the right place in index.html."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "# splash page\n",
    "# FILL OUT THESE DETAILS\n",
    "large_title = '<i>E. coli</i> ROS'\n",
    "sub_title = 'Unpublished Reactive Oxygen Species Evolution Study'\n",
    "author = 'Rychel, et al.'\n",
    "\n",
    "cwd = os.getcwd().split('/')\n",
    "organism_dir = cwd[-2]\n",
    "dataset_dir = cwd[-1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "html = '<div class=\"col-md-4 col-lg-3\">\\n'\n",
    "html += '\\t<a role=\"button\" class=\"btn btn-primary btn-outline-light btn-block d-flex flex-column private_set\" href=\"dataset.html?organism='\n",
    "html += organism_dir\n",
    "html += '&dataset='\n",
    "html += dataset_dir\n",
    "html += '\">\\n\\t\\t<h2>'\n",
    "html += large_title\n",
    "html += '</h2>\\n\\t\\t<p>'\n",
    "html += sub_title\n",
    "html += '</p>\\n\\t\\t<p class=\"mt-auto\">'\n",
    "html += author\n",
    "html += '</p>\\n\\t</a>\\n</div>'\n",
    "\n",
    "file = open('html_for_splash.html', 'w')\n",
    "file.write(html)\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now, open the file 'html_for_splash.html' and copy its contents. Find the index.html page (in the root directory of the site) and look for the comment around line 200 that says 'INSERT NEW DATASETS...'. Paste the html there.\n",
    "\n",
    "### All Done!\n",
    "\n",
    "Return to [the wiki](https://github.com/SBRG/modulytics/wiki/Adding-a-Private-Project) or try viewing the site."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
