{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Nicolas File Generator\n",
    "\n",
    "This notebook generates all files for the Modulytics web page of Bacillus subtilis, based on ICA of Nicolas, et al., 2012 database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "# be sure that this point to a clone of github.com/SBRG/ICA\n",
    "sys.path.append('../../../../ica/')\n",
    "from icaviz.plotting import *\n",
    "DATA_DIR = 'data_files/'\n",
    "GENE_DIR = '../annotation/gene_files/'\n",
    "enrich = pd.read_csv(DATA_DIR + 'curated_enrichments.csv', index_col=0)\n",
    "names = enrich['name'].tolist()\n",
    "ica_data = load_data(X=DATA_DIR+'log_tpm.csv',\n",
    "                     S=DATA_DIR+'M.csv',\n",
    "                     A=DATA_DIR+'A.csv',\n",
    "                     metadata='sample_metadata.csv',\n",
    "                     annotation=GENE_DIR+'gene_info.csv',\n",
    "                     trn=GENE_DIR+'TRN.csv',\n",
    "                     cutoff=1300, \n",
    "                     organism='bsubtilis')\n",
    "ica_data.thresholds = enrich.threshold.to_dict()\n",
    "pd.set_option('display.max_rows', None)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# get functions from py files\n",
    "sys.path.append('../../../py')\n",
    "from gene_table import *\n",
    "from gene_histogram import *\n",
    "from gene_scatter import *\n",
    "from activity_bar import *\n",
    "from regulon_venn import *\n",
    "from regulon_scatter import *\n",
    "\n",
    "# gene dashboards\n",
    "from gene_activity_bar import *\n",
    "\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "links = pd.read_csv('../annotation/gene_links.csv', index_col = 0, header=None).to_dict()[1]\n",
    "sample_meta = ica_data.metadata.reset_index()\n",
    "gene_info = pd.read_csv(GENE_DIR+'gene_info.csv', index_col = 0)\n",
    "trn = pd.read_csv(GENE_DIR+'TRN.csv', index_col = 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Bacillus has overly complicated strings for regulation\n",
    "# E coli and future datasets: see any other file generator\n",
    "# for a different version of this function\n",
    "def tf_with_links(k, row):\n",
    "    tf_str = row.TF\n",
    "\n",
    "    if not(type(tf_str) == str):\n",
    "        return tf_str\n",
    "\n",
    "    # get a list of transcription factors\n",
    "    def convert_to_link(tf):\n",
    "        pre = 'http://subtiwiki.uni-goettingen.de/v3/gene/search/exact/'\n",
    "        link = pre + tf\n",
    "        return '<a href=\"' + link + '\" target=\"_blank\">'+ tf + '</a>'\n",
    "    \n",
    "    i = 0;\n",
    "    res = ''\n",
    "    while i < len(tf_str):\n",
    "        if tf_str[i] in '[ ]+/':\n",
    "            res += tf_str[i]\n",
    "            i += 1\n",
    "        else:\n",
    "            tf = tf_str[i:].partition(']')[0].partition(' ')[0]\n",
    "            res += convert_to_link(tf)\n",
    "            i += len(tf)\n",
    "    return res"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:17: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    }
   ],
   "source": [
    "# dataset_meta: Stores information for the header of the dataset page\n",
    "num_genes = ica_data.X.shape[0]\n",
    "num_samps = ica_data.X.shape[1]\n",
    "num_conds = 0;\n",
    "for n, g in ica_data.metadata.groupby(['condition_id', 'project_id']):\n",
    "    num_conds += 1\n",
    "num_ims = ica_data.S.shape[1]\n",
    "\n",
    "dataset_meta = pd.Series({'Title':'<i>B. subtilis</i> Microarray',\n",
    "                          'Organism': '<i>Bacillus subtilis</i>',\n",
    "                          'Strain': '168/BSB1',\n",
    "                          'Publication':'<a href=\"https://doi.org/10.1101/2020.04.26.062638\">Rychel, et al., 2020</a>',\n",
    "                          'Number of Samples': num_samps,\n",
    "                          'Number of Unique Conditions': num_conds,\n",
    "                          'Number of Genes':num_genes,\n",
    "                          'Number of iModulons': num_ims})\n",
    "dataset_meta.to_csv('dataset_meta.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "im_table = enrich[['name', 'Regulator', 'Function', 'Category', 'n_genes', 'precision', 'recall']]\n",
    "im_table = im_table.rename(columns={'name':'Name'})\n",
    "im_table.index.name = 'k'\n",
    "im_table['category_num'] = enrich.new_idx\n",
    "im_table.to_csv('iM_table.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "num_ims = im_table.shape[0]-1\n",
    "file = open('num_ims.txt', 'w')\n",
    "file.write(str(num_ims))\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>TF</th>\n",
       "      <th>high_conf</th>\n",
       "      <th>precision</th>\n",
       "      <th>recall</th>\n",
       "      <th>reg_type</th>\n",
       "      <th>Function</th>\n",
       "      <th>Category</th>\n",
       "      <th>category2</th>\n",
       "      <th>category3</th>\n",
       "      <th>new_idx</th>\n",
       "      <th>Regulator</th>\n",
       "      <th>threshold</th>\n",
       "      <th>n_genes</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>FadR – Fatty Acids</td>\n",
       "      <td>FadR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.739130</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Fatty acid degradation</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Fatty Acids</td>\n",
       "      <td>0</td>\n",
       "      <td>FadR</td>\n",
       "      <td>0.087360</td>\n",
       "      <td>17.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      <td>MalR – Malate</td>\n",
       "      <td>MalR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Malate uptake and utilization</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Malate</td>\n",
       "      <td>1</td>\n",
       "      <td>MalR</td>\n",
       "      <td>0.144234</td>\n",
       "      <td>5.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>38</td>\n",
       "      <td>GlpP – Glycerol</td>\n",
       "      <td>GlpP</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Glycerol uptake and utilization</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Glycerol</td>\n",
       "      <td>2</td>\n",
       "      <td>GlpP</td>\n",
       "      <td>0.106678</td>\n",
       "      <td>10.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>59</td>\n",
       "      <td>FruR – Fructose</td>\n",
       "      <td>FruR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Fructose uptake and utilization</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Fructose</td>\n",
       "      <td>3</td>\n",
       "      <td>FruR</td>\n",
       "      <td>0.206403</td>\n",
       "      <td>4.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>72</td>\n",
       "      <td>LevR – Levan and Fructose</td>\n",
       "      <td>LevR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Fuctose uptake and phosphorylation, degradatio...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Fructose</td>\n",
       "      <td>4</td>\n",
       "      <td>LevR</td>\n",
       "      <td>0.142297</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>61</td>\n",
       "      <td>FrlR – Amino Sugars</td>\n",
       "      <td>FrlR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Amino sugar uptake and metabolism</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Amino Sugars</td>\n",
       "      <td>5</td>\n",
       "      <td>FrlR</td>\n",
       "      <td>0.127036</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>30</td>\n",
       "      <td>ManR – Mannose</td>\n",
       "      <td>ManR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Utilization of mannose for cell wall synthesis...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Mannose</td>\n",
       "      <td>6</td>\n",
       "      <td>ManR</td>\n",
       "      <td>0.119812</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>AcoR – Acetoin</td>\n",
       "      <td>AcoR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.750000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Utilization of acetoin, extracellular energy s...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Acetoin</td>\n",
       "      <td>7</td>\n",
       "      <td>AcoR</td>\n",
       "      <td>0.153858</td>\n",
       "      <td>3.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80</td>\n",
       "      <td>LicR – Lichenan</td>\n",
       "      <td>LicR</td>\n",
       "      <td>1</td>\n",
       "      <td>0.444444</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>contains unknown</td>\n",
       "      <td>Uptake, phosphorylation, and utilization of le...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>Carbon Source</td>\n",
       "      <td>Lichenan</td>\n",
       "      <td>8</td>\n",
       "      <td>LicR</td>\n",
       "      <td>0.076635</td>\n",
       "      <td>9.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>82</td>\n",
       "      <td>CcpA-1 – Low Glucose 1</td>\n",
       "      <td>CcpA + SigA</td>\n",
       "      <td>0</td>\n",
       "      <td>0.794872</td>\n",
       "      <td>0.126531</td>\n",
       "      <td>subset</td>\n",
       "      <td>Uptake and utilization of alternative carbon s...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>General Carbon</td>\n",
       "      <td>Low glucose response</td>\n",
       "      <td>9</td>\n",
       "      <td>CcpA + SigA</td>\n",
       "      <td>0.057071</td>\n",
       "      <td>39.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      <td>CcpA-2 – Low Glucose 2</td>\n",
       "      <td>[CcpA + SigA] / [CcpN]</td>\n",
       "      <td>0</td>\n",
       "      <td>0.611765</td>\n",
       "      <td>0.203922</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Uptake, utilization, and regulation of alterna...</td>\n",
       "      <td>Carbon</td>\n",
       "      <td>General Carbon</td>\n",
       "      <td>Low glucose response</td>\n",
       "      <td>10</td>\n",
       "      <td>[CcpA + SigA] / [CcpN]</td>\n",
       "      <td>0.045382</td>\n",
       "      <td>85.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>52</td>\n",
       "      <td>His – Histidine Synthesis</td>\n",
       "      <td>YlxR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.186047</td>\n",
       "      <td>subset, operon</td>\n",
       "      <td>Synthesis of histidine</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Specific: Histidine</td>\n",
       "      <td>11</td>\n",
       "      <td>YlxR</td>\n",
       "      <td>0.129423</td>\n",
       "      <td>8.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>56</td>\n",
       "      <td>AhrC – Arginine</td>\n",
       "      <td>AhrC + YlxR</td>\n",
       "      <td>1</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Uptake and biosynthesis of arginine</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Specific: Arginine</td>\n",
       "      <td>12</td>\n",
       "      <td>AhrC + YlxR</td>\n",
       "      <td>0.105072</td>\n",
       "      <td>15.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>39</td>\n",
       "      <td>MtrB – Tryptophan</td>\n",
       "      <td>MtrB</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.555556</td>\n",
       "      <td>subset, operon</td>\n",
       "      <td>Biosynthesis of tryptophan</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Specific: Tryptophan</td>\n",
       "      <td>13</td>\n",
       "      <td>MtrB</td>\n",
       "      <td>0.174997</td>\n",
       "      <td>5.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>41</td>\n",
       "      <td>S-Box – Methionine</td>\n",
       "      <td>S-box</td>\n",
       "      <td>1</td>\n",
       "      <td>0.840000</td>\n",
       "      <td>0.525000</td>\n",
       "      <td>subset</td>\n",
       "      <td>Methionine salvage and biosynthesis</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Specific: Methionine</td>\n",
       "      <td>14</td>\n",
       "      <td>S-box</td>\n",
       "      <td>0.076127</td>\n",
       "      <td>25.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>71</td>\n",
       "      <td>TnrA + CodY – Leucine, Methionine, and Threonine</td>\n",
       "      <td>[TnrA + CodY + ThrR] / [TnrA + CodY + T-box + ...</td>\n",
       "      <td>1</td>\n",
       "      <td>0.423077</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>contains unknown</td>\n",
       "      <td>Synthesis and salvage of leucine, methionine, ...</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Several</td>\n",
       "      <td>15</td>\n",
       "      <td>[TnrA + CodY + ThrR] / [TnrA + CodY + T-box + ...</td>\n",
       "      <td>0.065844</td>\n",
       "      <td>26.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>CodY – BCAA Limitation</td>\n",
       "      <td>CodY</td>\n",
       "      <td>1</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.148936</td>\n",
       "      <td>subset</td>\n",
       "      <td>Uptake of di-/oligopeptides, uptake of guanosi...</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Synthesis</td>\n",
       "      <td>Several</td>\n",
       "      <td>16</td>\n",
       "      <td>CodY</td>\n",
       "      <td>0.065552</td>\n",
       "      <td>40.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>75</td>\n",
       "      <td>HutP – Histidine Utilization</td>\n",
       "      <td>HutP</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.857143</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Histidine uptake and utilization</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Use</td>\n",
       "      <td>Histidine</td>\n",
       "      <td>17</td>\n",
       "      <td>HutP</td>\n",
       "      <td>0.086899</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>60</td>\n",
       "      <td>RocR / PutR – Arginine and Proline Utilization</td>\n",
       "      <td>[SigL + AhrC + RocR] / [Spo0A + SigA + CodY + ...</td>\n",
       "      <td>1</td>\n",
       "      <td>0.923077</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Utilization of arginine, ornithine, citrulline...</td>\n",
       "      <td>Amino Acids</td>\n",
       "      <td>Amino Acid Use</td>\n",
       "      <td>Energy</td>\n",
       "      <td>18</td>\n",
       "      <td>[SigL + AhrC + RocR] / [Spo0A + SigA + CodY + ...</td>\n",
       "      <td>0.093177</td>\n",
       "      <td>13.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>50</td>\n",
       "      <td>PyrR – Pyrimidines</td>\n",
       "      <td>PyrR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.785714</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Synthesis of pyrimidines</td>\n",
       "      <td>Other Organics</td>\n",
       "      <td>Nucleotides</td>\n",
       "      <td>Pyrimidines</td>\n",
       "      <td>19</td>\n",
       "      <td>PyrR</td>\n",
       "      <td>0.117698</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>58</td>\n",
       "      <td>PurR – Purines</td>\n",
       "      <td>PurR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Synthesis of purine, xanthine and guanine upta...</td>\n",
       "      <td>Other Organics</td>\n",
       "      <td>Nucleotides</td>\n",
       "      <td>Purines</td>\n",
       "      <td>20</td>\n",
       "      <td>PurR</td>\n",
       "      <td>0.089225</td>\n",
       "      <td>20.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>70</td>\n",
       "      <td>Thi-Box – Thiamine</td>\n",
       "      <td>Thi-box</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.833333</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Synthesis and uptake of thiamine</td>\n",
       "      <td>Other Organics</td>\n",
       "      <td>Cofactors</td>\n",
       "      <td>Thiamine</td>\n",
       "      <td>21</td>\n",
       "      <td>Thi-box</td>\n",
       "      <td>0.095194</td>\n",
       "      <td>15.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>PchR – Pulcherrimin</td>\n",
       "      <td>PchR</td>\n",
       "      <td>1</td>\n",
       "      <td>0.857143</td>\n",
       "      <td>0.600000</td>\n",
       "      <td>subset</td>\n",
       "      <td>Extracellular synthesis and import of pulcherr...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Iron</td>\n",
       "      <td>22</td>\n",
       "      <td>PchR</td>\n",
       "      <td>0.081261</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>57</td>\n",
       "      <td>Fur – Iron Limitation</td>\n",
       "      <td>Fur</td>\n",
       "      <td>1</td>\n",
       "      <td>0.890909</td>\n",
       "      <td>0.597561</td>\n",
       "      <td>subset</td>\n",
       "      <td>Production and transport of siderophores for i...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Iron</td>\n",
       "      <td>23</td>\n",
       "      <td>Fur</td>\n",
       "      <td>0.047500</td>\n",
       "      <td>55.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>77</td>\n",
       "      <td>PhoP-1 – Phosphate Limitation 1</td>\n",
       "      <td>PhoP + SigA</td>\n",
       "      <td>1</td>\n",
       "      <td>0.727273</td>\n",
       "      <td>0.114286</td>\n",
       "      <td>subset</td>\n",
       "      <td>High-affinity phosphate uptake, teichuronic ac...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Phosphate</td>\n",
       "      <td>24</td>\n",
       "      <td>PhoP + SigA</td>\n",
       "      <td>0.086751</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74</td>\n",
       "      <td>PhoP-2 – Phosphate Limitation 2</td>\n",
       "      <td>PhoP + SigA</td>\n",
       "      <td>1</td>\n",
       "      <td>0.954545</td>\n",
       "      <td>0.300000</td>\n",
       "      <td>subset</td>\n",
       "      <td>Degradation of wall teichoic acid to salvage p...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Phosphate</td>\n",
       "      <td>25</td>\n",
       "      <td>PhoP + SigA</td>\n",
       "      <td>0.089585</td>\n",
       "      <td>22.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>45</td>\n",
       "      <td>Zur – Zinc Limitation</td>\n",
       "      <td>Zur</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.923077</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Zinc uptake, zinc metallochaperones, alternati...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Zinc</td>\n",
       "      <td>26</td>\n",
       "      <td>Zur</td>\n",
       "      <td>0.104663</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>32</td>\n",
       "      <td>CzrA – Toxic Cation Export</td>\n",
       "      <td>CzrA</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.833333</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Resistance to Zn, Cu, Co, Ni, and Cd</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Ion Homeostasis</td>\n",
       "      <td>Toxic Cations</td>\n",
       "      <td>27</td>\n",
       "      <td>CzrA</td>\n",
       "      <td>0.155808</td>\n",
       "      <td>5.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>69</td>\n",
       "      <td>CymR – Sulfur</td>\n",
       "      <td>CymR</td>\n",
       "      <td>1</td>\n",
       "      <td>0.755556</td>\n",
       "      <td>0.693878</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Sulfur metabolism in response to oxidative str...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Sulfur Homeostasis</td>\n",
       "      <td>Oxidative Stress</td>\n",
       "      <td>28</td>\n",
       "      <td>CymR</td>\n",
       "      <td>0.046438</td>\n",
       "      <td>45.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      <td>Yrk – Putative Sulfur Carriers</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>unknown</td>\n",
       "      <td>Putative sulfur carriers. High activity under ...</td>\n",
       "      <td>Elements</td>\n",
       "      <td>Sulfur Homeostasis</td>\n",
       "      <td>Putative</td>\n",
       "      <td>29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.126741</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>ResD – Oxygen Limitation</td>\n",
       "      <td>ResD</td>\n",
       "      <td>0</td>\n",
       "      <td>0.434211</td>\n",
       "      <td>0.478261</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Anaerobic nitrate respiration, bacteriocin pro...</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Pleiotropic Energy</td>\n",
       "      <td>30</td>\n",
       "      <td>ResD</td>\n",
       "      <td>0.046826</td>\n",
       "      <td>76.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      <td>Fnr – Nitrate Respiration</td>\n",
       "      <td>Fnr + SigA</td>\n",
       "      <td>1</td>\n",
       "      <td>0.888889</td>\n",
       "      <td>0.888889</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Nitrate respiration and nitrite extrusion</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Nitrate Respiration</td>\n",
       "      <td>31</td>\n",
       "      <td>Fnr + SigA</td>\n",
       "      <td>0.120142</td>\n",
       "      <td>9.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>44</td>\n",
       "      <td>Rex – Overflow Metabolism</td>\n",
       "      <td>Rex</td>\n",
       "      <td>1</td>\n",
       "      <td>0.900000</td>\n",
       "      <td>0.529412</td>\n",
       "      <td>subset</td>\n",
       "      <td>Overflow metabolism (LDH) and cytochrome BD ox...</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Oxygen Limitation</td>\n",
       "      <td>Overflow</td>\n",
       "      <td>32</td>\n",
       "      <td>Rex</td>\n",
       "      <td>0.081571</td>\n",
       "      <td>10.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>49</td>\n",
       "      <td>LexA – DNA Repair</td>\n",
       "      <td>LexA</td>\n",
       "      <td>1</td>\n",
       "      <td>0.777778</td>\n",
       "      <td>0.272727</td>\n",
       "      <td>subset</td>\n",
       "      <td>SOS response for DNA protection and repair</td>\n",
       "      <td>Stress</td>\n",
       "      <td>Specific Stressors</td>\n",
       "      <td>DNA Damage</td>\n",
       "      <td>33</td>\n",
       "      <td>LexA</td>\n",
       "      <td>0.071558</td>\n",
       "      <td>27.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>SigW – Cell Wall Stress</td>\n",
       "      <td>SigW</td>\n",
       "      <td>1</td>\n",
       "      <td>0.940000</td>\n",
       "      <td>0.435185</td>\n",
       "      <td>subset</td>\n",
       "      <td>Adaptation to membrane active agents such as c...</td>\n",
       "      <td>Stress</td>\n",
       "      <td>Specific Stressors</td>\n",
       "      <td>Wall/Membrane Stress</td>\n",
       "      <td>34</td>\n",
       "      <td>SigW</td>\n",
       "      <td>0.061503</td>\n",
       "      <td>50.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>47</td>\n",
       "      <td>SigM – Membrane Stress</td>\n",
       "      <td>SigM</td>\n",
       "      <td>1</td>\n",
       "      <td>0.954545</td>\n",
       "      <td>0.162791</td>\n",
       "      <td>subset</td>\n",
       "      <td>General stress proteins, alarmones, lipid carr...</td>\n",
       "      <td>Stress</td>\n",
       "      <td>Specific Stressors</td>\n",
       "      <td>Wall/Membrane Stress</td>\n",
       "      <td>35</td>\n",
       "      <td>SigM</td>\n",
       "      <td>0.077209</td>\n",
       "      <td>22.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      <td>Ybc – Uncharacterized Operon</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>unknown</td>\n",
       "      <td>Contains one subunit of NADH dehydrogenase wit...</td>\n",
       "      <td>Stress</td>\n",
       "      <td>Specific Stressors</td>\n",
       "      <td>Possible Heat Shock</td>\n",
       "      <td>36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.162882</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>SigB-1 – General Stress 1</td>\n",
       "      <td>SigB</td>\n",
       "      <td>1</td>\n",
       "      <td>0.864407</td>\n",
       "      <td>0.323810</td>\n",
       "      <td>subset</td>\n",
       "      <td>Survival of general stressors including ethano...</td>\n",
       "      <td>Stress</td>\n",
       "      <td>General Stress</td>\n",
       "      <td>General Stress</td>\n",
       "      <td>37</td>\n",
       "      <td>SigB</td>\n",
       "      <td>0.033604</td>\n",
       "      <td>118.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>46</td>\n",
       "      <td>SigB-2 – General Stress 2</td>\n",
       "      <td>SigB</td>\n",
       "      <td>1</td>\n",
       "      <td>0.600000</td>\n",
       "      <td>0.095238</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Likely accounting for microarray saturation in...</td>\n",
       "      <td>Stress</td>\n",
       "      <td>General Stress</td>\n",
       "      <td>General Stress</td>\n",
       "      <td>38</td>\n",
       "      <td>SigB</td>\n",
       "      <td>0.057723</td>\n",
       "      <td>50.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      <td>StrRes – Translation</td>\n",
       "      <td>[stringent response] / [RplJ]</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.260000</td>\n",
       "      <td>subset</td>\n",
       "      <td>Production of ribosomes, translation initiatio...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Translation</td>\n",
       "      <td>Translation</td>\n",
       "      <td>39</td>\n",
       "      <td>[stringent response] / [RplJ]</td>\n",
       "      <td>0.062382</td>\n",
       "      <td>52.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>SigD – Motility</td>\n",
       "      <td>SigD</td>\n",
       "      <td>1</td>\n",
       "      <td>0.914894</td>\n",
       "      <td>0.573333</td>\n",
       "      <td>subset</td>\n",
       "      <td>Production of flagella components and export m...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Motility</td>\n",
       "      <td>Motility</td>\n",
       "      <td>40</td>\n",
       "      <td>SigD</td>\n",
       "      <td>0.056403</td>\n",
       "      <td>94.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>Eps – Exopolymeric Substances</td>\n",
       "      <td>SigA + AbrB + RemA + SinR</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>0.950000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Biosynthesis of tasA and extracellular polysac...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Biofilms</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41</td>\n",
       "      <td>SigA + AbrB + RemA + SinR</td>\n",
       "      <td>0.116051</td>\n",
       "      <td>19.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>53</td>\n",
       "      <td>ComA – Surfactants</td>\n",
       "      <td>[ComA + CcpA + TnrA] / [Abh + SigA + ComA + Sp...</td>\n",
       "      <td>1</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Activated during transition to stationary phas...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Biofilms</td>\n",
       "      <td>NaN</td>\n",
       "      <td>42</td>\n",
       "      <td>[ComA + CcpA + TnrA] / [Abh + SigA + ComA + Sp...</td>\n",
       "      <td>0.141925</td>\n",
       "      <td>7.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>54</td>\n",
       "      <td>DegU – Toxins and Colony Establishment</td>\n",
       "      <td>DegU</td>\n",
       "      <td>0</td>\n",
       "      <td>0.325000</td>\n",
       "      <td>0.122642</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Response to arrested flagellar motor. Secretio...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Biofilms</td>\n",
       "      <td>NaN</td>\n",
       "      <td>43</td>\n",
       "      <td>DegU</td>\n",
       "      <td>0.065658</td>\n",
       "      <td>40.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36</td>\n",
       "      <td>AbrB – Transition State</td>\n",
       "      <td>AbrB</td>\n",
       "      <td>1</td>\n",
       "      <td>0.833333</td>\n",
       "      <td>0.106667</td>\n",
       "      <td>subset</td>\n",
       "      <td>Various transition state roles: spore killing,...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Biofilms</td>\n",
       "      <td>NaN</td>\n",
       "      <td>44</td>\n",
       "      <td>AbrB</td>\n",
       "      <td>0.053203</td>\n",
       "      <td>48.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>79</td>\n",
       "      <td>Alb – Antilisterial Bacteriocin</td>\n",
       "      <td>Rok + ResD + AbrB + SigA</td>\n",
       "      <td>1</td>\n",
       "      <td>0.666667</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Produces and exports subtilosin, which kills a...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Toxin Production</td>\n",
       "      <td>NaN</td>\n",
       "      <td>45</td>\n",
       "      <td>Rok + ResD + AbrB + SigA</td>\n",
       "      <td>0.069789</td>\n",
       "      <td>12.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      <td>WapA – Intercellular Competition</td>\n",
       "      <td>SigA + YvrHb + DegU + WalR</td>\n",
       "      <td>0</td>\n",
       "      <td>0.176471</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>contains unknown</td>\n",
       "      <td>Activation of WapA and its operon of unknown g...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Toxin Production</td>\n",
       "      <td>NaN</td>\n",
       "      <td>46</td>\n",
       "      <td>SigA + YvrHb + DegU + WalR</td>\n",
       "      <td>0.116118</td>\n",
       "      <td>17.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>51</td>\n",
       "      <td>ComK – Competence</td>\n",
       "      <td>ComK</td>\n",
       "      <td>1</td>\n",
       "      <td>0.612245</td>\n",
       "      <td>0.405405</td>\n",
       "      <td>closest match</td>\n",
       "      <td>DNA transport, pseudopilus, recombination mach...</td>\n",
       "      <td>Lifestyles</td>\n",
       "      <td>Competence</td>\n",
       "      <td>NaN</td>\n",
       "      <td>47</td>\n",
       "      <td>ComK</td>\n",
       "      <td>0.050085</td>\n",
       "      <td>49.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35</td>\n",
       "      <td>Spo0A – Sporulation 1</td>\n",
       "      <td>[Spo0A] / [DnaA]</td>\n",
       "      <td>0</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>0.051724</td>\n",
       "      <td>subset</td>\n",
       "      <td>Initiation of sporulation through sigma factor...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>0</td>\n",
       "      <td>48</td>\n",
       "      <td>[Spo0A] / [DnaA]</td>\n",
       "      <td>0.076391</td>\n",
       "      <td>15.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>48</td>\n",
       "      <td>SigE – Sporulation 2</td>\n",
       "      <td>SigE</td>\n",
       "      <td>0</td>\n",
       "      <td>0.638710</td>\n",
       "      <td>0.327815</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Spore maturation through core dehydration, nut...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>1</td>\n",
       "      <td>49</td>\n",
       "      <td>SigE</td>\n",
       "      <td>0.042301</td>\n",
       "      <td>155.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>68</td>\n",
       "      <td>SigE / G – Sporulation 3</td>\n",
       "      <td>[SigE] / [SigG]</td>\n",
       "      <td>0</td>\n",
       "      <td>0.794118</td>\n",
       "      <td>0.113924</td>\n",
       "      <td>subset</td>\n",
       "      <td>Spore coat production, transcription factor an...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>2</td>\n",
       "      <td>50</td>\n",
       "      <td>[SigE] / [SigG]</td>\n",
       "      <td>0.049619</td>\n",
       "      <td>68.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>66</td>\n",
       "      <td>SigG – Sporulation 4</td>\n",
       "      <td>SigG</td>\n",
       "      <td>0</td>\n",
       "      <td>0.620915</td>\n",
       "      <td>0.524862</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Calcium and nutrient delivery to the spore, en...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>3</td>\n",
       "      <td>51</td>\n",
       "      <td>SigG</td>\n",
       "      <td>0.042330</td>\n",
       "      <td>153.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      <td>SigK-1 – Sporulation 5</td>\n",
       "      <td>SigK</td>\n",
       "      <td>0</td>\n",
       "      <td>0.618705</td>\n",
       "      <td>0.480447</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Production of spore coat and crust proteins, L...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>5</td>\n",
       "      <td>52</td>\n",
       "      <td>SigK</td>\n",
       "      <td>0.034650</td>\n",
       "      <td>139.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      <td>SigK-2 – Sporulation 6</td>\n",
       "      <td>SigK</td>\n",
       "      <td>0</td>\n",
       "      <td>0.556701</td>\n",
       "      <td>0.301676</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Production of outermost spore coat and crust p...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>6</td>\n",
       "      <td>53</td>\n",
       "      <td>SigK</td>\n",
       "      <td>0.035362</td>\n",
       "      <td>97.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      <td>GerE / GerR – Mid-late Spore Coat</td>\n",
       "      <td>[SigK + GerE] / [SigK + GerR]</td>\n",
       "      <td>0</td>\n",
       "      <td>0.764706</td>\n",
       "      <td>0.127451</td>\n",
       "      <td>subset</td>\n",
       "      <td>Production of spore coat and crust proteins. O...</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>Sporulation</td>\n",
       "      <td>4</td>\n",
       "      <td>54</td>\n",
       "      <td>[SigK + GerE] / [SigK + GerR]</td>\n",
       "      <td>0.092769</td>\n",
       "      <td>17.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>65</td>\n",
       "      <td>Xpf – PBSX Prophage</td>\n",
       "      <td>Xpf</td>\n",
       "      <td>1</td>\n",
       "      <td>0.878788</td>\n",
       "      <td>1.000000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Prophage element that induces production of ba...</td>\n",
       "      <td>Mobile Genes</td>\n",
       "      <td>Deleterious Prophages</td>\n",
       "      <td>PBSX</td>\n",
       "      <td>55</td>\n",
       "      <td>Xpf</td>\n",
       "      <td>0.081181</td>\n",
       "      <td>33.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      <td>Spβ-1 – SP-β Prophage 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>unknown</td>\n",
       "      <td>Temperature-responsive SP-beta prophage. Respo...</td>\n",
       "      <td>Mobile Genes</td>\n",
       "      <td>Deleterious Prophages</td>\n",
       "      <td>SPBeta</td>\n",
       "      <td>56</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.100877</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>64</td>\n",
       "      <td>Spβ-2 – SP-β Prophage 2</td>\n",
       "      <td>CsoR + SigA + SigK + SigE</td>\n",
       "      <td>0</td>\n",
       "      <td>0.093960</td>\n",
       "      <td>0.933333</td>\n",
       "      <td>contains unknown</td>\n",
       "      <td>Temperature-responsive SP-beta prophage. Proph...</td>\n",
       "      <td>Mobile Genes</td>\n",
       "      <td>Deleterious Prophages</td>\n",
       "      <td>SPBeta</td>\n",
       "      <td>57</td>\n",
       "      <td>CsoR + SigA + SigK + SigE</td>\n",
       "      <td>0.036485</td>\n",
       "      <td>149.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      <td>ImmR – ICEBs1</td>\n",
       "      <td>ImmR</td>\n",
       "      <td>1</td>\n",
       "      <td>0.950000</td>\n",
       "      <td>0.950000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Mobile genetic element that participates in ho...</td>\n",
       "      <td>Mobile Genes</td>\n",
       "      <td>Other Prophages</td>\n",
       "      <td>NaN</td>\n",
       "      <td>58</td>\n",
       "      <td>ImmR</td>\n",
       "      <td>0.078043</td>\n",
       "      <td>20.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      <td>SknR – Skin Element</td>\n",
       "      <td>SknR</td>\n",
       "      <td>0</td>\n",
       "      <td>0.352941</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Prophage element that disrupts the SigK gene b...</td>\n",
       "      <td>Mobile Genes</td>\n",
       "      <td>Other Prophages</td>\n",
       "      <td>NaN</td>\n",
       "      <td>59</td>\n",
       "      <td>SknR</td>\n",
       "      <td>0.052144</td>\n",
       "      <td>17.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      <td>IolR / MelR – Inositol, Melibiose, and Acetoin</td>\n",
       "      <td>[IolR] / [MelR] / [AcoR]</td>\n",
       "      <td>1</td>\n",
       "      <td>0.896552</td>\n",
       "      <td>0.928571</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Uptake and utilization of inositol, melibiose/...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Metabolism</td>\n",
       "      <td>Several Specific Sources</td>\n",
       "      <td>60</td>\n",
       "      <td>[IolR] / [MelR] / [AcoR]</td>\n",
       "      <td>0.076520</td>\n",
       "      <td>29.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>40</td>\n",
       "      <td>NadR / BirA – NAD and Biotin</td>\n",
       "      <td>[NadR] / [BirA]</td>\n",
       "      <td>1</td>\n",
       "      <td>0.681818</td>\n",
       "      <td>0.714286</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Biosynthesis of NAD and biotin, plus safA &amp; co...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Metabolism</td>\n",
       "      <td>NAD, Biotin</td>\n",
       "      <td>61</td>\n",
       "      <td>[NadR] / [BirA]</td>\n",
       "      <td>0.098662</td>\n",
       "      <td>22.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>81</td>\n",
       "      <td>MtlR / AnsR – Mannitol, Asparagine, and Aspartate</td>\n",
       "      <td>[MtlR] / [AnsR + SigA]</td>\n",
       "      <td>1</td>\n",
       "      <td>0.727273</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>well defined</td>\n",
       "      <td>Uptake and utilization of mannitol, degradatio...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Metabolism</td>\n",
       "      <td>Mannitol, Asparagine, Aspartate</td>\n",
       "      <td>62</td>\n",
       "      <td>[MtlR] / [AnsR + SigA]</td>\n",
       "      <td>0.091097</td>\n",
       "      <td>11.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>73</td>\n",
       "      <td>TnrA / PucR – Nitrogen Limitation</td>\n",
       "      <td>[SigA + TnrA] / [SigA + PucR]</td>\n",
       "      <td>1</td>\n",
       "      <td>0.900000</td>\n",
       "      <td>0.169811</td>\n",
       "      <td>subset</td>\n",
       "      <td>Ammonium uptake, utilization of nitrate and xa...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Metabolism</td>\n",
       "      <td>Nitrogen Ions</td>\n",
       "      <td>63</td>\n",
       "      <td>[SigA + TnrA] / [SigA + PucR]</td>\n",
       "      <td>0.101788</td>\n",
       "      <td>10.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>67</td>\n",
       "      <td>CymR / NsrR – Sulfur and Nitrate Respiration</td>\n",
       "      <td>[SigA + CymR] / [ResD + SigA + NsrR]</td>\n",
       "      <td>1</td>\n",
       "      <td>0.909091</td>\n",
       "      <td>0.338983</td>\n",
       "      <td>subset</td>\n",
       "      <td>Import and utilization of sulfonate and cystin...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Metabolism</td>\n",
       "      <td>Sulfur Limitation</td>\n",
       "      <td>64</td>\n",
       "      <td>[SigA + CymR] / [ResD + SigA + NsrR]</td>\n",
       "      <td>0.081852</td>\n",
       "      <td>22.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      <td>ArsR+ – Diamide Stress</td>\n",
       "      <td>[ArsR] / [YodB] / [SigA + CtsR] / [SigA + CsoR]</td>\n",
       "      <td>0</td>\n",
       "      <td>0.466667</td>\n",
       "      <td>0.280000</td>\n",
       "      <td>closest match</td>\n",
       "      <td>Detoxification in response to: oxidative prote...</td>\n",
       "      <td>Multi-Purpose</td>\n",
       "      <td>Multi-Purpose Stress</td>\n",
       "      <td>Oxidative Stress</td>\n",
       "      <td>65</td>\n",
       "      <td>[ArsR] / [YodB] / [SigA + CtsR] / [SigA + CsoR]</td>\n",
       "      <td>0.064922</td>\n",
       "      <td>30.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      <td>Uncharacterized 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains a SigE/SigG inhibitor, spore coat pro...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>66</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.065454</td>\n",
       "      <td>30.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>43</td>\n",
       "      <td>Uncharacterized 2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains cold shock, sporulation onset, toxin,...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>NaN</td>\n",
       "      <td>67</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.054663</td>\n",
       "      <td>40.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>33</td>\n",
       "      <td>Uncharacterized 3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains many poorly characterized RNAs with n...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>Putative Noise</td>\n",
       "      <td>68</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.048538</td>\n",
       "      <td>80.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>34</td>\n",
       "      <td>Uncharacterized 4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains many poorly characterized RNAs with n...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>Putative Noise</td>\n",
       "      <td>69</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.049752</td>\n",
       "      <td>62.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      <td>Uncharacterized 5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains many poorly characterized RNAs with n...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>Putative Noise</td>\n",
       "      <td>70</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.069265</td>\n",
       "      <td>19.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      <td>Uncharacterized 6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5’ ends of tRNAs and independent transcripts</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Groups of Genes</td>\n",
       "      <td>Putative Noise</td>\n",
       "      <td>71</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.074451</td>\n",
       "      <td>6.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>62</td>\n",
       "      <td>Noise – S144</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Likely accounting for noisey RNA S144, an inde...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Single RNA</td>\n",
       "      <td>72</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.217807</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      <td>Noise – S415</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Noisy independent transcript S415</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Single RNA</td>\n",
       "      <td>73</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.112846</td>\n",
       "      <td>2.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      <td>Noise – S1235</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Independent transcript S1235, behaves similar ...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Single RNA</td>\n",
       "      <td>74</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.122629</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>31</td>\n",
       "      <td>Noise – S1210</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Noisy independent transcript S1210</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Single RNA</td>\n",
       "      <td>75</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.142092</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>42</td>\n",
       "      <td>Noise – Carbon Source Transition 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Likely accounting for noise in the carbon sour...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Samples</td>\n",
       "      <td>76</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.073841</td>\n",
       "      <td>22.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>63</td>\n",
       "      <td>Noise – Carbon Source Transition 2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Likely accounting for noise in carbon source t...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Samples</td>\n",
       "      <td>77</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.060006</td>\n",
       "      <td>30.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>55</td>\n",
       "      <td>Noise – LBGexp_1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Likely accounting for noise in the first sampl...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Samples</td>\n",
       "      <td>78</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.055977</td>\n",
       "      <td>48.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>37</td>\n",
       "      <td>Noise – Diamide 15</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Likely accounting for noise in the diamide aft...</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Noisy Genes</td>\n",
       "      <td>Samples</td>\n",
       "      <td>79</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.095907</td>\n",
       "      <td>5.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      <td>Empty 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains no genes</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Empty</td>\n",
       "      <td>NaN</td>\n",
       "      <td>80</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.126665</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>76</td>\n",
       "      <td>Empty 2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains no genes</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Empty</td>\n",
       "      <td>NaN</td>\n",
       "      <td>81</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.163114</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>78</td>\n",
       "      <td>Empty 3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Contains no genes</td>\n",
       "      <td>Uncharacterized</td>\n",
       "      <td>Empty</td>\n",
       "      <td>NaN</td>\n",
       "      <td>82</td>\n",
       "      <td>NaN</td>\n",
       "      <td>0.123938</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 name  \\\n",
       "1                                  FadR – Fatty Acids   \n",
       "6                                       MalR – Malate   \n",
       "38                                    GlpP – Glycerol   \n",
       "59                                    FruR – Fructose   \n",
       "72                          LevR – Levan and Fructose   \n",
       "61                                FrlR – Amino Sugars   \n",
       "30                                     ManR – Mannose   \n",
       "3                                      AcoR – Acetoin   \n",
       "80                                    LicR – Lichenan   \n",
       "82                             CcpA-1 – Low Glucose 1   \n",
       "10                             CcpA-2 – Low Glucose 2   \n",
       "52                          His – Histidine Synthesis   \n",
       "56                                    AhrC – Arginine   \n",
       "39                                  MtrB – Tryptophan   \n",
       "41                                 S-Box – Methionine   \n",
       "71   TnrA + CodY – Leucine, Methionine, and Threonine   \n",
       "2                              CodY – BCAA Limitation   \n",
       "75                       HutP – Histidine Utilization   \n",
       "60     RocR / PutR – Arginine and Proline Utilization   \n",
       "50                                 PyrR – Pyrimidines   \n",
       "58                                     PurR – Purines   \n",
       "70                                 Thi-Box – Thiamine   \n",
       "0                                 PchR – Pulcherrimin   \n",
       "57                              Fur – Iron Limitation   \n",
       "77                    PhoP-1 – Phosphate Limitation 1   \n",
       "74                    PhoP-2 – Phosphate Limitation 2   \n",
       "45                              Zur – Zinc Limitation   \n",
       "32                         CzrA – Toxic Cation Export   \n",
       "69                                      CymR – Sulfur   \n",
       "12                     Yrk – Putative Sulfur Carriers   \n",
       "9                            ResD – Oxygen Limitation   \n",
       "8                           Fnr – Nitrate Respiration   \n",
       "44                          Rex – Overflow Metabolism   \n",
       "49                                  LexA – DNA Repair   \n",
       "4                             SigW – Cell Wall Stress   \n",
       "47                             SigM – Membrane Stress   \n",
       "16                       Ybc – Uncharacterized Operon   \n",
       "7                           SigB-1 – General Stress 1   \n",
       "46                          SigB-2 – General Stress 2   \n",
       "21                               StrRes – Translation   \n",
       "23                                    SigD – Motility   \n",
       "5                       Eps – Exopolymeric Substances   \n",
       "53                                 ComA – Surfactants   \n",
       "54             DegU – Toxins and Colony Establishment   \n",
       "36                            AbrB – Transition State   \n",
       "79                    Alb – Antilisterial Bacteriocin   \n",
       "14                   WapA – Intercellular Competition   \n",
       "51                                  ComK – Competence   \n",
       "35                              Spo0A – Sporulation 1   \n",
       "48                               SigE – Sporulation 2   \n",
       "68                           SigE / G – Sporulation 3   \n",
       "66                               SigG – Sporulation 4   \n",
       "13                             SigK-1 – Sporulation 5   \n",
       "25                             SigK-2 – Sporulation 6   \n",
       "15                  GerE / GerR – Mid-late Spore Coat   \n",
       "65                                Xpf – PBSX Prophage   \n",
       "22                            Spβ-1 – SP-β Prophage 1   \n",
       "64                            Spβ-2 – SP-β Prophage 2   \n",
       "18                                      ImmR – ICEBs1   \n",
       "27                                SknR – Skin Element   \n",
       "20     IolR / MelR – Inositol, Melibiose, and Acetoin   \n",
       "40                       NadR / BirA – NAD and Biotin   \n",
       "81  MtlR / AnsR – Mannitol, Asparagine, and Aspartate   \n",
       "73                  TnrA / PucR – Nitrogen Limitation   \n",
       "67       CymR / NsrR – Sulfur and Nitrate Respiration   \n",
       "11                             ArsR+ – Diamide Stress   \n",
       "17                                  Uncharacterized 1   \n",
       "43                                  Uncharacterized 2   \n",
       "33                                  Uncharacterized 3   \n",
       "34                                  Uncharacterized 4   \n",
       "24                                  Uncharacterized 5   \n",
       "26                                  Uncharacterized 6   \n",
       "62                                       Noise – S144   \n",
       "28                                       Noise – S415   \n",
       "29                                      Noise – S1235   \n",
       "31                                      Noise – S1210   \n",
       "42                 Noise – Carbon Source Transition 1   \n",
       "63                 Noise – Carbon Source Transition 2   \n",
       "55                                   Noise – LBGexp_1   \n",
       "37                                 Noise – Diamide 15   \n",
       "19                                            Empty 1   \n",
       "76                                            Empty 2   \n",
       "78                                            Empty 3   \n",
       "\n",
       "                                                   TF  high_conf  precision  \\\n",
       "1                                                FadR          1   1.000000   \n",
       "6                                                MalR          1   1.000000   \n",
       "38                                               GlpP          1   1.000000   \n",
       "59                                               FruR          1   1.000000   \n",
       "72                                               LevR          1   1.000000   \n",
       "61                                               FrlR          1   1.000000   \n",
       "30                                               ManR          1   1.000000   \n",
       "3                                                AcoR          1   1.000000   \n",
       "80                                               LicR          1   0.444444   \n",
       "82                                        CcpA + SigA          0   0.794872   \n",
       "10                             [CcpA + SigA] / [CcpN]          0   0.611765   \n",
       "52                                               YlxR          1   1.000000   \n",
       "56                                        AhrC + YlxR          1   0.666667   \n",
       "39                                               MtrB          1   1.000000   \n",
       "41                                              S-box          1   0.840000   \n",
       "71  [TnrA + CodY + ThrR] / [TnrA + CodY + T-box + ...          1   0.423077   \n",
       "2                                                CodY          1   0.875000   \n",
       "75                                               HutP          1   1.000000   \n",
       "60  [SigL + AhrC + RocR] / [Spo0A + SigA + CodY + ...          1   0.923077   \n",
       "50                                               PyrR          1   1.000000   \n",
       "58                                               PurR          1   1.000000   \n",
       "70                                            Thi-box          1   1.000000   \n",
       "0                                                PchR          1   0.857143   \n",
       "57                                                Fur          1   0.890909   \n",
       "77                                        PhoP + SigA          1   0.727273   \n",
       "74                                        PhoP + SigA          1   0.954545   \n",
       "45                                                Zur          1   1.000000   \n",
       "32                                               CzrA          1   1.000000   \n",
       "69                                               CymR          1   0.755556   \n",
       "12                                                NaN          0   0.000000   \n",
       "9                                                ResD          0   0.434211   \n",
       "8                                          Fnr + SigA          1   0.888889   \n",
       "44                                                Rex          1   0.900000   \n",
       "49                                               LexA          1   0.777778   \n",
       "4                                                SigW          1   0.940000   \n",
       "47                                               SigM          1   0.954545   \n",
       "16                                                NaN          0   0.000000   \n",
       "7                                                SigB          1   0.864407   \n",
       "46                                               SigB          1   0.600000   \n",
       "21                      [stringent response] / [RplJ]          1   1.000000   \n",
       "23                                               SigD          1   0.914894   \n",
       "5                           SigA + AbrB + RemA + SinR          1   1.000000   \n",
       "53  [ComA + CcpA + TnrA] / [Abh + SigA + ComA + Sp...          1   1.000000   \n",
       "54                                               DegU          0   0.325000   \n",
       "36                                               AbrB          1   0.833333   \n",
       "79                           Rok + ResD + AbrB + SigA          1   0.666667   \n",
       "14                         SigA + YvrHb + DegU + WalR          0   0.176471   \n",
       "51                                               ComK          1   0.612245   \n",
       "35                                   [Spo0A] / [DnaA]          0   0.800000   \n",
       "48                                               SigE          0   0.638710   \n",
       "68                                    [SigE] / [SigG]          0   0.794118   \n",
       "66                                               SigG          0   0.620915   \n",
       "13                                               SigK          0   0.618705   \n",
       "25                                               SigK          0   0.556701   \n",
       "15                      [SigK + GerE] / [SigK + GerR]          0   0.764706   \n",
       "65                                                Xpf          1   0.878788   \n",
       "22                                                NaN          0   0.000000   \n",
       "64                          CsoR + SigA + SigK + SigE          0   0.093960   \n",
       "18                                               ImmR          1   0.950000   \n",
       "27                                               SknR          0   0.352941   \n",
       "20                           [IolR] / [MelR] / [AcoR]          1   0.896552   \n",
       "40                                    [NadR] / [BirA]          1   0.681818   \n",
       "81                             [MtlR] / [AnsR + SigA]          1   0.727273   \n",
       "73                      [SigA + TnrA] / [SigA + PucR]          1   0.900000   \n",
       "67               [SigA + CymR] / [ResD + SigA + NsrR]          1   0.909091   \n",
       "11    [ArsR] / [YodB] / [SigA + CtsR] / [SigA + CsoR]          0   0.466667   \n",
       "17                                                NaN          0        NaN   \n",
       "43                                                NaN          0        NaN   \n",
       "33                                                NaN          0        NaN   \n",
       "34                                                NaN          0        NaN   \n",
       "24                                                NaN          0        NaN   \n",
       "26                                                NaN          0        NaN   \n",
       "62                                                NaN          0        NaN   \n",
       "28                                                NaN          0        NaN   \n",
       "29                                                NaN          0        NaN   \n",
       "31                                                NaN          0        NaN   \n",
       "42                                                NaN          0        NaN   \n",
       "63                                                NaN          0        NaN   \n",
       "55                                                NaN          0        NaN   \n",
       "37                                                NaN          0        NaN   \n",
       "19                                                NaN          0        NaN   \n",
       "76                                                NaN          0        NaN   \n",
       "78                                                NaN          0        NaN   \n",
       "\n",
       "      recall          reg_type  \\\n",
       "1   0.739130      well defined   \n",
       "6   1.000000      well defined   \n",
       "38  1.000000      well defined   \n",
       "59  1.000000      well defined   \n",
       "72  1.000000      well defined   \n",
       "61  1.000000      well defined   \n",
       "30  1.000000      well defined   \n",
       "3   0.750000      well defined   \n",
       "80  0.800000  contains unknown   \n",
       "82  0.126531            subset   \n",
       "10  0.203922     closest match   \n",
       "52  0.186047    subset, operon   \n",
       "56  1.000000      well defined   \n",
       "39  0.555556    subset, operon   \n",
       "41  0.525000            subset   \n",
       "71  1.000000  contains unknown   \n",
       "2   0.148936            subset   \n",
       "75  0.857143      well defined   \n",
       "60  0.800000      well defined   \n",
       "50  0.785714      well defined   \n",
       "58  0.666667      well defined   \n",
       "70  0.833333      well defined   \n",
       "0   0.600000            subset   \n",
       "57  0.597561            subset   \n",
       "77  0.114286            subset   \n",
       "74  0.300000            subset   \n",
       "45  0.923077      well defined   \n",
       "32  0.833333      well defined   \n",
       "69  0.693878      well defined   \n",
       "12  0.000000           unknown   \n",
       "9   0.478261     closest match   \n",
       "8   0.888889      well defined   \n",
       "44  0.529412            subset   \n",
       "49  0.272727            subset   \n",
       "4   0.435185            subset   \n",
       "47  0.162791            subset   \n",
       "16  0.000000           unknown   \n",
       "7   0.323810            subset   \n",
       "46  0.095238     closest match   \n",
       "21  0.260000            subset   \n",
       "23  0.573333            subset   \n",
       "5   0.950000      well defined   \n",
       "53  1.000000      well defined   \n",
       "54  0.122642     closest match   \n",
       "36  0.106667            subset   \n",
       "79  0.800000      well defined   \n",
       "14  1.000000  contains unknown   \n",
       "51  0.405405     closest match   \n",
       "35  0.051724            subset   \n",
       "48  0.327815     closest match   \n",
       "68  0.113924            subset   \n",
       "66  0.524862     closest match   \n",
       "13  0.480447     closest match   \n",
       "25  0.301676     closest match   \n",
       "15  0.127451            subset   \n",
       "65  1.000000      well defined   \n",
       "22  0.000000           unknown   \n",
       "64  0.933333  contains unknown   \n",
       "18  0.950000      well defined   \n",
       "27  0.500000     closest match   \n",
       "20  0.928571      well defined   \n",
       "40  0.714286      well defined   \n",
       "81  0.800000      well defined   \n",
       "73  0.169811            subset   \n",
       "67  0.338983            subset   \n",
       "11  0.280000     closest match   \n",
       "17       NaN               NaN   \n",
       "43       NaN               NaN   \n",
       "33       NaN               NaN   \n",
       "34       NaN               NaN   \n",
       "24       NaN               NaN   \n",
       "26       NaN               NaN   \n",
       "62       NaN               NaN   \n",
       "28       NaN               NaN   \n",
       "29       NaN               NaN   \n",
       "31       NaN               NaN   \n",
       "42       NaN               NaN   \n",
       "63       NaN               NaN   \n",
       "55       NaN               NaN   \n",
       "37       NaN               NaN   \n",
       "19       NaN               NaN   \n",
       "76       NaN               NaN   \n",
       "78       NaN               NaN   \n",
       "\n",
       "                                             Function           Category  \\\n",
       "1                              Fatty acid degradation             Carbon   \n",
       "6                       Malate uptake and utilization             Carbon   \n",
       "38                    Glycerol uptake and utilization             Carbon   \n",
       "59                    Fructose uptake and utilization             Carbon   \n",
       "72  Fuctose uptake and phosphorylation, degradatio...             Carbon   \n",
       "61                  Amino sugar uptake and metabolism             Carbon   \n",
       "30  Utilization of mannose for cell wall synthesis...             Carbon   \n",
       "3   Utilization of acetoin, extracellular energy s...             Carbon   \n",
       "80  Uptake, phosphorylation, and utilization of le...             Carbon   \n",
       "82  Uptake and utilization of alternative carbon s...             Carbon   \n",
       "10  Uptake, utilization, and regulation of alterna...             Carbon   \n",
       "52                             Synthesis of histidine        Amino Acids   \n",
       "56                Uptake and biosynthesis of arginine        Amino Acids   \n",
       "39                         Biosynthesis of tryptophan        Amino Acids   \n",
       "41                Methionine salvage and biosynthesis        Amino Acids   \n",
       "71  Synthesis and salvage of leucine, methionine, ...        Amino Acids   \n",
       "2   Uptake of di-/oligopeptides, uptake of guanosi...        Amino Acids   \n",
       "75                   Histidine uptake and utilization        Amino Acids   \n",
       "60  Utilization of arginine, ornithine, citrulline...        Amino Acids   \n",
       "50                           Synthesis of pyrimidines     Other Organics   \n",
       "58  Synthesis of purine, xanthine and guanine upta...     Other Organics   \n",
       "70                   Synthesis and uptake of thiamine     Other Organics   \n",
       "0   Extracellular synthesis and import of pulcherr...           Elements   \n",
       "57  Production and transport of siderophores for i...           Elements   \n",
       "77  High-affinity phosphate uptake, teichuronic ac...           Elements   \n",
       "74  Degradation of wall teichoic acid to salvage p...           Elements   \n",
       "45  Zinc uptake, zinc metallochaperones, alternati...           Elements   \n",
       "32               Resistance to Zn, Cu, Co, Ni, and Cd           Elements   \n",
       "69  Sulfur metabolism in response to oxidative str...           Elements   \n",
       "12  Putative sulfur carriers. High activity under ...           Elements   \n",
       "9   Anaerobic nitrate respiration, bacteriocin pro...  Oxygen Limitation   \n",
       "8           Nitrate respiration and nitrite extrusion  Oxygen Limitation   \n",
       "44  Overflow metabolism (LDH) and cytochrome BD ox...  Oxygen Limitation   \n",
       "49         SOS response for DNA protection and repair             Stress   \n",
       "4   Adaptation to membrane active agents such as c...             Stress   \n",
       "47  General stress proteins, alarmones, lipid carr...             Stress   \n",
       "16  Contains one subunit of NADH dehydrogenase wit...             Stress   \n",
       "7   Survival of general stressors including ethano...             Stress   \n",
       "46  Likely accounting for microarray saturation in...             Stress   \n",
       "21  Production of ribosomes, translation initiatio...         Lifestyles   \n",
       "23  Production of flagella components and export m...         Lifestyles   \n",
       "5   Biosynthesis of tasA and extracellular polysac...         Lifestyles   \n",
       "53  Activated during transition to stationary phas...         Lifestyles   \n",
       "54  Response to arrested flagellar motor. Secretio...         Lifestyles   \n",
       "36  Various transition state roles: spore killing,...         Lifestyles   \n",
       "79  Produces and exports subtilosin, which kills a...         Lifestyles   \n",
       "14  Activation of WapA and its operon of unknown g...         Lifestyles   \n",
       "51  DNA transport, pseudopilus, recombination mach...         Lifestyles   \n",
       "35  Initiation of sporulation through sigma factor...        Sporulation   \n",
       "48  Spore maturation through core dehydration, nut...        Sporulation   \n",
       "68  Spore coat production, transcription factor an...        Sporulation   \n",
       "66  Calcium and nutrient delivery to the spore, en...        Sporulation   \n",
       "13  Production of spore coat and crust proteins, L...        Sporulation   \n",
       "25  Production of outermost spore coat and crust p...        Sporulation   \n",
       "15  Production of spore coat and crust proteins. O...        Sporulation   \n",
       "65  Prophage element that induces production of ba...       Mobile Genes   \n",
       "22  Temperature-responsive SP-beta prophage. Respo...       Mobile Genes   \n",
       "64  Temperature-responsive SP-beta prophage. Proph...       Mobile Genes   \n",
       "18  Mobile genetic element that participates in ho...       Mobile Genes   \n",
       "27  Prophage element that disrupts the SigK gene b...       Mobile Genes   \n",
       "20  Uptake and utilization of inositol, melibiose/...      Multi-Purpose   \n",
       "40  Biosynthesis of NAD and biotin, plus safA & co...      Multi-Purpose   \n",
       "81  Uptake and utilization of mannitol, degradatio...      Multi-Purpose   \n",
       "73  Ammonium uptake, utilization of nitrate and xa...      Multi-Purpose   \n",
       "67  Import and utilization of sulfonate and cystin...      Multi-Purpose   \n",
       "11  Detoxification in response to: oxidative prote...      Multi-Purpose   \n",
       "17  Contains a SigE/SigG inhibitor, spore coat pro...    Uncharacterized   \n",
       "43  Contains cold shock, sporulation onset, toxin,...    Uncharacterized   \n",
       "33  Contains many poorly characterized RNAs with n...    Uncharacterized   \n",
       "34  Contains many poorly characterized RNAs with n...    Uncharacterized   \n",
       "24  Contains many poorly characterized RNAs with n...    Uncharacterized   \n",
       "26       5’ ends of tRNAs and independent transcripts    Uncharacterized   \n",
       "62  Likely accounting for noisey RNA S144, an inde...    Uncharacterized   \n",
       "28                  Noisy independent transcript S415    Uncharacterized   \n",
       "29  Independent transcript S1235, behaves similar ...    Uncharacterized   \n",
       "31                 Noisy independent transcript S1210    Uncharacterized   \n",
       "42  Likely accounting for noise in the carbon sour...    Uncharacterized   \n",
       "63  Likely accounting for noise in carbon source t...    Uncharacterized   \n",
       "55  Likely accounting for noise in the first sampl...    Uncharacterized   \n",
       "37  Likely accounting for noise in the diamide aft...    Uncharacterized   \n",
       "19                                  Contains no genes    Uncharacterized   \n",
       "76                                  Contains no genes    Uncharacterized   \n",
       "78                                  Contains no genes    Uncharacterized   \n",
       "\n",
       "                   category2                        category3  new_idx  \\\n",
       "1              Carbon Source                      Fatty Acids        0   \n",
       "6              Carbon Source                           Malate        1   \n",
       "38             Carbon Source                         Glycerol        2   \n",
       "59             Carbon Source                         Fructose        3   \n",
       "72             Carbon Source                         Fructose        4   \n",
       "61             Carbon Source                     Amino Sugars        5   \n",
       "30             Carbon Source                          Mannose        6   \n",
       "3              Carbon Source                          Acetoin        7   \n",
       "80             Carbon Source                         Lichenan        8   \n",
       "82            General Carbon             Low glucose response        9   \n",
       "10            General Carbon             Low glucose response       10   \n",
       "52      Amino Acid Synthesis              Specific: Histidine       11   \n",
       "56      Amino Acid Synthesis               Specific: Arginine       12   \n",
       "39      Amino Acid Synthesis             Specific: Tryptophan       13   \n",
       "41      Amino Acid Synthesis             Specific: Methionine       14   \n",
       "71      Amino Acid Synthesis                          Several       15   \n",
       "2       Amino Acid Synthesis                          Several       16   \n",
       "75            Amino Acid Use                        Histidine       17   \n",
       "60            Amino Acid Use                           Energy       18   \n",
       "50               Nucleotides                      Pyrimidines       19   \n",
       "58               Nucleotides                          Purines       20   \n",
       "70                 Cofactors                         Thiamine       21   \n",
       "0            Ion Homeostasis                             Iron       22   \n",
       "57           Ion Homeostasis                             Iron       23   \n",
       "77           Ion Homeostasis                        Phosphate       24   \n",
       "74           Ion Homeostasis                        Phosphate       25   \n",
       "45           Ion Homeostasis                             Zinc       26   \n",
       "32           Ion Homeostasis                    Toxic Cations       27   \n",
       "69        Sulfur Homeostasis                 Oxidative Stress       28   \n",
       "12        Sulfur Homeostasis                         Putative       29   \n",
       "9          Oxygen Limitation               Pleiotropic Energy       30   \n",
       "8          Oxygen Limitation              Nitrate Respiration       31   \n",
       "44         Oxygen Limitation                         Overflow       32   \n",
       "49        Specific Stressors                       DNA Damage       33   \n",
       "4         Specific Stressors             Wall/Membrane Stress       34   \n",
       "47        Specific Stressors             Wall/Membrane Stress       35   \n",
       "16        Specific Stressors              Possible Heat Shock       36   \n",
       "7             General Stress                   General Stress       37   \n",
       "46            General Stress                   General Stress       38   \n",
       "21               Translation                      Translation       39   \n",
       "23                  Motility                         Motility       40   \n",
       "5                   Biofilms                              NaN       41   \n",
       "53                  Biofilms                              NaN       42   \n",
       "54                  Biofilms                              NaN       43   \n",
       "36                  Biofilms                              NaN       44   \n",
       "79          Toxin Production                              NaN       45   \n",
       "14          Toxin Production                              NaN       46   \n",
       "51                Competence                              NaN       47   \n",
       "35               Sporulation                                0       48   \n",
       "48               Sporulation                                1       49   \n",
       "68               Sporulation                                2       50   \n",
       "66               Sporulation                                3       51   \n",
       "13               Sporulation                                5       52   \n",
       "25               Sporulation                                6       53   \n",
       "15               Sporulation                                4       54   \n",
       "65     Deleterious Prophages                             PBSX       55   \n",
       "22     Deleterious Prophages                           SPBeta       56   \n",
       "64     Deleterious Prophages                           SPBeta       57   \n",
       "18           Other Prophages                              NaN       58   \n",
       "27           Other Prophages                              NaN       59   \n",
       "20  Multi-Purpose Metabolism         Several Specific Sources       60   \n",
       "40  Multi-Purpose Metabolism                      NAD, Biotin       61   \n",
       "81  Multi-Purpose Metabolism  Mannitol, Asparagine, Aspartate       62   \n",
       "73  Multi-Purpose Metabolism                    Nitrogen Ions       63   \n",
       "67  Multi-Purpose Metabolism                Sulfur Limitation       64   \n",
       "11      Multi-Purpose Stress                 Oxidative Stress       65   \n",
       "17     Noisy Groups of Genes                              NaN       66   \n",
       "43     Noisy Groups of Genes                              NaN       67   \n",
       "33     Noisy Groups of Genes                   Putative Noise       68   \n",
       "34     Noisy Groups of Genes                   Putative Noise       69   \n",
       "24     Noisy Groups of Genes                   Putative Noise       70   \n",
       "26     Noisy Groups of Genes                   Putative Noise       71   \n",
       "62               Noisy Genes                       Single RNA       72   \n",
       "28               Noisy Genes                       Single RNA       73   \n",
       "29               Noisy Genes                       Single RNA       74   \n",
       "31               Noisy Genes                       Single RNA       75   \n",
       "42               Noisy Genes                          Samples       76   \n",
       "63               Noisy Genes                          Samples       77   \n",
       "55               Noisy Genes                          Samples       78   \n",
       "37               Noisy Genes                          Samples       79   \n",
       "19                     Empty                              NaN       80   \n",
       "76                     Empty                              NaN       81   \n",
       "78                     Empty                              NaN       82   \n",
       "\n",
       "                                            Regulator  threshold  n_genes  \n",
       "1                                                FadR   0.087360     17.0  \n",
       "6                                                MalR   0.144234      5.0  \n",
       "38                                               GlpP   0.106678     10.0  \n",
       "59                                               FruR   0.206403      4.0  \n",
       "72                                               LevR   0.142297      6.0  \n",
       "61                                               FrlR   0.127036      7.0  \n",
       "30                                               ManR   0.119812      6.0  \n",
       "3                                                AcoR   0.153858      3.0  \n",
       "80                                               LicR   0.076635      9.0  \n",
       "82                                        CcpA + SigA   0.057071     39.0  \n",
       "10                             [CcpA + SigA] / [CcpN]   0.045382     85.0  \n",
       "52                                               YlxR   0.129423      8.0  \n",
       "56                                        AhrC + YlxR   0.105072     15.0  \n",
       "39                                               MtrB   0.174997      5.0  \n",
       "41                                              S-box   0.076127     25.0  \n",
       "71  [TnrA + CodY + ThrR] / [TnrA + CodY + T-box + ...   0.065844     26.0  \n",
       "2                                                CodY   0.065552     40.0  \n",
       "75                                               HutP   0.086899      6.0  \n",
       "60  [SigL + AhrC + RocR] / [Spo0A + SigA + CodY + ...   0.093177     13.0  \n",
       "50                                               PyrR   0.117698     11.0  \n",
       "58                                               PurR   0.089225     20.0  \n",
       "70                                            Thi-box   0.095194     15.0  \n",
       "0                                                PchR   0.081261      7.0  \n",
       "57                                                Fur   0.047500     55.0  \n",
       "77                                        PhoP + SigA   0.086751     11.0  \n",
       "74                                        PhoP + SigA   0.089585     22.0  \n",
       "45                                                Zur   0.104663     12.0  \n",
       "32                                               CzrA   0.155808      5.0  \n",
       "69                                               CymR   0.046438     45.0  \n",
       "12                                                NaN   0.126741      7.0  \n",
       "9                                                ResD   0.046826     76.0  \n",
       "8                                          Fnr + SigA   0.120142      9.0  \n",
       "44                                                Rex   0.081571     10.0  \n",
       "49                                               LexA   0.071558     27.0  \n",
       "4                                                SigW   0.061503     50.0  \n",
       "47                                               SigM   0.077209     22.0  \n",
       "16                                                NaN   0.162882      7.0  \n",
       "7                                                SigB   0.033604    118.0  \n",
       "46                                               SigB   0.057723     50.0  \n",
       "21                      [stringent response] / [RplJ]   0.062382     52.0  \n",
       "23                                               SigD   0.056403     94.0  \n",
       "5                           SigA + AbrB + RemA + SinR   0.116051     19.0  \n",
       "53  [ComA + CcpA + TnrA] / [Abh + SigA + ComA + Sp...   0.141925      7.0  \n",
       "54                                               DegU   0.065658     40.0  \n",
       "36                                               AbrB   0.053203     48.0  \n",
       "79                           Rok + ResD + AbrB + SigA   0.069789     12.0  \n",
       "14                         SigA + YvrHb + DegU + WalR   0.116118     17.0  \n",
       "51                                               ComK   0.050085     49.0  \n",
       "35                                   [Spo0A] / [DnaA]   0.076391     15.0  \n",
       "48                                               SigE   0.042301    155.0  \n",
       "68                                    [SigE] / [SigG]   0.049619     68.0  \n",
       "66                                               SigG   0.042330    153.0  \n",
       "13                                               SigK   0.034650    139.0  \n",
       "25                                               SigK   0.035362     97.0  \n",
       "15                      [SigK + GerE] / [SigK + GerR]   0.092769     17.0  \n",
       "65                                                Xpf   0.081181     33.0  \n",
       "22                                                NaN   0.100877      2.0  \n",
       "64                          CsoR + SigA + SigK + SigE   0.036485    149.0  \n",
       "18                                               ImmR   0.078043     20.0  \n",
       "27                                               SknR   0.052144     17.0  \n",
       "20                           [IolR] / [MelR] / [AcoR]   0.076520     29.0  \n",
       "40                                    [NadR] / [BirA]   0.098662     22.0  \n",
       "81                             [MtlR] / [AnsR + SigA]   0.091097     11.0  \n",
       "73                      [SigA + TnrA] / [SigA + PucR]   0.101788     10.0  \n",
       "67               [SigA + CymR] / [ResD + SigA + NsrR]   0.081852     22.0  \n",
       "11    [ArsR] / [YodB] / [SigA + CtsR] / [SigA + CsoR]   0.064922     30.0  \n",
       "17                                                NaN   0.065454     30.0  \n",
       "43                                                NaN   0.054663     40.0  \n",
       "33                                                NaN   0.048538     80.0  \n",
       "34                                                NaN   0.049752     62.0  \n",
       "24                                                NaN   0.069265     19.0  \n",
       "26                                                NaN   0.074451      6.0  \n",
       "62                                                NaN   0.217807      1.0  \n",
       "28                                                NaN   0.112846      2.0  \n",
       "29                                                NaN   0.122629      1.0  \n",
       "31                                                NaN   0.142092      1.0  \n",
       "42                                                NaN   0.073841     22.0  \n",
       "63                                                NaN   0.060006     30.0  \n",
       "55                                                NaN   0.055977     48.0  \n",
       "37                                                NaN   0.095907      5.0  \n",
       "19                                                NaN   0.126665      0.0  \n",
       "76                                                NaN   0.163114      0.0  \n",
       "78                                                NaN   0.123938      0.0  "
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "enrich"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_directory(ica_data, k, row, links, sample_meta):\n",
    "    # generate the plot files\n",
    "    gene_table = gene_table_df(ica_data, k, row, links = links, operon_commas=False)\n",
    "    gene_hist = gene_hist_df(ica_data, k, row)\n",
    "    base_conds = ['M9exp_1', 'M9exp_2', 'M9exp_3']\n",
    "    gene_scatter = gene_scatter_df(ica_data, k, base_conds, links)\n",
    "    act_bar = activity_bar_df(ica_data, k, sample_meta)\n",
    "\n",
    "    reg_venn = regulon_venn_df(ica_data, k, row)\n",
    "    reg_scatter = regulon_scatter_df(ica_data, k, row)\n",
    "\n",
    "    # generate a basic data df\n",
    "    res = pd.Series(index=['name', 'TF', 'Regulator',\n",
    "                           'Function', 'Category', \n",
    "                           'has_venn', 'scatter', 'has_meme',\n",
    "                           'precision', 'recall'])\n",
    "    res.loc['name'] = row.loc['name']\n",
    "    res.loc['TF'] = row.TF\n",
    "    res.loc['Regulator'] = tf_with_links(k, row)\n",
    "    res.loc['Function'] = row.Function\n",
    "    res.loc['Category'] = row.Category\n",
    "    res.loc['has_venn'] = not(reg_venn is None)\n",
    "    if reg_scatter is None:\n",
    "        res.loc['scatter'] = 0\n",
    "    else:\n",
    "        res.loc['scatter'] = reg_scatter.shape[1] - 1\n",
    "    res.loc['has_meme'] = False # update later\n",
    "    res.loc['precision'] = row.precision\n",
    "    res.loc['recall'] = row.recall\n",
    "\n",
    "    # save output\n",
    "    folder = 'iModulon_files/'+str(k)+'/'\n",
    "    if not(os.path.isdir(folder)):\n",
    "        os.mkdir(folder)\n",
    "    res.to_csv(folder+str(k)+'_meta.csv')\n",
    "    gene_table.to_csv(folder+str(k)+'_gene_table.csv')\n",
    "    gene_hist.to_csv(folder+str(k)+'_gene_hist.csv')\n",
    "    gene_scatter.to_csv(folder+str(k)+'_gene_scatter.csv')\n",
    "    act_bar.to_csv(folder+str(k)+'_activity_bar.csv')\n",
    "    if not(reg_venn is None):\n",
    "        reg_venn.to_csv(folder+str(k)+'_reg_venn.csv')\n",
    "    if not(reg_scatter is None):\n",
    "        reg_scatter.to_csv(folder+str(k)+'_reg_scatter.csv')\n",
    "    ica_data.S[k].to_csv(folder+str(k)+'_gene_weights.csv')\n",
    "    ica_data.A.loc[k].to_csv(folder+str(k)+'_activity.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MISSING TF: stringentresponse\n",
      "MISSING TF: stringentresponse\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/scipy/optimize/minpack.py:795: OptimizeWarning: Covariance of the parameters could not be estimated\n",
      "  category=OptimizeWarning)\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:35: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:44: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "/home/kevin/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:45: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.\n",
      "../../../py/gene_scatter.py:61: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  res.link[g] = links[g]\n"
     ]
    }
   ],
   "source": [
    "for k, row in enrich.iterrows():\n",
    "    make_directory(ica_data, k, row, links, sample_meta)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Gene Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Filter out genes with no ICA data\n",
    "reduced_gene_info = gene_info.copy()\n",
    "reduced_gene_info = reduced_gene_info[reduced_gene_info.index.isin(ica_data.X.index)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Information for Gene Metadata\n",
    "\n",
    "def make_gene_files(ica_data, k, row, links, sample_meta):\n",
    "\n",
    "    # generate metadata df\n",
    "    res = pd.Series(index=['gene_id', 'name', 'operon', 'product',\n",
    "                           'cog', 'regulator(s)', 'link'])\n",
    "    res.loc['gene_id'] = k\n",
    "    res.loc['name'] = row.gene_name\n",
    "    res.loc['operon'] = row.operon\n",
    "    res.loc['product'] = row['product']\n",
    "    res.loc['cog'] = row.cog\n",
    "    res.loc['regulator(s)'] = \", \".join(trn[trn.gene_id == k].TF.to_list())\n",
    "    try:\n",
    "        res.loc['link'] = '<a href=\"' + str(links[k]) + '\">'+ 'SubtiWiki' + '</a>'\n",
    "    except:\n",
    "        res.loc['link'] = None\n",
    "    res.fillna(value=\"<i> Not Available </i>\", inplace = True) \n",
    "    \n",
    "    # save output\n",
    "    folder = 'gene_page_files/'+str(k)+'/'\n",
    "    if not(os.path.isdir(folder)):\n",
    "        os.mkdir(folder)\n",
    "    res.to_csv(folder+str(k)+'_meta.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/katherine/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py:7 \u001b[1;31mDeprecationWarning\u001b[0m: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.\n"
     ]
    }
   ],
   "source": [
    "# generate metadata csv for each gene\n",
    "for k, row in reduced_gene_info.iterrows():\n",
    "    make_gene_files(ica_data, k, row, links, sample_meta)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "# activity plot + expression for each gene\n",
    "for gene_id in reduced_gene_info.index:\n",
    "    this_fig = gene_activity_bar_df(ica_data, gene_id, sample_meta)\n",
    "    folder = 'gene_page_files/'+str(gene_id)+'/'\n",
    "    this_fig.to_csv(folder + gene_id + '_activity_bar.csv')\n",
    "    ica_data.X.loc[gene_id].to_csv(folder+gene_id+'_expression.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# table for each gene\n",
    "\n",
    "# generate df with all iM genes indicated T/F \n",
    "im_genes= ica_data.S.T.copy()\n",
    "for k,row in ica_data.S.T.iterrows():\n",
    "    im_genes.loc[k,:] = abs(row) > abs(ica_data.thresholds[k])\n",
    "    \n",
    "# iM_table: start with main component of iM page\n",
    "im_table = enrich[['name', 'Regulator', 'Function', 'Category']]\n",
    "im_table = im_table.rename(columns={'name':'Name'})\n",
    "im_table.index.name = 'k'\n",
    "\n",
    "#loop through genes:\n",
    "for gene_id in reduced_gene_info.index:\n",
    "    perGene_table = im_table.copy()\n",
    "    perGene_table.insert(column ='in_iM', value = im_genes.loc[:, gene_id], loc = 1)\n",
    "    perGene_table.insert(column ='gene_weight', value = ica_data.S.loc[gene_id, :], loc = 2)\n",
    "\n",
    "    #sort\n",
    "    perGene_table = (perGene_table.assign(A=perGene_table['gene_weight'].abs())\n",
    "            .sort_values(['in_iM','A'],ascending=[False, False])\n",
    "            .drop('A', 1))\n",
    "    folder = 'gene_page_files/'+str(gene_id)+'/'\n",
    "    perGene_table.to_csv(folder + gene_id + '_perGene_table.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### make json file for Search"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get df in correct format\n",
    "gene_df = reduced_gene_info.copy()\n",
    "gene_df[\"gene_id\"] = reduced_gene_info.index\n",
    "gene_df = gene_df[['gene_name', \"gene_id\", \"product\"]]\n",
    "gene_df = gene_df.sort_values(by=\"gene_name\")\n",
    "gene_df[\"gene_name\"] = gene_df[\"gene_name\"].fillna(value = \"Unnamed Gene\")\n",
    "gene_df[\"product\"] = gene_df[\"product\"].fillna(value = \"not available\")\n",
    "\n",
    "#create gene info json\n",
    "gene_df.to_json('./gene_page_files/gene_list.json', orient=\"records\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### iModulons"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "enrich_df = enrich.copy()\n",
    "enrich_df['component'] = enrich_df.index\n",
    "enrich_df = enrich_df[['component', 'name', 'Regulator', 'Function']]\n",
    "enrich_df = enrich_df.sort_values(by=\"name\").fillna(value = \"N/A\")\n",
    "\n",
    "enrich_df.to_json('./iModulon_files/im_list.json', orient=\"records\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
